{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "_cell_guid": "b1076dfc-b9ad-4769-8c92-a6c4dae69d19",
    "_uuid": "8f2839f25d086af736a60e9eeb907d3b93b6e0e5"
   },
   "outputs": [],
   "source": [
    "# System packages\n",
    "import os\n",
    "import sys\n",
    "import warnings\n",
    "# Data related\n",
    "import numpy as np \n",
    "import pandas as pd \n",
    "import pprint as pp\n",
    "\n",
    "\n",
    "# sklearn \n",
    "\n",
    "# sklearn tools \n",
    "from sklearn.metrics import  accuracy_score\n",
    "from sklearn.model_selection import train_test_split, cross_val_predict,cross_val_score,  StratifiedKFold, GridSearchCV\n",
    "from sklearn.feature_extraction.text import CountVectorizer, TfidfTransformer, TfidfVectorizer\n",
    "from sklearn.pipeline import FeatureUnion, Pipeline, make_pipeline, make_union\n",
    "from sklearn.preprocessing import FunctionTransformer, OneHotEncoder\n",
    "from sklearn.metrics import accuracy_score\n",
    "import scikitplot.plotters as skplt\n",
    "\n",
    "# nlp\n",
    "from gensim.models import KeyedVectors\n",
    "\n",
    "# models\n",
    "import xgboost as xgb\n",
    "import eli5\n",
    "from eli5.explain import explain_weights\n",
    "from eli5.formatters import explain_weights_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "warnings.filterwarnings('ignore')\n",
    "\n",
    "from utils_functions import *"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Load data "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('../data/processed/train_variants_text.csv')\n",
    "df=df.dropna(subset=['Text'])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Process data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "df['Class'] = df['Class'].astype(int)\n",
    "df['Gene'] = df['Gene'].astype(str)\n",
    "df['Variation'] = df['Variation'].astype(str)\n",
    "\n",
    "y= df['Class']\n",
    "X= df.drop(['Class','ID'],axis =1)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>FAM58A</td>\n",
       "      <td>Truncating Mutations</td>\n",
       "      <td>Cyclin-dependent kinases (CDKs) regulate a var...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     Gene             Variation  \\\n",
       "0  FAM58A  Truncating Mutations   \n",
       "\n",
       "                                                Text  \n",
       "0  Cyclin-dependent kinases (CDKs) regulate a var...  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "X.head(1)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Model"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.1 Bag-of-words 1 Group+xgboost"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Split into Train amd Validation data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_tr, X_val, y_tr, y_val = split_data(df,\n",
    "                                      'Text',\n",
    "                                      'Class',\n",
    "                                      0.1,\n",
    "                                      0,\n",
    "                                      stratify='Class')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {},
   "outputs": [],
   "source": [
    "clf = Pipeline([('vect', CountVectorizer(preprocessor=clean_text_stemmed, stop_words =stop_words)),\n",
    "                ('tfidf', TfidfTransformer()),\n",
    "                ('clf',  xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42))])                           "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.6445783132530121\n"
     ]
    }
   ],
   "source": [
    "clf.fit(X_tr, y_tr)  \n",
    "predicted = clf.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_te = test.Text.values"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_te.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "metadata": {},
   "outputs": [],
   "source": [
    "y_te = clf.predict_proba(X_te)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(986, 9)"
      ]
     },
     "execution_count": 119,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "y_te.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "metadata": {},
   "outputs": [],
   "source": [
    "submission_df = pd.DataFrame(y_te, columns=['class'+str(c+1) for c in range(9)])\n",
    "\n",
    "submission_df['ID'] = df_test['ID']\n",
    "submission_df.head()\n",
    "\n",
    "mysubset = [\"ID\"]\n",
    "mask = submission_df.columns.isin(mysubset)\n",
    "submission_df =pd.concat([submission_df.loc[:,mask], submission_df.loc[:,~mask]], axis=1)\n",
    "submission_df.to_csv('../data/processed/stage2_submission_3.1_new.csv', index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.2 Bag-of-words *3 groups +xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Select one dataframe column for vectorization\n",
    "def build_preprocessor(df,field):\n",
    "    field_idx = list(df.columns).index(field)\n",
    "    return lambda x: default_preprocessor(x[field_idx])\n",
    "default_preprocessor = CountVectorizer().build_preprocessor()    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "vectorizer = FeatureUnion([\n",
    "    ('Variation',TfidfVectorizer(preprocessor=build_preprocessor(X,'Variation'))),\n",
    "        ('Gene',TfidfVectorizer(preprocessor=build_preprocessor(X,'Gene'))),\n",
    "        ('Text',TfidfVectorizer(preprocessor=build_preprocessor(X,'Text'))),\n",
    "])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_v = vectorizer.fit_transform(X.values)\n",
    "X_tr, X_val, y_tr, y_val = train_test_split(X_v,\n",
    "                                            y,\n",
    "                                            test_size=0.2,\n",
    "                                            stratify=y,\n",
    "                                            random_state=42)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "XGBClassifier(base_score=0.5, booster='gbtree', colsample_bylevel=1,\n",
       "       colsample_bytree=1, gamma=0, learning_rate=0.1, max_delta_step=0,\n",
       "       max_depth=3, min_child_weight=1, missing=None, n_estimators=100,\n",
       "       n_jobs=1, nthread=None, objective='multi:softprob', random_state=42,\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1)"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model = xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42)\n",
    "model.fit(X_tr, y_tr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.6385542168674698"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Accuracy score \n",
    "model.score(X_val,y_val)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>feature</th>\n",
       "      <th>weight</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Gene__pten</td>\n",
       "      <td>0.026807</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Text__57</td>\n",
       "      <td>0.023044</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Text__unclassified</td>\n",
       "      <td>0.014653</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Text__1jm7</td>\n",
       "      <td>0.011033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Text__expected</td>\n",
       "      <td>0.006895</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Text__cause</td>\n",
       "      <td>0.006807</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Text__steric</td>\n",
       "      <td>0.006590</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Text__treatment</td>\n",
       "      <td>0.006474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Text__bioinformatic</td>\n",
       "      <td>0.006417</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Text__variants</td>\n",
       "      <td>0.006098</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "               feature    weight\n",
       "0           Gene__pten  0.026807\n",
       "1             Text__57  0.023044\n",
       "2   Text__unclassified  0.014653\n",
       "3           Text__1jm7  0.011033\n",
       "4       Text__expected  0.006895\n",
       "5          Text__cause  0.006807\n",
       "6         Text__steric  0.006590\n",
       "7      Text__treatment  0.006474\n",
       "8  Text__bioinformatic  0.006417\n",
       "9       Text__variants  0.006098"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_name = explain_weights_df(model, vec=vectorizer, top=10, feature_filter=lambda x: x != '<BIAS>')\n",
    "                \n",
    "df_name.to_csv('../data/features/20190609full_union_3groups_tfidf_feature_weights.csv')\n",
    "\n",
    "df_name.head(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <style>\n",
       "    table.eli5-weights tr:hover {\n",
       "        filter: brightness(85%);\n",
       "    }\n",
       "</style>\n",
       "\n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "        \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=1\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.081</b>, score <b>-0.615</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 94.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.505\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.18%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.111\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__clinical\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.062\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inhibitor\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.056\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__active\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.25%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.031\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__months\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.41%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.022\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__constitutive\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.55%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__advanced\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__proto\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__histidine\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__retrovirus\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__school\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__doses\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bilateral\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__glutamate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__noti\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__dead\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__linear\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__still\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cardiac\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__shorter\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__hydrogen\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__plasma\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Variation__amplification\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__independence\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__respond\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__230\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stimulating\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__randomly\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.96%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.000\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inside\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.95%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bethesda\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.95%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__111\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__05\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__aggregation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__osteosarcoma\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__transactivation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__silencing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__tryptophan\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__scaffold\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__gal\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__panels\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2100\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__extends\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__amls\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__wildtype\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__algorithms\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ss\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__tt\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cultures\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__aggregates\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inactivation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__sirnas\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mediate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mediates\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__processing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__skipping\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__asparagine\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__probands\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__close\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__contrast\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__sizes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__prone\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stop\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nuclear\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__structure\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.49%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__interaction\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.31%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.028\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__interior\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.18%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.036\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__surface\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.12%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.039\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stability\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.11%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.040\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Variation__mutations\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.073\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Variation__deletion\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.32%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.099\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Variation__truncating\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 91.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.987\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">w802*</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">cbl</span><br/><b>Text:</b> <span style=\"opacity: 0.80\"> abstract background  non-small cell lung cancer (nsclc) is a heterogeneous group of disorders with a number of genetic and proteomic alterations. c-cbl is an e3 ubiquitin ligase and adaptor molecule important in normal homeostasis and cancer. we determined the genetic </span><span style=\"background-color: hsl(120, 100.00%, 97.16%); opacity: 0.80\" title=\"0.020\">variations</span><span style=\"opacity: 0.80\"> of c-cbl, relationship to receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.41%); opacity: 0.81\" title=\"-0.028\">tyrosine</span><span style=\"opacity: 0.80\"> kinases (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> and met), and functionality in nsclc.  methods and findings  using archival formalin-fixed paraffin embedded (ffpe) extracted </span><span style=\"background-color: hsl(120, 100.00%, 98.85%); opacity: 0.80\" title=\"0.005\">genomic</span><span style=\"opacity: 0.80\"> dna, we show that c-cbl mutations occur in somatic fashion for lung cancers. c-cbl mutations were not mutually exclusive of met or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutations; however they were independent of </span><span style=\"background-color: hsl(0, 100.00%, 99.78%); opacity: 0.80\" title=\"-0.001\">p53</span><span style=\"opacity: 0.80\"> and kras mutations. in normal/tumor pairwise analysis, </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> was significant loss of heterozygosity (loh) for the c-cbl locus (22%, n = 8/37) and none of these samples revealed any mutation in the remaining copy of c-cbl. the c-cbl loh also positively correlated with </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> and met mutations observed in the same samples. using select c-cbl somatic mutations such </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> s80n/h94y, q249e and w802* (obtained from caucasian, taiwanese and african-american samples, respectively) transfected in nsclc cell lines, </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> was increased cell viability and cell motility.  conclusions  taking the overall mutation rate of c-cbl to be a combination </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> somatic </span><span style=\"background-color: hsl(120, 100.00%, 98.73%); opacity: 0.80\" title=\"0.006\">missense</span><span style=\"opacity: 0.80\"> mutation and loh, it is clear that c-cbl is highly mutated in lung cancers and may play an essential role in lung tumorigenesis and metastasis.  go to: introduction in the us alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which </span><span style=\"background-color: hsl(0, 100.00%, 99.47%); opacity: 0.80\" title=\"-0.002\">more</span><span style=\"opacity: 0.80\"> than 145,000 of them succumb to the disease [1]. this number is roughly equivalent to the combined mortality rates of cancers of the breast, prostate, colon, liver, kidney and melanoma [1]. in addition the prognosis is usually poor and the five-year survival rate is less than 15%. </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> are also significant ethnic differences for lung cancer, and the outcome is worse for blacks compared to whites. gender differences are also striking with women having significantly better prognosis </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> compared to men. </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> are a number of genetic alterations that can occur in lung cancer. </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> an example, in nsclc, mutations in kras, </span><span style=\"background-color: hsl(0, 100.00%, 99.78%); opacity: 0.80\" title=\"-0.001\">p53</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> and met </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> been identified. many of these pathways, especially receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.41%); opacity: 0.81\" title=\"-0.028\">tyrosine</span><span style=\"opacity: 0.80\"> kinases (rtks) are controlled by c-cbl.  cbl (casitas b-lineage lymphoma) is a </span><span style=\"background-color: hsl(120, 100.00%, 97.66%); opacity: 0.80\" title=\"0.015\">mammalian</span><span style=\"opacity: 0.80\"> gene located on human chromosome 11q23.3 [2] and is involved in cell signaling and </span><span style=\"background-color: hsl(120, 100.00%, 99.83%); opacity: 0.80\" title=\"0.000\">protein</span><span style=\"opacity: 0.80\"> ubiquitination [3]. cbl proteins belong to the ring finger class of ubiquitin ligases (e3) and </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> are </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> homologues c-cbl, cbl-b, cbl-3 [4]. the c-cbl and cbl-b genes are ubiquitously expressed with the highest levels in hematopoietic tissues [5]. c-cbl consists of four regions encoding for functionally distinct </span><span style=\"background-color: hsl(120, 100.00%, 99.83%); opacity: 0.80\" title=\"0.000\">protein</span><span style=\"opacity: 0.80\"> domains: the n-terminal </span><span style=\"background-color: hsl(0, 100.00%, 96.41%); opacity: 0.81\" title=\"-0.028\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.60%); opacity: 0.81\" title=\"-0.026\">kinase</span><span style=\"opacity: 0.80\"> binding (tkb) domain, the </span><span style=\"background-color: hsl(120, 100.00%, 99.21%); opacity: 0.80\" title=\"0.003\">linker</span><span style=\"opacity: 0.80\"> region, the catalytic ring finger domain, the proline-rich region and the c-terminal ubiquitin-associated (uba) domain that also overlaps with a leucine-zipper (lz) domain [3]. both tkb and ring finger domains are essential for ligand-induced ubiquitination of rtks [6], [7], [8], [9]. the ring finger domain is required for the recruitment of e2 ubiquitin-conjugating enzymes. the tkb domain includes four-helix bundle (4h), a calcium-biding ef hand, and a modified sh2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. in addition, the proline-rich region of c-cbl can associate with the sh3 domain of grb2, which can indirectly recruit c-cbl to rtks via the grb2 adaptor </span><span style=\"background-color: hsl(120, 100.00%, 99.83%); opacity: 0.80\" title=\"0.000\">protein</span><span style=\"opacity: 0.80\"> [7], [13], [14].  c-cbl also binds to </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> and acts </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> the e3 that targets </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> for ubiquitination and </span><span style=\"background-color: hsl(120, 100.00%, 98.48%); opacity: 0.80\" title=\"0.008\">degradation</span><span style=\"opacity: 0.80\">. furthermore, cbl desensitizes egf signaling and opposes cellular proliferation induced by egf [15]. egf activation also appears to activate the </span><span style=\"background-color: hsl(0, 100.00%, 96.41%); opacity: 0.81\" title=\"-0.028\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.60%); opacity: 0.81\" title=\"-0.026\">kinase</span><span style=\"opacity: 0.80\"> src, which phosphorylates c-cbl and in </span><span style=\"background-color: hsl(0, 100.00%, 99.43%); opacity: 0.80\" title=\"-0.002\">turn</span><span style=\"opacity: 0.80\"> activates the ubiquitination and </span><span style=\"background-color: hsl(120, 100.00%, 98.48%); opacity: 0.80\" title=\"0.008\">degradation</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> [16], [17], [18]. a recent study shows that defective endocytosis of </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> is characterized by a deletion mutant and the point mutation l858r, whereby its association with c-cbl and subsequent ubiquitination are impaired [19]. recently, the first human c-cbl mutations were reported in acute myeloid leukemia (aml) patients [20]. the mutation r420q inhibits fms-like </span><span style=\"background-color: hsl(0, 100.00%, 96.41%); opacity: 0.81\" title=\"-0.028\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.60%); opacity: 0.81\" title=\"-0.026\">kinase</span><span style=\"opacity: 0.80\"> 3 (flt3) internalization and ubiquitination [20].  not only can e3 activity be important in oncogenesis, c-cbl has a dual but separate </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.006\">function</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> a signal transduction molecule. we </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> previously shown that c-cbl is important in binding crkl and bcr/abl in hematopoietic cells. also, it can bind and modulate </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">functions</span><span style=\"opacity: 0.80\"> of cytoskeleton by binding to proteins like talin and paxillin. the tkb domain is important in binding to a number of </span><span style=\"background-color: hsl(0, 100.00%, 98.45%); opacity: 0.80\" title=\"-0.008\">molecules</span><span style=\"opacity: 0.80\">, and they then </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.006\">function</span><span style=\"opacity: 0.80\"> in signal transduction.  given the critical role of cbl in normal homeostasis and cancer, we hypothesized that it might be mutated in lung cancers. in this study, we report novel c-cbl somatic mutations s80n/h94y, q249e and w802* in caucasian, taiwanese and african-american lung cancer patients, respectively. expressing these mutations in nsclc cell lines lead to increased proliferation and cell motility. we show that c-cbl mutations occur with or without met or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutations but are mutually exclusive of a loh at the c-cbl locus. additionally, c-cbl loh is associated with either met or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutations. we thus hypothesize that c-cbl mutations might contribute to the </span><span style=\"background-color: hsl(0, 100.00%, 94.68%); opacity: 0.81\" title=\"-0.049\">oncogenic</span><span style=\"opacity: 0.80\"> potential of met and </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> in lung cancer.  go to: methods ethics statement  written consent on all research on human subjects has been obtained from the institutional review board, university of chicago and covers all research performed in the laboratory. the following is their contact information: institutional review board, the university of chicago, mcgiffert hall, 5751 s. woodlawn ave., 2nd floor, chicago, il 60637. written informed consents were received from all patients whose </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">tissue</span><span style=\"opacity: 0.80\"> samples were used for this study.  </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">tissue</span><span style=\"opacity: 0.80\"> samples  lung cancer </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">tissue</span><span style=\"opacity: 0.80\"> and paired adjacent normal lung tissues were obtained from 50 caucasian, 29 african-americans and 40 taiwanese nsclc patients who were recruited at the university of chicago hospital (chicago, usa) (caucasian and african-american patients) and taipei veterans general hospital of taiwan (taiwanese patients) after obtaining appropriate institutional review board permission and informed consent from the patients. out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 years. in terms of tumor types, 53 were adenocarcinoma, 32 were squamous cell carcinoma and 34 were large cell carcinoma. 49 were stage i, 14 were stage ii, 34 were stage iii, and 13 were stage iv (table s1).  cell culture  human non-small cell lung carcinoma cells a549 and h358 were maintained in dmem and rpmi-1640, respectively. human embryonic kidney 293t cells were cultured in dmem. media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (invitrogen, carlsbad, ca). cells were cultured at 37°c in a humidified incubator containing 5% co2.  c-cbl gene mutational analysis  exons 2 to 16 of c-cbl gene were individually amplified by polymerase chain reaction (pcr). primers are listed in table s2. pcr conditions were 1 cycle of 95°c for 5 minutes; 35 cycles of 94°c for 30 seconds, 58°c for 30 seconds and 72°c for 2 minutes; and </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">one</span><span style=\"opacity: 0.80\"> cycle of 72°c for 10 minutes. pcr products were treated with exosap-it (usb corporation, cleveland, oh) and sequenced by big-dye terminator chemistry (applied biosystems, foster city, ca). sequencing was performed on the forward coding strand with confirmation of c-cbl alterations performed by sequencing the reverse strand </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> well. chromatograms were analyzed for mutations using mutation surveyor v2.61 (softgenetics, state college, pa).  plasmid constructs and site-directed mutagenesis  the wild-type c-cbl cdna insert was subcloned into the paltermax expression vector using xhoi and sali restriction enzyme sites (promega, madison, wi). using this parental plasmid paltermax-c-cbl, the tkb domain double mutation (s80n/h94y), the point mutation (q249e), and the c-terminal point mutation w802* of c-cbl were created using the following primers: 5′-gctggcgctaaagaataacccaccttatatcttagac-3′ and 5′-ctaccagatacctaccagtatctccgtactatcttgtc-3′ for the double mutation s80n/h94y; 5′-ctttacccgactctttgagccctggtcctctttgc-3′ for q249e, and 5′-cagctcctcctttggctgattgtctctggatggtgatc-3′ for w802* </span><span style=\"background-color: hsl(0, 100.00%, 93.19%); opacity: 0.82\" title=\"-0.069\">along</span><span style=\"opacity: 0.80\"> with their complementary primers using the quickchange site-directed mutagenesis xl kit (stratagene, </span><span style=\"background-color: hsl(0, 100.00%, 89.19%); opacity: 0.83\" title=\"-0.134\">la</span><span style=\"opacity: 0.80\"> jolla, ca) according to the manufacturer&#x27;s instructions. the constructs were confirmed for the point mutations by standard dna sequencing of both strands.  loss of heterozygosity (loh) analysis  five microsatellites on chromosome 11 (3 on 11q at or within 200 </span><span style=\"background-color: hsl(0, 100.00%, 85.16%); opacity: 0.85\" title=\"-0.211\">kb</span><span style=\"opacity: 0.80\"> up or downstream of the c-cbl gene and 2 control markers on 11p) were selected for analysis (table s3). established microsatellite markers and respective primer sequences were selected from the geneloc database (http://genecards.weizmann.ac.il/geneloc/index.shtml, weizmann institute of science, rehovot, israel). primers were custom designed and each forward primer was fluorescently labeled at the 5′ end with fam, pet, ned, or vic (applied biosystems). primer annealing temperatures and duplex scores were evaluated with nist primer tools (http://yellow.nist.gov:8444/dnaanalysis/primertoolspage.do; national institute of standards and technology, gaithersburg, md). primers were verified by performing pcr with control dna (isolated from tk6 cells) and resolving the products on agarose gels. bands were visualized with an </span><span style=\"background-color: hsl(0, 100.00%, 98.52%); opacity: 0.80\" title=\"-0.008\">uv</span><span style=\"opacity: 0.80\"> transilluminator. </span><span style=\"background-color: hsl(120, 100.00%, 98.85%); opacity: 0.80\" title=\"0.005\">genomic</span><span style=\"opacity: 0.80\"> dna was extracted from tumor samples and paired normal lung </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">tissue</span><span style=\"opacity: 0.80\">. primers were grouped into multiplex combinations shown in table s4. marker d11s929 served </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> an internal control to check for consistency in pcrs and of peaks from capillary electrophoresis. multiplex pcrs were carried out in a volume of 10 µl that contained 1 µl </span><span style=\"background-color: hsl(120, 100.00%, 98.85%); opacity: 0.80\" title=\"0.005\">genomic</span><span style=\"opacity: 0.80\"> dna (20–50 ng), 0.5 µm of each primer (1.0 µm total for each primer pair), 400 µm dntps, 1x pcr buffer containing mgcl2, and 0.2 u taq dna polymerase. pcr was performed on the abi geneamp 9700 pcr system under the following conditions: 5 min at 94°c; 30 cycles of 30 sec at 94°c, 1 min at 60°c, 1 min at 72°c; and 5 min at 72°c. the pcr products were separated by capillary electrophoresis on an abi 3130xl dna analyzer. chromatograms were analyzed with peak scanner 1.0 and genemapper 3.7 software (applied biosystems) for allelic alterations. the area of the peaks produced by the dna pcr products was quantified for each allele. the ratio of the allelic areas was calculated for each tumor and paired normal dna sample. </span><span style=\"background-color: hsl(0, 100.00%, 99.31%); opacity: 0.80\" title=\"-0.003\">when</span><span style=\"opacity: 0.80\"> the qloh (allelic ratio for the tumor peaks divided by the allelic ratio of paired normal sample) was ≤0.5 or ≥2.0 for c-cbl and at least </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">one</span><span style=\"opacity: 0.80\"> other 11q marker in at least two separate experiments, the sample was considered </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> having an allelic imbalance and interpreted </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> loh. samples were evaluated in at least two separate experiments and samples showing prospective loh at c-cbl repeated a third time which included a new control marker at the bax locus (data not shown) on chromosome 19 to verify integrity of sample dna.  transfection of c-cbl constructs  the a549 cell line was transfected using the fugene hd (roche, nutley, nj) reagent according to the manufacturer&#x27;s instructions. eight µg of plasmid dna, containing either no insert (empty vector), wild-type c-cbl, s80n/h94y c-cbl, q249e c-cbl or w802* cbl was used for transfection in a 6-well culture plate. cells were harvested 48 h after transfection and analyzed for expression.  c-cbl knockdown  c-cbl knockdown was performed using lentiviral transduction using mission lentiviral transduction particles (sigma-aldrich, st. louis, mo) </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> per manufacturer&#x27;s instructions. briefly, 1×105 h358 cells/well were seeded in 6-well plates and infected the following day with c-cbl lentiviral shrna constructs. to generate stable c-cbl knockdown cell lines, cells were selected for 2 </span><span style=\"background-color: hsl(0, 100.00%, 96.90%); opacity: 0.81\" title=\"-0.023\">days</span><span style=\"opacity: 0.80\"> with 1 µg/ml puromycin. c-cbl levels were determined using whole cell lysates by immunoblotting with anti-cbl antibody (santa cruz biotechnologies, santa cruz, ca).  cell viability assay  cells were transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> described above in the transfection assay. forty-eight hours after transfection, viability of cells was assessed using trypan blue exclusion.  wound healing assay  a549 cells were seeded in 6-well plates and cultured for 48 h until 100% confluent. the medium was then changed and the cells were transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> described in the transfection assay. twelve hours after transfection, a straight scratch was </span><span style=\"background-color: hsl(120, 100.00%, 93.83%); opacity: 0.81\" title=\"0.060\">made</span><span style=\"opacity: 0.80\"> across the cell layer using a 1 ml pipette tip. the cells were then gently washed with 1× pbs to remove cellular debris and the media was replaced. photographs were taken of the wound region every 12 h until 48 h.  western blot analysis  forty eight hours after transfection, cells were collected and washed twice in 1x pbs, then lysed in ice-cold lysis buffer (0.5m tris-hcl with ph 7.4, 1.5 m nacl, 2.5% deoxycholic acid, 10 mm edta, 10% np-40, 0.5 mm dtt, 1 mm phenylmethylsulfonyl fluoride, 5 µg/ml leupeptin, and 10 µg/ml aprotinin) for 5 minutes. the lysate was centrifuged at 13,000 rpm for 20 minutes at 4°c, and </span><span style=\"background-color: hsl(120, 100.00%, 99.83%); opacity: 0.80\" title=\"0.000\">protein</span><span style=\"opacity: 0.80\"> content of the supernatant was measured. total cell lysates (50 µg/well) were separated by sds-page electrophoresis and the gels transferred onto nitrocellulose membranes (whatman, piscataway, nj). membranes were blocked with 5% non-fat dry milk in phosphate-buffered saline containing tween-20 (pbst) (1x pbs, 0.1% tween-20) for 1 h at room temperature and </span><span style=\"background-color: hsl(120, 100.00%, 95.51%); opacity: 0.81\" title=\"0.038\">incubated</span><span style=\"opacity: 0.80\"> with the appropriate primary antibody at 4°c overnight. membranes then were washed </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> times with pbst and probed with appropriate horseradish peroxidase (hrp)-conjugated secondary antibody for 1 h at room temperature. the membranes were again washed </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> times in pbst and bands were visualized using western blot chemiluminescence reagent (biorad, valencia, ca) on a chemidoc gel documentation system (biorad, valencia, ca). antibodies were obtained from santa cruz biotechnologies and used at the following dilutions (c-cbl, 1∶5000; c-met, 1∶5000; </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\">, 1∶5000; ubiquitin, 1∶1000; ha, 1∶5000 and β-actin, 1∶10,000).  flow cytometry  cell cycle analysis was carried out by flow cytometry. approximately 2×106 cells were grown in media containing 10% fbs. cells were harvested by trypsin/edta treatment, washed with 1x pbs </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> times and fixed with ice-cold 70% ethanol for 2 h. cells were washed again with cold pbs and stained with a solution containing 25 µg/ml propidium iodide, 200 µg/ml rnase a, and 0.1% triton x-100 for 30 minutes in the dark. cell cycle analysis was performed using a guava pca-96 flow cytometer (guava technologies, millipore, billerica, ma).  ubiquitin ligase activity  293t cells were maintained in culture in dmem supplemented with 10% fbs and 1% penicillin (100 units/ml) and streptomycin (100 µg/ml) were transfected with 0.2 µg </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\">-pcdna3 and 2 µg ha-tagged c-cbl constructs </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> indicated using calcium phosphate according to manufacturer&#x27;s protocol (profection, promega, madison, wi). </span><span style=\"background-color: hsl(0, 100.00%, 99.33%); opacity: 0.80\" title=\"-0.003\">twenty</span><span style=\"opacity: 0.80\">-four hours post-transfection, cells were starved overnight in dmem supplemented with 0.5% fbs, and then treated with or without egf (100 ng/ml) for 15 min. the cells were collected and washed two times in ice-cold pbs containing 0.2 mm sodium orthovanadate then lysed in ice-cold lysis buffer (10 mm tris hcl, ph 7.5, 150 mm nacl, 5 mm edta, 1% triton x100, 10% glycerol, 2 mm sodium orthovanadate and protease </span><span style=\"background-color: hsl(0, 100.00%, 91.60%); opacity: 0.82\" title=\"-0.094\">inhibitors</span><span style=\"opacity: 0.80\">). lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4°c. </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> immunoprecipitations were performed on 200 µg of cleared lysate using 250 ng of rabbit-anti-</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> and </span><span style=\"background-color: hsl(120, 100.00%, 99.83%); opacity: 0.80\" title=\"0.000\">protein</span><span style=\"opacity: 0.80\"> a/g plus sepharose overnight at 4°c. precipitations were washed 5 times in lysis buffer before boiling in laemmli buffer. elutions were immunoblotted with anti-ubiquitin and </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\">. </span><span style=\"background-color: hsl(0, 100.00%, 99.33%); opacity: 0.80\" title=\"-0.003\">twenty</span><span style=\"opacity: 0.80\"> micrograms of cleared lysate were immunoblotted for each of the c-cbl constructs using anti-ha.  statistical analysis  mutation rates between different groups were compared using fisher&#x27;s exact test. for continuous variables, group comparisons were performed using analysis of variance (anova) followed by sidak&#x27;s adjustment for multiple comparisons. experiments involving repeated measurements over time were analyzed using repeated measures anova with the greenhouse-geisser adjustment to the degrees of freedom. analyses were conducted using stata (v10.1) software (stata corporation, college station, tx).  go to: results c-cbl gene mutations in lung cancer  to investigate the role of c-cbl in lung cancer, we analyzed its </span><span style=\"background-color: hsl(120, 100.00%, 98.85%); opacity: 0.80\" title=\"0.005\">genomic</span><span style=\"opacity: 0.80\"> dna in tumor and paired normal samples drawn from multiple ethnicities. the lung tumor samples represented caucasians (n = 50), african-americans (n = 29), and taiwanese (n = 40) lung cancer patients. we designed 12 pairs of primers to </span><span style=\"background-color: hsl(120, 100.00%, 98.64%); opacity: 0.80\" title=\"0.007\">sequence</span><span style=\"opacity: 0.80\"> the coding region of c-cbl gene that spans exons 2 to 16 (table s2). we identified 8 unique somatic mutations in c-cbl exons among 8 different patients. a variation l620f, a known snp (rs2227988) in exon 11 was also detected. importantly, the eight novel non-synonymous mutations were confirmed by sequencing both strands of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.85%); opacity: 0.80\" title=\"0.005\">genomic</span><span style=\"opacity: 0.80\"> dna obtained from lung tumor samples (table 1). moreover, none of the 8 mutations were detected in the corresponding normal </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">tissue</span><span style=\"opacity: 0.80\">, indicating that these were somatic mutations. four synonymous single nucleotide </span><span style=\"background-color: hsl(120, 100.00%, 97.16%); opacity: 0.80\" title=\"0.020\">variations</span><span style=\"opacity: 0.80\"> (snvs) were also identified but were not used further in this study.  table 1 table 1 c-cbl mutation analysis in 119 lung cancer patient tumor tissues. </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> of the 8 novel non-synonymous mutations were located in the tkb (</span><span style=\"background-color: hsl(0, 100.00%, 96.41%); opacity: 0.81\" title=\"-0.028\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.60%); opacity: 0.81\" title=\"-0.026\">kinase</span><span style=\"opacity: 0.80\"> binding) domain (s80n, h94y, and q249e), </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">one</span><span style=\"opacity: 0.80\"> in the ring finger domain (v391i), </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">one</span><span style=\"opacity: 0.80\"> in the proline-rich region (72515_72517 del atg), and </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> in the c-terminal region (w802*, r830k, and a848t) of the c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.83%); opacity: 0.80\" title=\"0.000\">protein</span><span style=\"opacity: 0.80\"> (figure 1a and figure s1). in figure 1b, we show model chromatograms of </span><span style=\"background-color: hsl(120, 100.00%, 99.66%); opacity: 0.80\" title=\"0.001\">representative</span><span style=\"opacity: 0.80\"> samples.  figure 1 figure 1 c-cbl mutations and loh in non-small cell lung cancer. 11q loh of c-cbl gene  paired lung tumor and normal lung </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">tissue</span><span style=\"opacity: 0.80\"> samples from taiwanese patients (n = 37) were investigated for loh. eight (21.6%) showed loh at the c-cbl locus on chromosome 11 while 29 samples (78.4%) revealed normal allelic contribution at the microsatellite markers (</span><span style=\"background-color: hsl(0, 100.00%, 98.67%); opacity: 0.80\" title=\"-0.007\">figures</span><span style=\"opacity: 0.80\"> 1c and d).  c-cbl mutations in different ethnic groups  the c-cbl double mutant s80n/h94y was found in the same patient and the overall mutation rate for c-cbl in lung tumors was 6.7% (8/119). the frequency of c-cbl mutation was highest in large cell carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the least was observed in adenocarcinoma (ad) (1.8%; 1 of 53 patients), although these rates were not statistically significant (p = 0.292). mutation rates were 6.0% among caucasians (0 of 20 in ad; 0 of 10 in sq; and 3 of 20 in lc), 13.8% in african-americans (1 of 10 in ad; 1 of 10 in sq; and 2 of 9 in lc), and 2.5% (0 of 23 in ad; 1 of 12 in sq; and 0 of 5 in lc) in the taiwanese population. additionally two taiwanese patients with lung cancer (</span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">one</span><span style=\"opacity: 0.80\"> squamous and </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">one</span><span style=\"opacity: 0.80\"> adenocarcinoma) had the known snp l620f. ethnic differences were not statistically significant, however the power to detect differences was low.  mutations in met and </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> can be co-associated with c-cbl alterations  since east asians with lung cancer </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> a higher frequency of </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> and met mutations in lung tumors [21], [22], we also determined mutations in </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> and met in the same taiwanese cohort samples and compared the results with the observed c-cbl alterations (loh and/or mutations). in the 37 samples tested, we did not find any overlap between c-cbl mutations and c-cbl loh (figure 2). of the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> c-cbl mutants (including the known l620f snp, rs2227988), </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">one</span><span style=\"opacity: 0.80\"> of the samples had a met mutation (n375s) and the other had an </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutation (l858r). among the 8 samples that had a loh at the c-cbl locus 5 had an additional mutation in met (n375s) and 2 had an </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> exon 19 deletion. </span><span style=\"background-color: hsl(0, 100.00%, 99.33%); opacity: 0.80\" title=\"-0.003\">twenty</span><span style=\"opacity: 0.80\">-six samples had neither c-cbl mutation nor c-cbl loh (3 patients had a c-cbl mutation but no c-cbl loh). among these 26 samples 9 had a met mutation (8 n375s, 1 l211w), 13 had an </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutation (7 exon 9 deletion, 6 l858r) and 4 had no other met or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutation. thus the rate of met or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutations among patients with loh at the c-cbl locus (7 of 8) was similar to that seen in patients without c-cbl mutation or loh (22 of 26 patients) (p = 0.99). these 4 patients with no identifiable mutation in c-cbl, met or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> represented 10.8% of the 37 patients analyzed in the taiwanese patient cohort. conversely, 89.2% taiwanese lung cancer patients </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> an identifiable mutation in either c-cbl, met or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> or a combination of the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> genes (figure 2). additionally, we determined </span><span style=\"background-color: hsl(0, 100.00%, 99.78%); opacity: 0.80\" title=\"-0.001\">p53</span><span style=\"opacity: 0.80\"> and kras mutations in these taiwanese cohorts. two </span><span style=\"background-color: hsl(0, 100.00%, 99.78%); opacity: 0.80\" title=\"-0.001\">p53</span><span style=\"opacity: 0.80\"> and 1 kras mutation were detected. the single kras mutation overlapped with </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">one</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.78%); opacity: 0.80\" title=\"-0.001\">p53</span><span style=\"opacity: 0.80\"> mutation. this patient also had the </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> exon 19 deletion but had no c-cbl mutation. the other </span><span style=\"background-color: hsl(0, 100.00%, 99.78%); opacity: 0.80\" title=\"-0.001\">p53</span><span style=\"opacity: 0.80\"> mutation sample had a c-cbl loh with concurrent met n375s mutation. thus, in the taiwanese samples analyzed, </span><span style=\"background-color: hsl(0, 100.00%, 99.78%); opacity: 0.80\" title=\"-0.001\">p53</span><span style=\"opacity: 0.80\">/kras mutations and c-cbl mutations were mutually exclusive (data not shown).  figure 2 figure 2 c-cbl mutations and relationship to met and </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutations in lung cancer. cellular </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">functions</span><span style=\"opacity: 0.80\"> of c-cbl alterations in the context of lung tumorigenesis  a. e3 activity is intact in the mutant c-cbl proteins to investigate whether the different c-cbl mutations affect the e3 activity, </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> was chosen </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> a model substrate for c-cbl e3 </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.006\">function</span><span style=\"opacity: 0.80\">. all of the c-cbl mutants tested enhanced ubiquitination of the activated </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> similar to the wild-type c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.83%); opacity: 0.80\" title=\"0.000\">protein</span><span style=\"opacity: 0.80\">. this result demonstrates that the catalytic activity of the c-cbl mutants is not impaired </span><span style=\"background-color: hsl(0, 100.00%, 99.31%); opacity: 0.80\" title=\"-0.003\">when</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> was the substrate. (figure 3a).  figure 3 figure 3 ubiquitination, viability, expression and cell cycle analysis of various c-cbl mutants. b. effect on lung cancer cell viability the effect of a </span><span style=\"background-color: hsl(120, 100.00%, 99.66%); opacity: 0.80\" title=\"0.001\">representative</span><span style=\"opacity: 0.80\"> c-cbl mutant from each of the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> ethnic backgrounds on lung cancer cell viability in cell lines was determined. s80n/h94y double mutation, q249e, and w802* were identified in lung tumor samples obtained from a caucasian, a taiwanese and an african-american, respectively. </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> described in methods, the c-cbl wild-type (wt) and the above </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> mutants were expressed after cloning them into paltermax vector in a549 cells. these cells express relatively low basal levels of endogenous c-cbl (data not shown). transfection efficiency was comparable between different groups and the number of cells transfected with c-cbl wild-type construct was about 70% compared to control cells that were transfected with the empty vector. cells transfected with s80n/h94y, q249e and w802* c-cbl mutant constructs resulted in increased number of viable cells that was 132.3%, 120.8% and 147.9% higher respectively, relative to the empty vector control transfected cells and significantly different from the wild-type construct (p = 0.022, p = 0.049, and p = 0.008, respectively) (figure 3b). relative levels of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.83%); opacity: 0.80\" title=\"0.000\">protein</span><span style=\"opacity: 0.80\"> in whole cell lysates prepared from samples obtained from a parallel experiment was determined. the c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.83%); opacity: 0.80\" title=\"0.000\">protein</span><span style=\"opacity: 0.80\"> levels in samples representing untransfected and empty vector transfected cells were comparable and those representing the c-cbl wt and the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> c-cbl mutants were comparable (figure 3c).  c. effect on cell cycle to investigate if the increases in cell viability in different c-cbl mutants are due to increased cellular proliferation, a cell cycle analysis was performed. a549 cells were transfected with the c-cbl wt or the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> different mutants: s80n/h94y, q249e and w802*. the empty vector transfectant was used </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> a control. forty-eight hours after transfection cell cycle analysis was performed </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> described in materials and methods. </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> was no significant change in the subg1, g1 or the s phase of the cell cycle among the different mutants compared to the wt construct (p = 0.64, p = 0.40, and p = 0.28, respectively). the g2/m phase of the cell cycle showed an increase in cell numbers for the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> mutants, s80n/h94y, q249e and w802*, </span><span style=\"background-color: hsl(0, 100.00%, 99.31%); opacity: 0.80\" title=\"-0.003\">when</span><span style=\"opacity: 0.80\"> compared to the wt but again the difference was not statistically significant (p = 0.25) (figure 3d).  d. effect on cell motility to investigate the effect of the expression of the above </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> c-cbl mutants on cell migration, we carried out wound healing assay </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> described in materials and methods. the closing of the scratch or the wound was monitored at 0, 12, 24, 36, and 48 h. (figure 4a). in all the samples, that represented cells transfected with mutants, the wound gap was much smaller than that seen in the sample that represented cells transfected with c-cbl wt (p&lt;0.001). we also determined the rate of wound closure for all the five groups. at 48 h, wild-type c-cbl transfectants showed 61.1% open wound while the s80n/h94y, q249e and w802* mutants showed 18.7%, 23.9% and 34.3% open wound respectively (p&lt;0.001) (figure 4b).  figure 4 figure 4 c-cbl mutations affect wound healing in a549 cells. e. c-cbl knockdown increases cell viability it is hypothesized that the loh seen in </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> samples could lead to decreased expression of c-cbl. thus we tested the effect of c-cbl knockdown in lung cancer cells. compared to a549, h358 lung cancer cells express relatively high levels of endogenous c-cbl (data not shown). c-cbl expression was knocked down using lentiviral construct that expressed c-cbl specific shrna and compared the results with those that were transduced with scrambled shrna and the results are shown in figure 5. we identified several clones that revealed varying degrees of c-cbl knockdown showing different sets of c-cbl lentiviral shrna knockdown efficiency (figure 5a). of all the clones tested, clone 27 was chosen for further experiments. equal amount of cells were seeded in a 6-well plate and the cell proliferation was measured at various times and the results are depicted in figure 5b. </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> expected, number of cells increased in a time dependent fashion from 100 to 190% relative to scrambled shrna </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> control in a span of 48 h (p = 0.0002) (figure 5b). the cell cycle phases in h358 cells that were knocked down with c-cbl shrna were looked at and compared with the scrambled shrna. </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> were no discernable differences between these two constructs in the different phases of the cell cycle (data not shown).  figure 5 figure 5 knockdown of c-cbl using an shrna increases cell proliferation. go to: discussion </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> results demonstrate that c-cbl is somatically mutated (or has loh) in lung cancers, and can significantly contribute to enhanced cell viability and motility. </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> was also a high prevalence of loh with respect to c-cbl in lung tumors that harbored met or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutation.  in the present study, we </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> demonstrated the occurrence of c-cbl mutations in lung cancer patients, especially with different ancestral </span><span style=\"background-color: hsl(120, 100.00%, 97.16%); opacity: 0.80\" title=\"0.020\">variations</span><span style=\"opacity: 0.80\">. mutations in c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> been recently reported in juvenile myelomonocytic leukemia and myeloid malignancies. in the aml study, the mutation r420q located in the junction of the ring finger and the </span><span style=\"background-color: hsl(120, 100.00%, 99.21%); opacity: 0.80\" title=\"0.003\">linker</span><span style=\"opacity: 0.80\"> region inhibited fms-like </span><span style=\"background-color: hsl(0, 100.00%, 96.41%); opacity: 0.81\" title=\"-0.028\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.60%); opacity: 0.81\" title=\"-0.026\">kinase</span><span style=\"opacity: 0.80\"> 3 (flt3) internalization and ubiquitination [20], thus contributing to the gain-in-</span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.006\">function</span><span style=\"opacity: 0.80\"> for the rtk. in addition, mutations such </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> h398y, c384r, and l380p mapped to the ring finger domain and the </span><span style=\"background-color: hsl(120, 100.00%, 99.21%); opacity: 0.80\" title=\"0.003\">linker</span><span style=\"opacity: 0.80\"> region of c-cbl that is required for its e3 activity [23], [24], [25], [26], [27]. additionally, homozygous mutations in the ring finger domain of the c-cbl gene were described </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> a result of acquired uniparental disomy (upd) [26]. it is important to note that </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> results indicate loh at 11q23 locus and these are mutually exclusive from </span><span style=\"background-color: hsl(120, 100.00%, 98.73%); opacity: 0.80\" title=\"0.006\">missense</span><span style=\"opacity: 0.80\"> mutations of c-cbl. the somatic mutations were all heterozygous. the mutations in aml led to abrogation of the e3 activity leading to prolonged rtk activation. in addition mutants located on the </span><span style=\"background-color: hsl(120, 100.00%, 99.21%); opacity: 0.80\" title=\"0.003\">linker</span><span style=\"opacity: 0.80\"> region surrounding the ring finger domain exhibited enhanced akt signaling in response to cytokine stimulation [26]. in addition, it was shown in nh3t3 cells, that neither mutations in the ring finger nor the </span><span style=\"background-color: hsl(120, 100.00%, 99.21%); opacity: 0.80\" title=\"0.003\">linker</span><span style=\"opacity: 0.80\"> region causes transformation, however while certain mutations perturbs the ubiquitination, others affect receptor recycling and prolong </span><span style=\"background-color: hsl(0, 100.00%, 96.60%); opacity: 0.81\" title=\"-0.026\">kinase</span><span style=\"opacity: 0.80\"> activity [28].  we report here c-cbl mutations that mapped not only to the ring finger domain, but also to the tkb domain, proline-rich domain and the c-terminal region, but none mapped to the </span><span style=\"background-color: hsl(120, 100.00%, 99.21%); opacity: 0.80\" title=\"0.003\">linker</span><span style=\"opacity: 0.80\"> region </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> reported in the aml studies described above [23], [24], [25], [26], [29]. in addition, 8 mutants that we detected were found in different ethnic backgrounds. for example, s80n/h94y, q249e, w802* were detected in caucasians, taiwanese and african-americans, respectively. the results point out not only the difference between lung cancer and other cancers, but also genetic polymorphism among different races in the same cancer. interestingly, </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> is a large disparity between african-american and other ethnic populations with lung cancer [30]. we </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> previously shown that </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> was a low frequency of </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> and met mutation in african-americans </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> compared with taiwanese and caucasians [31]. in this study, the number of african-american samples analyzed was relatively fewer and we found 3 mutations that are unique to this ethnicity. it would now behoove us to further study the genetic alterations that can occur and determine the targeted therapeutics for african-americans. </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> results provide </span><span style=\"background-color: hsl(120, 100.00%, 96.71%); opacity: 0.81\" title=\"0.025\">evidence</span><span style=\"opacity: 0.80\"> of the importance of c-cbl in tumorigenesis and potential signaling. </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> prediction, based on the aml data, would be that the v391i ring finger domain mutation would affect the e3 activity. also, it will be important to determine the binding partners for c-cbl in the tkb domain and proline rich domain mutations. it has previously been shown that the tkb domain can bind to growth factor receptors and it will be important to determine the cross-binding of these mutants to met and or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\">. it would also be important in the future to look at fluorescence in situ </span><span style=\"background-color: hsl(0, 100.00%, 93.33%); opacity: 0.82\" title=\"-0.067\">hybridization</span><span style=\"opacity: 0.80\">/copy number changes in c-cbl in lung cancer.  c-cbl plays an important role in down regulating rtk-mediated signaling through k63 poly-ubiquitination and subsequent downregulation of rtks followed by lysosomal degredation [3]. mono-ubiquitination or ubiquitinated with k63-linked chains of substrates by c-cbl may lead to enhancement of biological and biochemical </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">functions</span><span style=\"opacity: 0.80\"> (reviewed in hermann et al, 2007 [32]). the mutations that we analyzed in </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> studies all point out to the fact that the e3 activity of c-cbl on </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> is intact; the </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> levels in the various mutants remain same (figure s2). multiple kinases, both rtks and non-rtks could be acted upon by c-cbl, including erbs, pdgfr, fms, met, c-kit, vegfr, flt-1, ron, fgfr, ir, </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> well </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> syk, fyn, lck, fgr, lyn and c-abl [3]. in lung cancers the relevant substrates of c-cbl in terms of </span><span style=\"background-color: hsl(120, 100.00%, 98.48%); opacity: 0.80\" title=\"0.008\">degradation</span><span style=\"opacity: 0.80\"> or signal transduction are yet to be identified.  the observation that c-cbl somatic mutations, especially s80n/h94y, q249e and w802* showed increased cell viability and cell motility that is in agreement with the physiological role for d-cbl in the regulation of apoptosis and differentiation identified in drosophila is very significant [33]. it has been previously shown that </span><span style=\"background-color: hsl(120, 100.00%, 99.53%); opacity: 0.80\" title=\"0.002\">activating</span><span style=\"opacity: 0.80\"> c-cbl mutation downregulates </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> signaling and decreases cellular proliferation and migration in breast cancer cell lines [34]. although the role of c-cbl in the negative regulation of rtks is well substantiated, thereby suggesting that it is a natural tumor </span><span style=\"background-color: hsl(120, 100.00%, 95.97%); opacity: 0.81\" title=\"0.033\">suppressor</span><span style=\"opacity: 0.80\">, studies in cancer cells </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> revealed both tumor </span><span style=\"background-color: hsl(120, 100.00%, 95.97%); opacity: 0.81\" title=\"0.033\">suppressor</span><span style=\"opacity: 0.80\"> and tumor promoting activities depending on the type of c-cbl mutation and the number of alleles at the c-cbl locus [24]. in agreement with the above, the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\"> c-cbl mutants described here appear to </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> tumor growth and metastasis promoting properties. although these mutants are outside of the ring finger or the </span><span style=\"background-color: hsl(120, 100.00%, 99.21%); opacity: 0.80\" title=\"0.003\">linker</span><span style=\"opacity: 0.80\"> region of c-cbl, their downstream effects are significant so </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> to cause increased proliferation and migration, but the substrate affected by these mutations are not known yet. this raises the possibility that some of the cellular </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">functions</span><span style=\"opacity: 0.80\"> of c-cbl are independent of its ubiquitin-ligase activity, an area that we are currently investigating. the </span><span style=\"background-color: hsl(0, 100.00%, 94.68%); opacity: 0.81\" title=\"-0.049\">oncogenic</span><span style=\"opacity: 0.80\"> nature of rtks, addiction of cancers to growth signals and given the clustering of c-cbl, </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> and met mutations, it is possible that the transforming effect of c-cbl mutations is most likely a combinatorial effect of the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">three</span><span style=\"opacity: 0.80\">. we also show that loh for c-cbl was found in a significant number of samples that harbored met or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutations. the fact that about 7% of lung tumor samples are likely to </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> c-cbl mutations and an additional 22% are likely to harbor c-cbl-related loh, makes c-cbl a highly mutated molecule in lung cancer. since loh alone is not enough to cause a transforming event [35], [36], [37], associated mutation in the met or </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> locus or yet another rtk discussed above may play a role in carcinogenesis. we predict that this loh in c-cbl results in haploinsufficiency that downplays rtk ubiquitination leading to hyperactivity of the rtks. however, whether this is sufficient cause for tumorigenesis remains to be determined. consistent with </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> hypothesis is the fact that c-cbl-/- mice </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> increased </span><span style=\"background-color: hsl(0, 100.00%, 96.60%); opacity: 0.81\" title=\"-0.026\">kinase</span><span style=\"opacity: 0.80\"> activity in lymphocytes, but is not sufficient for tumor formation [35], [36], [37]. c-cbl loh could also lead to increased expression of c-cbl from the other allele to compensate for a loss of an allele. alternately, </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> could be some form of synergy working with reduced c-cbl levels and mutated receptors that exacerbate the phenotype of each alone.  previous studies from </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> lab and others </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> shown that east asians with lung cancers </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> relatively high frequencies of gain-of-</span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.006\">function</span><span style=\"opacity: 0.80\"> of mutations in rtks such </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> and met [31]. in a cohort of japanese patients an </span><span style=\"background-color: hsl(120, 100.00%, 99.53%); opacity: 0.80\" title=\"0.002\">activating</span><span style=\"opacity: 0.80\"> met mutation was identified in the splice region that deletes the juxtamembrane domain that is involved e3 activity of c-cbl [38]. this study also found that activation of met is mutually exclusive of </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\">, kras and her2 gene mutations [38]. we failed to detect such mutations in significant numbers in lung tumor samples obtained from african-americans (n = 29) and caucasian (n = 50) patients. </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">one</span><span style=\"opacity: 0.80\"> met mutation was identified in each of the groups whereas 1 and 3 </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutations were identified in the african-american and caucasian cohorts respectively. </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\"> mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> earlier been identified </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">one</span><span style=\"opacity: 0.80\"> of the key mutations affecting lung adenocarcinoma patients in a comprehensive study of 188 patients [39]. </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> study encompasses different histologies of nsclc. however, the published series did not find any mutations in c-cbl or met unlike </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> study that encompassed different subtypes of nsclc. it is important to note, we </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> recently shown that met mutations in lung cancer are in majority germline [31]. we </span><span style=\"background-color: hsl(0, 100.00%, 99.83%); opacity: 0.80\" title=\"-0.000\">have</span><span style=\"opacity: 0.80\"> reported earlier c-cbl mutations in a small cohort of taiwanese lung cancer samples [40]. in </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> efforts to understand the ethnic differences in the lung oncogenome, we also looked at pax </span><span style=\"background-color: hsl(0, 100.00%, 91.68%); opacity: 0.82\" title=\"-0.092\">transcription</span><span style=\"opacity: 0.80\"> factors such </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> pax5 and pax8 that are highly expressed in lung cancers; however </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">there</span><span style=\"opacity: 0.80\"> was no preferential expression or mutations of the above genes in lung tumor samples of african-americans. in this study, we show relatively high frequency of c-cbl mutations in lung cancers, especially in the large cell type among caucasians and particularly among african-americans. we therefore propose c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> an efficacious target for lung cancers in african-americans that needs to be further substantiated. this is all the </span><span style=\"background-color: hsl(0, 100.00%, 99.47%); opacity: 0.80\" title=\"-0.002\">more</span><span style=\"opacity: 0.80\"> important because the prognosis for african-americans with lung cancer, especially for men is much poorer compared to their caucasian counterparts [41].  in conclusion, the results presented in this study demonstrate that c-cbl is frequently mutated or even lost in lung cancers. </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.004\">our</span><span style=\"opacity: 0.80\"> results support a role for c-cbl mutants that are independent of its ubiquitination activity. given the relatively high mutation rates in c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> well </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> rtks such </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">as</span><span style=\"opacity: 0.80\"> met and </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.001\">egfr</span><span style=\"opacity: 0.80\">, it is likely that their combined effect could be synergistic in promoting tumorigenesis.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=2\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.615</b>, score <b>1.418</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 91.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.940\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 96.20%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.317\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 97.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.158\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__functional\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.051\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__length\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.12%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.039\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__truncating\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.15%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.037\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__useful\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.16%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.036\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__studied\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.18%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.035\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pedigrees\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.32%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.027\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__core\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.35%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.026\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__top\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.40%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.023\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__folding\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.46%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.019\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__syndrome\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.53%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__localization\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.55%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__limitations\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.55%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stability\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__variant\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inactivation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__unable\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__consensus\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__localized\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stat5\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ectopic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__codon\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__display\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nuclear\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__4e\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ability\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__aa\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pathway\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__transiently\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__proper\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__frameshift\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__breakpoint\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__almost\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__lumbar\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__prime\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__lyl1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stimulated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cyclic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__walking\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__imaging\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__limitation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__purified\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__anchorage\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mixture\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__towards\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__utilizing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__classification\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__chimeric\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__percentile\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__suspected\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__subject\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__glycine\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__intracellular\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__resistant\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__rbd\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Variation__fusions\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__rapid\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__gc\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__facial\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__consequently\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__extraction\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__identification\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__frequent\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__hras\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__discovery\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bhlh\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__rearranged\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.55%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__advanced\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.53%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__drug\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.50%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__transcripts\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.50%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__compound\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.44%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__achieved\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.44%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__heterodimerization\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.42%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__part\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.35%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.026\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fusions\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.28%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.030\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__need\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.27%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.030\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__qiagen\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.060\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__metastatic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">w802*</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">cbl</span><br/><b>Text:</b> <span style=\"opacity: 0.80\"> abstract background  non-small cell lung cancer (nsclc) is a heterogeneous group of disorders with a number of genetic and proteomic alterations. c-cbl is an </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> ubiquitin ligase and adaptor molecule important </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> normal homeostasis and cancer. we determined the genetic variations of c-cbl, relationship to receptor tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.003\">kinases</span><span style=\"opacity: 0.80\"> (egfr and met), and functionality </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> nsclc.  methods and </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.006\">findings</span><span style=\"opacity: 0.80\">  using archival formalin-fixed paraffin embedded (ffpe) extracted genomic </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\">, we show that c-cbl mutations occur </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> somatic fashion for lung cancers. c-cbl mutations were not mutually exclusive of met or egfr mutations; however they were independent of p53 and kras mutations. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> normal/tumor pairwise analysis, there was significant loss of heterozygosity (loh) for the c-cbl locus (22%, n = 8/37) and none of these samples revealed any mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the remaining copy of c-cbl. the c-cbl loh also positively correlated with egfr and met mutations observed </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the same samples. using select c-cbl somatic mutations such as s80n/h94y, q249e and w802* (obtained from caucasian, taiwanese and african-american samples, respectively) transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> nsclc cell lines, there was increased cell </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">viability</span><span style=\"opacity: 0.80\"> and cell motility.  conclusions  taking the overall mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.22%); opacity: 0.80\" title=\"0.003\">rate</span><span style=\"opacity: 0.80\"> of c-cbl to be a combination as somatic </span><span style=\"background-color: hsl(0, 100.00%, 98.76%); opacity: 0.80\" title=\"-0.006\">missense</span><span style=\"opacity: 0.80\"> mutation and loh, it is clear that c-cbl is highly mutated </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers and may </span><span style=\"background-color: hsl(120, 100.00%, 97.68%); opacity: 0.80\" title=\"0.015\">play</span><span style=\"opacity: 0.80\"> an essential role </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung tumorigenesis and metastasis.  go to: introduction </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the us alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which more than 145,000 of them succumb to the disease [1]. this number is roughly equivalent to the combined mortality rates of cancers of the breast, prostate, colon, liver, kidney and melanoma [1]. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> addition the prognosis is usually poor and the five-year survival </span><span style=\"background-color: hsl(120, 100.00%, 99.22%); opacity: 0.80\" title=\"0.003\">rate</span><span style=\"opacity: 0.80\"> is less than 15%. there are also significant ethnic differences for lung cancer, and the outcome is worse for blacks compared to whites. gender differences are also striking with women having significantly better prognosis as compared to men. there are a number of genetic alterations that can occur </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer. as an example, </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> nsclc, mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> kras, p53, egfr and met have been identified. many of these pathways, especially receptor tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.003\">kinases</span><span style=\"opacity: 0.80\"> (rtks) are controlled by c-cbl.  cbl (casitas b-lineage lymphoma) is a mammalian gene located on human chromosome 11q23.3 [2] and is involved </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> cell signaling and protein ubiquitination [3]. cbl proteins belong to the ring finger class of ubiquitin ligases (</span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\">) and there are three homologues c-cbl, cbl-b, cbl-3 [4]. the c-cbl and cbl-b genes are ubiquitously expressed with the highest levels </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> hematopoietic tissues [5]. c-cbl consists of four regions encoding for functionally distinct protein domains: the n-terminal tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.57%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> binding (tkb) domain, the </span><span style=\"background-color: hsl(120, 100.00%, 99.50%); opacity: 0.80\" title=\"0.002\">linker</span><span style=\"opacity: 0.80\"> region, the catalytic ring finger domain, the proline-rich region and the c-terminal ubiquitin-associated (uba) domain that also overlaps with a leucine-zipper (lz) domain [3]. both tkb and ring finger domains are essential for ligand-induced ubiquitination of rtks [6], [7], [8], [9]. the ring finger domain is required for the recruitment of e2 ubiquitin-conjugating enzymes. the tkb domain includes four-helix bundle (4h), a calcium-biding ef hand, and a modified sh2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> addition, the proline-rich region of c-cbl can associate with the sh3 domain of grb2, which can indirectly recruit c-cbl to rtks via the grb2 adaptor protein [7], [13], [14].  c-cbl also binds to egfr and acts as the </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> that targets egfr for ubiquitination and degradation. furthermore, cbl </span><span style=\"background-color: hsl(120, 100.00%, 86.04%); opacity: 0.84\" title=\"0.194\">desensitizes</span><span style=\"opacity: 0.80\"> egf signaling and opposes cellular proliferation induced by egf [15]. egf </span><span style=\"background-color: hsl(120, 100.00%, 99.53%); opacity: 0.80\" title=\"0.002\">activation</span><span style=\"opacity: 0.80\"> also appears to activate the tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.57%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> src, which phosphorylates c-cbl and </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> turn activates the ubiquitination and degradation of egfr [16], [</span><span style=\"background-color: hsl(0, 100.00%, 98.78%); opacity: 0.80\" title=\"-0.006\">17</span><span style=\"opacity: 0.80\">], [18]. a recent </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\"> shows that defective endocytosis of egfr is characterized by a deletion mutant and the point mutation l858r, whereby its association with c-cbl and subsequent ubiquitination are impaired [19]. recently, the first human c-cbl mutations were reported </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 94.49%); opacity: 0.81\" title=\"0.051\">acute</span><span style=\"opacity: 0.80\"> myeloid leukemia (aml) patients [20]. the mutation r420q inhibits fms-like tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.57%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> 3 (flt3) internalization and ubiquitination [20].  not only can </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> activity be important </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> oncogenesis, c-cbl has a dual but separate </span><span style=\"background-color: hsl(120, 100.00%, 97.63%); opacity: 0.80\" title=\"0.015\">function</span><span style=\"opacity: 0.80\"> as a signal transduction molecule. we have previously shown that c-cbl is important </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> binding crkl and bcr/</span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.002\">abl</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> hematopoietic cells. also, it can bind and modulate functions of cytoskeleton by binding to proteins like talin and paxillin. the tkb domain is important </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> binding to a number of molecules, and they then </span><span style=\"background-color: hsl(120, 100.00%, 97.63%); opacity: 0.80\" title=\"0.015\">function</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> signal transduction.  given the critical role of cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> normal homeostasis and cancer, we hypothesized that it might be mutated </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> this </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\">, we report novel c-cbl somatic mutations s80n/h94y, q249e and w802* </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> caucasian, taiwanese and african-american lung cancer patients, respectively. expressing these mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> nsclc cell lines lead to increased proliferation and cell motility. we show that c-cbl mutations occur with or without met or egfr mutations but are mutually exclusive of a loh at the c-cbl locus. additionally, c-cbl loh is associated with either met or egfr mutations. we thus hypothesize that c-cbl mutations might contribute to the </span><span style=\"background-color: hsl(120, 100.00%, 96.54%); opacity: 0.81\" title=\"0.026\">oncogenic</span><span style=\"opacity: 0.80\"> potential of met and egfr </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer.  go to: methods ethics statement  written consent on all research on human subjects has been obtained from the institutional review board, university of chicago and covers all research performed </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the laboratory. the following is their contact information: institutional review board, the university of chicago, mcgiffert hall, 5751 s. woodlawn ave., 2nd floor, chicago, </span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">il</span><span style=\"opacity: 0.80\"> 60637. written informed consents were received from all patients whose tissue samples were used for this </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\">.  tissue samples  lung cancer tissue and paired adjacent normal lung tissues were obtained from 50 caucasian, 29 african-americans and 40 taiwanese nsclc patients who were recruited at the university of chicago hospital (chicago, usa) (caucasian and african-american patients) and taipei veterans general hospital of taiwan (taiwanese patients) after obtaining appropriate institutional review board permission and informed consent from the patients. out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 </span><span style=\"background-color: hsl(0, 100.00%, 98.58%); opacity: 0.80\" title=\"-0.007\">years</span><span style=\"opacity: 0.80\">. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> terms of tumor types, 53 were adenocarcinoma, 32 were squamous cell carcinoma and 34 were large cell carcinoma. 49 were stage i, 14 were stage ii, 34 were stage iii, and 13 were stage iv (table s1).  cell culture  human non-small cell lung carcinoma cells a549 and h358 were maintained </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> dmem and rpmi-1640, respectively. human </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">embryonic</span><span style=\"opacity: 0.80\"> kidney 293t cells were </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">cultured</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> dmem. media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (invitrogen, carlsbad, ca). cells were </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">cultured</span><span style=\"opacity: 0.80\"> at 37°c </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a humidified incubator containing 5% co2.  c-cbl gene mutational analysis  </span><span style=\"background-color: hsl(0, 100.00%, 99.57%); opacity: 0.80\" title=\"-0.001\">exons</span><span style=\"opacity: 0.80\"> 2 to 16 of c-cbl gene were individually amplified by polymerase chain reaction (pcr). primers are listed </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> table s2. pcr conditions were 1 cycle of 95°c for 5 minutes; 35 cycles of 94°c for 30 seconds, 58°c for 30 seconds and 72°c for 2 minutes; and one cycle of 72°c for 10 minutes. pcr products were treated with exosap-it (usb corporation, cleveland, oh) and sequenced by </span><span style=\"background-color: hsl(120, 100.00%, 94.71%); opacity: 0.81\" title=\"0.048\">big</span><span style=\"opacity: 0.80\">-dye terminator chemistry (applied biosystems, foster city, ca). sequencing was performed on the forward coding strand with confirmation of c-cbl alterations performed by sequencing the reverse strand as well. chromatograms were analyzed for mutations using mutation surveyor v2.61 (softgenetics, state college, pa).  plasmid constructs and site-directed mutagenesis  the </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">wild</span><span style=\"opacity: 0.80\">-type c-cbl cdna insert was subcloned into the paltermax </span><span style=\"background-color: hsl(0, 100.00%, 99.46%); opacity: 0.80\" title=\"-0.002\">expression</span><span style=\"opacity: 0.80\"> vector using xhoi and sali restriction enzyme sites (promega, madison, wi). using this parental plasmid paltermax-c-cbl, the tkb domain double mutation (s80n/h94y), the point mutation (q249e), and the c-terminal point mutation w802* of c-cbl were created using the following primers: 5′-gctggcgctaaagaataacccaccttatatcttagac-3′ and 5′-ctaccagatacctaccagtatctccgtactatcttgtc-3′ for the double mutation s80n/h94y; 5′-ctttacccgactctttgagccctggtcctctttgc-3′ for q249e, and 5′-cagctcctcctttggctgattgtctctggatggtgatc-3′ for w802* along with their complementary primers using the quickchange site-directed mutagenesis xl kit (stratagene, la jolla, ca) according to the manufacturer&#x27;s instructions. the constructs were confirmed for the point mutations by standard </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\"> sequencing of both strands.  loss of heterozygosity (loh) analysis  five microsatellites on chromosome 11 (3 on 11q at or within 200 kb up or downstream of the c-cbl gene and 2 control markers on 11p) were selected for analysis (table s3). established microsatellite markers and respective primer sequences were selected from the geneloc database (http://genecards.weizmann.ac.</span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">il</span><span style=\"opacity: 0.80\">/geneloc/index.shtml, weizmann institute of science, rehovot, israel). primers were custom designed and each forward primer was fluorescently labeled at the 5′ end with fam, pet, ned, or vic (applied biosystems). primer annealing temperatures and duplex scores were evaluated with nist primer tools (http://yellow.nist.gov:8444/dnaanalysis/primertoolspage.do; national institute of standards and technology, gaithersburg, md). primers were verified by performing pcr with control </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\"> (isolated from tk6 cells) and resolving the products on agarose gels. bands were visualized with an uv transilluminator. genomic </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\"> was extracted from tumor samples and paired normal lung tissue. primers were grouped into multiplex combinations shown </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> table s4. marker d11s929 served as an internal control to check for consistency </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> pcrs and of peaks from capillary electrophoresis. multiplex pcrs were carried out </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a volume of 10 µl that contained 1 µl genomic </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\"> (20–50 ng), 0.5 µm of each primer (1.0 µm total for each primer pair), 400 µm dntps, 1x pcr buffer containing mgcl2, and 0.2 u taq </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\"> polymerase. pcr was performed on the abi geneamp 9700 pcr system under the following conditions: 5 min at 94°c; 30 cycles of 30 sec at 94°c, 1 min at 60°c, 1 min at 72°c; and 5 min at 72°c. the pcr products were separated by capillary electrophoresis on an abi 3130xl </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\"> analyzer. chromatograms were analyzed with peak scanner 1.0 and genemapper 3.7 software (applied biosystems) for allelic alterations. the area of the peaks produced by the </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\"> pcr products was quantified for each </span><span style=\"background-color: hsl(120, 100.00%, 98.95%); opacity: 0.80\" title=\"0.005\">allele</span><span style=\"opacity: 0.80\">. the ratio of the allelic areas was calculated for each tumor and paired normal </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\"> sample. when the qloh (allelic ratio for the tumor peaks divided by the allelic ratio of paired normal sample) was ≤0.5 or ≥2.0 for c-cbl and at least one other 11q marker </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> at least two separate experiments, the sample was considered as having an allelic imbalance and interpreted as loh. samples were evaluated </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> at least two separate experiments and samples showing prospective loh at c-cbl repeated a third </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.005\">time</span><span style=\"opacity: 0.80\"> which included a new control marker at the bax locus (data not shown) on chromosome 19 to verify integrity of sample </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\">.  transfection of c-cbl constructs  the a549 cell line was transfected using the fugene hd (roche, nutley, nj) reagent according to the manufacturer&#x27;s instructions. eight µg of plasmid </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\">, containing either no insert (empty vector), </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">wild</span><span style=\"opacity: 0.80\">-type c-cbl, s80n/h94y c-cbl, q249e c-cbl or w802* cbl was used for transfection </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a 6-well culture plate. cells were harvested 48 h after transfection and analyzed for </span><span style=\"background-color: hsl(0, 100.00%, 99.46%); opacity: 0.80\" title=\"-0.002\">expression</span><span style=\"opacity: 0.80\">.  c-cbl knockdown  c-cbl knockdown was performed using lentiviral transduction using mission lentiviral transduction </span><span style=\"background-color: hsl(120, 100.00%, 94.59%); opacity: 0.81\" title=\"0.050\">particles</span><span style=\"opacity: 0.80\"> (sigma-aldrich, st. louis, mo) as per manufacturer&#x27;s instructions. briefly, 1×</span><span style=\"background-color: hsl(120, 100.00%, 99.10%); opacity: 0.80\" title=\"0.004\">105</span><span style=\"opacity: 0.80\"> h358 cells/well were seeded </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> 6-well plates and infected the following day with c-cbl lentiviral shrna constructs. to generate stable c-cbl knockdown cell lines, cells were selected for 2 days with 1 µg/ml puromycin. c-cbl levels were determined using whole cell lysates by immunoblotting with anti-cbl antibody (santa cruz biotechnologies, santa cruz, ca).  cell </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">viability</span><span style=\"opacity: 0.80\"> assay  cells were transfected as described above </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the transfection assay. forty-eight hours after transfection, </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">viability</span><span style=\"opacity: 0.80\"> of cells was assessed using trypan blue exclusion.  wound healing assay  a549 cells were seeded </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> 6-well plates and </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">cultured</span><span style=\"opacity: 0.80\"> for 48 h until 100% confluent. the medium was then changed and the cells were transfected as described </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the transfection assay. twelve hours after transfection, a straight scratch was made across the cell layer using a 1 ml pipette tip. the cells were then gently washed with 1× pbs to remove cellular debris and the media was replaced. photographs were taken of the wound region every 12 h until 48 h.  western blot analysis  forty eight hours after transfection, cells were collected and washed twice </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> 1x pbs, then lysed </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> ice-cold lysis buffer (0.5m tris-hcl with ph 7.4, 1.5 m nacl, 2.5% deoxycholic acid, 10 </span><span style=\"background-color: hsl(0, 100.00%, 98.80%); opacity: 0.80\" title=\"-0.006\">mm</span><span style=\"opacity: 0.80\"> edta, 10% np-40, 0.5 </span><span style=\"background-color: hsl(0, 100.00%, 98.80%); opacity: 0.80\" title=\"-0.006\">mm</span><span style=\"opacity: 0.80\"> dtt, 1 </span><span style=\"background-color: hsl(0, 100.00%, 98.80%); opacity: 0.80\" title=\"-0.006\">mm</span><span style=\"opacity: 0.80\"> phenylmethylsulfonyl fluoride, 5 µg/ml leupeptin, and 10 µg/ml aprotinin) for 5 minutes. the lysate was centrifuged at 13,000 rpm for 20 minutes at 4°c, and protein content of the supernatant was measured. total cell lysates (50 µg/well) were separated by sds-</span><span style=\"background-color: hsl(120, 100.00%, 96.97%); opacity: 0.81\" title=\"0.022\">page</span><span style=\"opacity: 0.80\"> electrophoresis and the gels transferred onto nitrocellulose membranes (whatman, piscataway, nj). membranes were blocked with 5% non-fat dry milk </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> phosphate-buffered saline containing tween-20 (pbst) (1x pbs, 0.1% tween-20) for 1 h at room temperature and incubated with the appropriate primary antibody at 4°c overnight. membranes then were washed three times with pbst and </span><span style=\"background-color: hsl(0, 100.00%, 99.06%); opacity: 0.80\" title=\"-0.004\">probed</span><span style=\"opacity: 0.80\"> with appropriate horseradish peroxidase (hrp)-conjugated secondary antibody for 1 h at room temperature. the membranes were again washed three times </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> pbst and bands were visualized using western blot chemiluminescence reagent (biorad, valencia, ca) on a chemidoc gel documentation system (biorad, valencia, ca). antibodies were obtained from santa cruz biotechnologies and used at the following dilutions (c-cbl, 1∶5000; c-met, 1∶5000; egfr, 1∶5000; ubiquitin, 1∶1000; ha, 1∶5000 and β-actin, 1∶10,000).  flow cytometry  cell cycle analysis was carried out by flow cytometry. approximately 2×106 cells were grown </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> media containing 10% fbs. cells were harvested by trypsin/edta </span><span style=\"background-color: hsl(0, 100.00%, 89.81%); opacity: 0.83\" title=\"-0.123\">treatment</span><span style=\"opacity: 0.80\">, washed with 1x pbs three times and fixed with ice-cold 70% ethanol for 2 h. cells were washed again with cold pbs and stained with a solution containing 25 µg/ml propidium iodide, 200 µg/ml rnase a, and 0.1% triton x-100 for 30 minutes </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the dark. cell cycle analysis was performed using a guava pca-96 flow cytometer (guava technologies, millipore, billerica, ma).  ubiquitin ligase activity  293t cells were maintained </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> culture </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> dmem supplemented with 10% fbs and 1% penicillin (100 units/ml) and streptomycin (100 µg/ml) were transfected with 0.2 µg egfr-pcdna3 and 2 µg ha-tagged c-cbl constructs as indicated using calcium phosphate according to manufacturer&#x27;s protocol (profection, promega, madison, wi). twenty-four hours post-transfection, cells were starved overnight </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> dmem supplemented with 0.5% fbs, and then treated with or without egf (100 ng/ml) for 15 min. the cells were collected and washed two times </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> ice-cold pbs containing 0.2 </span><span style=\"background-color: hsl(0, 100.00%, 98.80%); opacity: 0.80\" title=\"-0.006\">mm</span><span style=\"opacity: 0.80\"> sodium orthovanadate then lysed </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> ice-cold lysis buffer (10 </span><span style=\"background-color: hsl(0, 100.00%, 98.80%); opacity: 0.80\" title=\"-0.006\">mm</span><span style=\"opacity: 0.80\"> tris hcl, ph 7.5, 150 </span><span style=\"background-color: hsl(0, 100.00%, 98.80%); opacity: 0.80\" title=\"-0.006\">mm</span><span style=\"opacity: 0.80\"> nacl, 5 </span><span style=\"background-color: hsl(0, 100.00%, 98.80%); opacity: 0.80\" title=\"-0.006\">mm</span><span style=\"opacity: 0.80\"> edta, 1% triton x100, 10% glycerol, 2 </span><span style=\"background-color: hsl(0, 100.00%, 98.80%); opacity: 0.80\" title=\"-0.006\">mm</span><span style=\"opacity: 0.80\"> sodium orthovanadate and protease inhibitors). lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4°c. egfr immunoprecipitations were performed on 200 µg of cleared lysate using 250 ng of rabbit-anti-egfr and protein a/g plus sepharose overnight at 4°c. precipitations were washed 5 times </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lysis buffer before boiling </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> laemmli buffer. elutions were immunoblotted with anti-ubiquitin and egfr. twenty micrograms of cleared lysate were immunoblotted for each of the c-cbl constructs using anti-ha.  statistical analysis  mutation rates between different groups were compared using fisher&#x27;s exact test. for continuous variables, group comparisons were performed using analysis of variance (anova) followed by sidak&#x27;s adjustment for multiple comparisons. experiments involving repeated measurements over </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.005\">time</span><span style=\"opacity: 0.80\"> were analyzed using repeated measures anova with the greenhouse-geisser adjustment to the degrees of freedom. analyses were conducted using stata (v10.1) software (stata corporation, college station, tx).  go to: results c-cbl gene mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer  to investigate the role of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer, we analyzed its genomic </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> tumor and paired normal samples drawn from multiple ethnicities. the lung tumor samples represented caucasians (n = 50), african-americans (n = 29), and taiwanese (n = 40) lung cancer patients. we designed 12 pairs of primers to sequence the coding region of c-cbl gene that spans </span><span style=\"background-color: hsl(0, 100.00%, 99.57%); opacity: 0.80\" title=\"-0.001\">exons</span><span style=\"opacity: 0.80\"> 2 to 16 (table s2). we identified 8 </span><span style=\"background-color: hsl(120, 100.00%, 97.70%); opacity: 0.80\" title=\"0.015\">unique</span><span style=\"opacity: 0.80\"> somatic mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.57%); opacity: 0.80\" title=\"-0.001\">exons</span><span style=\"opacity: 0.80\"> among 8 different patients. a variation l620f, a </span><span style=\"background-color: hsl(120, 100.00%, 98.15%); opacity: 0.80\" title=\"0.011\">known</span><span style=\"opacity: 0.80\"> snp (rs2227988) </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> exon 11 was also detected. importantly, the eight novel non-synonymous mutations were confirmed by sequencing both strands of c-cbl genomic </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">dna</span><span style=\"opacity: 0.80\"> obtained from lung tumor samples (table 1). moreover, none of the 8 mutations were detected </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the corresponding normal tissue, indicating that these were somatic mutations. four synonymous single nucleotide variations (snvs) were also identified but were not used further </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> this </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\">.  table 1 table 1 c-cbl mutation analysis </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> 119 lung cancer </span><span style=\"background-color: hsl(120, 100.00%, 97.45%); opacity: 0.80\" title=\"0.017\">patient</span><span style=\"opacity: 0.80\"> tumor tissues. three of the 8 novel non-synonymous mutations were located </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the tkb (tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.57%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> binding) domain (s80n, h94y, and q249e), one </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the ring finger domain (v391i), one </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the proline-rich region (72515_72517 del atg), and three </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the c-terminal region (w802*, r830k, and a848t) of the c-cbl protein (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 1a and </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> s1). </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.85%); opacity: 0.80\" title=\"0.013\">1b</span><span style=\"opacity: 0.80\">, we show model chromatograms of representative samples.  </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 1 </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 1 c-cbl mutations and loh </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> non-small cell lung cancer. 11q loh of c-cbl gene  paired lung tumor and normal lung tissue samples from taiwanese patients (n = 37) were investigated for loh. eight (21.6%) showed loh at the c-cbl locus on chromosome 11 while 29 samples (78.4%) revealed normal allelic contribution at the microsatellite markers (figures 1c and d).  c-cbl mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> different ethnic groups  the c-cbl double mutant s80n/h94y was found </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the same </span><span style=\"background-color: hsl(120, 100.00%, 97.45%); opacity: 0.80\" title=\"0.017\">patient</span><span style=\"opacity: 0.80\"> and the overall mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.22%); opacity: 0.80\" title=\"0.003\">rate</span><span style=\"opacity: 0.80\"> for c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung tumors was 6.7% (8/119). the frequency of c-cbl mutation was highest </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> large cell carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the least was observed </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> adenocarcinoma (ad) (1.8%; 1 of 53 patients), although these rates were not statistically significant (p = 0.292). mutation rates were 6.0% among caucasians (0 of 20 </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> ad; 0 of 10 </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> sq; and 3 of 20 </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lc), 13.8% </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> african-americans (1 of 10 </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> ad; 1 of 10 </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> sq; and 2 of 9 </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lc), and 2.5% (0 of 23 </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> ad; 1 of 12 </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> sq; and 0 of 5 </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lc) </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the taiwanese population. additionally two taiwanese patients with lung cancer (one squamous and one adenocarcinoma) had the </span><span style=\"background-color: hsl(120, 100.00%, 98.15%); opacity: 0.80\" title=\"0.011\">known</span><span style=\"opacity: 0.80\"> snp l620f. ethnic differences were not statistically significant, however the power to detect differences was low.  mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> met and egfr can be co-associated with c-cbl alterations  since east asians with lung cancer have a higher frequency of egfr and met mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung tumors [21], [22], we also determined mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> egfr and met </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the same taiwanese cohort samples and compared the results with the observed c-cbl alterations (loh and/or mutations). </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the 37 samples tested, we did not find any overlap between c-cbl mutations and c-cbl loh (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 2). of the three c-cbl mutants (including the </span><span style=\"background-color: hsl(120, 100.00%, 98.15%); opacity: 0.80\" title=\"0.011\">known</span><span style=\"opacity: 0.80\"> l620f snp, rs2227988), one of the samples had a met mutation (n375s) and the other had an egfr mutation (l858r). among the 8 samples that had a loh at the c-cbl locus 5 had an additional mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> met (n375s) and 2 had an egfr exon 19 deletion. twenty-six samples had </span><span style=\"background-color: hsl(120, 100.00%, 95.61%); opacity: 0.81\" title=\"0.037\">neither</span><span style=\"opacity: 0.80\"> c-cbl mutation nor c-cbl loh (3 patients had a c-cbl mutation but no c-cbl loh). among these 26 samples 9 had a met mutation (8 n375s, 1 l211w), 13 had an egfr mutation (7 exon 9 deletion, 6 l858r) and 4 had no other met or egfr mutation. thus the </span><span style=\"background-color: hsl(120, 100.00%, 99.22%); opacity: 0.80\" title=\"0.003\">rate</span><span style=\"opacity: 0.80\"> of met or egfr mutations among patients with loh at the c-cbl locus (7 of 8) was similar to that seen </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> patients without c-cbl mutation or loh (22 of 26 patients) (p = 0.99). these 4 patients with no identifiable mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl, met or egfr represented 10.8% of the 37 patients analyzed </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the taiwanese </span><span style=\"background-color: hsl(120, 100.00%, 97.45%); opacity: 0.80\" title=\"0.017\">patient</span><span style=\"opacity: 0.80\"> cohort. conversely, 89.2% taiwanese lung cancer patients have an identifiable mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> either c-cbl, met or egfr or a combination of the three genes (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 2). additionally, we determined p53 and kras mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> these taiwanese cohorts. two p53 and 1 kras mutation were detected. the single kras mutation overlapped with one p53 mutation. this </span><span style=\"background-color: hsl(120, 100.00%, 97.45%); opacity: 0.80\" title=\"0.017\">patient</span><span style=\"opacity: 0.80\"> also had the egfr exon 19 deletion but had no c-cbl mutation. the other p53 mutation sample had a c-cbl loh with concurrent met n375s mutation. thus, </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the taiwanese samples analyzed, p53/kras mutations and c-cbl mutations were mutually exclusive (data not shown).  </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 2 </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 2 c-cbl mutations and relationship to met and egfr mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer. cellular functions of c-cbl alterations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the context of lung tumorigenesis  a. </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> activity is intact </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the mutant c-cbl proteins to investigate whether the different c-cbl mutations affect the </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> activity, egfr was chosen as a model substrate for c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.63%); opacity: 0.80\" title=\"0.015\">function</span><span style=\"opacity: 0.80\">. all of the c-cbl mutants tested enhanced ubiquitination of the activated egfr similar to the </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">wild</span><span style=\"opacity: 0.80\">-type c-cbl protein. this result demonstrates that the catalytic activity of the c-cbl mutants is not impaired when egfr was the substrate. (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 3a).  </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 3 </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 3 ubiquitination, </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">viability</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(0, 100.00%, 99.46%); opacity: 0.80\" title=\"-0.002\">expression</span><span style=\"opacity: 0.80\"> and cell cycle analysis of various c-cbl mutants. b. effect on lung cancer cell </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">viability</span><span style=\"opacity: 0.80\"> the effect of a representative c-cbl mutant from each of the three ethnic backgrounds on lung cancer cell </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">viability</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> cell lines was determined. s80n/h94y double mutation, q249e, and w802* were identified </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung tumor samples obtained from a caucasian, a taiwanese and an african-american, respectively. as described </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> methods, the c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">wild</span><span style=\"opacity: 0.80\">-type (</span><span style=\"background-color: hsl(120, 100.00%, 99.52%); opacity: 0.80\" title=\"0.002\">wt</span><span style=\"opacity: 0.80\">) and the above three mutants were expressed after cloning them into paltermax vector </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a549 cells. these cells express relatively low basal levels of endogenous c-cbl (data not shown). transfection efficiency was comparable between different groups and the number of cells transfected with c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">wild</span><span style=\"opacity: 0.80\">-type construct was about 70% compared to control cells that were transfected with the empty vector. cells transfected with s80n/h94y, q249e and w802* c-cbl mutant constructs resulted </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> increased number of viable cells that was 132.3%, 120.8% and 147.9% higher respectively, relative to the empty vector control transfected cells and significantly different from the </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">wild</span><span style=\"opacity: 0.80\">-type construct (p = 0.022, p = 0.049, and p = 0.008, respectively) (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 3b). relative levels of c-cbl protein </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> whole cell lysates prepared from samples obtained from a parallel experiment was determined. the c-cbl protein levels </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> samples representing untransfected and empty vector transfected cells were comparable and those representing the c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.52%); opacity: 0.80\" title=\"0.002\">wt</span><span style=\"opacity: 0.80\"> and the three c-cbl mutants were comparable (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 3c).  c. effect on cell cycle to investigate if the increases </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> cell </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">viability</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> different c-cbl mutants are due to increased cellular proliferation, a cell cycle analysis was performed. a549 cells were transfected with the c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.52%); opacity: 0.80\" title=\"0.002\">wt</span><span style=\"opacity: 0.80\"> or the three different mutants: s80n/h94y, q249e and w802*. the empty vector transfectant was used as a control. forty-eight hours after transfection cell cycle analysis was performed as described </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.23%); opacity: 0.80\" title=\"0.003\">materials</span><span style=\"opacity: 0.80\"> and methods. there was no significant change </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the subg1, g1 or the s phase of the cell cycle among the different mutants compared to the </span><span style=\"background-color: hsl(120, 100.00%, 99.52%); opacity: 0.80\" title=\"0.002\">wt</span><span style=\"opacity: 0.80\"> construct (p = 0.64, p = 0.40, and p = 0.28, respectively). the g2/m phase of the cell cycle showed an increase </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> cell numbers for the three mutants, s80n/h94y, q249e and w802*, when compared to the </span><span style=\"background-color: hsl(120, 100.00%, 99.52%); opacity: 0.80\" title=\"0.002\">wt</span><span style=\"opacity: 0.80\"> but again the difference was not statistically significant (p = 0.25) (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 3d).  d. effect on cell motility to investigate the effect of the </span><span style=\"background-color: hsl(0, 100.00%, 99.46%); opacity: 0.80\" title=\"-0.002\">expression</span><span style=\"opacity: 0.80\"> of the above three c-cbl mutants on cell migration, we carried out wound healing assay as described </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.23%); opacity: 0.80\" title=\"0.003\">materials</span><span style=\"opacity: 0.80\"> and methods. the closing of the scratch or the wound was monitored at 0, 12, 24, 36, and 48 h. (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 4a). </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> all the samples, that represented cells transfected with mutants, the wound gap was much smaller than that seen </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the sample that represented cells transfected with c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.52%); opacity: 0.80\" title=\"0.002\">wt</span><span style=\"opacity: 0.80\"> (p&lt;0.001). we also determined the </span><span style=\"background-color: hsl(120, 100.00%, 99.22%); opacity: 0.80\" title=\"0.003\">rate</span><span style=\"opacity: 0.80\"> of wound closure for all the five groups. at 48 h, </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">wild</span><span style=\"opacity: 0.80\">-type c-cbl transfectants showed 61.1% open wound while the s80n/h94y, q249e and w802* mutants showed 18.7%, 23.9% and 34.3% open wound respectively (p&lt;0.001) (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 4b).  </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 4 </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 4 c-cbl mutations affect wound healing </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a549 cells. e. c-cbl knockdown increases cell </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">viability</span><span style=\"opacity: 0.80\"> it is hypothesized that the loh seen </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> samples could lead to decreased </span><span style=\"background-color: hsl(0, 100.00%, 99.46%); opacity: 0.80\" title=\"-0.002\">expression</span><span style=\"opacity: 0.80\"> of c-cbl. thus we tested the effect of c-cbl knockdown </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer cells. compared to a549, h358 lung cancer cells express relatively high levels of endogenous c-cbl (data not shown). c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.46%); opacity: 0.80\" title=\"-0.002\">expression</span><span style=\"opacity: 0.80\"> was knocked down using lentiviral construct that expressed c-cbl specific shrna and compared the results with those that were transduced with scrambled shrna and the results are shown </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 5. we identified several clones that revealed varying degrees of c-cbl knockdown showing different sets of c-cbl lentiviral shrna knockdown efficiency (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 5a). of all the clones tested, clone 27 was chosen for further experiments. equal amount of cells were seeded </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a 6-well plate and the cell proliferation was measured at various times and the results are depicted </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 5b. as expected, number of cells increased </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.005\">time</span><span style=\"opacity: 0.80\"> dependent fashion from 100 to 190% relative to scrambled shrna as control </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a span of 48 h (p = 0.0002) (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 5b). the cell cycle phases </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> h358 cells that were knocked down with c-cbl shrna were looked at and compared with the scrambled shrna. there were no discernable differences between these two constructs </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the different phases of the cell cycle (data not shown).  </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 5 </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> 5 knockdown of c-cbl using an shrna increases cell proliferation. go to: discussion </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> results demonstrate that c-cbl is somatically mutated (or has loh) </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers, and can significantly contribute to enhanced cell </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">viability</span><span style=\"opacity: 0.80\"> and motility. there was also a high prevalence of loh with respect to c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung tumors that harbored met or egfr mutation.  </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the present </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\">, we have demonstrated the occurrence of c-cbl mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer patients, especially with different ancestral variations. mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl have been recently reported </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> juvenile myelomonocytic leukemia and myeloid malignancies. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the aml </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\">, the mutation r420q located </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the junction of the ring finger and the </span><span style=\"background-color: hsl(120, 100.00%, 99.50%); opacity: 0.80\" title=\"0.002\">linker</span><span style=\"opacity: 0.80\"> region inhibited fms-like tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.57%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> 3 (flt3) internalization and ubiquitination [20], thus contributing to the gain-</span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\">-</span><span style=\"background-color: hsl(120, 100.00%, 97.63%); opacity: 0.80\" title=\"0.015\">function</span><span style=\"opacity: 0.80\"> for the rtk. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> addition, mutations such as h398y, c384r, and l380p mapped to the ring finger domain and the </span><span style=\"background-color: hsl(120, 100.00%, 99.50%); opacity: 0.80\" title=\"0.002\">linker</span><span style=\"opacity: 0.80\"> region of c-cbl that is required for its </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> activity [23], [24], [25], [26], [27]. additionally, homozygous mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the ring finger domain of the c-cbl gene were described as a result of </span><span style=\"background-color: hsl(0, 100.00%, 98.92%); opacity: 0.80\" title=\"-0.005\">acquired</span><span style=\"opacity: 0.80\"> uniparental disomy (upd) [26]. it is important to note that </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> results indicate loh at 11q23 locus and these are mutually exclusive from </span><span style=\"background-color: hsl(0, 100.00%, 98.76%); opacity: 0.80\" title=\"-0.006\">missense</span><span style=\"opacity: 0.80\"> mutations of c-cbl. the somatic mutations were all heterozygous. the mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> aml led to abrogation of the </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> activity leading to prolonged rtk </span><span style=\"background-color: hsl(120, 100.00%, 99.53%); opacity: 0.80\" title=\"0.002\">activation</span><span style=\"opacity: 0.80\">. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> addition mutants located on the </span><span style=\"background-color: hsl(120, 100.00%, 99.50%); opacity: 0.80\" title=\"0.002\">linker</span><span style=\"opacity: 0.80\"> region surrounding the ring finger domain exhibited enhanced akt signaling </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 94.27%); opacity: 0.81\" title=\"-0.054\">response</span><span style=\"opacity: 0.80\"> to cytokine stimulation [26]. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> addition, it was shown </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> nh3t3 cells, that </span><span style=\"background-color: hsl(120, 100.00%, 95.61%); opacity: 0.81\" title=\"0.037\">neither</span><span style=\"opacity: 0.80\"> mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the ring finger nor the </span><span style=\"background-color: hsl(120, 100.00%, 99.50%); opacity: 0.80\" title=\"0.002\">linker</span><span style=\"opacity: 0.80\"> region causes transformation, however while certain mutations perturbs the ubiquitination, others affect receptor recycling and prolong </span><span style=\"background-color: hsl(120, 100.00%, 99.57%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity [28].  we report here c-cbl mutations that mapped not only to the ring finger domain, but also to the tkb domain, proline-rich domain and the c-terminal region, but none mapped to the </span><span style=\"background-color: hsl(120, 100.00%, 99.50%); opacity: 0.80\" title=\"0.002\">linker</span><span style=\"opacity: 0.80\"> region as reported </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the aml studies described above [23], [24], [25], [26], [29]. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> addition, 8 mutants that we detected were found </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> different ethnic backgrounds. for example, s80n/h94y, q249e, w802* were detected </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> caucasians, taiwanese and african-americans, respectively. the results point out not only the difference between lung cancer and other cancers, but also genetic polymorphism among different races </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the same cancer. interestingly, there is a large disparity between african-american and other ethnic populations with lung cancer [30]. we have previously shown that there was a low frequency of egfr and met mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> african-americans as compared with taiwanese and caucasians [31]. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> this </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\">, the number of african-american samples analyzed was relatively fewer and we found 3 mutations that are </span><span style=\"background-color: hsl(120, 100.00%, 97.70%); opacity: 0.80\" title=\"0.015\">unique</span><span style=\"opacity: 0.80\"> to this ethnicity. it </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.005\">would</span><span style=\"opacity: 0.80\"> now behoove us to further </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\"> the genetic alterations that can occur and </span><span style=\"background-color: hsl(120, 100.00%, 99.34%); opacity: 0.80\" title=\"0.002\">determine</span><span style=\"opacity: 0.80\"> the targeted therapeutics for african-americans. </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> results provide evidence of the importance of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> tumorigenesis and potential signaling. </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> prediction, based on the aml data, </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.005\">would</span><span style=\"opacity: 0.80\"> be that the v391i ring finger domain mutation </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.005\">would</span><span style=\"opacity: 0.80\"> affect the </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> activity. also, it will be important to </span><span style=\"background-color: hsl(120, 100.00%, 99.34%); opacity: 0.80\" title=\"0.002\">determine</span><span style=\"opacity: 0.80\"> the binding partners for c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the tkb domain and proline rich domain mutations. it has previously been shown that the tkb domain can bind to </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">growth</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.64%); opacity: 0.80\" title=\"0.007\">factor</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">receptors</span><span style=\"opacity: 0.80\"> and it will be important to </span><span style=\"background-color: hsl(120, 100.00%, 99.34%); opacity: 0.80\" title=\"0.002\">determine</span><span style=\"opacity: 0.80\"> the cross-binding of these mutants to met and or egfr. it </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.005\">would</span><span style=\"opacity: 0.80\"> also be important </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the future to look at fluorescence </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> situ hybridization/copy number changes </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer.  c-cbl plays an important role </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> down regulating rtk-mediated signaling through k63 poly-ubiquitination and subsequent downregulation of rtks followed by lysosomal degredation [3]. mono-ubiquitination or ubiquitinated with k63-linked chains of substrates by c-cbl may lead to enhancement of biological and biochemical functions (reviewed </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> hermann et </span><span style=\"background-color: hsl(120, 100.00%, 96.93%); opacity: 0.81\" title=\"0.022\">al</span><span style=\"opacity: 0.80\">, 2007 [32]). the mutations that we analyzed </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> studies all point out to the fact that the </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> activity of c-cbl on egfr is intact; the egfr levels </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the various mutants remain same (</span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">figure</span><span style=\"opacity: 0.80\"> s2). multiple </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.003\">kinases</span><span style=\"opacity: 0.80\">, both rtks and non-rtks could be acted upon by c-cbl, including erbs, pdgfr, fms, met, c-kit, vegfr, flt-1, ron, fgfr, ir, as well as syk, fyn, lck, fgr, lyn and c-</span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.002\">abl</span><span style=\"opacity: 0.80\"> [3]. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers the relevant substrates of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> terms of degradation or signal transduction are yet to be identified.  the observation that c-cbl somatic mutations, especially s80n/h94y, q249e and w802* showed increased cell </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">viability</span><span style=\"opacity: 0.80\"> and cell motility that is </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> agreement with the physiological role for d-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the regulation of apoptosis and differentiation identified </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> drosophila is very significant [33]. it has been previously shown that </span><span style=\"background-color: hsl(120, 100.00%, 95.85%); opacity: 0.81\" title=\"0.034\">activating</span><span style=\"opacity: 0.80\"> c-cbl mutation downregulates egfr signaling and decreases cellular proliferation and migration </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> breast cancer cell lines [34]. although the role of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the negative regulation of rtks is well substantiated, thereby suggesting that it is a natural tumor </span><span style=\"background-color: hsl(0, 100.00%, 93.12%); opacity: 0.82\" title=\"-0.070\">suppressor</span><span style=\"opacity: 0.80\">, studies </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> cancer cells have revealed both tumor </span><span style=\"background-color: hsl(0, 100.00%, 93.12%); opacity: 0.82\" title=\"-0.070\">suppressor</span><span style=\"opacity: 0.80\"> and tumor promoting activities depending on the type of c-cbl mutation and the number of alleles at the c-cbl locus [24]. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> agreement with the above, the three c-cbl mutants described here appear to have tumor </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">growth</span><span style=\"opacity: 0.80\"> and metastasis promoting properties. although these mutants are outside of the ring finger or the </span><span style=\"background-color: hsl(120, 100.00%, 99.50%); opacity: 0.80\" title=\"0.002\">linker</span><span style=\"opacity: 0.80\"> region of c-cbl, their downstream </span><span style=\"background-color: hsl(120, 100.00%, 91.15%); opacity: 0.82\" title=\"0.101\">effects</span><span style=\"opacity: 0.80\"> are significant so as to cause increased proliferation and migration, but the substrate affected by these mutations are not </span><span style=\"background-color: hsl(120, 100.00%, 98.15%); opacity: 0.80\" title=\"0.011\">known</span><span style=\"opacity: 0.80\"> yet. this raises the possibility that some of the cellular functions of c-cbl are independent of its ubiquitin-ligase activity, an area that we are currently investigating. the </span><span style=\"background-color: hsl(120, 100.00%, 96.54%); opacity: 0.81\" title=\"0.026\">oncogenic</span><span style=\"opacity: 0.80\"> nature of rtks, addiction of cancers to </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">growth</span><span style=\"opacity: 0.80\"> signals and given the clustering of c-cbl, egfr and met mutations, it is possible that the transforming effect of c-cbl mutations is most likely a combinatorial effect of the three. we also show that loh for c-cbl was found </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a significant number of samples that harbored met or egfr mutations. the fact that about 7% of lung tumor samples are likely to have c-cbl mutations and an additional 22% are likely to harbor c-cbl-related loh, makes c-cbl a highly mutated molecule </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer. since loh alone is not enough to cause a transforming event [35], [36], [37], associated mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the met or egfr locus or yet another rtk discussed above may </span><span style=\"background-color: hsl(120, 100.00%, 97.68%); opacity: 0.80\" title=\"0.015\">play</span><span style=\"opacity: 0.80\"> a role </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> carcinogenesis. we predict that this loh </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl results </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> haploinsufficiency that downplays rtk ubiquitination leading to hyperactivity of the rtks. however, whether this is sufficient cause for tumorigenesis remains to be determined. consistent with </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> hypothesis is the fact that c-cbl-/- mice have increased </span><span style=\"background-color: hsl(120, 100.00%, 99.57%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lymphocytes, but is not sufficient for tumor formation [35], [36], [37]. c-cbl loh could also lead to increased </span><span style=\"background-color: hsl(0, 100.00%, 99.46%); opacity: 0.80\" title=\"-0.002\">expression</span><span style=\"opacity: 0.80\"> of c-cbl from the other </span><span style=\"background-color: hsl(120, 100.00%, 98.95%); opacity: 0.80\" title=\"0.005\">allele</span><span style=\"opacity: 0.80\"> to compensate for a loss of an </span><span style=\"background-color: hsl(120, 100.00%, 98.95%); opacity: 0.80\" title=\"0.005\">allele</span><span style=\"opacity: 0.80\">. alternately, there could be some form of synergy working with reduced c-cbl levels and mutated </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">receptors</span><span style=\"opacity: 0.80\"> that exacerbate the phenotype of each alone.  previous studies from </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> lab and others have shown that east asians with lung cancers have relatively high frequencies of gain-of-</span><span style=\"background-color: hsl(120, 100.00%, 97.63%); opacity: 0.80\" title=\"0.015\">function</span><span style=\"opacity: 0.80\"> of mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> rtks such as egfr and met [31]. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a cohort of japanese patients an </span><span style=\"background-color: hsl(120, 100.00%, 95.85%); opacity: 0.81\" title=\"0.034\">activating</span><span style=\"opacity: 0.80\"> met mutation was identified </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(120, 100.00%, 96.22%); opacity: 0.81\" title=\"0.030\">splice</span><span style=\"opacity: 0.80\"> region that deletes the juxtamembrane domain that is involved </span><span style=\"background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80\" title=\"0.009\">e3</span><span style=\"opacity: 0.80\"> activity of c-cbl [38]. this </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\"> also found that </span><span style=\"background-color: hsl(120, 100.00%, 99.53%); opacity: 0.80\" title=\"0.002\">activation</span><span style=\"opacity: 0.80\"> of met is mutually exclusive of egfr, kras and her2 gene mutations [38]. we failed to detect such mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> significant numbers </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung tumor samples obtained from african-americans (n = 29) and caucasian (n = 50) patients. one met mutation was identified </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> each of the groups whereas 1 and 3 egfr mutations were identified </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the african-american and caucasian cohorts respectively. egfr mutations have earlier been identified as one of the key mutations affecting lung adenocarcinoma patients </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a comprehensive </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\"> of 188 patients [39]. </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\"> encompasses different histologies of nsclc. however, the published series did not find any mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl or met unlike </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\"> that encompassed different subtypes of nsclc. it is important to note, we have recently shown that met mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer are </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> majority germline [31]. we have reported earlier c-cbl mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> a small cohort of taiwanese lung cancer samples [40]. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> efforts to understand the ethnic differences </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the lung oncogenome, we also looked at pax transcription </span><span style=\"background-color: hsl(0, 100.00%, 99.18%); opacity: 0.80\" title=\"-0.003\">factors</span><span style=\"opacity: 0.80\"> such as pax5 and pax8 that are highly expressed </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers; however there was no preferential </span><span style=\"background-color: hsl(0, 100.00%, 99.46%); opacity: 0.80\" title=\"-0.002\">expression</span><span style=\"opacity: 0.80\"> or mutations of the above genes </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung tumor samples of african-americans. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> this </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\">, we show relatively high frequency of c-cbl mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers, especially </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> the large cell type among caucasians and particularly among african-americans. we therefore propose c-cbl as an </span><span style=\"background-color: hsl(120, 100.00%, 91.29%); opacity: 0.82\" title=\"0.099\">efficacious</span><span style=\"opacity: 0.80\"> target for lung cancers </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> african-americans that needs to be further substantiated. this is all the more important because the prognosis for african-americans with lung cancer, especially for men is much poorer compared to their caucasian counterparts [41].  </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> conclusion, the results presented </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> this </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">study</span><span style=\"opacity: 0.80\"> demonstrate that c-cbl is frequently mutated or even lost </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers. </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">our</span><span style=\"opacity: 0.80\"> results support a role for c-cbl mutants that are independent of its ubiquitination activity. given the relatively high mutation rates </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl as well as rtks such as met and egfr, it is likely that their combined effect could be synergistic </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">in</span><span style=\"opacity: 0.80\"> promoting tumorigenesis.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=3\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.006</b>, score <b>-3.175</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 97.35%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.189\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__studied\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.46%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.019\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__core\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.50%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__roles\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__full\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__characteristic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__features\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__4c\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__intensity\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__modification\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mechanisms\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__provides\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__use\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__immediately\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inc\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__detailed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__initiation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__see\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__16p13\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__y105c\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__394\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__characterised\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__reliant\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__383\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__recognizing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__dimer\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__spanning\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__autoinhibited\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__controls\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__characterise\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__characterisation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__transactivation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__deficient\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bis\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.56%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__where\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cerevisiae\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.50%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__soda\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.45%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__gfp\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.25%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.031\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inhibition\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.18%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.035\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__126\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.16%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.036\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__tuberous\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.16%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.037\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__second\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.12%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.039\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__phosphorylation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.04%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.044\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__reports\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.058\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fully\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 89.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -1.323\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 87.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -1.741\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">w802*</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">cbl</span><br/><b>Text:</b> <span style=\"opacity: 0.80\"> abstract background  non-small cell lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> (nsclc) is a heterogeneous group of disorders with a number of </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">genetic</span><span style=\"opacity: 0.80\"> and proteomic alterations. c-cbl is an e3 ubiquitin ligase and adaptor molecule important in normal homeostasis and </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">. we determined the </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">genetic</span><span style=\"opacity: 0.80\"> variations of c-cbl, relationship to receptor tyrosine kinases (egfr and met), and functionality in nsclc.  methods and findings  using archival formalin-fixed paraffin embedded (ffpe) extracted genomic dna, we show that c-cbl mutations occur in somatic fashion for lung cancers. c-cbl mutations were </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> mutually exclusive of met or egfr mutations; however they were independent of p53 and kras mutations. in normal/tumor pairwise analysis, there was significant loss of heterozygosity (loh) for the c-cbl locus (22%, n = 8/37) and none of these samples revealed any mutation in the remaining copy of c-cbl. the c-cbl loh also positively correlated with egfr and met mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.22%); opacity: 0.80\" title=\"0.003\">observed</span><span style=\"opacity: 0.80\"> in the same samples. using select c-cbl somatic mutations such as s80n/h94y, q249e and w802* (obtained from caucasian, taiwanese and african-american samples, respectively) transfected in nsclc cell lines, there was increased cell viability and cell motility.  conclusions  taking the overall mutation rate of c-cbl to be a combination as somatic missense mutation and loh, it is clear that c-cbl is highly mutated in lung cancers and may play an essential role in lung tumorigenesis and metastasis.  go to: introduction in the us alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which more than 145,000 of them succumb to the disease [1]. this number is roughly equivalent to the combined mortality rates of cancers of the breast, prostate, colon, liver, kidney and melanoma [1]. in addition the prognosis is usually poor and the five-year survival rate is less than 15%. there are also significant ethnic differences for lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">, and the outcome is worse for blacks compared to whites. gender differences are also striking with women having significantly better prognosis as compared to men. there are a number of </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">genetic</span><span style=\"opacity: 0.80\"> alterations that can occur in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">. as an example, in nsclc, mutations in kras, p53, egfr and met have been identified. many of these pathways, especially receptor tyrosine kinases (rtks) are controlled by c-cbl.  cbl (casitas b-lineage lymphoma) is a mammalian gene located on human chromosome 11q23.3 [2] and is involved in cell signaling and protein ubiquitination [3]. cbl proteins belong to the ring finger class of ubiquitin ligases (e3) and there are three homologues c-cbl, cbl-b, cbl-3 [4]. the c-cbl and cbl-b genes are ubiquitously expressed with the highest levels in hematopoietic tissues [5]. c-cbl consists of four regions encoding for functionally distinct protein domains: the n-terminal tyrosine kinase binding (tkb) domain, the linker region, the catalytic ring finger domain, the proline-rich region and the c-terminal ubiquitin-associated (uba) domain that also overlaps with a leucine-zipper (lz) domain [3]. both tkb and ring finger domains are essential for ligand-induced ubiquitination of rtks [6], [7], [8], [9]. the ring finger domain is required for the recruitment of e2 ubiquitin-conjugating enzymes. the tkb domain includes four-helix bundle (4h), a calcium-biding ef hand, and a modified sh2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. in addition, the proline-rich region of c-cbl can associate with the sh3 domain of grb2, which can indirectly recruit c-cbl to rtks via the grb2 adaptor protein [7], [13], [14].  c-cbl also binds to egfr and acts as the e3 that targets egfr for ubiquitination and </span><span style=\"background-color: hsl(0, 100.00%, 97.57%); opacity: 0.80\" title=\"-0.016\">degradation</span><span style=\"opacity: 0.80\">. furthermore, cbl desensitizes egf signaling and opposes cellular proliferation induced by egf [15]. egf activation also appears to activate the tyrosine kinase src, which phosphorylates c-cbl and in turn activates the ubiquitination and </span><span style=\"background-color: hsl(0, 100.00%, 97.57%); opacity: 0.80\" title=\"-0.016\">degradation</span><span style=\"opacity: 0.80\"> of egfr [16], [</span><span style=\"background-color: hsl(120, 100.00%, 98.05%); opacity: 0.80\" title=\"0.012\">17</span><span style=\"opacity: 0.80\">], [18]. a recent study shows that </span><span style=\"background-color: hsl(0, 100.00%, 98.52%); opacity: 0.80\" title=\"-0.008\">defective</span><span style=\"opacity: 0.80\"> endocytosis of egfr is characterized by a deletion mutant and the point mutation l858r, whereby its association with c-cbl and subsequent ubiquitination are impaired [19]. recently, the first human c-cbl mutations were reported in acute myeloid leukemia (aml) patients [20]. the mutation r420q inhibits fms-like tyrosine kinase 3 (</span><span style=\"background-color: hsl(0, 100.00%, 94.14%); opacity: 0.81\" title=\"-0.056\">flt3</span><span style=\"opacity: 0.80\">) internalization and ubiquitination [20].  </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> only can e3 activity be important in oncogenesis, c-cbl has a dual but separate function as a signal transduction molecule. we have previously shown that c-cbl is important in binding crkl and bcr/abl in hematopoietic cells. also, it can bind and modulate functions of cytoskeleton by binding to proteins like talin and paxillin. the tkb domain is important in binding to a number of molecules, and they then function in signal transduction.  given the critical role of cbl in normal homeostasis and </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">, we hypothesized that it might be mutated in lung cancers. in this study, we report novel c-cbl somatic mutations s80n/h94y, q249e and w802* in caucasian, taiwanese and african-american lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> patients, respectively. expressing these mutations in nsclc cell lines lead to increased proliferation and cell motility. we show that c-cbl mutations occur with or without met or egfr mutations but are mutually exclusive of a loh at the c-cbl locus. additionally, c-cbl loh is associated with either met or egfr mutations. we thus hypothesize that c-cbl mutations might contribute to the oncogenic potential of met and egfr in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">.  go to: methods ethics statement  written consent on all research on human subjects has been obtained from the institutional review board, university of chicago and covers all research performed in the laboratory. the following is their contact information: institutional review board, the university of chicago, mcgiffert hall, 5751 s. woodlawn ave., 2nd floor, chicago, </span><span style=\"background-color: hsl(0, 100.00%, 96.55%); opacity: 0.81\" title=\"-0.026\">il</span><span style=\"opacity: 0.80\"> 60637. written informed consents were received from all patients whose tissue samples were used for this study.  tissue samples  lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> tissue and paired adjacent normal lung tissues were obtained from 50 caucasian, 29 african-americans and 40 taiwanese nsclc patients who were recruited at the university of chicago hospital (chicago, usa) (caucasian and african-american patients) and taipei veterans general hospital of taiwan (taiwanese patients) after obtaining appropriate institutional review board permission and informed consent from the patients. out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 years. in terms of tumor types, 53 were adenocarcinoma, 32 were squamous cell carcinoma and 34 were </span><span style=\"background-color: hsl(120, 100.00%, 95.93%); opacity: 0.81\" title=\"0.033\">large</span><span style=\"opacity: 0.80\"> cell carcinoma. 49 were stage i, 14 were stage ii, 34 were stage iii, and 13 were stage iv (table s1).  cell culture  human non-small cell lung carcinoma cells a549 and h358 were maintained in dmem and rpmi-1640, respectively. human embryonic kidney </span><span style=\"background-color: hsl(0, 100.00%, 98.92%); opacity: 0.80\" title=\"-0.005\">293t</span><span style=\"opacity: 0.80\"> cells were cultured in dmem. media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (invitrogen, carlsbad, ca). cells were cultured at 37°c in a humidified incubator containing 5% co2.  c-cbl gene mutational analysis  exons 2 to 16 of c-cbl gene were individually amplified by polymerase </span><span style=\"background-color: hsl(120, 100.00%, 96.66%); opacity: 0.81\" title=\"0.025\">chain</span><span style=\"opacity: 0.80\"> reaction (pcr). primers are listed in table s2. pcr conditions were 1 </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> of 95°c for 5 minutes; 35 cycles of 94°c for 30 seconds, 58°c for 30 seconds and 72°c for 2 minutes; and one </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> of 72°c for 10 minutes. pcr products were treated with exosap-it (usb corporation, cleveland, oh) and sequenced by big-dye terminator chemistry (</span><span style=\"background-color: hsl(0, 100.00%, 97.67%); opacity: 0.80\" title=\"-0.015\">applied</span><span style=\"opacity: 0.80\"> biosystems, foster city, ca). sequencing was performed on the forward coding strand with confirmation of c-cbl alterations performed by sequencing the </span><span style=\"background-color: hsl(0, 100.00%, 95.47%); opacity: 0.81\" title=\"-0.039\">reverse</span><span style=\"opacity: 0.80\"> strand as well. chromatograms were </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">analyzed</span><span style=\"opacity: 0.80\"> for mutations using mutation surveyor v2.61 (softgenetics, state college, pa).  plasmid constructs and site-directed mutagenesis  the wild-type c-cbl cdna insert was subcloned into the paltermax expression vector using xhoi and sali restriction enzyme sites (promega, madison, wi). using this parental plasmid paltermax-c-cbl, the tkb domain double mutation (s80n/h94y), the point mutation (q249e), and the c-terminal point mutation w802* of c-cbl were created using the following primers: 5′-gctggcgctaaagaataacccaccttatatcttagac-3′ and 5′-ctaccagatacctaccagtatctccgtactatcttgtc-3′ for the double mutation s80n/h94y; 5′-ctttacccgactctttgagccctggtcctctttgc-3′ for q249e, and 5′-cagctcctcctttggctgattgtctctggatggtgatc-3′ for w802* along with their complementary primers using the quickchange site-directed mutagenesis xl kit (stratagene, la jolla, ca) according to the manufacturer&#x27;s instructions. the constructs were confirmed for the point mutations by standard dna sequencing of both strands.  loss of heterozygosity (loh) analysis  five microsatellites on chromosome 11 (3 on 11q at or within 200 kb up or downstream of the c-cbl gene and 2 control markers on 11p) were selected for analysis (table s3). established microsatellite markers and respective primer sequences were selected from the geneloc database (http://genecards.weizmann.ac.</span><span style=\"background-color: hsl(0, 100.00%, 96.55%); opacity: 0.81\" title=\"-0.026\">il</span><span style=\"opacity: 0.80\">/geneloc/index.shtml, weizmann institute of science, rehovot, israel). primers were custom designed and each forward primer was fluorescently labeled at the 5′ end with fam, pet, ned, or vic (</span><span style=\"background-color: hsl(0, 100.00%, 97.67%); opacity: 0.80\" title=\"-0.015\">applied</span><span style=\"opacity: 0.80\"> biosystems). primer annealing temperatures and duplex scores were evaluated with nist primer tools (http://yellow.nist.gov:8444/dnaanalysis/primertoolspage.do; national institute of standards and technology, gaithersburg, md). primers were verified by performing pcr with control dna (isolated from tk6 cells) and resolving the products on agarose gels. bands were visualized with an uv transilluminator. genomic dna was extracted from tumor samples and paired normal lung tissue. primers were grouped into multiplex combinations shown in table s4. marker d11s929 served as an internal control to check for consistency in pcrs and of peaks from capillary electrophoresis. multiplex pcrs were </span><span style=\"background-color: hsl(0, 100.00%, 99.63%); opacity: 0.80\" title=\"-0.001\">carried</span><span style=\"opacity: 0.80\"> out in a volume of 10 µl that </span><span style=\"background-color: hsl(120, 100.00%, 95.53%); opacity: 0.81\" title=\"0.038\">contained</span><span style=\"opacity: 0.80\"> 1 µl genomic dna (20–50 ng), 0.5 µm of each primer (1.0 µm total for each primer pair), 400 µm dntps, 1x pcr buffer containing mgcl2, and 0.2 u taq dna polymerase. pcr was performed on the abi geneamp 9700 pcr system under the following conditions: 5 min at 94°c; 30 cycles of 30 sec at 94°c, 1 min at 60°c, 1 min at 72°c; and 5 min at 72°c. the pcr products were separated by capillary electrophoresis on an abi 3130xl dna analyzer. chromatograms were </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">analyzed</span><span style=\"opacity: 0.80\"> with peak scanner 1.0 and genemapper 3.7 software (</span><span style=\"background-color: hsl(0, 100.00%, 97.67%); opacity: 0.80\" title=\"-0.015\">applied</span><span style=\"opacity: 0.80\"> biosystems) for allelic alterations. the area of the peaks produced by the dna pcr products was quantified for each allele. the ratio of the allelic areas was calculated for each tumor and paired normal dna sample. when the qloh (allelic ratio for the tumor peaks divided by the allelic ratio of paired normal sample) was ≤0.5 or ≥2.0 for c-cbl and at </span><span style=\"background-color: hsl(0, 100.00%, 99.53%); opacity: 0.80\" title=\"-0.002\">least</span><span style=\"opacity: 0.80\"> one other 11q marker in at </span><span style=\"background-color: hsl(0, 100.00%, 99.53%); opacity: 0.80\" title=\"-0.002\">least</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">two</span><span style=\"opacity: 0.80\"> separate </span><span style=\"background-color: hsl(120, 100.00%, 99.76%); opacity: 0.80\" title=\"0.001\">experiments</span><span style=\"opacity: 0.80\">, the sample was considered as having an allelic imbalance and interpreted as loh. samples were evaluated in at </span><span style=\"background-color: hsl(0, 100.00%, 99.53%); opacity: 0.80\" title=\"-0.002\">least</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">two</span><span style=\"opacity: 0.80\"> separate </span><span style=\"background-color: hsl(120, 100.00%, 99.76%); opacity: 0.80\" title=\"0.001\">experiments</span><span style=\"opacity: 0.80\"> and samples showing prospective loh at c-cbl repeated a third time which included a new control marker at the bax locus (data </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> shown) on chromosome 19 to verify integrity of sample dna.  transfection of c-cbl constructs  the a549 cell line was transfected using the fugene hd (roche, nutley, nj) reagent according to the manufacturer&#x27;s instructions. eight µg of plasmid dna, containing either no insert (empty vector), wild-type c-cbl, s80n/h94y c-cbl, q249e c-cbl or w802* cbl was used for transfection in a 6-well culture plate. cells were harvested 48 h after transfection and </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">analyzed</span><span style=\"opacity: 0.80\"> for expression.  c-cbl knockdown  c-cbl knockdown was performed using lentiviral transduction using mission lentiviral transduction particles (sigma-aldrich, st. louis, mo) as per manufacturer&#x27;s instructions. briefly, 1×105 h358 cells/well were </span><span style=\"background-color: hsl(120, 100.00%, 98.87%); opacity: 0.80\" title=\"0.005\">seeded</span><span style=\"opacity: 0.80\"> in 6-well plates and </span><span style=\"background-color: hsl(0, 100.00%, 84.97%); opacity: 0.85\" title=\"-0.215\">infected</span><span style=\"opacity: 0.80\"> the following day with c-cbl lentiviral shrna constructs. to generate </span><span style=\"background-color: hsl(0, 100.00%, 88.80%); opacity: 0.83\" title=\"-0.141\">stable</span><span style=\"opacity: 0.80\"> c-cbl knockdown cell lines, cells were selected for 2 days with 1 µg/ml puromycin. c-cbl levels were determined using whole cell lysates by immunoblotting with anti-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.82%); opacity: 0.80\" title=\"0.006\">antibody</span><span style=\"opacity: 0.80\"> (santa cruz biotechnologies, santa cruz, ca).  cell viability assay  cells were transfected as described above in the transfection assay. forty-eight </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">hours</span><span style=\"opacity: 0.80\"> after transfection, viability of cells was assessed using trypan blue exclusion.  wound healing assay  a549 cells were </span><span style=\"background-color: hsl(120, 100.00%, 98.87%); opacity: 0.80\" title=\"0.005\">seeded</span><span style=\"opacity: 0.80\"> in 6-well plates and cultured for 48 h until 100% confluent. the medium was then changed and the cells were transfected as described in the transfection assay. twelve </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">hours</span><span style=\"opacity: 0.80\"> after transfection, a straight scratch was made across the cell layer using a 1 ml pipette tip. the cells were then gently washed with 1× pbs to remove cellular debris and the media was replaced. photographs were taken of the wound region every 12 h until 48 h.  western blot analysis  forty eight </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">hours</span><span style=\"opacity: 0.80\"> after transfection, cells were collected and washed twice in 1x pbs, then lysed in ice-cold lysis buffer (0.5m tris-hcl with ph 7.4, 1.5 m nacl, 2.5% deoxycholic acid, 10 mm edta, 10% np-40, 0.5 mm dtt, 1 mm phenylmethylsulfonyl fluoride, 5 µg/ml leupeptin, and 10 µg/ml aprotinin) for 5 minutes. the lysate was centrifuged at 13,000 rpm for 20 minutes at 4°c, and protein content of the supernatant was measured. total cell lysates (50 µg/well) were separated by sds-page electrophoresis and the gels transferred onto nitrocellulose membranes (whatman, piscataway, nj). membranes were </span><span style=\"background-color: hsl(0, 100.00%, 91.52%); opacity: 0.82\" title=\"-0.095\">blocked</span><span style=\"opacity: 0.80\"> with 5% non-fat dry milk in phosphate-buffered saline containing tween-20 (pbst) (1x pbs, 0.1% tween-20) for 1 h at room temperature and incubated with the appropriate primary </span><span style=\"background-color: hsl(120, 100.00%, 98.82%); opacity: 0.80\" title=\"0.006\">antibody</span><span style=\"opacity: 0.80\"> at 4°c overnight. membranes then were washed three times with pbst and probed with appropriate horseradish peroxidase (hrp)-conjugated secondary </span><span style=\"background-color: hsl(120, 100.00%, 98.82%); opacity: 0.80\" title=\"0.006\">antibody</span><span style=\"opacity: 0.80\"> for 1 h at room temperature. the membranes were again washed three times in pbst and bands were visualized using western blot chemiluminescence reagent (biorad, valencia, ca) on a chemidoc gel documentation system (biorad, valencia, ca). antibodies were obtained from santa cruz biotechnologies and used at the following dilutions (c-cbl, 1∶5000; c-met, 1∶5000; egfr, 1∶5000; ubiquitin, 1∶1000; ha, 1∶5000 and β-actin, 1∶10,000).  flow cytometry  cell </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> analysis was </span><span style=\"background-color: hsl(0, 100.00%, 99.63%); opacity: 0.80\" title=\"-0.001\">carried</span><span style=\"opacity: 0.80\"> out by flow cytometry. approximately 2×106 cells were grown in media containing 10% fbs. cells were harvested by trypsin/edta treatment, washed with 1x pbs three times and fixed with ice-cold 70% ethanol for 2 h. cells were washed again with cold pbs and stained with a solution containing 25 µg/ml propidium iodide, 200 µg/ml rnase a, and 0.1% triton x-100 for 30 minutes in the dark. cell </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> analysis was performed using a guava pca-96 flow cytometer (guava technologies, millipore, billerica, ma).  ubiquitin ligase activity  </span><span style=\"background-color: hsl(0, 100.00%, 98.92%); opacity: 0.80\" title=\"-0.005\">293t</span><span style=\"opacity: 0.80\"> cells were maintained in culture in dmem supplemented with 10% fbs and 1% penicillin (100 units/ml) and streptomycin (100 µg/ml) were transfected with 0.2 µg egfr-</span><span style=\"background-color: hsl(0, 100.00%, 98.42%); opacity: 0.80\" title=\"-0.009\">pcdna3</span><span style=\"opacity: 0.80\"> and 2 µg ha-tagged c-cbl constructs as </span><span style=\"background-color: hsl(0, 100.00%, 60.00%); opacity: 1.00\" title=\"-0.871\">indicated</span><span style=\"opacity: 0.80\"> using calcium phosphate according to manufacturer&#x27;s protocol (profection, promega, madison, wi). twenty-four </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">hours</span><span style=\"opacity: 0.80\"> post-transfection, cells were starved overnight in dmem supplemented with 0.5% fbs, and then treated with or without egf (100 ng/ml) for 15 min. the cells were collected and washed </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">two</span><span style=\"opacity: 0.80\"> times in ice-cold pbs containing 0.2 mm sodium orthovanadate then lysed in ice-cold lysis buffer (10 mm tris hcl, ph 7.5, 150 mm nacl, 5 mm edta, 1% triton x100, 10% glycerol, 2 mm sodium orthovanadate and protease inhibitors). lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4°c. egfr immunoprecipitations were performed on 200 µg of cleared lysate using 250 ng of rabbit-anti-egfr and protein a/g plus sepharose overnight at 4°c. precipitations were washed 5 times in lysis buffer before boiling in laemmli buffer. elutions were immunoblotted with anti-ubiquitin and egfr. twenty micrograms of cleared lysate were immunoblotted for each of the c-cbl constructs using anti-ha.  statistical analysis  mutation rates between different groups were compared using fisher&#x27;s exact test. for continuous variables, group comparisons were performed using analysis of variance (anova) followed by sidak&#x27;s adjustment for multiple comparisons. </span><span style=\"background-color: hsl(120, 100.00%, 99.76%); opacity: 0.80\" title=\"0.001\">experiments</span><span style=\"opacity: 0.80\"> involving repeated measurements over time were </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">analyzed</span><span style=\"opacity: 0.80\"> using repeated measures anova with the greenhouse-geisser adjustment to the degrees of freedom. analyses were conducted using stata (v10.1) software (stata corporation, college station, tx).  go to: results c-cbl gene mutations in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">  to investigate the role of c-cbl in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">, we </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">analyzed</span><span style=\"opacity: 0.80\"> its genomic dna in tumor and paired normal samples drawn from multiple ethnicities. the lung tumor samples represented caucasians (n = 50), african-americans (n = 29), and taiwanese (n = 40) lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> patients. we designed 12 pairs of primers to sequence the coding region of c-cbl gene that spans exons 2 to 16 (table s2). we identified 8 unique somatic mutations in c-cbl exons among 8 different patients. a variation l620f, a </span><span style=\"background-color: hsl(0, 100.00%, 97.99%); opacity: 0.80\" title=\"-0.012\">known</span><span style=\"opacity: 0.80\"> snp (rs2227988) in exon 11 was also detected. importantly, the eight novel non-synonymous mutations were confirmed by sequencing both strands of c-cbl genomic dna obtained from lung tumor samples (table 1). moreover, none of the 8 mutations were detected in the corresponding normal tissue, indicating that these were somatic mutations. four synonymous single nucleotide variations (snvs) were also identified but were </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> used </span><span style=\"background-color: hsl(0, 100.00%, 98.72%); opacity: 0.80\" title=\"-0.006\">further</span><span style=\"opacity: 0.80\"> in this study.  table 1 table 1 c-cbl mutation analysis in 119 lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> patient tumor tissues. three of the 8 novel non-synonymous mutations were located in the tkb (tyrosine kinase binding) domain (s80n, h94y, and q249e), one in the ring finger domain (v391i), one in the proline-rich region (72515_72517 del atg), and three in the c-terminal region (w802*, r830k, and a848t) of the c-cbl protein (figure 1a and figure s1). in figure 1b, we show model chromatograms of representative samples.  figure 1 figure 1 c-cbl mutations and loh in non-small cell lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">. 11q loh of c-cbl gene  paired lung tumor and normal lung tissue samples from taiwanese patients (n = 37) were investigated for loh. eight (21.6%) showed loh at the c-cbl locus on chromosome 11 while 29 samples (78.4%) revealed normal allelic contribution at the microsatellite markers (figures 1c and d).  c-cbl mutations in different ethnic groups  the c-cbl double mutant s80n/h94y was found in the same patient and the overall mutation rate for c-cbl in lung tumors was 6.7% (8/119). the frequency of c-cbl mutation was highest in </span><span style=\"background-color: hsl(120, 100.00%, 95.93%); opacity: 0.81\" title=\"0.033\">large</span><span style=\"opacity: 0.80\"> cell carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the </span><span style=\"background-color: hsl(0, 100.00%, 99.53%); opacity: 0.80\" title=\"-0.002\">least</span><span style=\"opacity: 0.80\"> was </span><span style=\"background-color: hsl(120, 100.00%, 99.22%); opacity: 0.80\" title=\"0.003\">observed</span><span style=\"opacity: 0.80\"> in adenocarcinoma (ad) (1.8%; 1 of 53 patients), although these rates were </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> statistically significant (p = 0.292). mutation rates were 6.0% among caucasians (0 of 20 in ad; 0 of 10 in sq; and 3 of 20 in lc), 13.8% in african-americans (1 of 10 in ad; 1 of 10 in sq; and 2 of 9 in lc), and 2.5% (0 of 23 in ad; 1 of 12 in sq; and 0 of 5 in lc) in the taiwanese population. additionally </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">two</span><span style=\"opacity: 0.80\"> taiwanese patients with lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> (one squamous and one adenocarcinoma) had the </span><span style=\"background-color: hsl(0, 100.00%, 97.99%); opacity: 0.80\" title=\"-0.012\">known</span><span style=\"opacity: 0.80\"> snp l620f. ethnic differences were </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> statistically significant, however the power to detect differences was low.  mutations in met and egfr can be co-associated with c-cbl alterations  since east asians with lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> have a </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.004\">higher</span><span style=\"opacity: 0.80\"> frequency of egfr and met mutations in lung tumors [21], [22], we also determined mutations in egfr and met in the same taiwanese cohort samples and compared the results with the </span><span style=\"background-color: hsl(120, 100.00%, 99.22%); opacity: 0.80\" title=\"0.003\">observed</span><span style=\"opacity: 0.80\"> c-cbl alterations (loh and/or mutations). in the 37 samples tested, we </span><span style=\"background-color: hsl(0, 100.00%, 97.94%); opacity: 0.80\" title=\"-0.013\">did</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> find any overlap between c-cbl mutations and c-cbl loh (figure 2). of the three c-cbl mutants (including the </span><span style=\"background-color: hsl(0, 100.00%, 97.99%); opacity: 0.80\" title=\"-0.012\">known</span><span style=\"opacity: 0.80\"> l620f snp, rs2227988), one of the samples had a met mutation (n375s) and the other had an egfr mutation (l858r). among the 8 samples that had a loh at the c-cbl locus 5 had an additional mutation in met (n375s) and 2 had an egfr exon 19 deletion. twenty-six samples had neither c-cbl mutation nor c-cbl loh (3 patients had a c-cbl mutation but no c-cbl loh). among these 26 samples 9 had a met mutation (8 n375s, 1 l211w), 13 had an egfr mutation (7 exon 9 deletion, 6 l858r) and 4 had no other met or egfr mutation. thus the rate of met or egfr mutations among patients with loh at the c-cbl locus (7 of 8) was similar to that seen in patients without c-cbl mutation or loh (22 of 26 patients) (p = 0.99). these 4 patients with no identifiable mutation in c-cbl, met or egfr represented 10.8% of the 37 patients </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">analyzed</span><span style=\"opacity: 0.80\"> in the taiwanese patient cohort. conversely, 89.2% taiwanese lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> patients have an identifiable mutation in either c-cbl, met or egfr or a combination of the three genes (figure 2). additionally, we determined p53 and kras mutations in these taiwanese cohorts. </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">two</span><span style=\"opacity: 0.80\"> p53 and 1 kras mutation were detected. the single kras mutation overlapped with one p53 mutation. this patient also had the egfr exon 19 deletion but had no c-cbl mutation. the other p53 mutation sample had a c-cbl loh with concurrent met n375s mutation. thus, in the taiwanese samples </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">analyzed</span><span style=\"opacity: 0.80\">, p53/kras mutations and c-cbl mutations were mutually exclusive (data </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> shown).  figure 2 figure 2 c-cbl mutations and relationship to met and egfr mutations in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">. cellular functions of c-cbl alterations in the context of lung tumorigenesis  a. e3 activity is intact in the mutant c-cbl proteins to investigate whether the different c-cbl mutations </span><span style=\"background-color: hsl(120, 100.00%, 98.96%); opacity: 0.80\" title=\"0.005\">affect</span><span style=\"opacity: 0.80\"> the e3 activity, egfr was chosen as a model substrate for c-cbl e3 function. all of the c-cbl mutants tested enhanced ubiquitination of the activated egfr similar to the wild-type c-cbl protein. this result demonstrates that the catalytic activity of the c-cbl mutants is </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> impaired when egfr was the substrate. (figure 3a).  figure 3 figure 3 ubiquitination, viability, expression and cell </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> analysis of various c-cbl mutants. b. effect on lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> cell viability the effect of a representative c-cbl mutant from each of the three ethnic backgrounds on lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> cell viability in cell lines was determined. s80n/h94y double mutation, q249e, and w802* were identified in lung tumor samples obtained from a caucasian, a taiwanese and an african-american, respectively. as described in methods, the c-cbl wild-type (wt) and the above three mutants were expressed after cloning them into paltermax vector in a549 cells. these cells express relatively low basal levels of endogenous c-cbl (data </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> shown). transfection efficiency was comparable between different groups and the number of cells transfected with c-cbl wild-type construct was about 70% compared to control cells that were transfected with the empty vector. cells transfected with s80n/h94y, q249e and w802* c-cbl mutant constructs resulted in increased number of viable cells that was 132.3%, 120.8% and 147.9% </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.004\">higher</span><span style=\"opacity: 0.80\"> respectively, relative to the empty vector control transfected cells and significantly different from the wild-type construct (p = 0.022, p = 0.049, and p = 0.008, respectively) (figure 3b). relative levels of c-cbl protein in whole cell lysates prepared from samples obtained from a parallel experiment was determined. the c-cbl protein levels in samples representing untransfected and empty vector transfected cells were comparable and those representing the c-cbl wt and the three c-cbl mutants were comparable (figure 3c).  c. effect on cell </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> to investigate if the increases in cell viability in different c-cbl mutants are due to increased cellular proliferation, a cell </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> analysis was performed. a549 cells were transfected with the c-cbl wt or the three different mutants: s80n/h94y, q249e and w802*. the empty vector transfectant was used as a control. forty-eight </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">hours</span><span style=\"opacity: 0.80\"> after transfection cell </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> analysis was performed as described in materials and methods. there was no significant change in the subg1, g1 or the s phase of the cell </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> among the different mutants compared to the wt construct (p = 0.64, p = 0.40, and p = 0.28, respectively). the g2/m phase of the cell </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> showed an increase in cell numbers for the three mutants, s80n/h94y, q249e and w802*, when compared to the wt but again the difference was </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> statistically significant (p = 0.25) (figure 3d).  d. effect on cell motility to investigate the effect of the expression of the above three c-cbl mutants on cell migration, we </span><span style=\"background-color: hsl(0, 100.00%, 99.63%); opacity: 0.80\" title=\"-0.001\">carried</span><span style=\"opacity: 0.80\"> out wound healing assay as described in materials and methods. the closing of the scratch or the wound was monitored at 0, 12, 24, 36, and 48 h. (figure 4a). in all the samples, that represented cells transfected with mutants, the wound gap was much smaller than that seen in the sample that represented cells transfected with c-cbl wt (p&lt;0.001). we also determined the rate of wound closure for all the five groups. at 48 h, wild-type c-cbl transfectants showed 61.1% open wound while the s80n/h94y, q249e and w802* mutants showed 18.7%, 23.9% and 34.3% open wound respectively (p&lt;0.001) (figure 4b).  figure 4 figure 4 c-cbl mutations </span><span style=\"background-color: hsl(120, 100.00%, 98.96%); opacity: 0.80\" title=\"0.005\">affect</span><span style=\"opacity: 0.80\"> wound healing in a549 cells. e. c-cbl knockdown increases cell viability it is hypothesized that the loh seen in our samples could lead to decreased expression of c-cbl. thus we tested the effect of c-cbl knockdown in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> cells. compared to a549, h358 lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> cells express relatively high levels of endogenous c-cbl (data </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> shown). c-cbl expression was knocked down using lentiviral construct that expressed c-cbl specific shrna and compared the results with those that were transduced with scrambled shrna and the results are shown in figure 5. we identified </span><span style=\"background-color: hsl(0, 100.00%, 98.86%); opacity: 0.80\" title=\"-0.005\">several</span><span style=\"opacity: 0.80\"> clones that revealed varying degrees of c-cbl knockdown showing different sets of c-cbl lentiviral shrna knockdown efficiency (figure 5a). of all the clones tested, clone 27 was chosen for </span><span style=\"background-color: hsl(0, 100.00%, 98.72%); opacity: 0.80\" title=\"-0.006\">further</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.76%); opacity: 0.80\" title=\"0.001\">experiments</span><span style=\"opacity: 0.80\">. equal </span><span style=\"background-color: hsl(0, 100.00%, 98.63%); opacity: 0.80\" title=\"-0.007\">amount</span><span style=\"opacity: 0.80\"> of cells were </span><span style=\"background-color: hsl(120, 100.00%, 98.87%); opacity: 0.80\" title=\"0.005\">seeded</span><span style=\"opacity: 0.80\"> in a 6-well plate and the cell proliferation was measured at various times and the results are depicted in figure 5b. as expected, number of cells increased in a time dependent fashion from 100 to 190% relative to scrambled shrna as control in a span of 48 h (p = 0.0002) (figure 5b). the cell </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> phases in h358 cells that were knocked down with c-cbl shrna were looked at and compared with the scrambled shrna. there were no discernable differences between these </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">two</span><span style=\"opacity: 0.80\"> constructs in the different phases of the cell </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">cycle</span><span style=\"opacity: 0.80\"> (data </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> shown).  figure 5 figure 5 knockdown of c-cbl using an shrna increases cell proliferation. go to: discussion our results demonstrate that c-cbl is somatically mutated (or has loh) in lung cancers, and can significantly contribute to enhanced cell viability and motility. there was also a high prevalence of loh with respect to c-cbl in lung tumors that harbored met or egfr mutation.  in the present study, we have demonstrated the occurrence of c-cbl mutations in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> patients, especially with different ancestral variations. mutations in c-cbl have been recently reported in juvenile myelomonocytic leukemia and myeloid malignancies. in the aml study, the mutation r420q located in the junction of the ring finger and the linker region inhibited fms-like tyrosine kinase 3 (</span><span style=\"background-color: hsl(0, 100.00%, 94.14%); opacity: 0.81\" title=\"-0.056\">flt3</span><span style=\"opacity: 0.80\">) internalization and ubiquitination [20], thus contributing to the gain-in-function for the rtk. in addition, mutations such as h398y, c384r, and l380p mapped to the ring finger domain and the linker region of c-cbl that is required for its e3 activity [23], [24], [25], [26], [27]. additionally, homozygous mutations in the ring finger domain of the c-cbl gene were described as a result of acquired uniparental disomy (upd) [26]. it is important to note that our results indicate loh at 11q23 locus and these are mutually exclusive from missense mutations of c-cbl. the somatic mutations were all heterozygous. the mutations in aml led to abrogation of the e3 activity leading to prolonged rtk activation. in addition mutants located on the linker region surrounding the ring finger domain </span><span style=\"background-color: hsl(0, 100.00%, 92.02%); opacity: 0.82\" title=\"-0.087\">exhibited</span><span style=\"opacity: 0.80\"> enhanced akt signaling in response to cytokine stimulation [26]. in addition, it was shown in nh3t3 cells, that neither mutations in the ring finger nor the linker region causes transformation, however while certain mutations perturbs the ubiquitination, others </span><span style=\"background-color: hsl(120, 100.00%, 98.96%); opacity: 0.80\" title=\"0.005\">affect</span><span style=\"opacity: 0.80\"> receptor recycling and prolong kinase activity [28].  we report here c-cbl mutations that mapped </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> only to the ring finger domain, but also to the tkb domain, proline-rich domain and the c-terminal region, but none mapped to the linker region as reported in the aml studies described above [23], [24], [25], [26], [29]. in addition, 8 mutants that we detected were found in different ethnic backgrounds. for example, s80n/h94y, q249e, w802* were detected in caucasians, taiwanese and african-americans, respectively. the results point out </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> only the difference between lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> and other cancers, but also </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">genetic</span><span style=\"opacity: 0.80\"> polymorphism among different races in the same </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">. interestingly, there is a </span><span style=\"background-color: hsl(120, 100.00%, 95.93%); opacity: 0.81\" title=\"0.033\">large</span><span style=\"opacity: 0.80\"> disparity between african-american and other ethnic populations with lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> [30]. we have previously shown that there was a low frequency of egfr and met mutation in african-americans as compared with taiwanese and caucasians [31]. in this study, the number of african-american samples </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">analyzed</span><span style=\"opacity: 0.80\"> was relatively fewer and we found 3 mutations that are unique to this ethnicity. it would now behoove us to </span><span style=\"background-color: hsl(0, 100.00%, 98.72%); opacity: 0.80\" title=\"-0.006\">further</span><span style=\"opacity: 0.80\"> study the </span><span style=\"background-color: hsl(120, 100.00%, 99.35%); opacity: 0.80\" title=\"0.002\">genetic</span><span style=\"opacity: 0.80\"> alterations that can occur and determine the targeted therapeutics for african-americans. our results provide </span><span style=\"background-color: hsl(120, 100.00%, 97.93%); opacity: 0.80\" title=\"0.013\">evidence</span><span style=\"opacity: 0.80\"> of the importance of c-cbl in tumorigenesis and potential signaling. our prediction, based on the aml data, would be that the v391i ring finger domain mutation would </span><span style=\"background-color: hsl(120, 100.00%, 98.96%); opacity: 0.80\" title=\"0.005\">affect</span><span style=\"opacity: 0.80\"> the e3 activity. also, it will be important to determine the binding partners for c-cbl in the tkb domain and proline rich domain mutations. it has previously been shown that the tkb domain can bind to growth factor receptors and it will be important to determine the cross-binding of these mutants to met and or egfr. it would also be important in the future to look at fluorescence in situ </span><span style=\"background-color: hsl(0, 100.00%, 95.25%); opacity: 0.81\" title=\"-0.041\">hybridization</span><span style=\"opacity: 0.80\">/copy number changes in c-cbl in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">.  c-cbl plays an important role in down regulating rtk-</span><span style=\"background-color: hsl(120, 100.00%, 99.38%); opacity: 0.80\" title=\"0.002\">mediated</span><span style=\"opacity: 0.80\"> signaling through k63 poly-ubiquitination and subsequent downregulation of rtks followed by lysosomal degredation [3]. mono-ubiquitination or ubiquitinated with k63-linked chains of substrates by c-cbl may lead to enhancement of biological and biochemical functions (reviewed in hermann et al, 2007 [32]). the mutations that we </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">analyzed</span><span style=\"opacity: 0.80\"> in our studies all point out to the fact that the e3 activity of c-cbl on egfr is intact; the egfr levels in the various mutants remain same (figure s2). multiple kinases, both rtks and non-rtks could be acted upon by c-cbl, including erbs, pdgfr, fms, met, c-kit, vegfr, flt-1, ron, fgfr, ir, as well as syk, fyn, lck, fgr, lyn and c-abl [3]. in lung cancers the relevant substrates of c-cbl in terms of </span><span style=\"background-color: hsl(0, 100.00%, 97.57%); opacity: 0.80\" title=\"-0.016\">degradation</span><span style=\"opacity: 0.80\"> or signal transduction are yet to be identified.  the observation that c-cbl somatic mutations, especially s80n/h94y, q249e and w802* showed increased cell viability and cell motility that is in agreement with the physiological role for d-cbl in the regulation of apoptosis and differentiation identified in drosophila is very significant [33]. it has been previously shown that </span><span style=\"background-color: hsl(0, 100.00%, 99.10%); opacity: 0.80\" title=\"-0.004\">activating</span><span style=\"opacity: 0.80\"> c-cbl mutation downregulates egfr signaling and decreases cellular proliferation and migration in breast </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> cell lines [34]. although the role of c-cbl in the negative regulation of rtks is well substantiated, thereby suggesting that it is a natural tumor suppressor, studies in </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> cells have revealed both tumor suppressor and tumor promoting activities depending on the type of c-cbl mutation and the number of alleles at the c-cbl locus [24]. in agreement with the above, the three c-cbl mutants described here appear to have tumor growth and metastasis promoting properties. although these mutants are outside of the ring finger or the linker region of c-cbl, their downstream effects are significant so as to cause increased proliferation and migration, but the substrate affected by these mutations are </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 97.99%); opacity: 0.80\" title=\"-0.012\">known</span><span style=\"opacity: 0.80\"> yet. this raises the possibility that </span><span style=\"background-color: hsl(120, 100.00%, 98.62%); opacity: 0.80\" title=\"0.007\">some</span><span style=\"opacity: 0.80\"> of the cellular functions of c-cbl are independent of its ubiquitin-ligase activity, an area that we are currently investigating. the oncogenic nature of rtks, addiction of cancers to growth signals and given the clustering of c-cbl, egfr and met mutations, it is possible that the transforming effect of c-cbl mutations is most likely a combinatorial effect of the three. we also show that loh for c-cbl was found in a significant number of samples that harbored met or egfr mutations. the fact that about 7% of lung tumor samples are likely to have c-cbl mutations and an additional 22% are likely to harbor c-cbl-related loh, makes c-cbl a highly mutated molecule in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">. since loh alone is </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> enough to cause a transforming event [35], [36], [37], associated mutation in the met or egfr locus or yet another rtk discussed above may play a role in carcinogenesis. we predict that this loh in c-cbl results in haploinsufficiency that downplays rtk ubiquitination leading to hyperactivity of the rtks. however, whether this is </span><span style=\"background-color: hsl(0, 100.00%, 94.73%); opacity: 0.81\" title=\"-0.048\">sufficient</span><span style=\"opacity: 0.80\"> cause for tumorigenesis remains to be determined. consistent with our hypothesis is the fact that c-cbl-/- mice have increased kinase activity in lymphocytes, but is </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 94.73%); opacity: 0.81\" title=\"-0.048\">sufficient</span><span style=\"opacity: 0.80\"> for tumor formation [35], [36], [37]. c-cbl loh could also lead to increased expression of c-cbl from the other allele to compensate for a loss of an allele. alternately, there could be </span><span style=\"background-color: hsl(120, 100.00%, 98.62%); opacity: 0.80\" title=\"0.007\">some</span><span style=\"opacity: 0.80\"> form of synergy working with reduced c-cbl levels and mutated receptors that exacerbate the phenotype of each alone.  previous studies from our lab and others have shown that east asians with lung cancers have relatively high frequencies of gain-of-function of mutations in rtks such as egfr and met [31]. in a cohort of japanese patients an </span><span style=\"background-color: hsl(0, 100.00%, 99.10%); opacity: 0.80\" title=\"-0.004\">activating</span><span style=\"opacity: 0.80\"> met mutation was identified in the splice region that deletes the juxtamembrane domain that is involved e3 activity of c-cbl [38]. this study also found that activation of met is mutually exclusive of egfr, kras and her2 gene mutations [38]. we </span><span style=\"background-color: hsl(0, 100.00%, 99.33%); opacity: 0.80\" title=\"-0.003\">failed</span><span style=\"opacity: 0.80\"> to detect such mutations in significant numbers in lung tumor samples obtained from african-americans (n = 29) and caucasian (n = 50) patients. one met mutation was identified in each of the groups whereas 1 and 3 egfr mutations were identified in the african-american and caucasian cohorts respectively. egfr mutations have earlier been identified as one of the key mutations </span><span style=\"background-color: hsl(0, 100.00%, 91.20%); opacity: 0.82\" title=\"-0.100\">affecting</span><span style=\"opacity: 0.80\"> lung adenocarcinoma patients in a comprehensive study of 188 patients [39]. our study encompasses different histologies of nsclc. however, the published series </span><span style=\"background-color: hsl(0, 100.00%, 97.94%); opacity: 0.80\" title=\"-0.013\">did</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.80%); opacity: 0.80\" title=\"0.000\">not</span><span style=\"opacity: 0.80\"> find any mutations in c-cbl or met unlike our study that encompassed different subtypes of nsclc. it is important to note, we have recently shown that met mutations in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> are in majority germline [31]. we have reported earlier c-cbl mutations in a small cohort of taiwanese lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\"> samples [40]. in our efforts to understand the ethnic differences in the lung oncogenome, we also looked at pax transcription factors such as pax5 and pax8 that are highly expressed in lung cancers; however there was no preferential expression or mutations of the above genes in lung tumor samples of african-americans. in this study, we show relatively high frequency of c-cbl mutations in lung cancers, especially in the </span><span style=\"background-color: hsl(120, 100.00%, 95.93%); opacity: 0.81\" title=\"0.033\">large</span><span style=\"opacity: 0.80\"> cell type among caucasians and particularly among african-americans. we therefore propose c-cbl as an efficacious target for lung cancers in african-americans that needs to be </span><span style=\"background-color: hsl(0, 100.00%, 98.72%); opacity: 0.80\" title=\"-0.006\">further</span><span style=\"opacity: 0.80\"> substantiated. this is all the more important because the prognosis for african-americans with lung </span><span style=\"background-color: hsl(0, 100.00%, 99.95%); opacity: 0.80\" title=\"-0.000\">cancer</span><span style=\"opacity: 0.80\">, especially for men is much poorer compared to their caucasian counterparts [41].  in conclusion, the results presented in this study demonstrate that c-cbl is frequently mutated or even lost in lung cancers. our results support a role for c-cbl mutants that are independent of its ubiquitination activity. given the relatively high mutation rates in c-cbl as well as rtks such as met and egfr, it is likely that their combined effect could be synergistic in promoting tumorigenesis.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=4\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.168</b>, score <b>0.119</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 93.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.669\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.060\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__efficacy\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.07%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.042\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__drug\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.10%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.041\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__amplification\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.34%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.026\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__increasing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.35%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.025\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__harboring\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.019\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__quantify\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.55%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__surface\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.56%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__soft\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__sensitivity\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inc\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__concentrations\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__recurrence\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__phospho\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__anchorage\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__conformation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__advanced\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__59\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__curves\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__kd\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__equilibrium\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__sf9\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__screening\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__resistance\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__reveals\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__options\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2b\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2f\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mg\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__rearrangements\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__symptoms\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__counting\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__challenge\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__terminally\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fgfr1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2006\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__grade\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__136\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__01\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__plane\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__song\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__shorter\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__norvegicus\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cycloheximide\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__relate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__rigid\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stabilization\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pancreatic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__list\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__p15ink4b\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__material\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__originally\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__resulting\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__calculate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__translocates\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__conjunction\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__agent\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__tgf\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__complexes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__microscopy\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__absent\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__abrogate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__evidenced\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__laboratories\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__lack\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bsa\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__heterodimer\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__modifications\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__pik3r1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pm\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__lanes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__4j\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.56%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__pten\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.53%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__deficient\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.53%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__null\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.52%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__major\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__massive\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.49%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bioinformatic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__caused\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.45%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__beads\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.45%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__unable\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.41%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.022\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inactivating\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.35%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.025\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stability\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.28%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.029\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__conserved\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.26%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.030\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__phosphoinositide\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.96%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.050\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inactivation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.076\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nonsense\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 95.33%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.426\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">w802*</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">cbl</span><br/><b>Text:</b> <span style=\"opacity: 0.80\"> abstract background  non-</span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.006\">small</span><span style=\"opacity: 0.80\"> cell lung cancer (nsclc) is a heterogeneous group of disorders with a number of genetic and proteomic alterations. c-cbl is an </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> ubiquitin ligase and adaptor molecule important in normal homeostasis and cancer. we determined the genetic variations of c-cbl, relationship to receptor tyrosine kinases (egfr and met), and functionality in nsclc.  methods and findings  using archival formalin-fixed paraffin embedded (ffpe) extracted genomic dna, we show that c-cbl mutations occur in somatic fashion for lung cancers. c-cbl mutations were not mutually exclusive of met or egfr mutations; however they were </span><span style=\"background-color: hsl(0, 100.00%, 98.96%); opacity: 0.80\" title=\"-0.005\">independent</span><span style=\"opacity: 0.80\"> of p53 and kras mutations. in normal/tumor pairwise analysis, there was significant loss of heterozygosity (loh) for the c-cbl locus (22%, n = 8/37) and none of these samples revealed any mutation in the remaining copy of c-cbl. the c-cbl loh also positively correlated with egfr and met mutations observed in the same samples. using select c-cbl somatic mutations such as s80n/h94y, q249e and w802* (</span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">obtained</span><span style=\"opacity: 0.80\"> from caucasian, taiwanese and african-american samples, respectively) transfected in nsclc cell lines, there was increased cell viability and cell motility.  conclusions  taking the overall mutation rate of c-cbl to be a combination as somatic missense mutation and loh, it is clear that c-cbl is highly mutated in lung cancers and may play an essential role in lung </span><span style=\"background-color: hsl(0, 100.00%, 99.85%); opacity: 0.80\" title=\"-0.000\">tumorigenesis</span><span style=\"opacity: 0.80\"> and metastasis.  go to: introduction in the us alone, each year approximately 219,400 people </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> diagnosed with lung cancers, out of which more than 145,000 of them succumb to the disease [1]. this number is roughly equivalent to the combined mortality rates of cancers of the breast, prostate, colon, liver, kidney and melanoma [1]. in addition the prognosis is usually poor and the five-year survival rate is less than 15%. there </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> also significant ethnic differences for lung cancer, and the outcome is worse for blacks compared to whites. gender differences </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> also striking with women having significantly better prognosis as compared to men. there </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> a number of genetic alterations that can occur in lung cancer. as an example, in nsclc, mutations in kras, p53, egfr and met have been identified. many of these pathways, especially receptor tyrosine kinases (rtks) </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> controlled by c-cbl.  cbl (casitas b-lineage lymphoma) is a mammalian gene located on human chromosome 11q23.3 [2] and is involved in cell signaling and protein ubiquitination [3]. cbl proteins belong to the ring finger class of ubiquitin ligases (</span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\">) and there </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> homologues c-cbl, cbl-b, cbl-3 [4]. the c-cbl and cbl-b genes </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> ubiquitously expressed with the highest levels in hematopoietic tissues [5]. c-cbl consists of four regions encoding for functionally distinct protein domains: the n-terminal tyrosine kinase binding (tkb) domain, the linker region, the catalytic ring finger domain, the proline-rich region and the c-terminal ubiquitin-associated (uba) domain that also overlaps with a leucine-zipper (lz) domain [3]. both tkb and ring finger domains </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> essential for </span><span style=\"background-color: hsl(0, 100.00%, 93.18%); opacity: 0.82\" title=\"-0.070\">ligand</span><span style=\"opacity: 0.80\">-induced ubiquitination of rtks [6], [7], [8], [9]. the ring finger domain is required for the recruitment of e2 ubiquitin-conjugating enzymes. the tkb domain includes four-helix bundle (4h), a calcium-biding ef hand, and a modified sh2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. in addition, the proline-rich region of c-cbl can associate with the sh3 domain of grb2, which can indirectly recruit c-cbl to rtks via the grb2 adaptor protein [7], [13], [</span><span style=\"background-color: hsl(120, 100.00%, 98.89%); opacity: 0.80\" title=\"0.005\">14</span><span style=\"opacity: 0.80\">].  c-cbl also binds to egfr and acts as the </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> that targets egfr for ubiquitination and degradation. furthermore, cbl desensitizes egf signaling and opposes cellular proliferation induced by egf [15]. egf </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.008\">activation</span><span style=\"opacity: 0.80\"> also appears to activate the tyrosine kinase src, which phosphorylates c-cbl and in turn activates the ubiquitination and degradation of egfr [16], [</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">17</span><span style=\"opacity: 0.80\">], [18]. a recent study shows that </span><span style=\"background-color: hsl(120, 100.00%, 94.57%); opacity: 0.81\" title=\"0.050\">defective</span><span style=\"opacity: 0.80\"> endocytosis of egfr is characterized by a deletion mutant and the point mutation l858r, whereby its association with c-cbl and subsequent ubiquitination </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> impaired [19]. </span><span style=\"background-color: hsl(0, 100.00%, 96.12%); opacity: 0.81\" title=\"-0.031\">recently</span><span style=\"opacity: 0.80\">, the first human c-cbl mutations were reported in acute myeloid leukemia (aml) patients [20]. the mutation r420q inhibits fms-like tyrosine kinase 3 (flt3) internalization and ubiquitination [20].  not only can </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> activity be important in oncogenesis, c-cbl has a dual but separate </span><span style=\"background-color: hsl(0, 100.00%, 98.96%); opacity: 0.80\" title=\"-0.005\">function</span><span style=\"opacity: 0.80\"> as a signal transduction molecule. we have previously shown that c-cbl is important in binding crkl and bcr/abl in hematopoietic cells. also, it can bind and modulate functions of cytoskeleton by binding to proteins like talin and paxillin. the tkb domain is important in binding to a number of molecules, and they then </span><span style=\"background-color: hsl(0, 100.00%, 98.96%); opacity: 0.80\" title=\"-0.005\">function</span><span style=\"opacity: 0.80\"> in signal transduction.  given the critical role of cbl in normal homeostasis and cancer, we hypothesized that it might be mutated in lung cancers. in this study, we report novel c-cbl somatic mutations s80n/h94y, q249e and w802* in caucasian, taiwanese and african-american lung cancer patients, respectively. expressing these mutations in nsclc cell lines lead to increased proliferation and cell motility. we show that c-cbl mutations occur with or without met or egfr mutations but </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> mutually exclusive of a loh at the c-cbl locus. additionally, c-cbl loh is associated with either met or egfr mutations. we thus hypothesize that c-cbl mutations might contribute to the oncogenic potential of met and egfr in lung cancer.  go to: methods ethics statement  written consent on all </span><span style=\"background-color: hsl(0, 100.00%, 97.11%); opacity: 0.80\" title=\"-0.020\">research</span><span style=\"opacity: 0.80\"> on human subjects has been </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">obtained</span><span style=\"opacity: 0.80\"> from the institutional review board, university of </span><span style=\"background-color: hsl(120, 100.00%, 99.81%); opacity: 0.80\" title=\"0.000\">chicago</span><span style=\"opacity: 0.80\"> and covers all </span><span style=\"background-color: hsl(0, 100.00%, 97.11%); opacity: 0.80\" title=\"-0.020\">research</span><span style=\"opacity: 0.80\"> performed in the laboratory. the following is their </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.002\">contact</span><span style=\"opacity: 0.80\"> information: institutional review board, the university of </span><span style=\"background-color: hsl(120, 100.00%, 99.81%); opacity: 0.80\" title=\"0.000\">chicago</span><span style=\"opacity: 0.80\">, mcgiffert hall, 5751 s. woodlawn ave., 2nd floor, </span><span style=\"background-color: hsl(120, 100.00%, 99.81%); opacity: 0.80\" title=\"0.000\">chicago</span><span style=\"opacity: 0.80\">, il 60637. written informed consents were received from all patients whose tissue samples were used for this study.  tissue samples  lung cancer tissue and paired adjacent normal lung tissues were </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">obtained</span><span style=\"opacity: 0.80\"> from 50 caucasian, 29 african-americans and 40 taiwanese nsclc patients who were recruited at the university of </span><span style=\"background-color: hsl(120, 100.00%, 99.81%); opacity: 0.80\" title=\"0.000\">chicago</span><span style=\"opacity: 0.80\"> hospital (</span><span style=\"background-color: hsl(120, 100.00%, 99.81%); opacity: 0.80\" title=\"0.000\">chicago</span><span style=\"opacity: 0.80\">, usa) (caucasian and african-american patients) and taipei veterans general hospital of taiwan (taiwanese patients) </span><span style=\"background-color: hsl(120, 100.00%, 99.72%); opacity: 0.80\" title=\"0.001\">after</span><span style=\"opacity: 0.80\"> obtaining appropriate institutional review board permission and informed consent from the patients. out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 years. in terms of tumor types, 53 were adenocarcinoma, 32 were squamous cell carcinoma and 34 were large cell carcinoma. 49 were stage i, </span><span style=\"background-color: hsl(120, 100.00%, 98.89%); opacity: 0.80\" title=\"0.005\">14</span><span style=\"opacity: 0.80\"> were stage ii, 34 were stage iii, and 13 were stage iv (table s1).  cell culture  human non-</span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.006\">small</span><span style=\"opacity: 0.80\"> cell lung carcinoma cells a549 and h358 were maintained in dmem and rpmi-1640, respectively. human embryonic kidney 293t cells were cultured in dmem. media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (invitrogen, carlsbad, ca). cells were cultured at 37°c in a humidified incubator containing 5% co2.  c-cbl gene mutational analysis  exons 2 to 16 of c-cbl gene were individually amplified by polymerase </span><span style=\"background-color: hsl(120, 100.00%, 99.45%); opacity: 0.80\" title=\"0.002\">chain</span><span style=\"opacity: 0.80\"> reaction (pcr). primers </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> listed in table s2. pcr conditions were 1 cycle of 95°c for 5 minutes; 35 cycles of 94°c for 30 seconds, 58°c for 30 seconds and 72°c for 2 minutes; and one cycle of 72°c for 10 minutes. pcr products were treated with exosap-it (usb corporation, cleveland, oh) and sequenced by </span><span style=\"background-color: hsl(0, 100.00%, 95.29%); opacity: 0.81\" title=\"-0.041\">big</span><span style=\"opacity: 0.80\">-dye terminator chemistry (applied biosystems, foster city, ca). sequencing was performed on the forward coding strand with confirmation of c-cbl alterations performed by sequencing the reverse strand as well. chromatograms were </span><span style=\"background-color: hsl(0, 100.00%, 99.73%); opacity: 0.80\" title=\"-0.001\">analyzed</span><span style=\"opacity: 0.80\"> for mutations using mutation surveyor v2.61 (softgenetics, state college, pa).  plasmid constructs and site-directed mutagenesis  the wild-type c-cbl cdna insert was subcloned into the paltermax expression vector using xhoi and sali restriction enzyme sites (promega, madison, wi). using this parental plasmid paltermax-c-cbl, the tkb domain double mutation (s80n/h94y), the point mutation (q249e), and the c-terminal point mutation w802* of c-cbl were created using the following primers: 5′-gctggcgctaaagaataacccaccttatatcttagac-3′ and 5′-ctaccagatacctaccagtatctccgtactatcttgtc-3′ for the double mutation s80n/h94y; 5′-ctttacccgactctttgagccctggtcctctttgc-3′ for q249e, and 5′-cagctcctcctttggctgattgtctctggatggtgatc-3′ for w802* along with their complementary primers using the quickchange site-directed mutagenesis xl kit (stratagene, la jolla, ca) according to the manufacturer&#x27;s instructions. the constructs were confirmed for the point mutations by standard dna sequencing of both strands.  loss of heterozygosity (loh) analysis  five microsatellites on chromosome 11 (3 on 11q at or </span><span style=\"background-color: hsl(120, 100.00%, 96.75%); opacity: 0.81\" title=\"0.024\">within</span><span style=\"opacity: 0.80\"> 200 kb up or downstream of the c-cbl gene and 2 control markers on 11p) were selected for analysis (table s3). established microsatellite markers and respective primer sequences were selected from the geneloc database (http://genecards.weizmann.ac.il/geneloc/index.shtml, weizmann institute of science, rehovot, israel). primers were custom designed and each forward primer was fluorescently labeled at the 5′ end with fam, pet, ned, or vic (applied biosystems). primer annealing temperatures and duplex scores were evaluated with nist primer tools (http://yellow.nist.gov:8444/dnaanalysis/primertoolspage.do; national institute of standards and technology, gaithersburg, md). primers were verified by performing pcr with control dna (isolated from tk6 cells) and resolving the products on agarose gels. bands were visualized with an uv transilluminator. genomic dna was extracted from tumor samples and paired normal lung tissue. primers were grouped into multiplex combinations shown in table s4. marker d11s929 served as an internal control to check for consistency in pcrs and of peaks from </span><span style=\"background-color: hsl(0, 100.00%, 94.67%); opacity: 0.81\" title=\"-0.049\">capillary</span><span style=\"opacity: 0.80\"> electrophoresis. multiplex pcrs were carried out in a volume of 10 µl that contained 1 µl genomic dna (20–50 ng), 0.5 µm of each primer (1.0 µm total for each primer pair), 400 µm dntps, 1x pcr buffer containing mgcl2, and 0.2 u taq dna polymerase. pcr was performed on the abi geneamp 9700 pcr system under the following conditions: 5 min at 94°c; 30 cycles of 30 sec at 94°c, 1 min at 60°c, 1 min at 72°c; and 5 min at 72°c. the pcr products were separated by </span><span style=\"background-color: hsl(0, 100.00%, 94.67%); opacity: 0.81\" title=\"-0.049\">capillary</span><span style=\"opacity: 0.80\"> electrophoresis on an abi 3130xl dna analyzer. chromatograms were </span><span style=\"background-color: hsl(0, 100.00%, 99.73%); opacity: 0.80\" title=\"-0.001\">analyzed</span><span style=\"opacity: 0.80\"> with peak scanner 1.0 and genemapper 3.7 software (applied biosystems) for allelic alterations. the area of the peaks produced by the dna pcr products was quantified for each allele. the ratio of the allelic areas was calculated for each tumor and paired normal dna sample. when the qloh (allelic ratio for the tumor peaks divided by the allelic ratio of paired normal sample) was ≤0.5 or ≥2.0 for c-cbl and at least one other 11q marker in at least two separate experiments, the sample was considered as having an allelic imbalance and interpreted as loh. samples were evaluated in at least two separate experiments and samples showing prospective loh at c-cbl repeated a third time which included a new control marker at the bax locus (data not shown) on chromosome 19 to verify integrity of sample dna.  transfection of c-cbl constructs  the a549 cell line was transfected using the fugene hd (roche, nutley, nj) reagent according to the manufacturer&#x27;s instructions. eight µg of plasmid dna, containing either no insert (empty vector), wild-type c-cbl, s80n/h94y c-cbl, q249e c-cbl or w802* cbl was used for transfection in a 6-well culture plate. cells were harvested 48 h </span><span style=\"background-color: hsl(120, 100.00%, 99.72%); opacity: 0.80\" title=\"0.001\">after</span><span style=\"opacity: 0.80\"> transfection and </span><span style=\"background-color: hsl(0, 100.00%, 99.73%); opacity: 0.80\" title=\"-0.001\">analyzed</span><span style=\"opacity: 0.80\"> for expression.  c-cbl knockdown  c-cbl knockdown was performed using lentiviral transduction using mission lentiviral transduction particles (</span><span style=\"background-color: hsl(0, 100.00%, 98.64%); opacity: 0.80\" title=\"-0.007\">sigma</span><span style=\"opacity: 0.80\">-aldrich, st. louis, mo) as per manufacturer&#x27;s instructions. briefly, 1×105 h358 cells/well were seeded in 6-well plates and infected the following day with c-cbl lentiviral shrna constructs. to generate stable c-cbl knockdown cell lines, cells were selected for 2 days with 1 µg/ml puromycin. c-cbl levels were determined using whole cell lysates by immunoblotting with anti-cbl antibody (santa cruz biotechnologies, santa cruz, ca).  cell viability assay  cells were transfected as described above in the transfection assay. forty-eight hours </span><span style=\"background-color: hsl(120, 100.00%, 99.72%); opacity: 0.80\" title=\"0.001\">after</span><span style=\"opacity: 0.80\"> transfection, viability of cells was assessed using </span><span style=\"background-color: hsl(0, 100.00%, 91.71%); opacity: 0.82\" title=\"-0.092\">trypan</span><span style=\"opacity: 0.80\"> blue exclusion.  wound healing assay  a549 cells were seeded in 6-well plates and cultured for 48 h until 100% confluent. the medium was then changed and the cells were transfected as described in the transfection assay. twelve hours </span><span style=\"background-color: hsl(120, 100.00%, 99.72%); opacity: 0.80\" title=\"0.001\">after</span><span style=\"opacity: 0.80\"> transfection, a straight scratch was made across the cell layer using a 1 ml pipette tip. the cells were then gently washed with 1× pbs to remove cellular debris and the media was replaced. photographs were taken of the wound region every 12 h until 48 h.  western blot analysis  forty eight hours </span><span style=\"background-color: hsl(120, 100.00%, 99.72%); opacity: 0.80\" title=\"0.001\">after</span><span style=\"opacity: 0.80\"> transfection, cells were collected and washed twice in 1x pbs, then lysed in ice-cold lysis buffer (0.5m tris-</span><span style=\"background-color: hsl(120, 100.00%, 97.95%); opacity: 0.80\" title=\"0.012\">hcl</span><span style=\"opacity: 0.80\"> with ph 7.4, 1.5 m nacl, 2.5% deoxycholic acid, 10 mm edta, 10% np-40, 0.5 mm </span><span style=\"background-color: hsl(0, 100.00%, 99.61%); opacity: 0.80\" title=\"-0.001\">dtt</span><span style=\"opacity: 0.80\">, 1 mm phenylmethylsulfonyl fluoride, 5 µg/ml leupeptin, and 10 µg/ml aprotinin) for 5 minutes. the lysate was centrifuged at 13,000 rpm for 20 minutes at 4°c, and protein content of the supernatant was measured. total cell lysates (50 µg/well) were separated by sds-page electrophoresis and the gels transferred onto nitrocellulose membranes (whatman, piscataway, nj). membranes were blocked with 5% non-fat dry milk in phosphate-buffered saline containing </span><span style=\"background-color: hsl(120, 100.00%, 93.07%); opacity: 0.82\" title=\"0.071\">tween</span><span style=\"opacity: 0.80\">-20 (pbst) (1x pbs, 0.1% </span><span style=\"background-color: hsl(120, 100.00%, 93.07%); opacity: 0.82\" title=\"0.071\">tween</span><span style=\"opacity: 0.80\">-20) for 1 h at room temperature and incubated with the appropriate primary antibody at 4°c overnight. membranes then were washed </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> times with pbst and probed with appropriate horseradish peroxidase (hrp)-conjugated secondary antibody for 1 h at room temperature. the membranes were again washed </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> times in pbst and bands were visualized using western blot chemiluminescence reagent (biorad, valencia, ca) on a chemidoc gel documentation system (biorad, valencia, ca). antibodies were </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">obtained</span><span style=\"opacity: 0.80\"> from santa cruz biotechnologies and used at the following dilutions (c-cbl, 1∶5000; c-met, 1∶5000; egfr, 1∶5000; ubiquitin, 1∶1000; ha, 1∶5000 and β-actin, 1∶10,000).  flow cytometry  cell cycle analysis was carried out by flow cytometry. approximately 2×106 cells were grown in media containing 10% fbs. cells were harvested by trypsin/edta treatment, washed with 1x pbs </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> times and fixed with ice-cold 70% ethanol for 2 h. cells were washed again with cold pbs and stained with a solution containing 25 µg/ml propidium iodide, 200 µg/ml rnase a, and 0.1% triton x-100 for 30 minutes in the dark. cell cycle analysis was performed using a guava pca-96 flow cytometer (guava technologies, millipore, billerica, ma).  ubiquitin ligase activity  293t cells were maintained in culture in dmem supplemented with 10% fbs and 1% penicillin (100 units/ml) and streptomycin (100 µg/ml) were transfected with 0.2 µg egfr-pcdna3 and 2 µg ha-</span><span style=\"background-color: hsl(120, 100.00%, 92.98%); opacity: 0.82\" title=\"0.073\">tagged</span><span style=\"opacity: 0.80\"> c-cbl constructs as indicated using calcium phosphate according to manufacturer&#x27;s protocol (profection, promega, madison, wi). twenty-four hours post-transfection, cells were starved overnight in dmem supplemented with 0.5% fbs, and then treated with or without egf (100 ng/ml) for 15 min. the cells were collected and washed two times in ice-cold pbs containing 0.2 mm sodium orthovanadate then lysed in ice-cold lysis buffer (10 mm tris </span><span style=\"background-color: hsl(120, 100.00%, 97.95%); opacity: 0.80\" title=\"0.012\">hcl</span><span style=\"opacity: 0.80\">, ph 7.5, 150 mm nacl, 5 mm edta, 1% triton x100, 10% glycerol, 2 mm sodium orthovanadate and protease inhibitors). lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4°c. egfr immunoprecipitations were performed on 200 µg of cleared lysate using 250 ng of rabbit-anti-egfr and protein a/g plus </span><span style=\"background-color: hsl(120, 100.00%, 96.57%); opacity: 0.81\" title=\"0.026\">sepharose</span><span style=\"opacity: 0.80\"> overnight at 4°c. precipitations were washed 5 times in lysis buffer before boiling in laemmli buffer. elutions were immunoblotted with anti-ubiquitin and egfr. twenty micrograms of cleared lysate were immunoblotted for each of the c-cbl constructs using anti-ha.  statistical analysis  mutation rates between different groups were compared using fisher&#x27;s exact test. for continuous variables, group comparisons were performed using analysis of variance (anova) followed by sidak&#x27;s adjustment for </span><span style=\"background-color: hsl(0, 100.00%, 99.02%); opacity: 0.80\" title=\"-0.004\">multiple</span><span style=\"opacity: 0.80\"> comparisons. experiments involving repeated measurements over time were </span><span style=\"background-color: hsl(0, 100.00%, 99.73%); opacity: 0.80\" title=\"-0.001\">analyzed</span><span style=\"opacity: 0.80\"> using repeated measures anova with the greenhouse-geisser adjustment to the degrees of freedom. analyses were conducted using stata (v10.1) software (stata corporation, college station, tx).  go to: results c-cbl gene mutations in lung cancer  to investigate the role of c-cbl in lung cancer, we </span><span style=\"background-color: hsl(0, 100.00%, 99.73%); opacity: 0.80\" title=\"-0.001\">analyzed</span><span style=\"opacity: 0.80\"> its genomic dna in tumor and paired normal samples drawn from </span><span style=\"background-color: hsl(0, 100.00%, 99.02%); opacity: 0.80\" title=\"-0.004\">multiple</span><span style=\"opacity: 0.80\"> ethnicities. the lung tumor samples represented caucasians (n = 50), african-americans (n = 29), and taiwanese (n = 40) lung cancer patients. we designed 12 pairs of primers to sequence the coding region of c-cbl gene that spans exons 2 to 16 (table s2). we identified 8 unique somatic mutations in c-cbl exons among 8 different patients. a variation l620f, a known snp (rs2227988) in exon 11 was also detected. importantly, the eight novel non-synonymous mutations were confirmed by sequencing both strands of c-cbl genomic dna </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">obtained</span><span style=\"opacity: 0.80\"> from lung tumor samples (table 1). moreover, none of the 8 mutations were detected in the corresponding normal tissue, indicating that these were somatic mutations. four synonymous single nucleotide variations (snvs) were also identified but were not used further in this study.  table 1 table 1 c-cbl mutation analysis in 119 lung cancer patient tumor tissues. </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> of the 8 novel non-synonymous mutations were located in the tkb (tyrosine kinase binding) domain (s80n, h94y, and q249e), one in the ring finger domain (v391i), one in the proline-rich region (72515_72517 del atg), and </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> in the c-terminal region (w802*, r830k, and a848t) of the c-cbl protein (figure 1a and figure s1). in figure 1b, we show model chromatograms of representative samples.  figure 1 figure 1 c-cbl mutations and loh in non-</span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.006\">small</span><span style=\"opacity: 0.80\"> cell lung cancer. 11q loh of c-cbl gene  paired lung tumor and normal lung tissue samples from taiwanese patients (n = 37) were investigated for loh. eight (21.6%) showed loh at the c-cbl locus on chromosome 11 while 29 samples (78.4%) revealed normal allelic contribution at the microsatellite markers (figures 1c and d).  c-cbl mutations in different ethnic groups  the c-cbl double mutant s80n/h94y was found in the same patient and the overall mutation rate for c-cbl in lung </span><span style=\"background-color: hsl(120, 100.00%, 94.73%); opacity: 0.81\" title=\"0.048\">tumors</span><span style=\"opacity: 0.80\"> was 6.7% (8/119). the frequency of c-cbl mutation was highest in large cell carcinoma (</span><span style=\"background-color: hsl(120, 100.00%, 98.89%); opacity: 0.80\" title=\"0.005\">14</span><span style=\"opacity: 0.80\">.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the least was observed in adenocarcinoma (ad) (1.8%; 1 of 53 patients), although these rates were not statistically significant (p = 0.292). mutation rates were 6.0% among caucasians (0 of 20 in ad; 0 of 10 in sq; and 3 of 20 in lc), 13.8% in african-americans (1 of 10 in ad; 1 of 10 in sq; and 2 of 9 in lc), and 2.5% (0 of 23 in ad; 1 of 12 in sq; and 0 of 5 in lc) in the taiwanese population. additionally two taiwanese patients with lung cancer (one squamous and one adenocarcinoma) had the known snp l620f. ethnic differences were not statistically significant, however the power to detect differences was low.  mutations in met and egfr can be co-associated with c-cbl alterations  since east asians with lung cancer have a higher frequency of egfr and met mutations in lung </span><span style=\"background-color: hsl(120, 100.00%, 94.73%); opacity: 0.81\" title=\"0.048\">tumors</span><span style=\"opacity: 0.80\"> [21], [22], we also determined mutations in egfr and met in the same taiwanese cohort samples and compared the results with the observed c-cbl alterations (loh and/or mutations). in the 37 samples tested, we did not find any overlap between c-cbl mutations and c-cbl loh (figure 2). of the </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> (including the known l620f snp, rs2227988), one of the samples had a met mutation (n375s) and the other had an egfr mutation (l858r). among the 8 samples that had a loh at the c-cbl locus 5 had an additional mutation in met (n375s) and 2 had an egfr exon 19 deletion. twenty-six samples had neither c-cbl mutation nor c-cbl loh (3 patients had a c-cbl mutation but no c-cbl loh). among these 26 samples 9 had a met mutation (8 n375s, 1 l211w), 13 had an egfr mutation (7 exon 9 deletion, 6 l858r) and 4 had no other met or egfr mutation. thus the rate of met or egfr mutations among patients with loh at the c-cbl locus (7 of 8) was similar to that </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.007\">seen</span><span style=\"opacity: 0.80\"> in patients without c-cbl mutation or loh (22 of 26 patients) (p = 0.99). these 4 patients with no identifiable mutation in c-cbl, met or egfr represented 10.8% of the 37 patients </span><span style=\"background-color: hsl(0, 100.00%, 99.73%); opacity: 0.80\" title=\"-0.001\">analyzed</span><span style=\"opacity: 0.80\"> in the taiwanese patient cohort. conversely, 89.2% taiwanese lung cancer patients have an identifiable mutation in either c-cbl, met or egfr or a combination of the </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> genes (figure 2). additionally, we determined p53 and kras mutations in these taiwanese cohorts. two p53 and 1 kras mutation were detected. the single kras mutation overlapped with one p53 mutation. this patient also had the egfr exon 19 deletion but had no c-cbl mutation. the other p53 mutation sample had a c-cbl loh with concurrent met n375s mutation. thus, in the taiwanese samples </span><span style=\"background-color: hsl(0, 100.00%, 99.73%); opacity: 0.80\" title=\"-0.001\">analyzed</span><span style=\"opacity: 0.80\">, p53/kras mutations and c-cbl mutations were mutually exclusive (data not shown).  figure 2 figure 2 c-cbl mutations and relationship to met and egfr mutations in lung cancer. cellular functions of c-cbl alterations in the context of lung </span><span style=\"background-color: hsl(0, 100.00%, 99.85%); opacity: 0.80\" title=\"-0.000\">tumorigenesis</span><span style=\"opacity: 0.80\">  a. </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> activity is intact in the mutant c-cbl proteins to investigate whether the different c-cbl mutations affect the </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> activity, egfr was chosen as a model substrate for c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.96%); opacity: 0.80\" title=\"-0.005\">function</span><span style=\"opacity: 0.80\">. all of the c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> tested enhanced ubiquitination of the activated egfr similar to the wild-type c-cbl protein. this result demonstrates that the catalytic activity of the c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> is not impaired when egfr was the substrate. (figure 3a).  figure 3 figure 3 ubiquitination, viability, expression and cell cycle analysis of various c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\">. b. effect on lung cancer cell viability the effect of a representative c-cbl mutant from each of the </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> ethnic backgrounds on lung cancer cell viability in cell lines was determined. s80n/h94y double mutation, q249e, and w802* were identified in lung tumor samples </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">obtained</span><span style=\"opacity: 0.80\"> from a caucasian, a taiwanese and an african-american, respectively. as described in methods, the c-cbl wild-type (wt) and the above </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> were expressed </span><span style=\"background-color: hsl(120, 100.00%, 99.72%); opacity: 0.80\" title=\"0.001\">after</span><span style=\"opacity: 0.80\"> cloning them into paltermax vector in a549 cells. these cells express relatively low basal levels of endogenous c-cbl (data not shown). transfection efficiency was comparable between different groups and the number of cells transfected with c-cbl wild-type construct was about 70% compared to control cells that were transfected with the empty vector. cells transfected with s80n/h94y, q249e and w802* c-cbl mutant constructs resulted in increased number of viable cells that was 132.3%, 120.8% and 147.9% higher respectively, relative to the empty vector control transfected cells and significantly different from the wild-type construct (p = 0.022, p = 0.049, and p = 0.008, respectively) (figure 3b). relative levels of c-cbl protein in whole cell lysates prepared from samples </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">obtained</span><span style=\"opacity: 0.80\"> from a parallel experiment was determined. the c-cbl protein levels in samples representing untransfected and empty vector transfected cells were comparable and those representing the c-cbl wt and the </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> were comparable (figure 3c).  c. effect on cell cycle to investigate if the increases in cell viability in different c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> due to increased cellular proliferation, a cell cycle analysis was performed. a549 cells were transfected with the c-cbl wt or the </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> different </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\">: s80n/h94y, q249e and w802*. the empty vector transfectant was used as a control. forty-eight hours </span><span style=\"background-color: hsl(120, 100.00%, 99.72%); opacity: 0.80\" title=\"0.001\">after</span><span style=\"opacity: 0.80\"> transfection cell cycle analysis was performed as described in materials and methods. there was no significant </span><span style=\"background-color: hsl(0, 100.00%, 97.02%); opacity: 0.80\" title=\"-0.021\">change</span><span style=\"opacity: 0.80\"> in the subg1, g1 or the s phase of the cell cycle among the different </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> compared to the wt construct (p = 0.64, p = 0.40, and p = 0.28, respectively). the g2/m phase of the cell cycle showed an increase in cell numbers for the </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\">, s80n/h94y, q249e and w802*, when compared to the wt but again the difference was not statistically significant (p = 0.25) (figure 3d).  d. effect on cell motility to investigate the effect of the expression of the above </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> on cell migration, we carried out wound healing assay as described in materials and methods. the closing of the scratch or the wound was monitored at 0, 12, 24, 36, and 48 h. (figure 4a). in all the samples, that represented cells transfected with </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\">, the wound gap was much smaller than that </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.007\">seen</span><span style=\"opacity: 0.80\"> in the sample that represented cells transfected with c-cbl wt (p&lt;0.001). we also determined the rate of wound closure for all the five groups. at 48 h, wild-type c-cbl transfectants showed 61.1% open wound while the s80n/h94y, q249e and w802* </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> showed 18.7%, 23.9% and 34.3% open wound respectively (p&lt;0.001) (figure 4b).  figure 4 figure 4 c-cbl mutations affect wound healing in a549 cells. e. c-cbl knockdown increases cell viability it is hypothesized that the loh </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.007\">seen</span><span style=\"opacity: 0.80\"> in our samples could lead to decreased expression of c-cbl. thus we tested the effect of c-cbl knockdown in lung cancer cells. compared to a549, h358 lung cancer cells express relatively high levels of endogenous c-cbl (data not shown). c-cbl expression was knocked down using lentiviral construct that expressed c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.62%); opacity: 0.80\" title=\"0.007\">specific</span><span style=\"opacity: 0.80\"> shrna and compared the results with those that were transduced with scrambled shrna and the results </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> shown in figure 5. we identified several clones that revealed varying degrees of c-cbl knockdown showing different sets of c-cbl lentiviral shrna knockdown efficiency (figure 5a). of all the clones tested, </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">clone</span><span style=\"opacity: 0.80\"> 27 was chosen for further experiments. equal amount of cells were seeded in a 6-well plate and the cell proliferation was measured at various times and the results </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> depicted in figure 5b. as expected, number of cells increased in a time dependent fashion from 100 to 190% relative to scrambled shrna as control in a </span><span style=\"background-color: hsl(0, 100.00%, 98.98%); opacity: 0.80\" title=\"-0.005\">span</span><span style=\"opacity: 0.80\"> of 48 h (p = 0.0002) (figure 5b). the cell cycle phases in h358 cells that were knocked down with c-cbl shrna were looked at and compared with the scrambled shrna. there were no discernable differences between these two constructs in the different phases of the cell cycle (data not shown).  figure 5 figure 5 knockdown of c-cbl using an shrna increases cell proliferation. go to: discussion our results demonstrate that c-cbl is somatically mutated (or has loh) in lung cancers, and can significantly contribute to enhanced cell viability and motility. there was also a high prevalence of loh with respect to c-cbl in lung </span><span style=\"background-color: hsl(120, 100.00%, 94.73%); opacity: 0.81\" title=\"0.048\">tumors</span><span style=\"opacity: 0.80\"> that harbored met or egfr mutation.  in the present study, we have demonstrated the occurrence of c-cbl mutations in lung cancer patients, especially with different ancestral variations. mutations in c-cbl have been </span><span style=\"background-color: hsl(0, 100.00%, 96.12%); opacity: 0.81\" title=\"-0.031\">recently</span><span style=\"opacity: 0.80\"> reported in juvenile myelomonocytic leukemia and myeloid malignancies. in the aml study, the mutation r420q located in the junction of the ring finger and the linker region inhibited fms-like tyrosine kinase 3 (flt3) internalization and ubiquitination [20], thus contributing to the gain-in-</span><span style=\"background-color: hsl(0, 100.00%, 98.96%); opacity: 0.80\" title=\"-0.005\">function</span><span style=\"opacity: 0.80\"> for the rtk. in addition, mutations such as h398y, c384r, and l380p mapped to the ring finger domain and the linker region of c-cbl that is required for its </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> activity [23], [24], [25], [26], [27]. additionally, </span><span style=\"background-color: hsl(120, 100.00%, 95.44%); opacity: 0.81\" title=\"0.039\">homozygous</span><span style=\"opacity: 0.80\"> mutations in the ring finger domain of the c-cbl gene were described as a result of acquired uniparental disomy (upd) [26]. it is important to note that our results indicate loh at 11q23 locus and these </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> mutually exclusive from missense mutations of c-cbl. the somatic mutations were all heterozygous. the mutations in aml led to abrogation of the </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> activity leading to prolonged rtk </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.008\">activation</span><span style=\"opacity: 0.80\">. in addition </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> located on the linker region surrounding the ring finger domain exhibited enhanced </span><span style=\"background-color: hsl(0, 100.00%, 93.46%); opacity: 0.82\" title=\"-0.065\">akt</span><span style=\"opacity: 0.80\"> signaling in </span><span style=\"background-color: hsl(0, 100.00%, 96.20%); opacity: 0.81\" title=\"-0.030\">response</span><span style=\"opacity: 0.80\"> to cytokine stimulation [26]. in addition, it was shown in nh3t3 cells, that neither mutations in the ring finger nor the linker region causes transformation, however while certain mutations perturbs the ubiquitination, others affect receptor recycling and prolong kinase activity [28].  we report here c-cbl mutations that mapped not only to the ring finger domain, but also to the tkb domain, proline-rich domain and the c-terminal region, but none mapped to the linker region as reported in the aml studies described above [23], [24], [25], [26], [29]. in addition, 8 </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> that we detected were found in different ethnic backgrounds. for example, s80n/h94y, q249e, w802* were detected in caucasians, taiwanese and african-americans, respectively. the results point out not only the difference between lung cancer and other cancers, but also genetic polymorphism among different races in the same cancer. interestingly, there is a large disparity between african-american and other ethnic populations with lung cancer [30]. we have previously shown that there was a low frequency of egfr and met mutation in african-americans as compared with taiwanese and caucasians [31]. in this study, the number of african-american samples </span><span style=\"background-color: hsl(0, 100.00%, 99.73%); opacity: 0.80\" title=\"-0.001\">analyzed</span><span style=\"opacity: 0.80\"> was relatively fewer and we found 3 mutations that </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> unique to this ethnicity. it would now behoove us to further study the genetic alterations that can occur and determine the targeted therapeutics for african-americans. our results provide evidence of the importance of c-cbl in </span><span style=\"background-color: hsl(0, 100.00%, 99.85%); opacity: 0.80\" title=\"-0.000\">tumorigenesis</span><span style=\"opacity: 0.80\"> and potential signaling. our prediction, based on the aml data, would be that the v391i ring finger domain mutation would affect the </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> activity. also, it will be important to determine the binding partners for c-cbl in the tkb domain and proline rich domain mutations. it has previously been shown that the tkb domain can bind to growth factor receptors and it will be important to determine the cross-binding of these </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> to met and or egfr. it would also be important in the future to look at fluorescence in situ hybridization/copy number changes in c-cbl in lung cancer.  c-cbl plays an important role in down regulating rtk-mediated signaling through k63 poly-ubiquitination and subsequent downregulation of rtks followed by lysosomal degredation [3]. mono-ubiquitination or ubiquitinated with k63-linked chains of substrates by c-cbl may lead to enhancement of biological and biochemical functions (reviewed in hermann et al, 2007 [32]). the mutations that we </span><span style=\"background-color: hsl(0, 100.00%, 99.73%); opacity: 0.80\" title=\"-0.001\">analyzed</span><span style=\"opacity: 0.80\"> in our studies all point out to the fact that the </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> activity of c-cbl on egfr is intact; the egfr levels in the various </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> remain same (figure s2). </span><span style=\"background-color: hsl(0, 100.00%, 99.02%); opacity: 0.80\" title=\"-0.004\">multiple</span><span style=\"opacity: 0.80\"> kinases, both rtks and non-rtks could be acted upon by c-cbl, including erbs, pdgfr, fms, met, c-kit, vegfr, flt-1, ron, fgfr, ir, as well as syk, fyn, lck, fgr, lyn and c-abl [3]. in lung cancers the relevant substrates of c-cbl in terms of degradation or signal transduction </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> yet to be identified.  the observation that c-cbl somatic mutations, especially s80n/h94y, q249e and w802* showed increased cell viability and cell motility that is in agreement with the physiological role for d-cbl in the regulation of apoptosis and differentiation identified in drosophila is very significant [33]. it has been previously shown that </span><span style=\"background-color: hsl(0, 100.00%, 90.93%); opacity: 0.82\" title=\"-0.105\">activating</span><span style=\"opacity: 0.80\"> c-cbl mutation downregulates egfr signaling and decreases cellular proliferation and migration in breast cancer cell lines [34]. although the role of c-cbl in the </span><span style=\"background-color: hsl(0, 100.00%, 98.23%); opacity: 0.80\" title=\"-0.010\">negative</span><span style=\"opacity: 0.80\"> regulation of rtks is well substantiated, thereby suggesting that it is a natural tumor </span><span style=\"background-color: hsl(120, 100.00%, 97.90%); opacity: 0.80\" title=\"0.013\">suppressor</span><span style=\"opacity: 0.80\">, studies in cancer cells have revealed both tumor </span><span style=\"background-color: hsl(120, 100.00%, 97.90%); opacity: 0.80\" title=\"0.013\">suppressor</span><span style=\"opacity: 0.80\"> and tumor promoting activities depending on the type of c-cbl mutation and the number of alleles at the c-cbl locus [24]. in agreement with the above, the </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> described here appear to have tumor growth and metastasis promoting properties. although these </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> outside of the ring finger or the linker region of c-cbl, their downstream effects </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> significant so as to cause increased proliferation and migration, but the substrate </span><span style=\"background-color: hsl(0, 100.00%, 98.18%); opacity: 0.80\" title=\"-0.011\">affected</span><span style=\"opacity: 0.80\"> by these mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> not known yet. this raises the possibility that some of the cellular functions of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.96%); opacity: 0.80\" title=\"-0.005\">independent</span><span style=\"opacity: 0.80\"> of its ubiquitin-ligase activity, an area that we </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> currently investigating. the oncogenic nature of rtks, addiction of cancers to growth signals and given the clustering of c-cbl, egfr and met mutations, it is possible that the transforming effect of c-cbl mutations is most likely a combinatorial effect of the </span><span style=\"background-color: hsl(120, 100.00%, 99.73%); opacity: 0.80\" title=\"0.001\">three</span><span style=\"opacity: 0.80\">. we also show that loh for c-cbl was found in a significant number of samples that harbored met or egfr mutations. the fact that about 7% of lung tumor samples </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> likely to have c-cbl mutations and an additional 22% </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> likely to harbor c-cbl-related loh, makes c-cbl a highly mutated molecule in lung cancer. since loh alone is not enough to cause a transforming event [35], [36], [37], associated mutation in the met or egfr locus or yet another rtk discussed above may play a role in carcinogenesis. we predict that this loh in c-cbl results in haploinsufficiency that downplays rtk ubiquitination leading to hyperactivity of the rtks. however, whether this is sufficient cause for </span><span style=\"background-color: hsl(0, 100.00%, 99.85%); opacity: 0.80\" title=\"-0.000\">tumorigenesis</span><span style=\"opacity: 0.80\"> remains to be determined. consistent with our hypothesis is the fact that c-cbl-/- mice have increased kinase activity in lymphocytes, but is not sufficient for tumor formation [35], [36], [37]. c-cbl loh could also lead to increased expression of c-cbl from the other allele to compensate for a loss of an allele. alternately, there could be some form of synergy working with reduced c-cbl levels and mutated receptors that exacerbate the phenotype of each alone.  previous studies from our lab and others have shown that east asians with lung cancers have relatively high frequencies of gain-of-</span><span style=\"background-color: hsl(0, 100.00%, 98.96%); opacity: 0.80\" title=\"-0.005\">function</span><span style=\"opacity: 0.80\"> of mutations in rtks such as egfr and met [31]. in a cohort of japanese patients an </span><span style=\"background-color: hsl(0, 100.00%, 90.93%); opacity: 0.82\" title=\"-0.105\">activating</span><span style=\"opacity: 0.80\"> met mutation was identified in the splice region that deletes the </span><span style=\"background-color: hsl(0, 100.00%, 83.44%); opacity: 0.86\" title=\"-0.247\">juxtamembrane</span><span style=\"opacity: 0.80\"> domain that is involved </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.007\">e3</span><span style=\"opacity: 0.80\"> activity of c-cbl [38]. this study also found that </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.008\">activation</span><span style=\"opacity: 0.80\"> of met is mutually exclusive of egfr, kras and her2 gene mutations [38]. we failed to detect such mutations in significant numbers in lung tumor samples </span><span style=\"background-color: hsl(0, 100.00%, 99.30%); opacity: 0.80\" title=\"-0.003\">obtained</span><span style=\"opacity: 0.80\"> from african-americans (n = 29) and caucasian (n = 50) patients. one met mutation was identified in each of the groups whereas 1 and 3 egfr mutations were identified in the african-american and caucasian cohorts respectively. egfr mutations have earlier been identified as one of the key mutations affecting lung adenocarcinoma patients in a comprehensive study of 188 patients [39]. our study encompasses different histologies of nsclc. however, the published series did not find any mutations in c-cbl or met unlike our study that encompassed different subtypes of nsclc. it is important to note, we have </span><span style=\"background-color: hsl(0, 100.00%, 96.12%); opacity: 0.81\" title=\"-0.031\">recently</span><span style=\"opacity: 0.80\"> shown that met mutations in lung cancer </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> in majority germline [31]. we have reported earlier c-cbl mutations in a </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.006\">small</span><span style=\"opacity: 0.80\"> cohort of taiwanese lung cancer samples [40]. in our efforts to understand the ethnic differences in the lung oncogenome, we also looked at pax transcription </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.005\">factors</span><span style=\"opacity: 0.80\"> such as pax5 and pax8 that </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> highly expressed in lung cancers; however there was no preferential expression or mutations of the above genes in lung tumor samples of african-americans. in this study, we show relatively high frequency of c-cbl mutations in lung cancers, especially in the large cell type among caucasians and particularly among african-americans. we therefore propose c-cbl as an efficacious target for lung cancers in african-americans that needs to be further substantiated. this is all the more important because the prognosis for african-americans with lung cancer, especially for men is much poorer compared to their caucasian counterparts [41].  in conclusion, the results presented in this study demonstrate that c-cbl is frequently mutated or even lost in lung cancers. our results support a role for c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.002\">mutants</span><span style=\"opacity: 0.80\"> that </span><span style=\"background-color: hsl(120, 100.00%, 99.90%); opacity: 0.80\" title=\"0.000\">are</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.96%); opacity: 0.80\" title=\"-0.005\">independent</span><span style=\"opacity: 0.80\"> of its ubiquitination activity. given the relatively high mutation rates in c-cbl as well as rtks such as met and egfr, it is likely that their combined effect could be synergistic in promoting </span><span style=\"background-color: hsl(0, 100.00%, 99.85%); opacity: 0.80\" title=\"-0.000\">tumorigenesis</span><span style=\"opacity: 0.80\">.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=5\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.019</b>, score <b>-2.074</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.32%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.027\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__summarized\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.33%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.026\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__months\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.45%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__material\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__specimen\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.52%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__apoptotic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inhibitor\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__possibly\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.63%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__73\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__united\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__variable\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__upstream\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__transiently\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__his\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__normally\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__sought\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nucleotides\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__virus\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__morphology\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__analysed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__enhancing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__chinese\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__polymorphisms\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__kda\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__350\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__preventive\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__trained\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__731\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nih\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cac\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__traces\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__anion\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__width\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__biolabs\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__suggested\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2008\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__charged\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__241\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__insights\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__releasing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__larger\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__coordinates\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fifty\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__needed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__improvements\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__polyacrylamide\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__purification\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bipartite\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__share\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ig\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__purchased\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pik3ca\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__channel\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stromal\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__designated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__focused\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.63%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__contrast\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fgfr2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__volumes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.55%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__applications\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.50%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__interior\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.50%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__level\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.46%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.019\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__allow\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.36%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.025\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__finally\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.33%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.026\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__being\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.145\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__variants\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.168\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__neutral\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 95.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.390\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 89.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -1.284\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">w802*</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">cbl</span><br/><b>Text:</b> <span style=\"opacity: 0.80\"> abstract background  non-small cell lung cancer (nsclc) is a heterogeneous </span><span style=\"background-color: hsl(0, 100.00%, 99.25%); opacity: 0.80\" title=\"-0.003\">group</span><span style=\"opacity: 0.80\"> of disorders with a number of genetic and proteomic alterations. c-cbl is an e3 ubiquitin ligase and adaptor molecule important in </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\"> homeostasis and cancer. we determined the genetic variations of c-cbl, relationship to receptor tyrosine kinases (egfr and met), and functionality in nsclc.  methods and findings  using archival formalin-fixed paraffin embedded (ffpe) extracted genomic dna, we show that c-cbl mutations occur in somatic fashion for lung cancers. c-cbl mutations were </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> mutually exclusive of met or egfr mutations; </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">however</span><span style=\"opacity: 0.80\"> they were independent of p53 and kras mutations. in </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\">/tumor pairwise analysis, there was significant loss of heterozygosity (loh) for the c-cbl locus (22%, n = 8/37) and </span><span style=\"background-color: hsl(0, 100.00%, 99.75%); opacity: 0.80\" title=\"-0.001\">none</span><span style=\"opacity: 0.80\"> of these samples revealed any mutation in the remaining copy of c-cbl. the c-cbl loh also positively correlated with egfr and met mutations observed in the same samples. using select c-cbl somatic mutations such as s80n/h94y, q249e and w802* (obtained from caucasian, taiwanese and african-american samples, respectively) transfected in nsclc cell </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">lines</span><span style=\"opacity: 0.80\">, there was increased cell viability and cell motility.  </span><span style=\"background-color: hsl(0, 100.00%, 93.26%); opacity: 0.82\" title=\"-0.068\">conclusions</span><span style=\"opacity: 0.80\">  taking the </span><span style=\"background-color: hsl(120, 100.00%, 98.53%); opacity: 0.80\" title=\"0.008\">overall</span><span style=\"opacity: 0.80\"> mutation rate of c-cbl to be a combination as somatic missense mutation and loh, it is clear that c-cbl is </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.001\">highly</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">mutated</span><span style=\"opacity: 0.80\"> in lung cancers and may play an essential </span><span style=\"background-color: hsl(0, 100.00%, 98.93%); opacity: 0.80\" title=\"-0.005\">role</span><span style=\"opacity: 0.80\"> in lung tumorigenesis and metastasis.  go to: introduction in the us alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which more than 145,000 of them succumb to the disease [1]. </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> number is roughly equivalent to the combined mortality rates of cancers of the breast, prostate, colon, liver, kidney and melanoma [1]. in addition the prognosis is usually poor and the five-year survival rate is less than 15%. there are also significant ethnic differences for lung cancer, and the outcome is worse for blacks compared to whites. gender differences are also striking with women having significantly better prognosis as compared to men. there are a number of genetic alterations that can occur in lung cancer. as an </span><span style=\"background-color: hsl(0, 100.00%, 98.44%); opacity: 0.80\" title=\"-0.008\">example</span><span style=\"opacity: 0.80\">, in nsclc, mutations in kras, p53, egfr and met have </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">been</span><span style=\"opacity: 0.80\"> identified. many of these pathways, especially receptor tyrosine kinases (rtks) are controlled by c-cbl.  cbl (casitas b-lineage lymphoma) is a mammalian </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">gene</span><span style=\"opacity: 0.80\"> located on human chromosome 11q23.3 [2] and is involved in cell signaling and protein ubiquitination [3]. cbl proteins belong to the ring finger class of ubiquitin ligases (e3) and there are three homologues c-cbl, cbl-b, cbl-3 [4]. the c-cbl and cbl-b genes are ubiquitously expressed with the highest levels in hematopoietic </span><span style=\"background-color: hsl(0, 100.00%, 98.11%); opacity: 0.80\" title=\"-0.011\">tissues</span><span style=\"opacity: 0.80\"> [5]. c-cbl consists of four regions encoding for functionally distinct protein domains: the n-terminal tyrosine kinase </span><span style=\"background-color: hsl(0, 100.00%, 98.95%); opacity: 0.80\" title=\"-0.005\">binding</span><span style=\"opacity: 0.80\"> (tkb) domain, the linker region, the catalytic ring finger domain, the proline-rich region and the c-terminal ubiquitin-associated (uba) domain that also overlaps with a leucine-zipper (lz) domain [3]. </span><span style=\"background-color: hsl(120, 100.00%, 98.64%); opacity: 0.80\" title=\"0.007\">both</span><span style=\"opacity: 0.80\"> tkb and ring finger domains are essential for ligand-induced ubiquitination of rtks [6], [7], [8], [9]. the ring finger domain is required for the recruitment of e2 ubiquitin-conjugating enzymes. the tkb domain includes four-helix bundle (4h), a calcium-biding ef hand, and a modified sh2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. in addition, the proline-rich region of c-cbl can associate with the sh3 domain of grb2, which can indirectly recruit c-cbl to rtks via the grb2 adaptor protein [7], [</span><span style=\"background-color: hsl(0, 100.00%, 99.22%); opacity: 0.80\" title=\"-0.003\">13</span><span style=\"opacity: 0.80\">], [14].  c-cbl also binds to egfr and acts as the e3 that targets egfr for ubiquitination and degradation. furthermore, cbl desensitizes egf signaling and opposes cellular proliferation induced by egf [15]. egf activation also appears to activate the tyrosine kinase src, which phosphorylates c-cbl and in turn activates the ubiquitination and degradation of egfr [16], [17], [18]. a recent study shows that defective endocytosis of egfr is characterized by a deletion mutant and the point mutation l858r, whereby its association with c-cbl and subsequent ubiquitination are impaired [19]. recently, the first human c-cbl mutations were reported in acute myeloid leukemia (aml) patients [20]. the mutation r420q inhibits fms-like tyrosine kinase 3 (flt3) internalization and ubiquitination [20].  </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> only can e3 activity be important in oncogenesis, c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 97.78%); opacity: 0.80\" title=\"-0.014\">has</span><span style=\"opacity: 0.80\"> a dual but separate function as a signal transduction molecule. we have </span><span style=\"background-color: hsl(120, 100.00%, 99.53%); opacity: 0.80\" title=\"0.002\">previously</span><span style=\"opacity: 0.80\"> shown that c-cbl is important in </span><span style=\"background-color: hsl(0, 100.00%, 98.95%); opacity: 0.80\" title=\"-0.005\">binding</span><span style=\"opacity: 0.80\"> crkl and bcr/abl in hematopoietic cells. also, it can bind and modulate functions of cytoskeleton by </span><span style=\"background-color: hsl(0, 100.00%, 98.95%); opacity: 0.80\" title=\"-0.005\">binding</span><span style=\"opacity: 0.80\"> to proteins like talin and paxillin. the tkb domain is important in </span><span style=\"background-color: hsl(0, 100.00%, 98.95%); opacity: 0.80\" title=\"-0.005\">binding</span><span style=\"opacity: 0.80\"> to a number of molecules, and they then function in signal transduction.  given the critical </span><span style=\"background-color: hsl(0, 100.00%, 98.93%); opacity: 0.80\" title=\"-0.005\">role</span><span style=\"opacity: 0.80\"> of cbl in </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\"> homeostasis and cancer, we hypothesized that it might be </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">mutated</span><span style=\"opacity: 0.80\"> in lung cancers. in </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> study, we report novel c-cbl somatic mutations s80n/h94y, q249e and w802* in caucasian, taiwanese and african-american lung cancer patients, respectively. expressing these mutations in nsclc cell </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">lines</span><span style=\"opacity: 0.80\"> lead to increased proliferation and cell motility. we show that c-cbl mutations occur with or without met or egfr mutations but are mutually exclusive of a loh at the c-cbl locus. additionally, c-cbl loh is associated with either met or egfr mutations. we thus hypothesize that c-cbl mutations might contribute to the oncogenic potential of met and egfr in lung cancer.  go to: methods ethics statement  written consent on all research on human subjects </span><span style=\"background-color: hsl(0, 100.00%, 97.78%); opacity: 0.80\" title=\"-0.014\">has</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">been</span><span style=\"opacity: 0.80\"> obtained from the institutional review board, university of chicago and covers all research performed in the laboratory. the following is their contact information: institutional review board, the university of chicago, mcgiffert hall, 5751 s. woodlawn ave., 2nd floor, chicago, il 60637. written informed consents were received from all patients whose tissue samples were used for </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> study.  tissue samples  lung cancer tissue and paired adjacent </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\"> lung </span><span style=\"background-color: hsl(0, 100.00%, 98.11%); opacity: 0.80\" title=\"-0.011\">tissues</span><span style=\"opacity: 0.80\"> were obtained from 50 caucasian, 29 african-americans and 40 taiwanese nsclc patients who were recruited at the university of chicago hospital (chicago, usa) (caucasian and african-american patients) and taipei veterans general hospital of taiwan (taiwanese patients) after obtaining appropriate institutional review board permission and informed consent from the patients. out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 </span><span style=\"background-color: hsl(120, 100.00%, 97.56%); opacity: 0.80\" title=\"0.016\">years</span><span style=\"opacity: 0.80\">. in terms of tumor types, </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">53</span><span style=\"opacity: 0.80\"> were adenocarcinoma, 32 were squamous cell carcinoma and 34 were large cell carcinoma. 49 were stage i, 14 were stage ii, 34 were stage iii, and </span><span style=\"background-color: hsl(0, 100.00%, 99.22%); opacity: 0.80\" title=\"-0.003\">13</span><span style=\"opacity: 0.80\"> were stage iv (table s1).  cell culture  human non-small cell lung carcinoma cells a549 and h358 were maintained in dmem and rpmi-1640, respectively. human embryonic kidney 293t cells were cultured in dmem. media were </span><span style=\"background-color: hsl(120, 100.00%, 96.18%); opacity: 0.81\" title=\"0.030\">supplemented</span><span style=\"opacity: 0.80\"> with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (invitrogen, carlsbad, ca). cells were cultured at 37°c in a humidified incubator containing 5% co2.  c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">gene</span><span style=\"opacity: 0.80\"> mutational analysis  exons 2 to 16 of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">gene</span><span style=\"opacity: 0.80\"> were individually amplified by polymerase chain reaction (pcr). primers are listed in table s2. pcr conditions were 1 cycle of 95°c for 5 minutes; 35 cycles of 94°c for 30 seconds, 58°c for 30 seconds and 72°c for 2 minutes; and one cycle of 72°c for 10 minutes. pcr products were treated with exosap-it (usb corporation, cleveland, oh) and sequenced by big-dye terminator chemistry (applied biosystems, foster city, ca). sequencing was performed on the forward coding strand with confirmation of c-cbl alterations performed by sequencing the reverse strand as well. chromatograms were analyzed for mutations using mutation surveyor v2.61 (softgenetics, state college, pa).  plasmid constructs and </span><span style=\"background-color: hsl(120, 100.00%, 98.77%); opacity: 0.80\" title=\"0.006\">site</span><span style=\"opacity: 0.80\">-directed mutagenesis  the wild-type c-cbl cdna insert was subcloned into the paltermax expression vector using xhoi and sali restriction enzyme sites (promega, madison, wi). using </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> parental plasmid paltermax-c-cbl, the tkb domain double mutation (s80n/h94y), the point mutation (q249e), and the c-terminal point mutation w802* of c-cbl were </span><span style=\"background-color: hsl(120, 100.00%, 96.19%); opacity: 0.81\" title=\"0.030\">created</span><span style=\"opacity: 0.80\"> using the following primers: 5′-gctggcgctaaagaataacccaccttatatcttagac-3′ and 5′-ctaccagatacctaccagtatctccgtactatcttgtc-3′ for the double mutation s80n/h94y; 5′-ctttacccgactctttgagccctggtcctctttgc-3′ for q249e, and 5′-cagctcctcctttggctgattgtctctggatggtgatc-3′ for w802* along with their complementary primers using the quickchange </span><span style=\"background-color: hsl(120, 100.00%, 98.77%); opacity: 0.80\" title=\"0.006\">site</span><span style=\"opacity: 0.80\">-directed mutagenesis xl kit (stratagene, la jolla, ca) according to the manufacturer&#x27;s instructions. the constructs were confirmed for the point mutations by standard dna sequencing of </span><span style=\"background-color: hsl(120, 100.00%, 98.64%); opacity: 0.80\" title=\"0.007\">both</span><span style=\"opacity: 0.80\"> strands.  loss of heterozygosity (loh) analysis  five microsatellites on chromosome 11 (3 on 11q at or within 200 kb up or downstream of the c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">gene</span><span style=\"opacity: 0.80\"> and 2 control markers on 11p) were selected for analysis (table s3). established microsatellite markers and respective primer sequences were selected from the geneloc database (http://genecards.weizmann.ac.il/geneloc/index.shtml, weizmann institute of science, rehovot, israel). primers were custom designed and each forward primer was fluorescently labeled at the 5′ </span><span style=\"background-color: hsl(120, 100.00%, 98.64%); opacity: 0.80\" title=\"0.007\">end</span><span style=\"opacity: 0.80\"> with fam, pet, ned, or vic (applied biosystems). primer annealing temperatures and duplex scores were evaluated with nist primer tools (http://yellow.nist.gov:8444/dnaanalysis/primertoolspage.do; national institute of standards and technology, gaithersburg, md). primers were verified by performing pcr with control dna (isolated from tk6 cells) and resolving the products on agarose gels. bands were visualized with an uv transilluminator. genomic dna was extracted from tumor samples and paired </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\"> lung tissue. primers were grouped into multiplex combinations shown in table s4. marker d11s929 served as an </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.001\">internal</span><span style=\"opacity: 0.80\"> control to check for consistency in pcrs and of peaks from capillary electrophoresis. multiplex pcrs were carried out in a volume of 10 µl that contained 1 µl genomic dna (20–50 ng), 0.5 µm of each primer (1.0 µm total for each primer pair), 400 µm dntps, 1x pcr buffer containing mgcl2, and 0.2 u taq dna polymerase. pcr was performed on the abi geneamp 9700 pcr system under the following conditions: 5 </span><span style=\"background-color: hsl(120, 100.00%, 99.75%); opacity: 0.80\" title=\"0.001\">min</span><span style=\"opacity: 0.80\"> at 94°c; 30 cycles of 30 sec at 94°c, 1 </span><span style=\"background-color: hsl(120, 100.00%, 99.75%); opacity: 0.80\" title=\"0.001\">min</span><span style=\"opacity: 0.80\"> at 60°c, 1 </span><span style=\"background-color: hsl(120, 100.00%, 99.75%); opacity: 0.80\" title=\"0.001\">min</span><span style=\"opacity: 0.80\"> at 72°c; and 5 </span><span style=\"background-color: hsl(120, 100.00%, 99.75%); opacity: 0.80\" title=\"0.001\">min</span><span style=\"opacity: 0.80\"> at 72°c. the pcr products were separated by capillary electrophoresis on an abi 3130xl dna analyzer. chromatograms were analyzed with peak scanner 1.0 and genemapper 3.7 software (applied biosystems) for allelic alterations. the area of the peaks produced by the dna pcr products was quantified for each allele. the ratio of the allelic areas was calculated for each tumor and paired </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\"> dna sample. when the qloh (allelic ratio for the tumor peaks divided by the allelic ratio of paired </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\"> sample) was ≤0.5 or ≥2.0 for c-cbl and at least one other 11q marker in at least two separate experiments, the sample was considered as having an allelic imbalance and interpreted as loh. samples were evaluated in at least two separate experiments and samples showing prospective loh at c-cbl repeated a third time which included a new control marker at the bax locus (</span><span style=\"background-color: hsl(0, 100.00%, 98.84%); opacity: 0.80\" title=\"-0.006\">data</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> shown) on chromosome 19 to verify integrity of sample dna.  transfection of c-cbl constructs  the a549 cell line was transfected using the fugene hd (roche, nutley, nj) reagent according to the manufacturer&#x27;s instructions. eight µg of plasmid dna, containing either </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">no</span><span style=\"opacity: 0.80\"> insert (empty vector), wild-type c-cbl, s80n/h94y c-cbl, q249e c-cbl or w802* cbl was used for transfection in a 6-well culture plate. cells were harvested 48 h after transfection and analyzed for expression.  c-cbl knockdown  c-cbl knockdown was performed using lentiviral transduction using mission lentiviral transduction particles (sigma-aldrich, st. louis, mo) as </span><span style=\"background-color: hsl(0, 100.00%, 96.04%); opacity: 0.81\" title=\"-0.032\">per</span><span style=\"opacity: 0.80\"> manufacturer&#x27;s instructions. briefly, 1×105 h358 cells/well were seeded in 6-well plates and infected the following day with c-cbl lentiviral shrna constructs. to generate stable c-cbl knockdown cell </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">lines</span><span style=\"opacity: 0.80\">, cells were selected for 2 days with 1 µg/ml puromycin. c-cbl levels were determined using whole cell lysates by immunoblotting with anti-cbl antibody (santa cruz biotechnologies, santa cruz, ca).  cell viability assay  cells were transfected as described above in the transfection assay. forty-eight hours after transfection, viability of cells was assessed using trypan blue exclusion.  wound healing assay  a549 cells were seeded in 6-well plates and cultured for 48 h until 100% confluent. the medium was then changed and the cells were transfected as described in the transfection assay. twelve hours after transfection, a straight scratch was made across the cell layer using a 1 ml pipette tip. the cells were then gently washed with 1× pbs to remove cellular debris and the media was replaced. photographs were taken of the wound region every 12 h until 48 h.  western blot analysis  forty eight hours after transfection, cells were collected and washed twice in 1x pbs, then lysed in ice-cold lysis buffer (0.5m tris-hcl with ph 7.4, 1.5 m nacl, 2.5% deoxycholic acid, 10 mm edta, 10% np-40, 0.5 mm dtt, 1 mm phenylmethylsulfonyl fluoride, 5 µg/ml leupeptin, and 10 µg/ml aprotinin) for 5 minutes. the lysate was centrifuged at </span><span style=\"background-color: hsl(0, 100.00%, 99.22%); opacity: 0.80\" title=\"-0.003\">13</span><span style=\"opacity: 0.80\">,000 rpm for 20 minutes at 4°c, and protein content of the supernatant was measured. total cell lysates (50 µg/well) were separated by sds-page electrophoresis and the gels transferred onto nitrocellulose membranes (whatman, piscataway, nj). membranes were </span><span style=\"background-color: hsl(0, 100.00%, 90.10%); opacity: 0.83\" title=\"-0.119\">blocked</span><span style=\"opacity: 0.80\"> with 5% non-fat dry milk in phosphate-buffered </span><span style=\"background-color: hsl(0, 100.00%, 99.13%); opacity: 0.80\" title=\"-0.004\">saline</span><span style=\"opacity: 0.80\"> containing tween-20 (pbst) (1x pbs, 0.1% tween-20) for 1 h at room temperature and incubated with the appropriate primary antibody at 4°c overnight. membranes then were washed three times with pbst and probed with appropriate horseradish peroxidase (hrp)-conjugated secondary antibody for 1 h at room temperature. the membranes were again washed three times in pbst and bands were visualized using western blot chemiluminescence reagent (biorad, valencia, ca) on a chemidoc gel documentation system (biorad, valencia, ca). antibodies were obtained from santa cruz biotechnologies and used at the following dilutions (c-cbl, 1∶5000; c-met, 1∶5000; egfr, 1∶5000; ubiquitin, 1∶1000; ha, 1∶5000 and β-</span><span style=\"background-color: hsl(0, 100.00%, 64.46%); opacity: 0.97\" title=\"-0.736\">actin</span><span style=\"opacity: 0.80\">, 1∶10,000).  flow cytometry  cell cycle analysis was carried out by flow cytometry. approximately 2×106 cells were grown in media containing 10% fbs. cells were harvested by trypsin/edta treatment, washed with 1x pbs three times and fixed with ice-cold 70% ethanol for 2 h. cells were washed again with cold pbs and stained with a solution containing 25 µg/ml propidium iodide, 200 µg/ml rnase a, and 0.1% triton x-100 for 30 minutes in the dark. cell cycle analysis was performed using a guava pca-96 flow cytometer (guava technologies, millipore, billerica, ma).  ubiquitin ligase activity  293t cells were maintained in culture in dmem </span><span style=\"background-color: hsl(120, 100.00%, 96.18%); opacity: 0.81\" title=\"0.030\">supplemented</span><span style=\"opacity: 0.80\"> with 10% fbs and 1% penicillin (100 units/ml) and streptomycin (100 µg/ml) were transfected with 0.2 µg egfr-pcdna3 and 2 µg ha-tagged c-cbl constructs as indicated using calcium phosphate according to manufacturer&#x27;s protocol (profection, promega, madison, wi). twenty-four hours post-transfection, cells were starved overnight in dmem </span><span style=\"background-color: hsl(120, 100.00%, 96.18%); opacity: 0.81\" title=\"0.030\">supplemented</span><span style=\"opacity: 0.80\"> with 0.5% fbs, and then treated with or without egf (100 ng/ml) for 15 </span><span style=\"background-color: hsl(120, 100.00%, 99.75%); opacity: 0.80\" title=\"0.001\">min</span><span style=\"opacity: 0.80\">. the cells were collected and washed two times in ice-cold pbs containing 0.2 mm sodium orthovanadate then lysed in ice-cold lysis buffer (10 mm tris hcl, ph 7.5, 150 mm nacl, 5 mm edta, 1% triton x100, 10% glycerol, 2 mm sodium orthovanadate and protease inhibitors). lysates were cleared of debris by centrifugation at 16,000 g for 10 </span><span style=\"background-color: hsl(120, 100.00%, 99.75%); opacity: 0.80\" title=\"0.001\">min</span><span style=\"opacity: 0.80\"> at 4°c. egfr immunoprecipitations were performed on 200 µg of cleared lysate using 250 ng of rabbit-anti-egfr and protein a/g plus sepharose overnight at 4°c. precipitations were washed 5 times in lysis buffer before boiling in laemmli buffer. elutions were immunoblotted with anti-ubiquitin and egfr. twenty micrograms of cleared lysate were immunoblotted for each of the c-cbl constructs using anti-ha.  statistical analysis  mutation rates between different groups were compared using fisher&#x27;s exact test. for continuous variables, </span><span style=\"background-color: hsl(0, 100.00%, 99.25%); opacity: 0.80\" title=\"-0.003\">group</span><span style=\"opacity: 0.80\"> comparisons were performed using analysis of variance (anova) followed by sidak&#x27;s adjustment for multiple comparisons. experiments involving repeated measurements over time were analyzed using repeated measures anova with the greenhouse-geisser adjustment to the </span><span style=\"background-color: hsl(0, 100.00%, 99.59%); opacity: 0.80\" title=\"-0.001\">degrees</span><span style=\"opacity: 0.80\"> of freedom. analyses were conducted using stata (v10.1) software (stata corporation, college station, tx).  go to: results c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">gene</span><span style=\"opacity: 0.80\"> mutations in lung cancer  to investigate the </span><span style=\"background-color: hsl(0, 100.00%, 98.93%); opacity: 0.80\" title=\"-0.005\">role</span><span style=\"opacity: 0.80\"> of c-cbl in lung cancer, we analyzed its genomic dna in tumor and paired </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\"> samples drawn from multiple ethnicities. the lung tumor samples represented caucasians (n = 50), african-americans (n = 29), and taiwanese (n = 40) lung cancer patients. we designed 12 pairs of primers to sequence the coding region of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">gene</span><span style=\"opacity: 0.80\"> that spans exons 2 to 16 (table s2). we identified 8 unique somatic mutations in c-cbl exons among 8 different patients. a variation l620f, a known snp (rs2227988) in exon 11 was also detected. importantly, the eight novel non-synonymous mutations were confirmed by sequencing </span><span style=\"background-color: hsl(120, 100.00%, 98.64%); opacity: 0.80\" title=\"0.007\">both</span><span style=\"opacity: 0.80\"> strands of c-cbl genomic dna obtained from lung tumor samples (table 1). moreover, </span><span style=\"background-color: hsl(0, 100.00%, 99.75%); opacity: 0.80\" title=\"-0.001\">none</span><span style=\"opacity: 0.80\"> of the 8 mutations were detected in the corresponding </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\"> tissue, indicating that these were somatic mutations. four synonymous single nucleotide variations (snvs) were also identified but were </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> used further in </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> study.  table 1 table 1 c-cbl mutation analysis in 119 lung cancer patient tumor </span><span style=\"background-color: hsl(0, 100.00%, 98.11%); opacity: 0.80\" title=\"-0.011\">tissues</span><span style=\"opacity: 0.80\">. three of the 8 novel non-synonymous mutations were located in the tkb (tyrosine kinase </span><span style=\"background-color: hsl(0, 100.00%, 98.95%); opacity: 0.80\" title=\"-0.005\">binding</span><span style=\"opacity: 0.80\">) domain (s80n, h94y, and q249e), one in the ring finger domain (v391i), one in the proline-rich region (72515_72517 del atg), and three in the c-terminal region (w802*, r830k, and a848t) of the c-cbl protein (figure 1a and figure s1). in figure 1b, we show model chromatograms of representative samples.  figure 1 figure 1 c-cbl mutations and loh in non-small cell lung cancer. 11q loh of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">gene</span><span style=\"opacity: 0.80\">  paired lung tumor and </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\"> lung tissue samples from taiwanese patients (n = 37) were investigated for loh. eight (21.6%) showed loh at the c-cbl locus on chromosome 11 while 29 samples (78.4%) revealed </span><span style=\"background-color: hsl(0, 100.00%, 99.04%); opacity: 0.80\" title=\"-0.004\">normal</span><span style=\"opacity: 0.80\"> allelic contribution at the microsatellite markers (figures 1c and d).  c-cbl mutations in different ethnic groups  the c-cbl double mutant s80n/h94y was found in the same patient and the </span><span style=\"background-color: hsl(120, 100.00%, 98.53%); opacity: 0.80\" title=\"0.008\">overall</span><span style=\"opacity: 0.80\"> mutation rate for c-cbl in lung tumors was 6.7% (8/119). the frequency of c-cbl mutation was highest in large cell carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the least was observed in adenocarcinoma (ad) (1.8%; 1 of </span><span style=\"background-color: hsl(120, 100.00%, 99.59%); opacity: 0.80\" title=\"0.001\">53</span><span style=\"opacity: 0.80\"> patients), although these rates were </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> statistically significant (p = 0.292). mutation rates were 6.0% among caucasians (0 of 20 in ad; 0 of 10 in sq; and 3 of 20 in lc), </span><span style=\"background-color: hsl(0, 100.00%, 99.22%); opacity: 0.80\" title=\"-0.003\">13</span><span style=\"opacity: 0.80\">.8% in african-americans (1 of 10 in ad; 1 of 10 in sq; and 2 of 9 in lc), and 2.5% (0 of 23 in ad; 1 of 12 in sq; and 0 of 5 in lc) in the taiwanese population. additionally two taiwanese patients with lung cancer (one squamous and one adenocarcinoma) had the known snp l620f. ethnic differences were </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> statistically significant, </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">however</span><span style=\"opacity: 0.80\"> the power to detect differences was </span><span style=\"background-color: hsl(120, 100.00%, 98.72%); opacity: 0.80\" title=\"0.006\">low</span><span style=\"opacity: 0.80\">.  mutations in met and egfr can be co-associated with c-cbl alterations  </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.006\">since</span><span style=\"opacity: 0.80\"> east asians with lung cancer have a higher frequency of egfr and met mutations in lung tumors [21], [22], we also determined mutations in egfr and met in the same taiwanese cohort samples and compared the results with the observed c-cbl alterations (loh and/or mutations). in the 37 samples </span><span style=\"background-color: hsl(120, 100.00%, 97.40%); opacity: 0.80\" title=\"0.018\">tested</span><span style=\"opacity: 0.80\">, we did </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> find any overlap between c-cbl mutations and c-cbl loh (figure 2). of the three c-cbl mutants (including the known l620f snp, rs2227988), one of the samples had a met mutation (n375s) and the other had an egfr mutation (l858r). among the 8 samples that had a loh at the c-cbl locus 5 had an additional mutation in met (n375s) and 2 had an egfr exon 19 deletion. twenty-six samples had neither c-cbl mutation nor c-cbl loh (3 patients had a c-cbl mutation but </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">no</span><span style=\"opacity: 0.80\"> c-cbl loh). among these 26 samples 9 had a met mutation (8 n375s, 1 l211w), </span><span style=\"background-color: hsl(0, 100.00%, 99.22%); opacity: 0.80\" title=\"-0.003\">13</span><span style=\"opacity: 0.80\"> had an egfr mutation (7 exon 9 deletion, 6 l858r) and 4 had </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">no</span><span style=\"opacity: 0.80\"> other met or egfr mutation. thus the rate of met or egfr mutations among patients with loh at the c-cbl locus (7 of 8) was similar to that seen in patients without c-cbl mutation or loh (22 of 26 patients) (p = 0.99). these 4 patients with </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">no</span><span style=\"opacity: 0.80\"> identifiable mutation in c-cbl, met or egfr represented 10.8% of the 37 patients analyzed in the taiwanese patient cohort. conversely, 89.2% taiwanese lung cancer patients have an identifiable mutation in either c-cbl, met or egfr or a combination of the three genes (figure 2). additionally, we determined p53 and kras mutations in these taiwanese cohorts. two p53 and 1 kras mutation were detected. the single kras mutation overlapped with one p53 mutation. </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> patient also had the egfr exon 19 deletion but had </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">no</span><span style=\"opacity: 0.80\"> c-cbl mutation. the other p53 mutation sample had a c-cbl loh with concurrent met n375s mutation. thus, in the taiwanese samples analyzed, p53/kras mutations and c-cbl mutations were mutually exclusive (</span><span style=\"background-color: hsl(0, 100.00%, 98.84%); opacity: 0.80\" title=\"-0.006\">data</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> shown).  figure 2 figure 2 c-cbl mutations and relationship to met and egfr mutations in lung cancer. cellular functions of c-cbl alterations in the context of lung tumorigenesis  a. e3 activity is intact in the mutant c-cbl proteins to investigate whether the different c-cbl mutations affect the e3 activity, egfr was chosen as a model substrate for c-cbl e3 function. all of the c-cbl mutants </span><span style=\"background-color: hsl(120, 100.00%, 97.40%); opacity: 0.80\" title=\"0.018\">tested</span><span style=\"opacity: 0.80\"> enhanced ubiquitination of the activated egfr similar to the wild-type c-cbl protein. </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> result demonstrates that the catalytic activity of the c-cbl mutants is </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> impaired when egfr was the substrate. (figure 3a).  figure 3 figure 3 ubiquitination, viability, expression and cell cycle analysis of various c-cbl mutants. b. effect on lung cancer cell viability the effect of a representative c-cbl mutant from each of the three ethnic backgrounds on lung cancer cell viability in cell </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">lines</span><span style=\"opacity: 0.80\"> was determined. s80n/h94y double mutation, q249e, and w802* were identified in lung tumor samples obtained from a caucasian, a taiwanese and an african-american, respectively. as described in methods, the c-cbl wild-type (wt) and the above three mutants were expressed after cloning them into paltermax vector in a549 cells. these cells express relatively </span><span style=\"background-color: hsl(120, 100.00%, 98.72%); opacity: 0.80\" title=\"0.006\">low</span><span style=\"opacity: 0.80\"> basal levels of endogenous c-cbl (</span><span style=\"background-color: hsl(0, 100.00%, 98.84%); opacity: 0.80\" title=\"-0.006\">data</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> shown). transfection efficiency was comparable between different groups and the number of cells transfected with c-cbl wild-type construct was about 70% compared to control cells that were transfected with the empty vector. cells transfected with s80n/h94y, q249e and w802* c-cbl mutant constructs resulted in increased number of viable cells that was 132.3%, </span><span style=\"background-color: hsl(120, 100.00%, 98.94%); opacity: 0.80\" title=\"0.005\">120</span><span style=\"opacity: 0.80\">.8% and 147.9% higher respectively, relative to the empty vector control transfected cells and significantly different from the wild-type construct (p = 0.022, p = 0.049, and p = 0.008, respectively) (figure 3b). relative levels of c-cbl protein in whole cell lysates prepared from samples obtained from a parallel experiment was determined. the c-cbl protein levels in samples representing untransfected and empty vector transfected cells were comparable and those representing the c-cbl wt and the three c-cbl mutants were comparable (figure 3c).  c. effect on cell cycle to investigate if the increases in cell viability in different c-cbl mutants are due to increased cellular proliferation, a cell cycle analysis was performed. a549 cells were transfected with the c-cbl wt or the three different mutants: s80n/h94y, q249e and w802*. the empty vector transfectant was used as a control. forty-eight hours after transfection cell cycle analysis was performed as described in materials and methods. there was </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">no</span><span style=\"opacity: 0.80\"> significant change in the subg1, g1 or the s phase of the cell cycle among the different mutants compared to the wt construct (p = 0.64, p = 0.40, and p = 0.28, respectively). the g2/m phase of the cell cycle showed an increase in cell numbers for the three mutants, s80n/h94y, q249e and w802*, when compared to the wt but again the difference was </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> statistically significant (p = 0.25) (figure 3d).  d. effect on cell motility to investigate the effect of the expression of the above three c-cbl mutants on cell migration, we carried out wound healing assay as described in materials and methods. the closing of the scratch or the wound was monitored at 0, 12, 24, 36, and 48 h. (figure 4a). in all the samples, that represented cells transfected with mutants, the wound gap was much smaller than that seen in the sample that represented cells transfected with c-cbl wt (p&lt;0.001). we also determined the rate of wound closure for all the five groups. at 48 h, wild-type c-cbl transfectants showed 61.1% open wound while the s80n/h94y, q249e and w802* mutants showed 18.7%, 23.9% and 34.3% open wound respectively (p&lt;0.001) (figure 4b).  figure 4 figure 4 c-cbl mutations affect wound healing in a549 cells. e. c-cbl knockdown increases cell viability it is hypothesized that the loh seen in our samples could lead to decreased expression of c-cbl. thus we </span><span style=\"background-color: hsl(120, 100.00%, 97.40%); opacity: 0.80\" title=\"0.018\">tested</span><span style=\"opacity: 0.80\"> the effect of c-cbl knockdown in lung cancer cells. compared to a549, h358 lung cancer cells express relatively high levels of endogenous c-cbl (</span><span style=\"background-color: hsl(0, 100.00%, 98.84%); opacity: 0.80\" title=\"-0.006\">data</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> shown). c-cbl expression was knocked down using lentiviral construct that expressed c-cbl specific shrna and compared the results with those that were transduced with scrambled shrna and the results are shown in figure 5. we identified several clones that revealed varying </span><span style=\"background-color: hsl(0, 100.00%, 99.59%); opacity: 0.80\" title=\"-0.001\">degrees</span><span style=\"opacity: 0.80\"> of c-cbl knockdown showing different sets of c-cbl lentiviral shrna knockdown efficiency (figure 5a). of all the clones </span><span style=\"background-color: hsl(120, 100.00%, 97.40%); opacity: 0.80\" title=\"0.018\">tested</span><span style=\"opacity: 0.80\">, clone 27 was chosen for further experiments. equal amount of cells were seeded in a 6-well plate and the cell proliferation was measured at various times and the results are depicted in figure 5b. as expected, number of cells increased in a time </span><span style=\"background-color: hsl(120, 100.00%, 98.37%); opacity: 0.80\" title=\"0.009\">dependent</span><span style=\"opacity: 0.80\"> fashion from 100 to 190% relative to scrambled shrna as control in a span of 48 h (p = 0.0002) (figure 5b). the cell cycle phases in h358 cells that were knocked down with c-cbl shrna were looked at and compared with the scrambled shrna. there were </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">no</span><span style=\"opacity: 0.80\"> discernable differences between these two constructs in the different phases of the cell cycle (</span><span style=\"background-color: hsl(0, 100.00%, 98.84%); opacity: 0.80\" title=\"-0.006\">data</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> shown).  figure 5 figure 5 knockdown of c-cbl using an shrna increases cell proliferation. go to: discussion our results demonstrate that c-cbl is somatically </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">mutated</span><span style=\"opacity: 0.80\"> (or </span><span style=\"background-color: hsl(0, 100.00%, 97.78%); opacity: 0.80\" title=\"-0.014\">has</span><span style=\"opacity: 0.80\"> loh) in lung cancers, and can significantly contribute to enhanced cell viability and motility. there was also a high prevalence of loh with respect to c-cbl in lung tumors that harbored met or egfr mutation.  in the present study, we have demonstrated the occurrence of c-cbl mutations in lung cancer patients, especially with different ancestral variations. mutations in c-cbl have </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">been</span><span style=\"opacity: 0.80\"> recently reported in juvenile myelomonocytic leukemia and myeloid malignancies. in the aml study, the mutation r420q located in the junction of the ring finger and the linker region inhibited fms-like tyrosine kinase 3 (flt3) internalization and ubiquitination [20], thus contributing to the gain-in-function for the rtk. in addition, mutations such as h398y, c384r, and l380p mapped to the ring finger domain and the linker region of c-cbl that is required for its e3 activity [23], [24], [25], [26], [27]. additionally, homozygous mutations in the ring finger domain of the c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">gene</span><span style=\"opacity: 0.80\"> were described as a result of acquired uniparental disomy (upd) [26]. it is important to note that our results indicate loh at 11q23 locus and these are mutually exclusive from missense mutations of c-cbl. the somatic mutations were all heterozygous. the mutations in aml led to abrogation of the e3 activity leading to prolonged rtk activation. in addition mutants located on the linker region surrounding the ring finger domain exhibited enhanced akt signaling in response to cytokine stimulation [26]. in addition, it was shown in nh3t3 cells, that neither mutations in the ring finger nor the linker region causes transformation, </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">however</span><span style=\"opacity: 0.80\"> while certain mutations perturbs the ubiquitination, others affect receptor recycling and prolong kinase activity [28].  we report </span><span style=\"background-color: hsl(0, 100.00%, 96.92%); opacity: 0.81\" title=\"-0.022\">here</span><span style=\"opacity: 0.80\"> c-cbl mutations that mapped </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> only to the ring finger domain, but also to the tkb domain, proline-rich domain and the c-terminal region, but </span><span style=\"background-color: hsl(0, 100.00%, 99.75%); opacity: 0.80\" title=\"-0.001\">none</span><span style=\"opacity: 0.80\"> mapped to the linker region as reported in the aml studies described above [23], [24], [25], [26], [29]. in addition, 8 mutants that we detected were found in different ethnic backgrounds. for </span><span style=\"background-color: hsl(0, 100.00%, 98.44%); opacity: 0.80\" title=\"-0.008\">example</span><span style=\"opacity: 0.80\">, s80n/h94y, q249e, w802* were detected in caucasians, taiwanese and african-americans, respectively. the results point out </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> only the difference between lung cancer and other cancers, but also genetic polymorphism among different races in the same cancer. interestingly, there is a large disparity between african-american and other ethnic populations with lung cancer [30]. we have </span><span style=\"background-color: hsl(120, 100.00%, 99.53%); opacity: 0.80\" title=\"0.002\">previously</span><span style=\"opacity: 0.80\"> shown that there was a </span><span style=\"background-color: hsl(120, 100.00%, 98.72%); opacity: 0.80\" title=\"0.006\">low</span><span style=\"opacity: 0.80\"> frequency of egfr and met mutation in african-americans as compared with taiwanese and caucasians [31]. in </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> study, the number of african-american samples analyzed was relatively fewer and we found 3 mutations that are unique to </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> ethnicity. it would now behoove us to further study the genetic alterations that can occur and determine the targeted therapeutics for african-americans. our results provide evidence of the importance of c-cbl in tumorigenesis and potential signaling. our prediction, </span><span style=\"background-color: hsl(0, 100.00%, 97.21%); opacity: 0.80\" title=\"-0.019\">based</span><span style=\"opacity: 0.80\"> on the aml </span><span style=\"background-color: hsl(0, 100.00%, 98.84%); opacity: 0.80\" title=\"-0.006\">data</span><span style=\"opacity: 0.80\">, would be that the v391i ring finger domain mutation would affect the e3 activity. also, it will be important to determine the </span><span style=\"background-color: hsl(0, 100.00%, 98.95%); opacity: 0.80\" title=\"-0.005\">binding</span><span style=\"opacity: 0.80\"> partners for c-cbl in the tkb domain and proline rich domain mutations. it </span><span style=\"background-color: hsl(0, 100.00%, 97.78%); opacity: 0.80\" title=\"-0.014\">has</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.53%); opacity: 0.80\" title=\"0.002\">previously</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">been</span><span style=\"opacity: 0.80\"> shown that the tkb domain can bind to growth factor receptors and it will be important to determine the cross-</span><span style=\"background-color: hsl(0, 100.00%, 98.95%); opacity: 0.80\" title=\"-0.005\">binding</span><span style=\"opacity: 0.80\"> of these mutants to met and or egfr. it would also be important in the future to look at fluorescence in situ hybridization/copy number changes in c-cbl in lung cancer.  c-cbl plays an important </span><span style=\"background-color: hsl(0, 100.00%, 98.93%); opacity: 0.80\" title=\"-0.005\">role</span><span style=\"opacity: 0.80\"> in down regulating rtk-mediated signaling through k63 poly-ubiquitination and subsequent downregulation of rtks followed by lysosomal degredation [3]. mono-ubiquitination or ubiquitinated with k63-linked chains of substrates by c-cbl may lead to enhancement of biological and biochemical functions (reviewed in hermann et al, 2007 [32]). the mutations that we analyzed in our studies all point out to the fact that the e3 activity of c-cbl on egfr is intact; the egfr levels in the various mutants remain same (figure s2). multiple kinases, </span><span style=\"background-color: hsl(120, 100.00%, 98.64%); opacity: 0.80\" title=\"0.007\">both</span><span style=\"opacity: 0.80\"> rtks and non-rtks could be acted </span><span style=\"background-color: hsl(0, 100.00%, 95.75%); opacity: 0.81\" title=\"-0.035\">upon</span><span style=\"opacity: 0.80\"> by c-cbl, including erbs, pdgfr, fms, met, c-kit, vegfr, flt-1, ron, fgfr, ir, as well as syk, fyn, lck, fgr, lyn and c-abl [3]. in lung cancers the relevant substrates of c-cbl in terms of degradation or signal transduction are yet to be identified.  the observation that c-cbl somatic mutations, especially s80n/h94y, q249e and w802* showed increased cell viability and cell motility that is in agreement with the physiological </span><span style=\"background-color: hsl(0, 100.00%, 98.93%); opacity: 0.80\" title=\"-0.005\">role</span><span style=\"opacity: 0.80\"> for d-cbl in the regulation of </span><span style=\"background-color: hsl(120, 100.00%, 99.18%); opacity: 0.80\" title=\"0.003\">apoptosis</span><span style=\"opacity: 0.80\"> and differentiation identified in drosophila is very significant [</span><span style=\"background-color: hsl(0, 100.00%, 99.49%); opacity: 0.80\" title=\"-0.002\">33</span><span style=\"opacity: 0.80\">]. it </span><span style=\"background-color: hsl(0, 100.00%, 97.78%); opacity: 0.80\" title=\"-0.014\">has</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">been</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.53%); opacity: 0.80\" title=\"0.002\">previously</span><span style=\"opacity: 0.80\"> shown that activating c-cbl mutation downregulates egfr signaling and decreases cellular proliferation and migration in breast cancer cell </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">lines</span><span style=\"opacity: 0.80\"> [34]. although the </span><span style=\"background-color: hsl(0, 100.00%, 98.93%); opacity: 0.80\" title=\"-0.005\">role</span><span style=\"opacity: 0.80\"> of c-cbl in the negative regulation of rtks is well substantiated, thereby suggesting that it is a natural tumor suppressor, studies in cancer cells have revealed </span><span style=\"background-color: hsl(120, 100.00%, 98.64%); opacity: 0.80\" title=\"0.007\">both</span><span style=\"opacity: 0.80\"> tumor suppressor and tumor promoting activities depending on the type of c-cbl mutation and the number of alleles at the c-cbl locus [24]. in agreement with the above, the three c-cbl mutants described </span><span style=\"background-color: hsl(0, 100.00%, 96.92%); opacity: 0.81\" title=\"-0.022\">here</span><span style=\"opacity: 0.80\"> appear to have tumor growth and metastasis promoting properties. although these mutants are outside of the ring finger or the linker region of c-cbl, their downstream effects are significant so as to cause increased proliferation and migration, but the substrate affected by these mutations are </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> known yet. </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> raises the possibility that some of the cellular functions of c-cbl are independent of its ubiquitin-ligase activity, an area that we are currently investigating. the oncogenic nature of rtks, addiction of cancers to growth signals and given the clustering of c-cbl, egfr and met mutations, it is possible that the transforming effect of c-cbl mutations is most likely a combinatorial effect of the three. we also show that loh for c-cbl was found in a significant number of samples that harbored met or egfr mutations. the fact that about 7% of lung tumor samples are likely to have c-cbl mutations and an additional 22% are likely to harbor c-cbl-related loh, makes c-cbl a </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.001\">highly</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">mutated</span><span style=\"opacity: 0.80\"> molecule in lung cancer. </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.006\">since</span><span style=\"opacity: 0.80\"> loh alone is </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> enough to cause a transforming event [35], [36], [37], associated mutation in the met or egfr locus or yet another rtk discussed above may play a </span><span style=\"background-color: hsl(0, 100.00%, 98.93%); opacity: 0.80\" title=\"-0.005\">role</span><span style=\"opacity: 0.80\"> in carcinogenesis. we predict that </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> loh in c-cbl results in haploinsufficiency that downplays rtk ubiquitination leading to hyperactivity of the rtks. </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">however</span><span style=\"opacity: 0.80\">, whether </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> is </span><span style=\"background-color: hsl(0, 100.00%, 99.46%); opacity: 0.80\" title=\"-0.002\">sufficient</span><span style=\"opacity: 0.80\"> cause for tumorigenesis remains to be determined. consistent with our hypothesis is the fact that c-cbl-/- mice have increased kinase activity in lymphocytes, but is </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.46%); opacity: 0.80\" title=\"-0.002\">sufficient</span><span style=\"opacity: 0.80\"> for tumor formation [35], [36], [37]. c-cbl loh could also lead to increased expression of c-cbl from the other allele to compensate for a loss of an allele. alternately, there could be some form of synergy working with reduced c-cbl levels and </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">mutated</span><span style=\"opacity: 0.80\"> receptors that exacerbate the phenotype of each alone.  previous studies from our lab and others have shown that east asians with lung cancers have relatively high frequencies of gain-of-function of mutations in rtks such as egfr and met [31]. in a cohort of japanese patients an activating met mutation was identified in the </span><span style=\"background-color: hsl(0, 100.00%, 96.66%); opacity: 0.81\" title=\"-0.025\">splice</span><span style=\"opacity: 0.80\"> region that deletes the juxtamembrane domain that is involved e3 activity of c-cbl [38]. </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> study also found that activation of met is mutually exclusive of egfr, kras and her2 </span><span style=\"background-color: hsl(120, 100.00%, 99.51%); opacity: 0.80\" title=\"0.002\">gene</span><span style=\"opacity: 0.80\"> mutations [38]. we failed to detect such mutations in significant numbers in lung tumor samples obtained from african-americans (n = 29) and caucasian (n = 50) patients. one met mutation was identified in each of the groups whereas 1 and 3 egfr mutations were identified in the african-american and caucasian cohorts respectively. egfr mutations have earlier </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">been</span><span style=\"opacity: 0.80\"> identified as one of the key mutations affecting lung adenocarcinoma patients in a comprehensive study of 188 patients [39]. our study encompasses different histologies of nsclc. </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">however</span><span style=\"opacity: 0.80\">, the published series did </span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">not</span><span style=\"opacity: 0.80\"> find any mutations in c-cbl or met unlike our study that encompassed different </span><span style=\"background-color: hsl(0, 100.00%, 89.72%); opacity: 0.83\" title=\"-0.125\">subtypes</span><span style=\"opacity: 0.80\"> of nsclc. it is important to note, we have recently shown that met mutations in lung cancer are in </span><span style=\"background-color: hsl(120, 100.00%, 98.10%); opacity: 0.80\" title=\"0.011\">majority</span><span style=\"opacity: 0.80\"> germline [31]. we have reported earlier c-cbl mutations in a small cohort of taiwanese lung cancer samples [40]. in our </span><span style=\"background-color: hsl(0, 100.00%, 99.24%); opacity: 0.80\" title=\"-0.003\">efforts</span><span style=\"opacity: 0.80\"> to understand the ethnic differences in the lung oncogenome, we also looked at pax transcription factors such as pax5 and pax8 that are </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.001\">highly</span><span style=\"opacity: 0.80\"> expressed in lung cancers; </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">however</span><span style=\"opacity: 0.80\"> there was </span><span style=\"background-color: hsl(0, 100.00%, 99.94%); opacity: 0.80\" title=\"-0.000\">no</span><span style=\"opacity: 0.80\"> preferential expression or mutations of the above genes in lung tumor samples of african-americans. in </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> study, we show relatively high frequency of c-cbl mutations in lung cancers, especially in the large cell type among caucasians and particularly among african-americans. we therefore propose c-cbl as an efficacious </span><span style=\"background-color: hsl(0, 100.00%, 98.15%); opacity: 0.80\" title=\"-0.011\">target</span><span style=\"opacity: 0.80\"> for lung cancers in african-americans that needs to be further substantiated. </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> is all the more important because the prognosis for african-americans with lung cancer, especially for men is much poorer compared to their caucasian counterparts [41].  in conclusion, the results presented in </span><span style=\"background-color: hsl(120, 100.00%, 99.85%); opacity: 0.80\" title=\"0.000\">this</span><span style=\"opacity: 0.80\"> study demonstrate that c-cbl is frequently </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">mutated</span><span style=\"opacity: 0.80\"> or even lost in lung cancers. our results support a </span><span style=\"background-color: hsl(0, 100.00%, 98.93%); opacity: 0.80\" title=\"-0.005\">role</span><span style=\"opacity: 0.80\"> for c-cbl mutants that are independent of its ubiquitination activity. given the relatively high mutation rates in c-cbl as well as rtks such as met and egfr, it is likely that their combined effect could be synergistic in promoting tumorigenesis.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=6\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.021</b>, score <b>-1.943</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.25%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.031\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__extraction\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.36%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.025\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__defects\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.41%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.022\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2a\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.43%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bar\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.53%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__43\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__induces\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__exclusively\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2b\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__μg\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__germany\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__harboring\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__gift\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__appeared\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__observations\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__establish\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__just\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__encodes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__base\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__enhances\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bio\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__jp\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__skp2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__hexagons\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mesh\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__capacity\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ionizing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__enriched\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mabs\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__tgg\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__kdr\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__denaturation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__quartz\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__believe\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fiber\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__superimposition\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bard1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__termini\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stem\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__occasionally\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__457\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__costs\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__superdex\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__warranted\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__specimen\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__hsp70\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__homologs\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__55\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pt\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__1jm7\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__eleven\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__arabidopsis\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__dntp\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__alkaline\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__han\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__her\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__intensity\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__indirect\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__physiologic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__processes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__presence\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__apparently\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__290\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__folded\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__braf\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__42\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__57\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__measurement\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.63%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__thermal\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__steric\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__discrete\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.53%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__thawed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.53%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__highlights\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__use\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__amino\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.47%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.019\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__decrease\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.46%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__networks\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.31%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.028\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__006\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.27%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.030\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__unclassified\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.26%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.031\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__length\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.11%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.040\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__structural\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.05%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.044\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__exposed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.054\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__period\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.132\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__brca1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 96.20%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.317\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 90.63%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -1.151\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">w802*</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">cbl</span><br/><b>Text:</b> <span style=\"opacity: 0.80\"> abstract background  non-small cell lung cancer (nsclc) is a heterogeneous </span><span style=\"background-color: hsl(120, 100.00%, 97.50%); opacity: 0.80\" title=\"0.017\">group</span><span style=\"opacity: 0.80\"> of disorders with a number of genetic and proteomic alterations. c-cbl is an e3 ubiquitin ligase and adaptor molecule important in normal homeostasis and cancer. we determined the genetic variations of c-cbl, relationship to receptor tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.72%); opacity: 0.80\" title=\"0.001\">kinases</span><span style=\"opacity: 0.80\"> (egfr and met), and functionality in nsclc.  methods and </span><span style=\"background-color: hsl(120, 100.00%, 94.09%); opacity: 0.81\" title=\"0.057\">findings</span><span style=\"opacity: 0.80\">  </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> archival formalin-fixed paraffin embedded (ffpe) extracted genomic </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\">, we show that c-cbl mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.002\">occur</span><span style=\"opacity: 0.80\"> in somatic fashion for lung cancers. c-cbl mutations were not mutually exclusive of met or egfr mutations; however they were independent of p53 and kras mutations. in normal/tumor pairwise analysis, there was </span><span style=\"background-color: hsl(0, 100.00%, 99.44%); opacity: 0.80\" title=\"-0.002\">significant</span><span style=\"opacity: 0.80\"> loss of heterozygosity (loh) for the c-cbl locus (22%, n = 8/37) and none of these samples revealed any mutation in the remaining copy of c-cbl. the c-cbl loh also positively correlated with egfr and met mutations observed in the same samples. </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> select c-cbl somatic mutations such as s80n/h94y, q249e and w802* (obtained from caucasian, taiwanese and african-american samples, respectively) </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> in nsclc cell lines, there was </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">increased</span><span style=\"opacity: 0.80\"> cell viability and cell motility.  conclusions  taking the overall mutation rate of c-cbl to be a combination as somatic missense mutation and loh, it is clear that c-cbl is highly mutated in lung cancers and </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.006\">may</span><span style=\"opacity: 0.80\"> play an essential role in lung tumorigenesis and metastasis.  go to: </span><span style=\"background-color: hsl(0, 100.00%, 95.05%); opacity: 0.81\" title=\"-0.044\">introduction</span><span style=\"opacity: 0.80\"> in the us alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which more than 145,000 of them succumb to the disease [1]. this number is roughly equivalent to the combined mortality rates of cancers of the </span><span style=\"background-color: hsl(0, 100.00%, 97.67%); opacity: 0.80\" title=\"-0.015\">breast</span><span style=\"opacity: 0.80\">, prostate, colon, liver, kidney and melanoma [1]. in addition the prognosis is usually poor and the five-year survival rate is less than 15%. there are also </span><span style=\"background-color: hsl(0, 100.00%, 99.44%); opacity: 0.80\" title=\"-0.002\">significant</span><span style=\"opacity: 0.80\"> ethnic differences for lung cancer, and the outcome is worse for blacks compared to whites. gender differences are also striking with women having significantly better prognosis as compared to men. there are a number of genetic alterations that can </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.002\">occur</span><span style=\"opacity: 0.80\"> in lung cancer. as an example, in nsclc, mutations in kras, p53, egfr and met have been identified. many of these pathways, especially receptor tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.72%); opacity: 0.80\" title=\"0.001\">kinases</span><span style=\"opacity: 0.80\"> (rtks) are controlled by c-cbl.  cbl (casitas b-lineage lymphoma) is a mammalian gene located on human </span><span style=\"background-color: hsl(0, 100.00%, 98.11%); opacity: 0.80\" title=\"-0.011\">chromosome</span><span style=\"opacity: 0.80\"> 11q23.3 [2] and is involved in cell signaling and protein ubiquitination [3]. cbl proteins belong to the ring finger class of ubiquitin ligases (e3) and there are three homologues c-cbl, cbl-b, cbl-3 [4]. the c-cbl and cbl-b genes are ubiquitously expressed with the highest levels in hematopoietic tissues [5]. c-cbl consists of four regions encoding for functionally distinct protein domains: the n-</span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">terminal</span><span style=\"opacity: 0.80\"> tyrosine kinase binding (tkb) domain, the linker region, the catalytic ring finger domain, the proline-rich region and the c-</span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">terminal</span><span style=\"opacity: 0.80\"> ubiquitin-associated (uba) domain that also overlaps with a leucine-zipper (lz) domain [3]. both tkb and ring finger domains are essential for ligand-induced ubiquitination of rtks [6], [7], [8], [9]. the ring finger domain is required for the recruitment of e2 ubiquitin-conjugating enzymes. the tkb domain includes four-helix bundle (4h), a calcium-biding ef hand, and a modified sh2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. in addition, the proline-rich region of c-cbl can associate with the sh3 domain of grb2, which can indirectly recruit c-cbl to rtks via the grb2 adaptor protein [7], [13], [14].  c-cbl also binds to egfr and acts as the e3 that </span><span style=\"background-color: hsl(0, 100.00%, 96.45%); opacity: 0.81\" title=\"-0.027\">targets</span><span style=\"opacity: 0.80\"> egfr for ubiquitination and degradation. furthermore, cbl desensitizes egf signaling and opposes cellular proliferation induced by egf [15]. egf activation also appears to activate the tyrosine kinase src, which phosphorylates c-cbl and in turn activates the ubiquitination and degradation of egfr [16], [17], [18]. a recent study shows that defective endocytosis of egfr is characterized by a deletion mutant and the point mutation l858r, whereby its association with c-cbl and subsequent ubiquitination are impaired [19]. recently, the first human c-cbl mutations were reported in acute myeloid leukemia (aml) patients [</span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">20</span><span style=\"opacity: 0.80\">]. the mutation r420q inhibits fms-</span><span style=\"background-color: hsl(120, 100.00%, 99.63%); opacity: 0.80\" title=\"0.001\">like</span><span style=\"opacity: 0.80\"> tyrosine kinase 3 (flt3) internalization and ubiquitination [</span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">20</span><span style=\"opacity: 0.80\">].  not only can e3 activity be important in oncogenesis, c-cbl has a dual but separate function as a signal transduction molecule. we have previously shown that c-cbl is important in binding crkl and bcr/abl in hematopoietic </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\">. also, it can bind and modulate functions of cytoskeleton by binding to proteins </span><span style=\"background-color: hsl(120, 100.00%, 99.63%); opacity: 0.80\" title=\"0.001\">like</span><span style=\"opacity: 0.80\"> talin and paxillin. the tkb domain is important in binding to a number of molecules, and they then function in signal transduction.  given the critical role of cbl in normal homeostasis and cancer, we hypothesized that it might be mutated in lung cancers. in this study, we report novel c-cbl somatic mutations s80n/h94y, q249e and w802* in caucasian, taiwanese and african-american lung cancer patients, respectively. expressing these mutations in nsclc cell lines lead to </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">increased</span><span style=\"opacity: 0.80\"> proliferation and cell motility. we show that c-cbl mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.002\">occur</span><span style=\"opacity: 0.80\"> with or without met or egfr mutations but are mutually exclusive of a loh at the c-cbl locus. </span><span style=\"background-color: hsl(0, 100.00%, 94.83%); opacity: 0.81\" title=\"-0.047\">additionally</span><span style=\"opacity: 0.80\">, c-cbl loh is associated with either met or egfr mutations. we thus hypothesize that c-cbl mutations might contribute to the </span><span style=\"background-color: hsl(120, 100.00%, 97.71%); opacity: 0.80\" title=\"0.015\">oncogenic</span><span style=\"opacity: 0.80\"> potential of met and egfr in lung cancer.  go to: methods ethics statement  written consent on all research on human subjects has been obtained from the institutional review board, university of chicago and covers all research performed in the laboratory. the following is their contact information: institutional review board, the university of chicago, mcgiffert hall, 5751 s. woodlawn ave., 2nd floor, chicago, il 60637. written informed consents were received from all patients whose tissue samples were used for this study.  tissue samples  lung cancer tissue and paired adjacent normal lung tissues were obtained from 50 caucasian, 29 african-americans and 40 taiwanese nsclc patients who were recruited at the university of chicago hospital (chicago, usa) (caucasian and african-american patients) and taipei veterans general hospital of taiwan (taiwanese patients) after obtaining appropriate institutional review board permission and informed consent from the patients. out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 years. in terms of tumor types, 53 were adenocarcinoma, 32 were squamous cell carcinoma and 34 were large cell carcinoma. 49 were stage i, 14 were stage ii, 34 were stage iii, and 13 were stage iv (table s1).  cell culture  human non-small cell lung carcinoma </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> a549 and h358 were maintained in dmem and rpmi-1640, respectively. human </span><span style=\"background-color: hsl(0, 100.00%, 96.44%); opacity: 0.81\" title=\"-0.027\">embryonic</span><span style=\"opacity: 0.80\"> kidney 293t </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were cultured in dmem. media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (invitrogen, carlsbad, ca). </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were cultured at 37°c in a humidified incubator containing 5% co2.  c-cbl gene mutational analysis  exons 2 to 16 of c-cbl gene were individually amplified by polymerase chain reaction (pcr). primers are listed in table s2. pcr conditions were 1 cycle of 95°c for 5 minutes; 35 cycles of 94°c for 30 seconds, 58°c for 30 seconds and 72°c for 2 minutes; and one cycle of 72°c for 10 minutes. pcr products were treated with exosap-it (usb corporation, cleveland, oh) and sequenced by big-dye terminator chemistry (applied biosystems, foster city, ca). sequencing was performed on the forward coding strand with confirmation of c-cbl alterations performed by sequencing the reverse strand as well. chromatograms were analyzed for mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> mutation surveyor v2.61 (softgenetics, state college, pa).  plasmid constructs and site-directed mutagenesis  the wild-type c-cbl cdna insert was subcloned into the paltermax expression vector </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> xhoi and sali restriction enzyme sites (promega, madison, wi). </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> this parental plasmid paltermax-c-cbl, the tkb domain double mutation (s80n/h94y), the point mutation (q249e), and the c-</span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">terminal</span><span style=\"opacity: 0.80\"> point mutation w802* of c-cbl were created </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> the following primers: 5′-gctggcgctaaagaataacccaccttatatcttagac-3′ and 5′-ctaccagatacctaccagtatctccgtactatcttgtc-3′ for the double mutation s80n/h94y; 5′-ctttacccgactctttgagccctggtcctctttgc-3′ for q249e, and 5′-cagctcctcctttggctgattgtctctggatggtgatc-3′ for w802* along with their complementary primers </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> the quickchange site-directed mutagenesis xl kit (stratagene, la jolla, ca) according to the manufacturer&#x27;s instructions. the constructs were confirmed for the point mutations by standard </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\"> sequencing of both strands.  loss of heterozygosity (loh) analysis  five microsatellites on </span><span style=\"background-color: hsl(0, 100.00%, 98.11%); opacity: 0.80\" title=\"-0.011\">chromosome</span><span style=\"opacity: 0.80\"> 11 (3 on 11q at or within 200 kb up or </span><span style=\"background-color: hsl(0, 100.00%, 98.30%); opacity: 0.80\" title=\"-0.010\">downstream</span><span style=\"opacity: 0.80\"> of the c-cbl gene and 2 </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">control</span><span style=\"opacity: 0.80\"> markers on 11p) were selected for analysis (table s3). established microsatellite markers and respective primer sequences were selected from the geneloc database (http://genecards.weizmann.ac.il/geneloc/index.shtml, weizmann institute of science, rehovot, israel). primers were custom designed and each forward primer was fluorescently labeled at the 5′ end with fam, pet, ned, or vic (applied biosystems). primer annealing temperatures and duplex scores were evaluated with nist primer tools (http://yellow.nist.gov:8444/dnaanalysis/primertoolspage.do; national institute of standards and technology, gaithersburg, md). primers were verified by performing pcr with </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">control</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\"> (isolated from tk6 </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\">) and resolving the products on agarose gels. bands were visualized with an uv transilluminator. genomic </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\"> was extracted from tumor samples and paired normal lung tissue. primers were grouped into multiplex combinations shown in table s4. marker d11s929 served as an internal </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">control</span><span style=\"opacity: 0.80\"> to check for consistency in pcrs and of peaks from capillary electrophoresis. multiplex pcrs were carried out in a volume of 10 µl that contained 1 µl genomic </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\"> (</span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">20</span><span style=\"opacity: 0.80\">–50 ng), 0.5 µm of each primer (1.0 µm total for each primer pair), 400 µm dntps, 1x pcr buffer containing mgcl2, and 0.2 u taq </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\"> polymerase. pcr was performed on the abi geneamp 9700 pcr system </span><span style=\"background-color: hsl(0, 100.00%, 96.43%); opacity: 0.81\" title=\"-0.028\">under</span><span style=\"opacity: 0.80\"> the following conditions: 5 min at 94°c; 30 cycles of 30 sec at 94°c, 1 min at 60°c, 1 min at 72°c; and 5 min at 72°c. the pcr products were separated by capillary electrophoresis on an abi 3130xl </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\"> analyzer. chromatograms were analyzed with peak scanner 1.0 and genemapper 3.7 software (applied biosystems) for allelic alterations. the area of the peaks </span><span style=\"background-color: hsl(0, 100.00%, 93.96%); opacity: 0.81\" title=\"-0.058\">produced</span><span style=\"opacity: 0.80\"> by the </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\"> pcr products was quantified for each allele. the ratio of the allelic areas was calculated for each tumor and paired normal </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\"> sample. when the qloh (allelic ratio for the tumor peaks divided by the allelic ratio of paired normal sample) was ≤0.5 or ≥2.0 for c-cbl and at least one other 11q marker in at least two separate experiments, the sample was considered as having an allelic imbalance and interpreted as loh. samples were evaluated in at least two separate experiments and samples showing prospective loh at c-cbl repeated a third time which included a new </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">control</span><span style=\"opacity: 0.80\"> marker at the bax locus (data not shown) on </span><span style=\"background-color: hsl(0, 100.00%, 98.11%); opacity: 0.80\" title=\"-0.011\">chromosome</span><span style=\"opacity: 0.80\"> 19 to verify integrity of sample </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\">.  transfection of c-cbl constructs  the a549 cell line was </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> the fugene hd (roche, nutley, nj) reagent according to the manufacturer&#x27;s instructions. eight µg of plasmid </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\">, containing either no insert (empty vector), wild-type c-cbl, s80n/h94y c-cbl, q249e c-cbl or w802* cbl was used for transfection in a 6-well culture plate. </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were harvested 48 h after transfection and analyzed for expression.  c-cbl knockdown  c-cbl knockdown was performed </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> lentiviral transduction </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> mission lentiviral transduction particles (sigma-</span><span style=\"background-color: hsl(0, 100.00%, 87.11%); opacity: 0.84\" title=\"-0.173\">aldrich</span><span style=\"opacity: 0.80\">, st. louis, mo) as per manufacturer&#x27;s instructions. briefly, 1×105 h358 </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\">/well were seeded in 6-well plates and infected the following day with c-cbl lentiviral shrna constructs. to generate stable c-cbl knockdown cell lines, </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were selected for 2 days with 1 µg/ml puromycin. c-cbl levels were determined </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.31%); opacity: 0.80\" title=\"-0.010\">whole</span><span style=\"opacity: 0.80\"> cell lysates by immunoblotting with anti-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.49%); opacity: 0.80\" title=\"0.002\">antibody</span><span style=\"opacity: 0.80\"> (santa cruz biotechnologies, santa cruz, ca).  cell viability assay  </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> as </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.007\">described</span><span style=\"opacity: 0.80\"> above in the transfection assay. forty-eight </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.004\">hours</span><span style=\"opacity: 0.80\"> after transfection, viability of </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> was assessed </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> trypan </span><span style=\"background-color: hsl(120, 100.00%, 94.89%); opacity: 0.81\" title=\"0.046\">blue</span><span style=\"opacity: 0.80\"> exclusion.  wound healing assay  a549 </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were seeded in 6-well plates and cultured for 48 h until 100% confluent. the medium was then changed and the </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> as </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.007\">described</span><span style=\"opacity: 0.80\"> in the transfection assay. twelve </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.004\">hours</span><span style=\"opacity: 0.80\"> after transfection, a straight scratch was made across the cell layer </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> a 1 ml pipette tip. the </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were then gently washed with 1× pbs to remove cellular debris and the media was replaced. photographs were taken of the wound region every 12 h until 48 h.  western blot analysis  forty eight </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.004\">hours</span><span style=\"opacity: 0.80\"> after transfection, </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were collected and washed twice in 1x pbs, then lysed in ice-cold lysis buffer (0.5m tris-hcl with ph 7.4, 1.5 m nacl, 2.5% deoxycholic acid, 10 mm edta, 10% np-40, 0.5 mm dtt, 1 mm phenylmethylsulfonyl fluoride, 5 µg/ml leupeptin, and 10 µg/ml aprotinin) for 5 minutes. the lysate was centrifuged at 13,000 rpm for </span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">20</span><span style=\"opacity: 0.80\"> minutes at 4°c, and protein content of the supernatant was measured. total cell lysates (50 µg/well) were separated by sds-page electrophoresis and the gels transferred onto nitrocellulose membranes (whatman, piscataway, nj). membranes were blocked with 5% non-fat dry milk in phosphate-buffered saline containing tween-</span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">20</span><span style=\"opacity: 0.80\"> (pbst) (1x pbs, 0.1% tween-</span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">20</span><span style=\"opacity: 0.80\">) for 1 h at room temperature and incubated with the appropriate primary </span><span style=\"background-color: hsl(120, 100.00%, 99.49%); opacity: 0.80\" title=\"0.002\">antibody</span><span style=\"opacity: 0.80\"> at 4°c overnight. membranes then were washed three times with pbst and probed with appropriate horseradish peroxidase (hrp)-conjugated secondary </span><span style=\"background-color: hsl(120, 100.00%, 99.49%); opacity: 0.80\" title=\"0.002\">antibody</span><span style=\"opacity: 0.80\"> for 1 h at room temperature. the membranes were again washed three times in pbst and bands were visualized </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> western blot chemiluminescence reagent (biorad, valencia, ca) on a chemidoc gel documentation system (biorad, valencia, ca). </span><span style=\"background-color: hsl(0, 100.00%, 95.58%); opacity: 0.81\" title=\"-0.037\">antibodies</span><span style=\"opacity: 0.80\"> were obtained from santa cruz biotechnologies and used at the following dilutions (c-cbl, 1∶5000; c-met, 1∶5000; egfr, 1∶5000; ubiquitin, 1∶1000; ha, 1∶5000 and β-actin, 1∶10,000).  flow cytometry  cell cycle analysis was carried out by flow cytometry. approximately 2×106 </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were grown in media containing 10% fbs. </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were harvested by trypsin/edta treatment, washed with 1x pbs three times and fixed with ice-cold 70% ethanol for 2 h. </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were washed again with cold pbs and stained with a solution containing 25 µg/ml propidium iodide, 200 µg/ml rnase a, and 0.1% triton x-100 for 30 minutes in the dark. cell cycle analysis was performed </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> a guava pca-96 flow cytometer (guava technologies, millipore, billerica, ma).  ubiquitin ligase activity  293t </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were maintained in culture in dmem supplemented with 10% fbs and 1% penicillin (100 units/ml) and streptomycin (100 µg/ml) were </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> with 0.2 µg egfr-pcdna3 and 2 µg ha-</span><span style=\"background-color: hsl(120, 100.00%, 98.86%); opacity: 0.80\" title=\"0.005\">tagged</span><span style=\"opacity: 0.80\"> c-cbl constructs as indicated </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> calcium phosphate according to manufacturer&#x27;s protocol (profection, promega, madison, wi). twenty-four </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.004\">hours</span><span style=\"opacity: 0.80\"> post-transfection, </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were starved overnight in dmem supplemented with 0.5% fbs, and then treated with or without egf (100 ng/ml) for 15 min. the </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were collected and washed two times in ice-cold pbs containing 0.2 mm sodium orthovanadate then lysed in ice-cold lysis buffer (10 mm tris hcl, ph 7.5, 150 mm nacl, 5 mm edta, 1% triton x100, 10% glycerol, 2 mm sodium orthovanadate and protease inhibitors). lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4°c. egfr immunoprecipitations were performed on 200 µg of cleared lysate </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> 250 ng of rabbit-anti-egfr and protein a/g plus sepharose overnight at 4°c. precipitations were washed 5 times in lysis buffer before boiling in laemmli buffer. elutions were immunoblotted with anti-ubiquitin and egfr. twenty micrograms of cleared lysate were immunoblotted for each of the c-cbl constructs </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> anti-ha.  statistical analysis  mutation rates between different groups were compared </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> fisher&#x27;s exact test. for continuous variables, </span><span style=\"background-color: hsl(120, 100.00%, 97.50%); opacity: 0.80\" title=\"0.017\">group</span><span style=\"opacity: 0.80\"> comparisons were performed </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> analysis of variance (anova) followed by sidak&#x27;s adjustment for multiple comparisons. experiments involving repeated measurements over time were analyzed </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> repeated measures anova with the greenhouse-geisser adjustment to the degrees of freedom. analyses were conducted </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> stata (v10.1) software (stata corporation, college station, tx).  go to: results c-cbl gene mutations in lung cancer  to investigate the role of c-cbl in lung cancer, we analyzed its genomic </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\"> in tumor and paired normal samples drawn from multiple ethnicities. the lung tumor samples represented caucasians (n = 50), african-americans (n = 29), and taiwanese (n = 40) lung cancer patients. we designed 12 pairs of primers to sequence the coding region of c-cbl gene that spans exons 2 to 16 (table s2). we identified 8 unique somatic mutations in c-cbl exons among 8 different patients. a variation l620f, a known snp (rs2227988) in exon 11 was also detected. importantly, the eight novel non-synonymous mutations were confirmed by sequencing both strands of c-cbl genomic </span><span style=\"background-color: hsl(120, 100.00%, 99.56%); opacity: 0.80\" title=\"0.001\">dna</span><span style=\"opacity: 0.80\"> obtained from lung tumor samples (table 1). moreover, none of the 8 mutations were detected in the corresponding normal tissue, indicating that these were somatic mutations. four synonymous single nucleotide variations (snvs) were also identified but were not used further in this study.  table 1 table 1 c-cbl mutation analysis in 119 lung cancer patient tumor tissues. three of the 8 novel non-synonymous mutations were located in the tkb (tyrosine kinase binding) domain (s80n, h94y, and q249e), one in the ring finger domain (v391i), one in the proline-rich region (72515_72517 del atg), and three in the c-</span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">terminal</span><span style=\"opacity: 0.80\"> region (w802*, r830k, and a848t) of the c-cbl protein (figure 1a and figure s1). in figure 1b, we show model chromatograms of representative samples.  figure 1 figure 1 c-cbl mutations and loh in non-small cell lung cancer. 11q loh of c-cbl gene  paired lung tumor and normal lung tissue samples from taiwanese patients (n = 37) were investigated for loh. eight (21.6%) showed loh at the c-cbl locus on </span><span style=\"background-color: hsl(0, 100.00%, 98.11%); opacity: 0.80\" title=\"-0.011\">chromosome</span><span style=\"opacity: 0.80\"> 11 while 29 samples (</span><span style=\"background-color: hsl(0, 100.00%, 96.31%); opacity: 0.81\" title=\"-0.029\">78</span><span style=\"opacity: 0.80\">.4%) revealed normal allelic contribution at the microsatellite markers (figures 1c and d).  c-cbl mutations in different ethnic groups  the c-cbl double mutant s80n/h94y was found in the same patient and the overall mutation rate for c-cbl in lung tumors was 6.7% (8/119). the frequency of c-cbl mutation was highest in large cell carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the least was observed in adenocarcinoma (ad) (1.8%; 1 of 53 patients), although these rates were not statistically </span><span style=\"background-color: hsl(0, 100.00%, 99.44%); opacity: 0.80\" title=\"-0.002\">significant</span><span style=\"opacity: 0.80\"> (p = 0.292). mutation rates were 6.0% among caucasians (0 of </span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">20</span><span style=\"opacity: 0.80\"> in ad; 0 of 10 in sq; and 3 of </span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">20</span><span style=\"opacity: 0.80\"> in lc), 13.8% in african-americans (1 of 10 in ad; 1 of 10 in sq; and 2 of 9 in lc), and 2.5% (0 of 23 in ad; 1 of 12 in sq; and 0 of 5 in lc) in the taiwanese population. </span><span style=\"background-color: hsl(0, 100.00%, 94.83%); opacity: 0.81\" title=\"-0.047\">additionally</span><span style=\"opacity: 0.80\"> two taiwanese patients with lung cancer (one squamous and one adenocarcinoma) had the known snp l620f. ethnic differences were not statistically </span><span style=\"background-color: hsl(0, 100.00%, 99.44%); opacity: 0.80\" title=\"-0.002\">significant</span><span style=\"opacity: 0.80\">, however the power to detect differences was low.  mutations in met and egfr can be co-associated with c-cbl alterations  since east asians with lung cancer have a higher frequency of egfr and met mutations in lung tumors [21], [22], we also determined mutations in egfr and met in the same taiwanese cohort samples and compared the results with the observed c-cbl alterations (loh and/or mutations). in the 37 samples tested, we did not find any overlap between c-cbl mutations and c-cbl loh (figure 2). of the three c-cbl mutants (including the known l620f snp, rs2227988), one of the samples had a met mutation (n375s) and the other had an egfr mutation (l858r). among the 8 samples that had a loh at the c-cbl locus 5 had an additional mutation in met (n375s) and 2 had an egfr exon 19 deletion. twenty-six samples had neither c-cbl mutation nor c-cbl loh (3 patients had a c-cbl mutation but no c-cbl loh). among these 26 samples 9 had a met mutation (8 n375s, 1 l211w), 13 had an egfr mutation (7 exon 9 deletion, 6 l858r) and 4 had no other met or egfr mutation. thus the rate of met or egfr mutations among patients with loh at the c-cbl locus (7 of 8) was similar to that seen in patients without c-cbl mutation or loh (22 of 26 patients) (p = 0.99). these 4 patients with no identifiable mutation in c-cbl, met or egfr represented 10.8% of the 37 patients analyzed in the taiwanese patient cohort. conversely, 89.2% taiwanese lung cancer patients have an identifiable mutation in either c-cbl, met or egfr or a combination of the three genes (figure 2). </span><span style=\"background-color: hsl(0, 100.00%, 94.83%); opacity: 0.81\" title=\"-0.047\">additionally</span><span style=\"opacity: 0.80\">, we determined p53 and kras mutations in these taiwanese cohorts. two p53 and 1 kras mutation were detected. the single kras mutation overlapped with one p53 mutation. this patient also had the egfr exon 19 deletion but had no c-cbl mutation. the other p53 mutation sample had a c-cbl loh with concurrent met n375s mutation. thus, in the taiwanese samples analyzed, p53/kras mutations and c-cbl mutations were mutually exclusive (data not shown).  figure 2 figure 2 c-cbl mutations and relationship to met and egfr mutations in lung cancer. cellular functions of c-cbl alterations in the context of lung tumorigenesis  a. e3 activity is intact in the mutant c-cbl proteins to investigate whether the different c-cbl mutations affect the e3 activity, egfr was chosen as a model substrate for c-cbl e3 function. all of the c-cbl mutants tested enhanced ubiquitination of the activated egfr similar to the wild-type c-cbl protein. this result demonstrates that the catalytic activity of the c-cbl mutants is not impaired when egfr was the substrate. (figure 3a).  figure 3 figure 3 ubiquitination, viability, expression and cell cycle analysis of various c-cbl mutants. b. effect on lung cancer cell viability the effect of a representative c-cbl mutant from each of the three ethnic backgrounds on lung cancer cell viability in cell lines was determined. s80n/h94y double mutation, q249e, and w802* were identified in lung tumor samples obtained from a caucasian, a taiwanese and an african-american, respectively. as </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.007\">described</span><span style=\"opacity: 0.80\"> in methods, the c-cbl wild-type (wt) and the above three mutants were expressed after cloning them into paltermax vector in a549 </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\">. these </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> express relatively low basal levels of endogenous c-cbl (data not shown). transfection efficiency was comparable between different groups and the number of </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> with c-cbl wild-type construct was about 70% compared to </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">control</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> that were </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> with the empty vector. </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> with s80n/h94y, q249e and w802* c-cbl mutant constructs resulted in </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">increased</span><span style=\"opacity: 0.80\"> number of viable </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> that was 132.3%, 120.8% and 147.9% higher respectively, relative to the empty vector </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">control</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> and significantly different from the wild-type construct (p = 0.022, p = 0.049, and p = 0.008, respectively) (figure </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.007\">3b</span><span style=\"opacity: 0.80\">). relative levels of c-cbl protein in </span><span style=\"background-color: hsl(0, 100.00%, 98.31%); opacity: 0.80\" title=\"-0.010\">whole</span><span style=\"opacity: 0.80\"> cell lysates prepared from samples obtained from a parallel experiment was determined. the c-cbl protein levels in samples representing untransfected and empty vector </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were comparable and those representing the c-cbl wt and the three c-cbl mutants were comparable (figure 3c).  c. effect on cell cycle to investigate if the increases in cell viability in different c-cbl mutants are due to </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">increased</span><span style=\"opacity: 0.80\"> cellular proliferation, a cell cycle analysis was performed. a549 </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> with the c-cbl wt or the three different mutants: s80n/h94y, q249e and w802*. the empty vector transfectant was used as a </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">control</span><span style=\"opacity: 0.80\">. forty-eight </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.004\">hours</span><span style=\"opacity: 0.80\"> after transfection cell cycle analysis was performed as </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.007\">described</span><span style=\"opacity: 0.80\"> in materials and methods. there was no </span><span style=\"background-color: hsl(0, 100.00%, 99.44%); opacity: 0.80\" title=\"-0.002\">significant</span><span style=\"opacity: 0.80\"> change in the subg1, g1 or the s phase of the cell cycle among the different mutants compared to the wt construct (p = 0.64, p = 0.40, and p = 0.28, respectively). the g2/m phase of the cell cycle showed an increase in cell numbers for the three mutants, s80n/h94y, q249e and w802*, when compared to the wt but again the difference was not statistically </span><span style=\"background-color: hsl(0, 100.00%, 99.44%); opacity: 0.80\" title=\"-0.002\">significant</span><span style=\"opacity: 0.80\"> (p = 0.25) (figure 3d).  d. effect on cell motility to investigate the effect of the expression of the above three c-cbl mutants on cell migration, we carried out wound healing assay as </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.007\">described</span><span style=\"opacity: 0.80\"> in materials and methods. the closing of the scratch or the wound was monitored at 0, 12, 24, 36, and 48 h. (figure 4a). in all the samples, that represented </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> with mutants, the wound gap was much smaller than that seen in the sample that represented </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.029\">transfected</span><span style=\"opacity: 0.80\"> with c-cbl wt (p&lt;0.001). we also determined the rate of wound closure for all the five groups. at 48 h, wild-type c-cbl transfectants showed 61.1% open wound while the s80n/h94y, q249e and w802* mutants showed 18.7%, 23.9% and 34.3% open wound respectively (p&lt;0.001) (figure 4b).  figure 4 figure 4 c-cbl mutations affect wound healing in a549 </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\">. e. c-cbl knockdown increases cell viability it is hypothesized that the loh seen in our samples could lead to decreased expression of c-cbl. thus we tested the effect of c-cbl knockdown in lung cancer </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\">. compared to a549, h358 lung cancer </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> express relatively high levels of endogenous c-cbl (data not shown). c-cbl expression was knocked down </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> lentiviral construct that expressed c-cbl specific shrna and compared the results with those that were transduced with scrambled shrna and the results are shown in figure 5. we identified several clones that revealed varying degrees of c-cbl knockdown showing different sets of c-cbl lentiviral shrna knockdown efficiency (figure 5a). of all the clones tested, clone 27 was chosen for further experiments. equal amount of </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> were seeded in a 6-well plate and the cell proliferation was measured at various times and the results are depicted in figure 5b. as </span><span style=\"background-color: hsl(0, 100.00%, 90.46%); opacity: 0.83\" title=\"-0.112\">expected</span><span style=\"opacity: 0.80\">, number of </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">increased</span><span style=\"opacity: 0.80\"> in a time dependent fashion from 100 to 190% relative to scrambled shrna as </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.003\">control</span><span style=\"opacity: 0.80\"> in a span of 48 h (p = 0.0002) (figure 5b). the cell cycle phases in h358 </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> that were knocked down with c-cbl shrna were looked at and compared with the scrambled shrna. there were no discernable differences between these two constructs in the different phases of the cell cycle (data not shown).  figure 5 figure 5 knockdown of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 99.60%); opacity: 0.80\" title=\"0.001\">using</span><span style=\"opacity: 0.80\"> an shrna increases cell proliferation. go to: discussion our results demonstrate that c-cbl is somatically mutated (or has loh) in lung cancers, and can significantly contribute to enhanced cell viability and motility. there was also a high prevalence of loh with respect to c-cbl in lung tumors that harbored met or egfr mutation.  in the present study, we have demonstrated the occurrence of c-cbl mutations in lung cancer patients, especially with different ancestral variations. mutations in c-cbl have been recently reported in juvenile myelomonocytic leukemia and myeloid malignancies. in the aml study, the mutation r420q located in the junction of the ring finger and the linker region inhibited fms-</span><span style=\"background-color: hsl(120, 100.00%, 99.63%); opacity: 0.80\" title=\"0.001\">like</span><span style=\"opacity: 0.80\"> tyrosine kinase 3 (flt3) internalization and ubiquitination [</span><span style=\"background-color: hsl(120, 100.00%, 99.11%); opacity: 0.80\" title=\"0.004\">20</span><span style=\"opacity: 0.80\">], thus contributing to the gain-in-function for the rtk. in addition, mutations such as h398y, c384r, and l380p mapped to the ring finger domain and the linker region of c-cbl that is required for its e3 activity [23], [24], [25], [26], [27]. </span><span style=\"background-color: hsl(0, 100.00%, 94.83%); opacity: 0.81\" title=\"-0.047\">additionally</span><span style=\"opacity: 0.80\">, homozygous mutations in the ring finger domain of the c-cbl gene were </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.007\">described</span><span style=\"opacity: 0.80\"> as a result of acquired uniparental disomy (upd) [26]. it is important to note that our results indicate loh at 11q23 locus and these are mutually exclusive from missense mutations of c-cbl. the somatic mutations were all heterozygous. the mutations in aml led to abrogation of the e3 activity </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.002\">leading</span><span style=\"opacity: 0.80\"> to prolonged rtk activation. in addition mutants located on the linker region surrounding the ring finger domain exhibited enhanced akt signaling in response to cytokine stimulation [26]. in addition, it was shown in nh3t3 </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\">, that neither mutations in the ring finger nor the linker region causes transformation, however while certain mutations perturbs the ubiquitination, others affect receptor recycling and prolong kinase activity [28].  we report </span><span style=\"background-color: hsl(0, 100.00%, 99.50%); opacity: 0.80\" title=\"-0.002\">here</span><span style=\"opacity: 0.80\"> c-cbl mutations that mapped not only to the ring finger domain, but also to the tkb domain, proline-rich domain and the c-</span><span style=\"background-color: hsl(0, 100.00%, 99.40%); opacity: 0.80\" title=\"-0.002\">terminal</span><span style=\"opacity: 0.80\"> region, but none mapped to the linker region as reported in the aml studies </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.007\">described</span><span style=\"opacity: 0.80\"> above [23], [24], [25], [26], [29]. in addition, 8 mutants that we detected were found in different ethnic backgrounds. for example, s80n/h94y, q249e, w802* were detected in caucasians, taiwanese and african-americans, respectively. the results point out not only the difference between lung cancer and other cancers, but also genetic polymorphism among different races in the same cancer. interestingly, there is a large disparity between african-american and other ethnic populations with lung cancer [30]. we have previously shown that there was a low frequency of egfr and met mutation in african-americans as compared with taiwanese and caucasians [31]. in this study, the number of african-american samples analyzed was relatively fewer and we found 3 mutations that are unique to this ethnicity. it would now behoove us to further study the genetic alterations that can </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.002\">occur</span><span style=\"opacity: 0.80\"> and determine the targeted therapeutics for african-americans. our results provide evidence of the importance of c-cbl in tumorigenesis and potential signaling. our prediction, based on the aml data, would be that the v391i ring finger domain mutation would affect the e3 activity. also, it will be important to determine the binding partners for c-cbl in the tkb domain and proline rich domain mutations. it has previously been shown that the tkb domain can bind to growth factor receptors and it will be important to determine the cross-binding of these mutants to met and or egfr. it would also be important in the future to look at fluorescence in situ hybridization/copy number changes in c-cbl in lung cancer.  c-cbl plays an important role in down regulating rtk-mediated signaling through k63 poly-ubiquitination and subsequent downregulation of rtks followed by lysosomal degredation [3]. mono-ubiquitination or ubiquitinated with k63-linked chains of substrates by c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.006\">may</span><span style=\"opacity: 0.80\"> lead to enhancement of biological and biochemical functions (reviewed in hermann et al, 2007 [32]). the mutations that we analyzed in our studies all point out to the fact that the e3 activity of c-cbl on egfr is intact; the egfr levels in the various mutants </span><span style=\"background-color: hsl(0, 100.00%, 90.13%); opacity: 0.83\" title=\"-0.118\">remain</span><span style=\"opacity: 0.80\"> same (figure s2). multiple </span><span style=\"background-color: hsl(120, 100.00%, 99.72%); opacity: 0.80\" title=\"0.001\">kinases</span><span style=\"opacity: 0.80\">, both rtks and non-rtks could be acted upon by c-cbl, including erbs, pdgfr, fms, met, c-kit, vegfr, flt-1, ron, fgfr, ir, as well as syk, fyn, lck, fgr, lyn and c-abl [3]. in lung cancers the relevant substrates of c-cbl in terms of degradation or signal transduction are yet to be identified.  the observation that c-cbl somatic mutations, especially s80n/h94y, q249e and w802* showed </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">increased</span><span style=\"opacity: 0.80\"> cell viability and cell motility that is in agreement with the physiological role for d-cbl in the regulation of apoptosis and differentiation identified in drosophila is very </span><span style=\"background-color: hsl(0, 100.00%, 99.44%); opacity: 0.80\" title=\"-0.002\">significant</span><span style=\"opacity: 0.80\"> [33]. it has been previously shown that activating c-cbl mutation downregulates egfr signaling and decreases cellular proliferation and migration in </span><span style=\"background-color: hsl(0, 100.00%, 97.67%); opacity: 0.80\" title=\"-0.015\">breast</span><span style=\"opacity: 0.80\"> cancer cell lines [34]. although the role of c-cbl in the negative regulation of rtks is well substantiated, thereby suggesting that it is a natural tumor suppressor, studies in cancer </span><span style=\"background-color: hsl(0, 100.00%, 99.81%); opacity: 0.80\" title=\"-0.000\">cells</span><span style=\"opacity: 0.80\"> have revealed both tumor suppressor and tumor promoting activities depending on the type of c-cbl mutation and the number of alleles at the c-cbl locus [24]. in agreement with the above, the three c-cbl mutants </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.007\">described</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.50%); opacity: 0.80\" title=\"-0.002\">here</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.97%); opacity: 0.80\" title=\"0.005\">appear</span><span style=\"opacity: 0.80\"> to have tumor growth and metastasis promoting properties. although these mutants are outside of the ring finger or the linker region of c-cbl, their </span><span style=\"background-color: hsl(0, 100.00%, 98.30%); opacity: 0.80\" title=\"-0.010\">downstream</span><span style=\"opacity: 0.80\"> effects are </span><span style=\"background-color: hsl(0, 100.00%, 99.44%); opacity: 0.80\" title=\"-0.002\">significant</span><span style=\"opacity: 0.80\"> so as to cause </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">increased</span><span style=\"opacity: 0.80\"> proliferation and migration, but the substrate affected by these mutations are not known yet. this raises the possibility that some of the cellular functions of c-cbl are independent of its ubiquitin-ligase activity, an area that we are </span><span style=\"background-color: hsl(120, 100.00%, 91.77%); opacity: 0.82\" title=\"0.091\">currently</span><span style=\"opacity: 0.80\"> investigating. the </span><span style=\"background-color: hsl(120, 100.00%, 97.71%); opacity: 0.80\" title=\"0.015\">oncogenic</span><span style=\"opacity: 0.80\"> nature of rtks, addiction of cancers to growth </span><span style=\"background-color: hsl(0, 100.00%, 96.69%); opacity: 0.81\" title=\"-0.025\">signals</span><span style=\"opacity: 0.80\"> and given the clustering of c-cbl, egfr and met mutations, it is possible that the </span><span style=\"background-color: hsl(0, 100.00%, 96.95%); opacity: 0.81\" title=\"-0.022\">transforming</span><span style=\"opacity: 0.80\"> effect of c-cbl mutations is most likely a combinatorial effect of the three. we also show that loh for c-cbl was found in a </span><span style=\"background-color: hsl(0, 100.00%, 99.44%); opacity: 0.80\" title=\"-0.002\">significant</span><span style=\"opacity: 0.80\"> number of samples that harbored met or egfr mutations. the fact that about 7% of lung tumor samples are likely to have c-cbl mutations and an additional 22% are likely to harbor c-cbl-related loh, </span><span style=\"background-color: hsl(0, 100.00%, 98.97%); opacity: 0.80\" title=\"-0.005\">makes</span><span style=\"opacity: 0.80\"> c-cbl a highly mutated molecule in lung cancer. since loh alone is not enough to cause a </span><span style=\"background-color: hsl(0, 100.00%, 96.95%); opacity: 0.81\" title=\"-0.022\">transforming</span><span style=\"opacity: 0.80\"> event [35], [36], [37], associated mutation in the met or egfr locus or yet another rtk discussed above </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.006\">may</span><span style=\"opacity: 0.80\"> play a role in carcinogenesis. we predict that this loh in c-cbl results in haploinsufficiency that downplays rtk ubiquitination </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.002\">leading</span><span style=\"opacity: 0.80\"> to hyperactivity of the rtks. however, whether this is sufficient cause for tumorigenesis remains to be determined. consistent with our hypothesis is the fact that c-cbl-/- mice have </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">increased</span><span style=\"opacity: 0.80\"> kinase activity in lymphocytes, but is not sufficient for tumor formation [35], [36], [37]. c-cbl loh could also lead to </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.002\">increased</span><span style=\"opacity: 0.80\"> expression of c-cbl from the other allele to compensate for a loss of an allele. alternately, there could be some form of synergy working with reduced c-cbl levels and mutated receptors that exacerbate the phenotype of each alone.  previous studies from our lab and others have shown that east asians with lung cancers have relatively high frequencies of gain-of-function of mutations in rtks such as egfr and met [31]. in a cohort of japanese patients an activating met mutation was identified in the splice region that deletes the juxtamembrane domain that is involved e3 activity of c-cbl [38]. this study also found that activation of met is mutually exclusive of egfr, kras and her2 gene mutations [38]. we failed to detect such mutations in </span><span style=\"background-color: hsl(0, 100.00%, 99.44%); opacity: 0.80\" title=\"-0.002\">significant</span><span style=\"opacity: 0.80\"> numbers in lung tumor samples obtained from african-americans (n = 29) and caucasian (n = 50) patients. one met mutation was identified in each of the groups </span><span style=\"background-color: hsl(120, 100.00%, 97.93%); opacity: 0.80\" title=\"0.013\">whereas</span><span style=\"opacity: 0.80\"> 1 and 3 egfr mutations were identified in the african-american and caucasian cohorts respectively. egfr mutations have earlier been identified as one of the key mutations affecting lung adenocarcinoma patients in a comprehensive study of 188 patients [39]. our study encompasses different histologies of nsclc. however, the published series did not find any mutations in c-cbl or met unlike our study that encompassed different subtypes of nsclc. it is important to note, we have recently shown that met mutations in lung cancer are in majority germline [31]. we have reported earlier c-cbl mutations in a small cohort of taiwanese lung cancer samples [40]. in our efforts to understand the ethnic differences in the lung oncogenome, we also looked at pax transcription factors such as pax5 and pax8 that are highly expressed in lung cancers; however there was no preferential expression or mutations of the above genes in lung tumor samples of african-americans. in this study, we show relatively high frequency of c-cbl mutations in lung cancers, especially in the large cell type among caucasians and particularly among african-americans. we therefore propose c-cbl as an efficacious target for lung cancers in african-americans that needs to be further substantiated. this is all the more important because the prognosis for african-americans with lung cancer, especially for men is much poorer compared to their caucasian counterparts [41].  in conclusion, the results presented in this study demonstrate that c-cbl is frequently mutated or even lost in lung cancers. our results support a role for c-cbl mutants that are independent of its ubiquitination activity. given the relatively high mutation rates in c-cbl as well as rtks such as met and egfr, it is likely that their combined effect could be synergistic in promoting tumorigenesis.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=7\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.087</b>, score <b>-0.540</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 91.43%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +1.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.98%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.048\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__yeast\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.09%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.041\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nonsense\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.46%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.019\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__retained\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.52%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__putative\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__being\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__alignment\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__complexes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__families\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pathogenic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__tp53\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__browser\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__conserved\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__brain\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__truncated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__loci\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__folded\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__migrating\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__smad3\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ucsc\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2011\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__brca1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__scaffold\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__utr\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__corroborating\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__originally\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__eco\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__remainder\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__doublet\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__lithium\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__t2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__neoplastic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nude\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__slowly\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fig\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__anchorage\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__versus\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__002\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__grew\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__exchange\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inactivating\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__adjuvant\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__thyroid\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__clinicaltrials\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__antigen\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__synthase\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bp\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__interval\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__grades\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__triphosphate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__proliferate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__initiates\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nucleotides\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__dl\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__independently\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__distribution\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.63%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__slightly\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__conferred\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2a\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fragment\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__egfrviii\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__gsk\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__driver\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.56%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__arise\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.54%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__activator\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.49%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__incidence\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.45%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__absence\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.40%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.023\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__1d\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.36%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.025\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__virus\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.17%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.036\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__therapeutic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.16%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.037\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__3t3\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.11%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.040\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Variation__fusion\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.053\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__erk\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.063\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__length\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.065\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2b\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.40%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.092\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__constitutive\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.07%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.120\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__phosphorylation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 92.21%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.884\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">w802*</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">cbl</span><br/><b>Text:</b> <span style=\"opacity: 0.80\"> abstract background  non-small </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lung cancer (nsclc) is a heterogeneous group of disorders with a number of genetic and proteomic alterations. c-cbl is an e3 ubiquitin ligase and adaptor molecule important in normal homeostasis and cancer. we determined the genetic variations of c-cbl, relationship to receptor tyrosine kinases (egfr and met), and </span><span style=\"background-color: hsl(0, 100.00%, 83.53%); opacity: 0.86\" title=\"-0.245\">functionality</span><span style=\"opacity: 0.80\"> in nsclc.  methods and findings  using archival formalin-fixed paraffin embedded (ffpe) extracted genomic dna, we show that c-cbl mutations occur in somatic fashion for lung cancers. c-cbl mutations were not mutually exclusive of met or egfr mutations; however they were independent of p53 and kras mutations. in normal/tumor pairwise analysis, there was significant </span><span style=\"background-color: hsl(0, 100.00%, 96.95%); opacity: 0.81\" title=\"-0.022\">loss</span><span style=\"opacity: 0.80\"> of heterozygosity (loh) for the c-cbl locus (22%, n = 8/37) and none of these samples revealed any mutation in the remaining copy of c-cbl. the c-cbl loh also positively correlated with egfr and met mutations observed in the same samples. using </span><span style=\"background-color: hsl(0, 100.00%, 90.01%); opacity: 0.83\" title=\"-0.120\">select</span><span style=\"opacity: 0.80\"> c-cbl somatic mutations such as s80n/h94y, q249e and w802* (obtained from caucasian, taiwanese and african-american samples, respectively) transfected in nsclc </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lines, there was increased </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> viability and </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> motility.  conclusions  taking the overall mutation rate of c-cbl to be a combination as somatic </span><span style=\"background-color: hsl(120, 100.00%, 96.80%); opacity: 0.81\" title=\"0.024\">missense</span><span style=\"opacity: 0.80\"> mutation and loh, it is clear that c-cbl is highly mutated in lung cancers and may play an essential role in lung tumorigenesis and metastasis.  go to: introduction in the us alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which more than 145,000 of them succumb to the disease [1]. this number is roughly equivalent to the combined mortality rates of cancers of the breast, prostate, colon, liver, kidney and melanoma [1]. in addition the prognosis is usually poor and the five-year survival rate is less than 15%. there are also significant ethnic differences for lung cancer, and the outcome is worse for blacks compared to whites. gender differences are also striking with women having significantly better prognosis as compared to men. there are a number of genetic alterations that can occur in lung cancer. as an example, in nsclc, mutations in kras, p53, egfr and met have been identified. many of these pathways, especially receptor tyrosine kinases (rtks) are controlled by c-cbl.  cbl (casitas b-lineage lymphoma) is a mammalian gene located on human chromosome 11q23.3 [2] and is </span><span style=\"background-color: hsl(0, 100.00%, 97.73%); opacity: 0.80\" title=\"-0.014\">involved</span><span style=\"opacity: 0.80\"> in </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> signaling and </span><span style=\"background-color: hsl(120, 100.00%, 98.56%); opacity: 0.80\" title=\"0.008\">protein</span><span style=\"opacity: 0.80\"> ubiquitination [3]. cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.80%); opacity: 0.80\" title=\"-0.000\">proteins</span><span style=\"opacity: 0.80\"> belong to the ring finger class of ubiquitin ligases (e3) and there are three homologues c-cbl, cbl-b, cbl-3 [4]. the c-cbl and cbl-b genes are ubiquitously expressed with the highest levels in hematopoietic tissues [5]. c-cbl consists of four regions encoding for functionally distinct </span><span style=\"background-color: hsl(120, 100.00%, 98.56%); opacity: 0.80\" title=\"0.008\">protein</span><span style=\"opacity: 0.80\"> domains: the n-terminal tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.013\">kinase</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.001\">binding</span><span style=\"opacity: 0.80\"> (tkb) domain, the linker region, the catalytic ring finger domain, the proline-rich region and the c-terminal ubiquitin-associated (uba) domain that also overlaps with a leucine-zipper (lz) domain [3]. both tkb and ring finger domains are essential for ligand-induced ubiquitination of rtks [6], [7], [8], [9]. the ring finger domain is required for the recruitment of e2 ubiquitin-conjugating enzymes. the tkb domain includes four-helix bundle (4h), a calcium-biding ef hand, and a modified sh2 domain, which binds to phosphotyrosine residues [3], [10], [11], [</span><span style=\"background-color: hsl(0, 100.00%, 98.98%); opacity: 0.80\" title=\"-0.005\">12</span><span style=\"opacity: 0.80\">]. in addition, the proline-rich region of c-cbl can associate with the sh3 domain of grb2, which can indirectly recruit c-cbl to rtks via the grb2 adaptor </span><span style=\"background-color: hsl(120, 100.00%, 98.56%); opacity: 0.80\" title=\"0.008\">protein</span><span style=\"opacity: 0.80\"> [7], [13], [14].  c-cbl also binds to egfr and acts as the e3 that targets egfr for ubiquitination and degradation. furthermore, cbl desensitizes egf signaling and opposes cellular proliferation induced by egf [15]. egf </span><span style=\"background-color: hsl(0, 100.00%, 96.91%); opacity: 0.81\" title=\"-0.022\">activation</span><span style=\"opacity: 0.80\"> also appears to </span><span style=\"background-color: hsl(0, 100.00%, 97.96%); opacity: 0.80\" title=\"-0.012\">activate</span><span style=\"opacity: 0.80\"> the tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.013\">kinase</span><span style=\"opacity: 0.80\"> src, which phosphorylates c-cbl and in turn activates the ubiquitination and degradation of egfr [16], [17], [18]. a recent study shows that </span><span style=\"background-color: hsl(0, 100.00%, 83.33%); opacity: 0.86\" title=\"-0.249\">defective</span><span style=\"opacity: 0.80\"> endocytosis of egfr is characterized by a deletion mutant and the point mutation l858r, whereby its association with c-cbl and subsequent ubiquitination are impaired [19]. recently, the first human c-cbl mutations were reported in acute myeloid leukemia (aml) patients [20]. the mutation r420q inhibits fms-like tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.013\">kinase</span><span style=\"opacity: 0.80\"> 3 (flt3) internalization and ubiquitination [20].  not only can e3 activity be important in oncogenesis, c-cbl has a dual but separate </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.005\">function</span><span style=\"opacity: 0.80\"> as a signal transduction molecule. we have previously shown that c-cbl is important in </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.001\">binding</span><span style=\"opacity: 0.80\"> crkl and bcr/abl in hematopoietic </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\">. also, it can bind and modulate </span><span style=\"background-color: hsl(0, 100.00%, 97.72%); opacity: 0.80\" title=\"-0.015\">functions</span><span style=\"opacity: 0.80\"> of cytoskeleton by </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.001\">binding</span><span style=\"opacity: 0.80\"> to </span><span style=\"background-color: hsl(0, 100.00%, 99.80%); opacity: 0.80\" title=\"-0.000\">proteins</span><span style=\"opacity: 0.80\"> like talin and paxillin. the tkb domain is important in </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.001\">binding</span><span style=\"opacity: 0.80\"> to a number of molecules, and they then </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.005\">function</span><span style=\"opacity: 0.80\"> in signal transduction.  given the critical role of cbl in normal homeostasis and cancer, we hypothesized that it might be mutated in lung cancers. in this study, we report novel c-cbl somatic mutations s80n/h94y, q249e and w802* in caucasian, taiwanese and african-american lung cancer patients, respectively. </span><span style=\"background-color: hsl(120, 100.00%, 97.55%); opacity: 0.80\" title=\"0.016\">expressing</span><span style=\"opacity: 0.80\"> these mutations in nsclc </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lines lead to increased proliferation and </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> motility. we show that c-cbl mutations occur with or without met or egfr mutations but are mutually exclusive of a loh at the c-cbl locus. additionally, c-cbl loh is associated with either met or egfr mutations. we thus hypothesize that c-cbl mutations might </span><span style=\"background-color: hsl(120, 100.00%, 97.30%); opacity: 0.80\" title=\"0.019\">contribute</span><span style=\"opacity: 0.80\"> to the oncogenic potential of met and egfr in lung cancer.  go to: methods ethics statement  written consent on all research on human subjects has been obtained from the institutional review board, university of chicago and covers all research performed in the laboratory. the following is their contact information: institutional review board, the university of chicago, mcgiffert hall, 5751 s. woodlawn ave., 2nd floor, chicago, il 60637. written informed consents were received from all patients whose tissue samples were used for this study.  tissue samples  lung cancer tissue and paired adjacent normal lung tissues were obtained from 50 caucasian, 29 african-americans and 40 taiwanese nsclc patients </span><span style=\"background-color: hsl(120, 100.00%, 97.32%); opacity: 0.80\" title=\"0.018\">who</span><span style=\"opacity: 0.80\"> were recruited at the university of chicago hospital (chicago, usa) (caucasian and african-american patients) and taipei veterans general hospital of taiwan (taiwanese patients) after obtaining appropriate institutional review board permission and informed consent from the patients. out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 years. in terms of tumor types, 53 were adenocarcinoma, 32 were squamous </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> carcinoma and 34 were large </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> carcinoma. 49 were stage i, 14 were stage ii, 34 were stage iii, and 13 were stage iv (table s1).  </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> culture  human non-small </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lung carcinoma </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> a549 and h358 were maintained in dmem and rpmi-1640, respectively. human embryonic kidney 293t </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were cultured in dmem. media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (invitrogen, carlsbad, ca). </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were cultured at 37°c in a humidified incubator containing 5% co2.  c-cbl gene mutational analysis  exons 2 to 16 of c-cbl gene were </span><span style=\"background-color: hsl(0, 100.00%, 96.49%); opacity: 0.81\" title=\"-0.027\">individually</span><span style=\"opacity: 0.80\"> amplified by polymerase chain reaction (pcr). primers are listed in table s2. pcr conditions were 1 cycle of 95°c for 5 minutes; 35 cycles of 94°c for 30 seconds, 58°c for 30 seconds and 72°c for 2 minutes; and one cycle of 72°c for 10 minutes. pcr products were treated with exosap-it (usb corporation, cleveland, oh) and sequenced by big-dye terminator chemistry (applied biosystems, foster city, ca). sequencing was performed on the forward coding </span><span style=\"background-color: hsl(0, 100.00%, 95.46%); opacity: 0.81\" title=\"-0.039\">strand</span><span style=\"opacity: 0.80\"> with confirmation of c-cbl alterations performed by sequencing the reverse </span><span style=\"background-color: hsl(0, 100.00%, 95.46%); opacity: 0.81\" title=\"-0.039\">strand</span><span style=\"opacity: 0.80\"> as well. chromatograms were analyzed for mutations using mutation surveyor v2.61 (softgenetics, state college, pa).  plasmid constructs and site-directed mutagenesis  the wild-type c-cbl cdna insert was subcloned into the paltermax expression vector using xhoi and sali restriction enzyme sites (promega, madison, wi). using this parental plasmid paltermax-c-cbl, the tkb domain double mutation (s80n/h94y), the point mutation (q249e), and the c-terminal point mutation w802* of c-cbl were created using the following primers: 5′-gctggcgctaaagaataacccaccttatatcttagac-3′ and 5′-ctaccagatacctaccagtatctccgtactatcttgtc-3′ for the double mutation s80n/h94y; 5′-ctttacccgactctttgagccctggtcctctttgc-3′ for q249e, and 5′-cagctcctcctttggctgattgtctctggatggtgatc-3′ for w802* </span><span style=\"background-color: hsl(0, 100.00%, 98.54%); opacity: 0.80\" title=\"-0.008\">along</span><span style=\"opacity: 0.80\"> with their complementary primers using the quickchange site-directed mutagenesis xl kit (stratagene, la jolla, ca) according to the manufacturer&#x27;s instructions. the constructs were confirmed for the point mutations by standard dna sequencing of both strands.  </span><span style=\"background-color: hsl(0, 100.00%, 96.95%); opacity: 0.81\" title=\"-0.022\">loss</span><span style=\"opacity: 0.80\"> of heterozygosity (loh) analysis  five microsatellites on chromosome 11 (3 on 11q at or within 200 kb up or downstream of the c-cbl gene and 2 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">control</span><span style=\"opacity: 0.80\"> markers on 11p) were selected for analysis (table s3). established microsatellite markers and respective primer sequences were selected from the geneloc database (http://genecards.weizmann.ac.il/geneloc/index.shtml, weizmann institute of science, rehovot, israel). primers were custom designed and each forward primer was fluorescently labeled at the 5′ end with fam, pet, ned, or vic (applied biosystems). primer annealing temperatures and duplex scores were evaluated with nist primer tools (http://yellow.nist.gov:8444/dnaanalysis/primertoolspage.do; national institute of standards and technology, gaithersburg, md). primers were verified by performing pcr with </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">control</span><span style=\"opacity: 0.80\"> dna (isolated from tk6 </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\">) and resolving the products on agarose gels. bands were visualized with an uv transilluminator. genomic dna was extracted from tumor samples and paired normal lung tissue. primers were grouped into multiplex combinations shown in table s4. marker d11s929 served as an internal </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">control</span><span style=\"opacity: 0.80\"> to check for consistency in pcrs and of peaks from capillary electrophoresis. multiplex pcrs were carried out in a volume of 10 µl that contained 1 µl genomic dna (20–50 ng), 0.5 µm of each primer (1.0 µm total for each primer pair), 400 µm dntps, 1x pcr buffer containing mgcl2, and 0.2 u taq dna polymerase. pcr was performed on the abi geneamp 9700 pcr system under the following conditions: 5 min at 94°c; 30 cycles of 30 sec at 94°c, 1 min at 60°c, 1 min at 72°c; and 5 min at 72°c. the pcr products were separated by capillary electrophoresis on an abi 3130xl dna analyzer. chromatograms were analyzed with peak scanner 1.0 and genemapper 3.7 software (applied biosystems) for allelic alterations. the area of the peaks produced by the dna pcr products was quantified for each allele. the ratio of the allelic areas was calculated for each tumor and paired normal dna sample. when the qloh (allelic ratio for the tumor peaks </span><span style=\"background-color: hsl(0, 100.00%, 93.45%); opacity: 0.82\" title=\"-0.066\">divided</span><span style=\"opacity: 0.80\"> by the allelic ratio of paired normal sample) was ≤0.5 or ≥2.0 for c-cbl and at least one other 11q marker in at least two separate experiments, the sample was considered as having an allelic imbalance and interpreted as loh. samples were evaluated in at least two separate experiments and samples showing prospective loh at c-cbl repeated a third time which included a new </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">control</span><span style=\"opacity: 0.80\"> marker at the bax locus (data not shown) on chromosome 19 to verify integrity of sample dna.  transfection of c-cbl constructs  the a549 </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> line was transfected using the fugene hd (roche, nutley, nj) reagent according to the manufacturer&#x27;s instructions. eight µg of plasmid dna, containing either no insert (empty vector), wild-type c-cbl, s80n/h94y c-cbl, q249e c-cbl or w802* cbl was used for transfection in a 6-well culture plate. </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were harvested 48 h after transfection and analyzed for expression.  c-cbl knockdown  c-cbl knockdown was performed using lentiviral transduction using mission lentiviral transduction particles (sigma-aldrich, st. louis, mo) as per manufacturer&#x27;s instructions. briefly, 1×105 h358 </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\">/well were seeded in 6-well plates and infected the following day with c-cbl lentiviral shrna constructs. to generate stable c-cbl knockdown </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lines, </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were selected for 2 days with 1 µg/ml puromycin. c-cbl levels were determined using whole </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lysates by immunoblotting with anti-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.81%); opacity: 0.80\" title=\"0.006\">antibody</span><span style=\"opacity: 0.80\"> (santa cruz biotechnologies, santa cruz, ca).  </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> viability assay  </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were transfected as described above in the transfection assay. forty-eight hours after transfection, viability of </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> was assessed using trypan blue exclusion.  wound healing assay  a549 </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were seeded in 6-well plates and cultured for 48 h until 100% confluent. the medium was then changed and the </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were transfected as described in the transfection assay. twelve hours after transfection, a straight scratch was made across the </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> layer using a 1 ml pipette tip. the </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were then gently washed with 1× pbs to remove cellular debris and the media was replaced. photographs were taken of the wound region every </span><span style=\"background-color: hsl(0, 100.00%, 98.98%); opacity: 0.80\" title=\"-0.005\">12</span><span style=\"opacity: 0.80\"> h until 48 h.  western blot analysis  forty eight hours after transfection, </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were collected and washed twice in 1x pbs, then lysed in ice-cold lysis buffer (0.5m tris-hcl with ph 7.4, 1.5 m nacl, 2.5% deoxycholic acid, 10 mm edta, 10% np-40, 0.5 mm dtt, 1 mm phenylmethylsulfonyl fluoride, 5 µg/ml leupeptin, and 10 µg/ml aprotinin) for 5 minutes. the lysate was centrifuged at 13,000 rpm for 20 minutes at 4°c, and </span><span style=\"background-color: hsl(120, 100.00%, 98.56%); opacity: 0.80\" title=\"0.008\">protein</span><span style=\"opacity: 0.80\"> content of the supernatant was measured. total </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lysates (50 µg/well) were separated by sds-page electrophoresis and the gels transferred onto nitrocellulose membranes (whatman, piscataway, nj). membranes were blocked with 5% non-fat dry milk in phosphate-buffered saline containing tween-20 (pbst) (1x pbs, 0.1% tween-20) for 1 h at room temperature and incubated with the appropriate primary </span><span style=\"background-color: hsl(120, 100.00%, 98.81%); opacity: 0.80\" title=\"0.006\">antibody</span><span style=\"opacity: 0.80\"> at 4°c overnight. membranes then were washed three times with pbst and probed with appropriate horseradish peroxidase (hrp)-conjugated secondary </span><span style=\"background-color: hsl(120, 100.00%, 98.81%); opacity: 0.80\" title=\"0.006\">antibody</span><span style=\"opacity: 0.80\"> for 1 h at room temperature. the membranes were again washed three times in pbst and bands were visualized using western blot chemiluminescence reagent (biorad, valencia, ca) on a chemidoc gel documentation system (biorad, valencia, ca). antibodies were obtained from santa cruz biotechnologies and used at the following dilutions (c-cbl, 1∶5000; c-met, 1∶5000; egfr, 1∶5000; ubiquitin, 1∶1000; ha, 1∶5000 and β-actin, 1∶10,000).  flow cytometry  </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> cycle analysis was carried out by flow cytometry. approximately 2×106 </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were grown in media containing 10% fbs. </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were harvested by trypsin/edta treatment, washed with 1x pbs three times and fixed with ice-cold 70% ethanol for 2 h. </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were washed again with cold pbs and stained with a solution containing 25 µg/ml propidium iodide, 200 µg/ml rnase a, and 0.1% triton x-100 for 30 minutes in the dark. </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> cycle analysis was performed using a guava pca-96 flow cytometer (guava technologies, millipore, billerica, ma).  ubiquitin ligase activity  293t </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were maintained in culture in dmem supplemented with 10% fbs and 1% penicillin (100 units/ml) and streptomycin (100 µg/ml) were transfected with 0.2 µg egfr-pcdna3 and 2 µg ha-tagged c-cbl constructs as indicated using calcium phosphate according to manufacturer&#x27;s protocol (profection, promega, madison, wi). twenty-four hours post-transfection, </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were starved overnight in dmem supplemented with 0.5% fbs, and then treated with or without egf (100 ng/ml) for 15 min. the </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were collected and washed two times in ice-cold pbs containing 0.2 mm sodium orthovanadate then lysed in ice-cold lysis buffer (10 mm tris hcl, ph 7.5, 150 mm nacl, 5 mm edta, 1% triton x100, 10% glycerol, 2 mm sodium orthovanadate and protease inhibitors). lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4°c. egfr immunoprecipitations were performed on 200 µg of cleared lysate using 250 ng of rabbit-anti-egfr and </span><span style=\"background-color: hsl(120, 100.00%, 98.56%); opacity: 0.80\" title=\"0.008\">protein</span><span style=\"opacity: 0.80\"> a/g plus sepharose overnight at 4°c. precipitations were washed 5 times in lysis buffer before boiling in laemmli buffer. elutions were immunoblotted with anti-ubiquitin and egfr. twenty micrograms of cleared lysate were immunoblotted for each of the c-cbl constructs using anti-ha.  statistical analysis  mutation rates between different groups were compared using fisher&#x27;s exact </span><span style=\"background-color: hsl(120, 100.00%, 97.38%); opacity: 0.80\" title=\"0.018\">test</span><span style=\"opacity: 0.80\">. for continuous variables, group comparisons were performed using analysis of variance (anova) followed by sidak&#x27;s adjustment for multiple comparisons. experiments involving repeated measurements over time were analyzed using repeated measures anova with the greenhouse-geisser adjustment to the degrees of freedom. analyses were conducted using stata (v10.1) software (stata corporation, college station, tx).  go to: results c-cbl gene mutations in lung cancer  to investigate the role of c-cbl in lung cancer, we analyzed its genomic dna in tumor and paired normal samples drawn from multiple ethnicities. the lung tumor samples represented caucasians (n = 50), african-americans (n = 29), and taiwanese (n = 40) lung cancer patients. we designed </span><span style=\"background-color: hsl(0, 100.00%, 98.98%); opacity: 0.80\" title=\"-0.005\">12</span><span style=\"opacity: 0.80\"> pairs of primers to sequence the coding region of c-cbl gene that spans exons 2 to 16 (table s2). we identified 8 unique somatic mutations in c-cbl exons among 8 different patients. a variation l620f, a known snp (rs2227988) in exon 11 was also detected. importantly, the eight novel non-synonymous mutations were confirmed by sequencing both strands of c-cbl genomic dna obtained from lung tumor samples (table 1). moreover, none of the 8 mutations were detected in the corresponding normal tissue, indicating that these were somatic mutations. four synonymous single nucleotide variations (snvs) were also identified but were not used further in this study.  table 1 table 1 c-cbl mutation analysis in 119 lung cancer patient tumor tissues. three of the 8 novel non-synonymous mutations were located in the tkb (tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.013\">kinase</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.001\">binding</span><span style=\"opacity: 0.80\">) domain (s80n, h94y, and q249e), one in the ring finger domain (v391i), one in the proline-rich region (72515_72517 del atg), and three in the c-terminal region (w802*, r830k, and a848t) of the c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.56%); opacity: 0.80\" title=\"0.008\">protein</span><span style=\"opacity: 0.80\"> (figure 1a and figure s1). in figure 1b, we show model chromatograms of representative samples.  figure 1 figure 1 c-cbl mutations and loh in non-small </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lung cancer. 11q loh of c-cbl gene  paired lung tumor and normal lung tissue samples from taiwanese patients (n = 37) were investigated for loh. eight (21.6%) showed loh at the c-cbl locus on chromosome 11 while 29 samples (78.4%) revealed normal allelic contribution at the microsatellite markers (figures 1c and d).  c-cbl mutations in different ethnic groups  the c-cbl double mutant s80n/h94y was found in the same patient and the overall mutation rate for c-cbl in lung tumors was 6.7% (8/119). the frequency of c-cbl mutation was highest in large </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the least was observed in adenocarcinoma (ad) (1.8%; 1 of 53 patients), although these rates were not statistically significant (p = 0.292). mutation rates were 6.0% among caucasians (0 of 20 in ad; 0 of 10 in sq; and 3 of 20 in lc), 13.8% in african-americans (1 of 10 in ad; 1 of 10 in sq; and 2 of 9 in lc), and 2.5% (0 of 23 in ad; 1 of </span><span style=\"background-color: hsl(0, 100.00%, 98.98%); opacity: 0.80\" title=\"-0.005\">12</span><span style=\"opacity: 0.80\"> in sq; and 0 of 5 in lc) in the taiwanese population. additionally two taiwanese patients with lung cancer (one squamous and one adenocarcinoma) had the known snp l620f. ethnic differences were not statistically significant, however the power to detect differences was low.  mutations in met and egfr can be co-associated with c-cbl alterations  since east asians with lung cancer have a higher frequency of egfr and met mutations in lung tumors [21], [22], we also determined mutations in egfr and met in the same taiwanese cohort samples and compared the results with the observed c-cbl alterations (loh and/or mutations). in the 37 samples </span><span style=\"background-color: hsl(0, 100.00%, 99.21%); opacity: 0.80\" title=\"-0.003\">tested</span><span style=\"opacity: 0.80\">, we did not find any overlap between c-cbl mutations and c-cbl loh (figure 2). of the three c-cbl mutants (</span><span style=\"background-color: hsl(0, 100.00%, 97.96%); opacity: 0.80\" title=\"-0.012\">including</span><span style=\"opacity: 0.80\"> the known l620f snp, rs2227988), one of the samples had a met mutation (n375s) and the other had an egfr mutation (l858r). among the 8 samples that had a loh at the c-cbl locus 5 had an additional mutation in met (n375s) and 2 had an egfr exon 19 deletion. twenty-six samples had neither c-cbl mutation nor c-cbl loh (3 patients had a c-cbl mutation but no c-cbl loh). among these 26 samples 9 had a met mutation (8 n375s, 1 l211w), 13 had an egfr mutation (7 exon 9 deletion, 6 l858r) and 4 had no other met or egfr mutation. thus the rate of met or egfr mutations among patients with loh at the c-cbl locus (7 of 8) was similar to that seen in patients without c-cbl mutation or loh (22 of 26 patients) (p = 0.99). these 4 patients with no identifiable mutation in c-cbl, met or egfr represented 10.8% of the 37 patients analyzed in the taiwanese patient cohort. conversely, 89.2% taiwanese lung cancer patients have an identifiable mutation in either c-cbl, met or egfr or a combination of the three genes (figure 2). additionally, we determined p53 and kras mutations in these taiwanese cohorts. two p53 and 1 kras mutation were detected. the single kras mutation overlapped with one p53 mutation. this patient also had the egfr exon 19 deletion but had no c-cbl mutation. the other p53 mutation sample had a c-cbl loh with concurrent met n375s mutation. thus, in the taiwanese samples analyzed, p53/kras mutations and c-cbl mutations were mutually exclusive (data not shown).  figure 2 figure 2 c-cbl mutations and relationship to met and egfr mutations in lung cancer. cellular </span><span style=\"background-color: hsl(0, 100.00%, 97.72%); opacity: 0.80\" title=\"-0.015\">functions</span><span style=\"opacity: 0.80\"> of c-cbl alterations in the context of lung tumorigenesis  a. e3 activity is intact in the mutant c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.80%); opacity: 0.80\" title=\"-0.000\">proteins</span><span style=\"opacity: 0.80\"> to investigate whether the different c-cbl mutations </span><span style=\"background-color: hsl(0, 100.00%, 98.36%); opacity: 0.80\" title=\"-0.009\">affect</span><span style=\"opacity: 0.80\"> the e3 activity, egfr was chosen as a model substrate for c-cbl e3 </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.005\">function</span><span style=\"opacity: 0.80\">. all of the c-cbl mutants </span><span style=\"background-color: hsl(0, 100.00%, 99.21%); opacity: 0.80\" title=\"-0.003\">tested</span><span style=\"opacity: 0.80\"> enhanced ubiquitination of the </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.014\">activated</span><span style=\"opacity: 0.80\"> egfr similar to the wild-type c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.56%); opacity: 0.80\" title=\"0.008\">protein</span><span style=\"opacity: 0.80\">. this result demonstrates that the catalytic activity of the c-cbl mutants is not impaired when egfr was the substrate. (figure 3a).  figure 3 figure 3 ubiquitination, viability, expression and </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> cycle analysis of various c-cbl mutants. b. effect on lung cancer </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> viability the effect of a representative c-cbl mutant from each of the three ethnic backgrounds on lung cancer </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> viability in </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lines was determined. s80n/h94y double mutation, q249e, and w802* were identified in lung tumor samples obtained from a caucasian, a taiwanese and an african-american, respectively. as described in methods, the c-cbl wild-type (wt) and the above three mutants were expressed after cloning them into paltermax vector in a549 </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\">. these </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> express relatively low basal levels of endogenous c-cbl (data not shown). transfection efficiency was comparable between different groups and the number of </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> transfected with c-cbl wild-type construct was about 70% compared to </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">control</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> that were transfected with the empty vector. </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> transfected with s80n/h94y, q249e and w802* c-cbl mutant constructs resulted in increased number of viable </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> that was 132.3%, 120.8% and 147.9% higher respectively, relative to the empty vector </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">control</span><span style=\"opacity: 0.80\"> transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> and significantly different from the wild-type construct (p = 0.022, p = 0.049, and p = 0.008, respectively) (figure 3b). relative levels of c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.56%); opacity: 0.80\" title=\"0.008\">protein</span><span style=\"opacity: 0.80\"> in whole </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lysates prepared from samples obtained from a parallel experiment was determined. the c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.56%); opacity: 0.80\" title=\"0.008\">protein</span><span style=\"opacity: 0.80\"> levels in samples representing untransfected and empty vector transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were comparable and those representing the c-cbl wt and the three c-cbl mutants were comparable (figure 3c).  c. effect on </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> cycle to investigate if the increases in </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> viability in different c-cbl mutants are due to increased cellular proliferation, a </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> cycle analysis was performed. a549 </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were transfected with the c-cbl wt or the three different mutants: s80n/h94y, q249e and w802*. the empty vector transfectant was used as a </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">control</span><span style=\"opacity: 0.80\">. forty-eight hours after transfection </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> cycle analysis was performed as described in materials and methods. there was no significant change in the subg1, g1 or the s phase of the </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> cycle among the different mutants compared to the wt construct (p = 0.64, p = 0.40, and p = 0.28, respectively). the g2/m phase of the </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> cycle showed an </span><span style=\"background-color: hsl(0, 100.00%, 96.73%); opacity: 0.81\" title=\"-0.024\">increase</span><span style=\"opacity: 0.80\"> in </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> numbers for the three mutants, s80n/h94y, q249e and w802*, when compared to the wt but again the difference was not statistically significant (p = 0.25) (figure 3d).  d. effect on </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> motility to investigate the effect of the expression of the above three c-cbl mutants on </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> migration, we carried out wound healing assay as described in materials and methods. the closing of the scratch or the wound was monitored at 0, </span><span style=\"background-color: hsl(0, 100.00%, 98.98%); opacity: 0.80\" title=\"-0.005\">12</span><span style=\"opacity: 0.80\">, 24, 36, and 48 h. (figure 4a). in all the samples, that represented </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> transfected with mutants, the wound gap was much smaller than that seen in the sample that represented </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> transfected with c-cbl wt (p&lt;0.001). we also determined the rate of wound closure for all the five groups. at 48 h, wild-type c-cbl transfectants showed 61.1% open wound while the s80n/h94y, q249e and w802* mutants showed 18.7%, 23.9% and 34.3% open wound respectively (p&lt;0.001) (figure 4b).  figure 4 figure 4 c-cbl mutations </span><span style=\"background-color: hsl(0, 100.00%, 98.36%); opacity: 0.80\" title=\"-0.009\">affect</span><span style=\"opacity: 0.80\"> wound healing in a549 </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\">. e. c-cbl knockdown increases </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> viability it is hypothesized that the loh seen in our samples could lead to decreased expression of c-cbl. thus we </span><span style=\"background-color: hsl(0, 100.00%, 99.21%); opacity: 0.80\" title=\"-0.003\">tested</span><span style=\"opacity: 0.80\"> the effect of c-cbl knockdown in lung cancer </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\">. compared to a549, h358 lung cancer </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> express relatively high levels of endogenous c-cbl (data not shown). c-cbl expression was knocked down using lentiviral construct that expressed c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 85.55%); opacity: 0.85\" title=\"-0.203\">specific</span><span style=\"opacity: 0.80\"> shrna and compared the results with those that were transduced with scrambled shrna and the results are shown in figure 5. we identified several clones that revealed varying degrees of c-cbl knockdown showing different sets of c-cbl lentiviral shrna knockdown efficiency (figure 5a). of all the clones </span><span style=\"background-color: hsl(0, 100.00%, 99.21%); opacity: 0.80\" title=\"-0.003\">tested</span><span style=\"opacity: 0.80\">, clone 27 was chosen for further experiments. equal amount of </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> were seeded in a 6-well plate and the </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> proliferation was measured at various times and the results are depicted in figure 5b. as expected, number of </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> increased in a time dependent fashion from 100 to 190% relative to scrambled shrna as </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.003\">control</span><span style=\"opacity: 0.80\"> in a span of 48 h (p = 0.0002) (figure 5b). the </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> cycle phases in h358 </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> that were knocked down with c-cbl shrna were looked at and compared with the scrambled shrna. there were no discernable differences between these two constructs in the different phases of the </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> cycle (data not shown).  figure 5 figure 5 knockdown of c-cbl using an shrna increases </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> proliferation. go to: discussion our results demonstrate that c-cbl is somatically mutated (or has loh) in lung cancers, and can significantly </span><span style=\"background-color: hsl(120, 100.00%, 97.30%); opacity: 0.80\" title=\"0.019\">contribute</span><span style=\"opacity: 0.80\"> to enhanced </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> viability and motility. there was also a high prevalence of loh with respect to c-cbl in lung tumors that harbored met or egfr mutation.  in the present study, we have demonstrated the occurrence of c-cbl mutations in lung cancer patients, especially with different ancestral variations. mutations in c-cbl have been recently reported in juvenile myelomonocytic leukemia and myeloid malignancies. in the aml study, the mutation r420q located in the junction of the ring finger and the linker region inhibited fms-like tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.013\">kinase</span><span style=\"opacity: 0.80\"> 3 (flt3) internalization and ubiquitination [20], thus contributing to the gain-in-</span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.005\">function</span><span style=\"opacity: 0.80\"> for the rtk. in addition, mutations such as h398y, c384r, and l380p mapped to the ring finger domain and the linker region of c-cbl that is required for its e3 activity [23], [24], [25], [26], [27]. additionally, homozygous mutations in the ring finger domain of the c-cbl gene were described as a result of acquired uniparental disomy (upd) [26]. it is important to note that our results indicate loh at 11q23 locus and these are mutually exclusive from </span><span style=\"background-color: hsl(120, 100.00%, 96.80%); opacity: 0.81\" title=\"0.024\">missense</span><span style=\"opacity: 0.80\"> mutations of c-cbl. the somatic mutations were all heterozygous. the mutations in aml led to abrogation of the e3 activity </span><span style=\"background-color: hsl(120, 100.00%, 98.65%); opacity: 0.80\" title=\"0.007\">leading</span><span style=\"opacity: 0.80\"> to prolonged rtk </span><span style=\"background-color: hsl(0, 100.00%, 96.91%); opacity: 0.81\" title=\"-0.022\">activation</span><span style=\"opacity: 0.80\">. in addition mutants located on the linker region surrounding the ring finger domain exhibited enhanced akt signaling in response to cytokine stimulation [26]. in addition, it was shown in nh3t3 </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\">, that neither mutations in the ring finger nor the linker region causes transformation, however while certain mutations perturbs the ubiquitination, others </span><span style=\"background-color: hsl(0, 100.00%, 98.36%); opacity: 0.80\" title=\"-0.009\">affect</span><span style=\"opacity: 0.80\"> receptor recycling and prolong </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.013\">kinase</span><span style=\"opacity: 0.80\"> activity [28].  we report here c-cbl mutations that mapped not only to the ring finger domain, but also to the tkb domain, proline-rich domain and the c-terminal region, but none mapped to the linker region as reported in the aml studies described above [23], [24], [25], [26], [29]. in addition, 8 mutants that we detected were found in different ethnic backgrounds. for example, s80n/h94y, q249e, w802* were detected in caucasians, taiwanese and african-americans, respectively. the results point out not only the difference between lung cancer and other cancers, but also genetic polymorphism among different races in the same cancer. interestingly, there is a large disparity between african-american and other ethnic populations with lung cancer [30]. we have previously shown that there was a low frequency of egfr and met mutation in african-americans as compared with taiwanese and caucasians [31]. in this study, the number of african-american samples analyzed was relatively fewer and we found 3 mutations that are unique to this ethnicity. it would now behoove us to further study the genetic alterations that can occur and determine the targeted therapeutics for african-americans. our results provide evidence of the importance of c-cbl in tumorigenesis and potential signaling. our prediction, based on the aml data, would be that the v391i ring finger domain mutation would </span><span style=\"background-color: hsl(0, 100.00%, 98.36%); opacity: 0.80\" title=\"-0.009\">affect</span><span style=\"opacity: 0.80\"> the e3 activity. also, it will be important to determine the </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.001\">binding</span><span style=\"opacity: 0.80\"> partners for c-cbl in the tkb domain and proline rich domain mutations. it has previously been shown that the tkb domain can bind to growth factor receptors and it will be important to determine the cross-</span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.001\">binding</span><span style=\"opacity: 0.80\"> of these mutants to met and or egfr. it would also be important in the future to look at fluorescence in situ hybridization/copy number changes in c-cbl in lung cancer.  c-cbl plays an important role in down regulating rtk-mediated signaling through k63 poly-ubiquitination and subsequent downregulation of rtks followed by lysosomal degredation [3]. mono-ubiquitination or ubiquitinated with k63-linked chains of substrates by c-cbl may lead to enhancement of biological and biochemical </span><span style=\"background-color: hsl(0, 100.00%, 97.72%); opacity: 0.80\" title=\"-0.015\">functions</span><span style=\"opacity: 0.80\"> (reviewed in hermann et al, 2007 [32]). the mutations that we analyzed in our studies all point out to the fact that the e3 activity of c-cbl on egfr is intact; the egfr levels in the various mutants remain same (figure s2). multiple kinases, both rtks and non-rtks could be acted upon by c-cbl, </span><span style=\"background-color: hsl(0, 100.00%, 97.96%); opacity: 0.80\" title=\"-0.012\">including</span><span style=\"opacity: 0.80\"> erbs, pdgfr, fms, met, c-kit, vegfr, flt-1, ron, fgfr, ir, as well as syk, fyn, lck, fgr, lyn and c-abl [3]. in lung cancers the relevant substrates of c-cbl in terms of degradation or signal transduction are yet to be identified.  the observation that c-cbl somatic mutations, especially s80n/h94y, q249e and w802* showed increased </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> viability and </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> motility that is in agreement with the physiological role for d-cbl in the regulation of apoptosis and differentiation identified in drosophila is very significant [33]. it has been previously shown that </span><span style=\"background-color: hsl(120, 100.00%, 94.70%); opacity: 0.81\" title=\"0.048\">activating</span><span style=\"opacity: 0.80\"> c-cbl mutation downregulates egfr signaling and decreases cellular proliferation and migration in breast cancer </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> lines [34]. although the role of c-cbl in the negative regulation of rtks is well substantiated, thereby suggesting that it is a natural tumor suppressor, studies in cancer </span><span style=\"background-color: hsl(120, 100.00%, 99.87%); opacity: 0.80\" title=\"0.000\">cells</span><span style=\"opacity: 0.80\"> have revealed both tumor suppressor and tumor promoting activities depending on the type of c-cbl mutation and the number of alleles at the c-cbl locus [24]. in agreement with the above, the three c-cbl mutants described here appear to have tumor growth and metastasis promoting properties. although these mutants are outside of the ring finger or the linker region of c-cbl, their downstream effects are significant so as to cause increased proliferation and migration, but the substrate affected by these mutations are not known yet. this raises the possibility that some of the cellular </span><span style=\"background-color: hsl(0, 100.00%, 97.72%); opacity: 0.80\" title=\"-0.015\">functions</span><span style=\"opacity: 0.80\"> of c-cbl are independent of its ubiquitin-ligase activity, an area that we are currently investigating. the oncogenic nature of rtks, addiction of cancers to growth signals and given the clustering of c-cbl, egfr and met mutations, it is possible that the </span><span style=\"background-color: hsl(0, 100.00%, 98.76%); opacity: 0.80\" title=\"-0.006\">transforming</span><span style=\"opacity: 0.80\"> effect of c-cbl mutations is most likely a combinatorial effect of the three. we also show that loh for c-cbl was found in a significant number of samples that harbored met or egfr mutations. the fact that about 7% of lung tumor samples are likely to have c-cbl mutations and an additional 22% are likely to harbor c-cbl-related loh, makes c-cbl a highly mutated molecule in lung cancer. since loh alone is not enough to cause a </span><span style=\"background-color: hsl(0, 100.00%, 98.76%); opacity: 0.80\" title=\"-0.006\">transforming</span><span style=\"opacity: 0.80\"> event [35], [36], [37], associated mutation in the met or egfr locus or yet another rtk discussed above may play a role in carcinogenesis. we </span><span style=\"background-color: hsl(0, 100.00%, 97.50%); opacity: 0.80\" title=\"-0.017\">predict</span><span style=\"opacity: 0.80\"> that this loh in c-cbl results in haploinsufficiency that downplays rtk ubiquitination </span><span style=\"background-color: hsl(120, 100.00%, 98.65%); opacity: 0.80\" title=\"0.007\">leading</span><span style=\"opacity: 0.80\"> to hyperactivity of the rtks. however, whether this is sufficient cause for tumorigenesis remains to be determined. consistent with our hypothesis is the fact that c-cbl-/- mice have increased </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.013\">kinase</span><span style=\"opacity: 0.80\"> activity in lymphocytes, but is not sufficient for tumor formation [35], [36], [37]. c-cbl loh could also lead to increased expression of c-cbl from the other allele to compensate for a </span><span style=\"background-color: hsl(0, 100.00%, 96.95%); opacity: 0.81\" title=\"-0.022\">loss</span><span style=\"opacity: 0.80\"> of an allele. alternately, there could be some form of synergy working with reduced c-cbl levels and mutated receptors that exacerbate the phenotype of each alone.  previous studies from our lab and others have shown that east asians with lung cancers have relatively high frequencies of gain-of-</span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.005\">function</span><span style=\"opacity: 0.80\"> of mutations in rtks such as egfr and met [31]. in a cohort of japanese patients an </span><span style=\"background-color: hsl(120, 100.00%, 94.70%); opacity: 0.81\" title=\"0.048\">activating</span><span style=\"opacity: 0.80\"> met mutation was identified in the splice region that deletes the juxtamembrane domain that is </span><span style=\"background-color: hsl(0, 100.00%, 97.73%); opacity: 0.80\" title=\"-0.014\">involved</span><span style=\"opacity: 0.80\"> e3 activity of c-cbl [38]. this study also found that </span><span style=\"background-color: hsl(0, 100.00%, 96.91%); opacity: 0.81\" title=\"-0.022\">activation</span><span style=\"opacity: 0.80\"> of met is mutually exclusive of egfr, kras and her2 gene mutations [38]. we failed to detect such mutations in significant numbers in lung tumor samples obtained from african-americans (n = 29) and caucasian (n = 50) patients. one met mutation was identified in each of the groups whereas 1 and 3 egfr mutations were identified in the african-american and caucasian cohorts respectively. egfr mutations have earlier been identified as one of the </span><span style=\"background-color: hsl(0, 100.00%, 99.52%); opacity: 0.80\" title=\"-0.002\">key</span><span style=\"opacity: 0.80\"> mutations affecting lung adenocarcinoma patients in a comprehensive study of 188 patients [39]. our study encompasses different histologies of nsclc. however, the published series did not find any mutations in c-cbl or met </span><span style=\"background-color: hsl(120, 100.00%, 97.19%); opacity: 0.80\" title=\"0.020\">unlike</span><span style=\"opacity: 0.80\"> our study that encompassed different subtypes of nsclc. it is important to note, we have recently shown that met mutations in lung cancer are in majority germline [31]. we have reported earlier c-cbl mutations in a small cohort of taiwanese lung cancer samples [40]. in our efforts to understand the ethnic differences in the lung oncogenome, we also looked at pax transcription factors such as pax5 and pax8 that are highly expressed in lung cancers; however there was no preferential expression or mutations of the above genes in lung tumor samples of african-americans. in this study, we show relatively high frequency of c-cbl mutations in lung cancers, especially in the large </span><span style=\"background-color: hsl(120, 100.00%, 99.92%); opacity: 0.80\" title=\"0.000\">cell</span><span style=\"opacity: 0.80\"> type among caucasians and particularly among african-americans. we therefore propose c-cbl as an efficacious target for lung cancers in african-americans that needs to be further substantiated. this is all the more important because the prognosis for african-americans with lung cancer, especially for men is much poorer compared to their caucasian counterparts [41].  in conclusion, the results presented in this study demonstrate that c-cbl is frequently mutated or even lost in lung cancers. our results support a role for c-cbl mutants that are independent of its ubiquitination activity. given the relatively high mutation rates in c-cbl as well as rtks such as met and egfr, it is likely that their combined effect could be synergistic in promoting tumorigenesis.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=8\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.001</b>, score <b>-4.734</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.95%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__marshall\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ccnb3\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__xp11\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__clai\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__polycomb\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__trastuzumab\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stats\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nested\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pakt1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__haploinsufficient\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__sterically\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__c9\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__phalloidin\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pharmaceuticals\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pairing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cancerres\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__metabolite\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__aacrjournals\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__max\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__glioblastomas\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__asp\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__preliminary\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__quadrupole\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__tukey\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__aggregated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nad\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__produces\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__banding\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__biosynthesis\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__r132h\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__canonical\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__overexpressing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__completed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__dataset\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__h3k36\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__called\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__consideration\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__kdm2b\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__q12\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__promyelocytic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__lesion\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.56%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__donkey\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.52%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__glioma\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.41%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.022\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.41%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.022\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2009\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.01%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.046\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__3t3\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.052\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2d\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.053\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__transcripts\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.055\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__collections\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.057\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__deep\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.065\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__accumulate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.073\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2b\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.077\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__subunit\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.078\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__trials\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.27%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.103\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__chimeric\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.33%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.192\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__di\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.13%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.212\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__describe\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 80.00%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -3.398\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">w802*</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">cbl</span><br/><b>Text:</b> <span style=\"opacity: 0.80\"> abstract background  non-small cell lung cancer (nsclc) is a heterogeneous group of disorders with a number of genetic and proteomic alterations. c-cbl is an e3 ubiquitin ligase and adaptor molecule important in normal homeostasis and cancer. we determined the genetic variations of c-cbl, relationship to receptor tyrosine kinases (egfr and met), and functionality in nsclc.  methods and findings  using archival formalin-fixed paraffin embedded (ffpe) extracted genomic dna, we show that c-cbl mutations occur in somatic fashion for lung cancers. c-cbl mutations were not mutually exclusive of met or egfr mutations; however they were independent of p53 and kras mutations. in normal/tumor pairwise analysis, there was significant loss of heterozygosity (loh) for the c-cbl locus (22%, n = 8/37) and none of these samples revealed any mutation in the remaining copy of c-cbl. the c-cbl loh also positively correlated with egfr and met mutations observed in the same samples. using select c-cbl somatic mutations such as s80n/h94y, q249e and w802* (obtained from caucasian, taiwanese and african-american samples, respectively) transfected in nsclc cell lines, there was increased cell viability and cell motility.  conclusions  taking the overall mutation rate of c-cbl to be a combination as somatic missense mutation and loh, it is clear that c-cbl is highly mutated in lung cancers and may play an essential role in lung tumorigenesis and metastasis.  go to: introduction in the us alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which more than 145,000 of them succumb to the disease [1]. this number is roughly equivalent to the combined mortality rates of cancers of the breast, prostate, colon, liver, kidney and melanoma [1]. in addition the prognosis is usually poor and the five-year survival rate is less than 15%. there are also significant ethnic differences for lung cancer, and the outcome is worse for blacks compared to whites. gender differences are also striking with women having significantly better prognosis as compared to men. there are a number of genetic alterations that can occur in lung cancer. as an example, in nsclc, mutations in kras, p53, egfr and met have been identified. many of these pathways, especially receptor tyrosine kinases (rtks) are controlled by c-cbl.  cbl (casitas b-lineage lymphoma) is a mammalian gene located on human chromosome 11q23.3 [2] and is involved in cell signaling and protein ubiquitination [3]. cbl proteins belong to the ring finger class of ubiquitin ligases (e3) and there are three homologues c-cbl, cbl-b, cbl-3 [4]. the c-cbl and cbl-b genes are ubiquitously expressed with the highest levels in hematopoietic tissues [5]. c-cbl consists of four regions encoding for functionally distinct protein domains: the n-terminal tyrosine kinase binding (tkb) domain, the linker region, the </span><span style=\"background-color: hsl(0, 100.00%, 99.42%); opacity: 0.80\" title=\"-0.002\">catalytic</span><span style=\"opacity: 0.80\"> ring finger domain, the proline-rich region and the c-terminal ubiquitin-associated (uba) domain that also overlaps with a leucine-zipper (lz) domain [3]. both tkb and ring finger domains are essential for ligand-induced ubiquitination of rtks [6], [7], [8], [9]. the ring finger domain is required for the recruitment of e2 ubiquitin-conjugating enzymes. the tkb domain includes four-helix bundle (4h), a calcium-biding ef hand, and a modified sh2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. in addition, the proline-rich region of c-cbl can associate with the sh3 domain of grb2, which can indirectly recruit c-cbl to rtks via the grb2 adaptor protein [7], [13], [14].  c-cbl also binds to egfr and acts as the e3 that targets egfr for ubiquitination and degradation. furthermore, cbl desensitizes egf signaling and opposes cellular proliferation induced by egf [15]. egf activation also appears to activate the tyrosine kinase src, which phosphorylates c-cbl and in turn activates the ubiquitination and degradation of egfr [16], [17], [18]. a recent study shows that defective endocytosis of egfr is characterized by a deletion mutant and the point mutation l858r, whereby its association with c-cbl and subsequent ubiquitination are impaired [19]. recently, the first human c-cbl mutations were reported in acute myeloid leukemia (</span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">aml</span><span style=\"opacity: 0.80\">) patients [20]. the mutation r420q inhibits fms-like tyrosine kinase 3 (flt3) internalization and ubiquitination [20].  not only can e3 activity be important in oncogenesis, c-cbl has a dual but separate function as a signal transduction molecule. we have previously shown that c-cbl is important in binding crkl and bcr/abl in hematopoietic cells. also, it can bind and modulate functions of cytoskeleton by binding to proteins like talin and paxillin. the tkb domain is important in binding to a number of molecules, and they then function in signal transduction.  given the critical role of cbl in normal homeostasis and cancer, we hypothesized that it might be mutated in lung cancers. in this study, we report novel c-cbl somatic mutations s80n/h94y, q249e and w802* in caucasian, taiwanese and african-american lung cancer patients, respectively. expressing these mutations in nsclc cell lines lead to increased proliferation and cell motility. we show that c-cbl mutations occur with or without met or egfr mutations but are mutually exclusive of a loh at the c-cbl locus. additionally, c-cbl loh is associated with either met or egfr mutations. we thus hypothesize that c-cbl mutations might contribute to the oncogenic potential of met and egfr in lung cancer.  go to: methods ethics statement  written consent on all research on human subjects has been obtained from the institutional review board, university of chicago and covers all research performed in the laboratory. the following is their contact information: institutional review board, the university of chicago, mcgiffert hall, 5751 s. woodlawn ave., 2nd floor, chicago, il 60637. written informed consents were received from all patients whose tissue samples were used for this study.  tissue samples  lung cancer tissue and paired adjacent normal lung tissues were obtained from 50 caucasian, 29 african-americans and 40 taiwanese nsclc patients who were recruited at the university of chicago hospital (chicago, usa) (caucasian and african-american patients) and taipei veterans general hospital of taiwan (taiwanese patients) after obtaining appropriate institutional review board permission and informed consent from the patients. out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 years. in terms of tumor types, 53 were adenocarcinoma, 32 were squamous cell carcinoma and 34 were large cell carcinoma. 49 were stage i, 14 were stage ii, 34 were stage iii, and 13 were stage iv (table s1).  cell culture  human non-small cell lung carcinoma cells a549 and h358 were maintained in dmem and rpmi-1640, respectively. human embryonic kidney 293t cells were cultured in dmem. media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (invitrogen, carlsbad, ca). cells were cultured at 37°c in a humidified incubator containing 5% co2.  c-cbl gene mutational analysis  exons 2 to 16 of c-cbl gene were individually amplified by polymerase chain reaction (pcr). primers are listed in table s2. pcr conditions were 1 cycle of 95°c for 5 minutes; 35 cycles of 94°c for 30 seconds, 58°c for 30 seconds and 72°c for 2 minutes; and one cycle of 72°c for 10 minutes. pcr products were treated with exosap-it (usb corporation, cleveland, oh) and sequenced by big-dye terminator chemistry (applied biosystems, foster city, ca). sequencing was performed on the forward coding strand with confirmation of c-cbl alterations performed by sequencing the reverse strand as well. chromatograms were analyzed for mutations using mutation surveyor v2.61 (softgenetics, state college, pa).  plasmid constructs and site-directed mutagenesis  the wild-type c-cbl cdna insert was subcloned into the paltermax expression vector using xhoi and sali restriction enzyme sites (promega, madison, wi). using this parental plasmid paltermax-c-cbl, the tkb domain double mutation (s80n/h94y), the point mutation (q249e), and the c-terminal point mutation w802* of c-cbl were created using the following primers: 5′-gctggcgctaaagaataacccaccttatatcttagac-3′ and 5′-ctaccagatacctaccagtatctccgtactatcttgtc-3′ for the double mutation s80n/h94y; 5′-ctttacccgactctttgagccctggtcctctttgc-3′ for q249e, and 5′-cagctcctcctttggctgattgtctctggatggtgatc-3′ for w802* along with their complementary primers using the quickchange site-directed mutagenesis xl kit (stratagene, la jolla, ca) according to the manufacturer&#x27;s instructions. the constructs were confirmed for the point mutations by standard dna sequencing of both strands.  loss of heterozygosity (loh) analysis  five microsatellites on chromosome 11 (3 on 11q at or within 200 kb up or downstream of the c-cbl gene and 2 control markers on 11p) were selected for analysis (table s3). established microsatellite markers and respective primer sequences were selected from the geneloc database (http://genecards.weizmann.ac.il/geneloc/index.shtml, weizmann institute of science, rehovot, israel). primers were custom designed and each forward primer was fluorescently labeled at the 5′ end with fam, pet, ned, or vic (applied biosystems). primer annealing temperatures and duplex scores were evaluated with nist primer tools (http://yellow.nist.gov:8444/dnaanalysis/primertoolspage.do; national institute of standards and technology, gaithersburg, md). primers were verified by performing pcr with control dna (</span><span style=\"background-color: hsl(0, 100.00%, 96.24%); opacity: 0.81\" title=\"-0.030\">isolated</span><span style=\"opacity: 0.80\"> from tk6 cells) and resolving the products on agarose gels. bands were visualized with an uv transilluminator. genomic dna was extracted from tumor samples and paired normal lung tissue. primers were grouped into multiplex combinations shown in table s4. marker d11s929 served as an internal control to check for consistency in pcrs and of peaks from capillary electrophoresis. multiplex pcrs were carried out in a volume of 10 µl that contained 1 µl genomic dna (20–50 ng), 0.5 µm of each primer (1.0 µm total for each primer pair), 400 µm dntps, 1x pcr buffer containing mgcl2, and 0.2 u taq dna polymerase. pcr was performed on the abi geneamp 9700 pcr system under the following conditions: 5 min at 94°c; 30 cycles of 30 sec at 94°c, 1 min at 60°c, 1 min at 72°c; and 5 min at 72°c. the pcr products were separated by capillary electrophoresis on an abi 3130xl dna analyzer. chromatograms were analyzed with peak scanner 1.0 and genemapper 3.7 software (applied biosystems) for allelic alterations. the area of the peaks produced by the dna pcr products was quantified for each allele. the ratio of the allelic areas was calculated for each tumor and paired normal dna sample. when the qloh (allelic ratio for the tumor peaks divided by the allelic ratio of paired normal sample) was ≤0.5 or ≥2.0 for c-cbl and at least one other 11q marker in at least two separate experiments, the sample was considered as having an allelic imbalance and interpreted as loh. samples were evaluated in at least two separate experiments and samples showing prospective loh at c-cbl repeated a third time which included a new control marker at the bax locus (data not shown) on chromosome 19 to verify integrity of sample dna.  transfection of c-cbl constructs  the a549 cell line was transfected using the fugene hd (roche, nutley, nj) reagent according to the manufacturer&#x27;s instructions. eight µg of plasmid dna, containing either no insert (empty vector), wild-type c-cbl, s80n/h94y c-cbl, q249e c-cbl or w802* cbl was used for transfection in a 6-well culture plate. cells were harvested 48 h after transfection and analyzed for expression.  c-cbl knockdown  c-cbl knockdown was performed using lentiviral transduction using mission lentiviral transduction particles (sigma-aldrich, st. louis, mo) as per manufacturer&#x27;s instructions. briefly, 1×105 h358 cells/well were seeded in 6-well plates and infected the following day with c-cbl lentiviral shrna constructs. to generate stable c-cbl knockdown cell lines, cells were selected for 2 days with 1 µg/ml puromycin. c-cbl levels were determined using whole cell lysates by immunoblotting with anti-cbl antibody (santa cruz biotechnologies, santa cruz, ca).  cell viability assay  cells were transfected as described above in the transfection assay. forty-eight hours after transfection, viability of cells was assessed using trypan blue exclusion.  wound healing assay  a549 cells were seeded in 6-well plates and cultured for 48 h until 100% confluent. the medium was then changed and the cells were transfected as described in the transfection assay. twelve hours after transfection, a straight scratch was made across the cell layer using a 1 ml pipette tip. the cells were then gently washed with 1× pbs to remove cellular debris and the media was replaced. photographs were taken of the wound region every 12 h until 48 h.  western blot analysis  forty eight hours after transfection, cells were collected and washed twice in 1x pbs, then lysed in ice-cold lysis buffer (0.5m tris-hcl with ph 7.4, 1.5 m nacl, 2.5% deoxycholic acid, 10 mm edta, 10% np-40, 0.5 mm dtt, 1 mm phenylmethylsulfonyl fluoride, 5 µg/ml leupeptin, and 10 µg/ml aprotinin) for 5 minutes. the lysate was centrifuged at 13,000 rpm for 20 minutes at 4°c, and protein content of the supernatant was measured. total cell lysates (50 µg/well) were separated by sds-page electrophoresis and the gels transferred onto nitrocellulose membranes (whatman, piscataway, nj). membranes were </span><span style=\"background-color: hsl(0, 100.00%, 96.46%); opacity: 0.81\" title=\"-0.027\">blocked</span><span style=\"opacity: 0.80\"> with 5% non-fat dry milk in phosphate-buffered saline containing tween-20 (pbst) (1x pbs, 0.1% tween-20) for 1 h at room temperature and incubated with the appropriate primary antibody at 4°c overnight. membranes then were washed three times with pbst and probed with appropriate horseradish peroxidase (hrp)-conjugated secondary antibody for 1 h at room temperature. the membranes were again washed three times in pbst and bands were visualized using western blot chemiluminescence reagent (biorad, valencia, ca) on a chemidoc gel documentation system (biorad, valencia, ca). antibodies were obtained from santa cruz biotechnologies and used at the following dilutions (c-cbl, 1∶5000; c-met, 1∶5000; egfr, 1∶5000; ubiquitin, 1∶1000; ha, 1∶5000 and β-actin, 1∶10,000).  flow cytometry  cell cycle analysis was carried out by flow cytometry. approximately 2×106 cells were grown in media containing 10% fbs. cells were harvested by trypsin/edta treatment, washed with 1x pbs three times and fixed with ice-cold 70% ethanol for 2 h. cells were washed again with cold pbs and stained with a solution containing 25 µg/ml propidium iodide, 200 µg/ml rnase a, and 0.1% triton x-100 for 30 minutes in the dark. cell cycle analysis was performed using a guava pca-96 flow cytometer (guava technologies, millipore, billerica, ma).  ubiquitin ligase activity  293t cells were maintained in culture in dmem supplemented with 10% fbs and 1% penicillin (100 units/ml) and streptomycin (100 µg/ml) were transfected with 0.2 µg egfr-pcdna3 and 2 µg ha-tagged c-cbl constructs as indicated using calcium phosphate according to manufacturer&#x27;s protocol (profection, promega, madison, wi). twenty-four hours post-transfection, cells were starved overnight in dmem supplemented with 0.5% fbs, and then treated with or without egf (100 ng/ml) for 15 min. the cells were collected and washed two times in ice-cold pbs containing 0.2 mm sodium orthovanadate then lysed in ice-cold lysis buffer (10 mm tris hcl, ph 7.5, 150 mm nacl, 5 mm edta, 1% triton x100, 10% glycerol, 2 mm sodium orthovanadate and protease inhibitors). lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4°c. egfr immunoprecipitations were performed on 200 µg of cleared lysate using 250 ng of rabbit-anti-egfr and protein a/g plus sepharose overnight at 4°c. precipitations were washed 5 times in lysis buffer before boiling in laemmli buffer. elutions were immunoblotted with anti-ubiquitin and egfr. twenty micrograms of cleared lysate were immunoblotted for each of the c-cbl constructs using anti-ha.  statistical analysis  mutation rates between different groups were compared using fisher&#x27;s exact test. for continuous variables, group comparisons were performed using analysis of variance (anova) followed by sidak&#x27;s adjustment for multiple comparisons. experiments involving repeated measurements over time were analyzed using repeated measures anova with the greenhouse-geisser adjustment to the degrees of freedom. analyses were conducted using stata (v10.1) software (stata corporation, college station, tx).  go to: results c-cbl gene mutations in lung cancer  to investigate the role of c-cbl in lung cancer, we analyzed its genomic dna in tumor and paired normal samples drawn from multiple ethnicities. the lung tumor samples represented caucasians (n = 50), african-americans (n = 29), and taiwanese (n = 40) lung cancer patients. we designed 12 pairs of primers to sequence the coding region of c-cbl gene that spans exons 2 to 16 (table s2). we identified 8 unique somatic mutations in c-cbl exons among 8 different patients. a variation l620f, a known snp (rs2227988) in exon 11 was also detected. importantly, the eight novel non-synonymous mutations were confirmed by sequencing both strands of c-cbl genomic dna obtained from lung tumor samples (table 1). moreover, none of the 8 mutations were detected in the corresponding normal tissue, indicating that these were somatic mutations. four synonymous single nucleotide variations (snvs) were also identified but were not used further in this study.  table 1 table 1 c-cbl mutation analysis in 119 lung cancer patient tumor tissues. three of the 8 novel non-synonymous mutations were located in the tkb (tyrosine kinase binding) domain (s80n, h94y, and q249e), one in the ring finger domain (v391i), one in the proline-rich region (72515_72517 del atg), and three in the c-terminal region (w802*, r830k, and a848t) of the c-cbl protein (figure 1a and figure s1). in figure 1b, we show model chromatograms of representative samples.  figure 1 figure 1 c-cbl mutations and loh in non-small cell lung cancer. 11q loh of c-cbl gene  paired lung tumor and normal lung tissue samples from taiwanese patients (n = 37) were investigated for loh. eight (21.6%) showed loh at the c-cbl locus on chromosome 11 while 29 samples (78.4%) revealed normal allelic contribution at the microsatellite markers (figures 1c and d).  c-cbl mutations in different ethnic groups  the c-cbl double mutant s80n/h94y was found in the same patient and the overall mutation rate for c-cbl in lung tumors was 6.7% (8/119). the frequency of c-cbl mutation was highest in large cell carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the least was observed in adenocarcinoma (ad) (1.8%; 1 of 53 patients), although these rates were not statistically significant (p = 0.292). mutation rates were 6.0% among caucasians (0 of 20 in ad; 0 of 10 in sq; and 3 of 20 in lc), 13.8% in african-americans (1 of 10 in ad; 1 of 10 in sq; and 2 of 9 in lc), and 2.5% (0 of 23 in ad; 1 of 12 in sq; and 0 of 5 in lc) in the taiwanese population. additionally two taiwanese patients with lung cancer (one squamous and one adenocarcinoma) had the known snp l620f. ethnic differences were not statistically significant, however the power to detect differences was low.  mutations in met and egfr can be co-associated with c-cbl alterations  since east asians with lung cancer have a higher frequency of egfr and met mutations in lung tumors [21], [22], we also determined mutations in egfr and met in the same taiwanese cohort samples and compared the results with the observed c-cbl alterations (loh and/or mutations). in the 37 samples tested, we did not find any overlap between c-cbl mutations and c-cbl loh (figure 2). of the three c-cbl mutants (including the known l620f snp, rs2227988), one of the samples had a met mutation (n375s) and the other had an egfr mutation (l858r). among the 8 samples that had a loh at the c-cbl locus 5 had an additional mutation in met (n375s) and 2 had an egfr exon 19 deletion. twenty-six samples had neither c-cbl mutation nor c-cbl loh (3 patients had a c-cbl mutation but no c-cbl loh). among these 26 samples 9 had a met mutation (8 n375s, 1 l211w), 13 had an egfr mutation (7 exon 9 deletion, 6 l858r) and 4 had no other met or egfr mutation. thus the rate of met or egfr mutations among patients with loh at the c-cbl locus (7 of 8) was similar to that seen in patients without c-cbl mutation or loh (22 of 26 patients) (p = 0.99). these 4 patients with no identifiable mutation in c-cbl, met or egfr represented 10.8% of the 37 patients analyzed in the taiwanese patient cohort. conversely, 89.2% taiwanese lung cancer patients have an identifiable mutation in either c-cbl, met or egfr or a combination of the three genes (figure 2). additionally, we determined p53 and kras mutations in these taiwanese cohorts. two p53 and 1 kras mutation were detected. the single kras mutation overlapped with one p53 mutation. this patient also had the egfr exon 19 deletion but had no c-cbl mutation. the other p53 mutation sample had a c-cbl loh with concurrent met n375s mutation. thus, in the taiwanese samples analyzed, p53/kras mutations and c-cbl mutations were mutually exclusive (data not shown).  figure 2 figure 2 c-cbl mutations and relationship to met and egfr mutations in lung cancer. cellular functions of c-cbl alterations in the context of lung tumorigenesis  a. e3 activity is intact in the mutant c-cbl proteins to investigate whether the different c-cbl mutations affect the e3 activity, egfr was chosen as a model substrate for c-cbl e3 function. all of the c-cbl mutants tested enhanced ubiquitination of the activated egfr similar to the wild-type c-cbl protein. this result demonstrates that the </span><span style=\"background-color: hsl(0, 100.00%, 99.42%); opacity: 0.80\" title=\"-0.002\">catalytic</span><span style=\"opacity: 0.80\"> activity of the c-cbl mutants is not impaired when egfr was the substrate. (figure 3a).  figure 3 figure 3 ubiquitination, viability, expression and cell cycle analysis of various c-cbl mutants. b. effect on lung cancer cell viability the effect of a representative c-cbl mutant from each of the three ethnic backgrounds on lung cancer cell viability in cell lines was determined. s80n/h94y double mutation, q249e, and w802* were identified in lung tumor samples obtained from a caucasian, a taiwanese and an african-american, respectively. as described in methods, the c-cbl wild-type (wt) and the above three mutants were expressed after cloning them into paltermax vector in a549 cells. these cells express relatively low basal levels of endogenous c-cbl (data not shown). transfection efficiency was comparable between different groups and the number of cells transfected with c-cbl wild-type construct was about 70% compared to control cells that were transfected with the empty vector. cells transfected with s80n/h94y, q249e and w802* c-cbl mutant constructs resulted in increased number of viable cells that was 132.3%, 120.8% and 147.9% higher respectively, relative to the empty vector control transfected cells and significantly different from the wild-type construct (p = 0.022, p = 0.049, and p = 0.008, respectively) (figure 3b). relative levels of c-cbl protein in whole cell lysates prepared from samples obtained from a parallel experiment was determined. the c-cbl protein levels in samples representing untransfected and empty vector transfected cells were comparable and those representing the c-cbl wt and the three c-cbl mutants were comparable (figure 3c).  c. effect on cell cycle to investigate if the increases in cell viability in different c-cbl mutants are due to increased cellular proliferation, a cell cycle analysis was performed. a549 cells were transfected with the c-cbl wt or the three different mutants: s80n/h94y, q249e and w802*. the empty vector transfectant was used as a control. forty-eight hours after transfection cell cycle analysis was performed as described in materials and methods. there was no significant change in the subg1, g1 or the s phase of the cell cycle among the different mutants compared to the wt construct (p = 0.64, p = 0.40, and p = 0.28, respectively). the g2/m phase of the cell cycle showed an increase in cell numbers for the three mutants, s80n/h94y, q249e and w802*, when compared to the wt but again the difference was not statistically significant (p = 0.25) (figure 3d).  d. effect on cell motility to investigate the effect of the expression of the above three c-cbl mutants on cell migration, we carried out wound healing assay as described in materials and methods. the closing of the scratch or the wound was monitored at 0, 12, 24, 36, and 48 h. (figure 4a). in all the samples, that represented cells transfected with mutants, the wound gap was much smaller than that seen in the sample that represented cells transfected with c-cbl wt (p&lt;0.001). we also determined the rate of wound closure for all the five groups. at 48 h, wild-type c-cbl transfectants showed 61.1% open wound while the s80n/h94y, q249e and w802* mutants showed 18.7%, 23.9% and 34.3% open wound respectively (p&lt;0.001) (figure 4b).  figure 4 figure 4 c-cbl mutations affect wound healing in a549 cells. e. c-cbl knockdown increases cell viability it is hypothesized that the loh seen in our samples could lead to decreased expression of c-cbl. thus we tested the effect of c-cbl knockdown in lung cancer cells. compared to a549, h358 lung cancer cells express relatively high levels of endogenous c-cbl (data not shown). c-cbl expression was knocked down using lentiviral construct that expressed c-cbl specific shrna and compared the results with those that were transduced with scrambled shrna and the results are shown in figure 5. we identified several clones that revealed varying degrees of c-cbl knockdown showing different sets of c-cbl lentiviral shrna knockdown efficiency (figure 5a). of all the clones tested, clone 27 was chosen for further experiments. equal amount of cells were seeded in a 6-well plate and the cell proliferation was measured at various times and the results are depicted in figure 5b. as expected, number of cells increased in a time dependent fashion from 100 to 190% relative to scrambled shrna as control in a span of 48 h (p = 0.0002) (figure 5b). the cell cycle phases in h358 cells that were knocked down with c-cbl shrna were looked at and compared with the scrambled shrna. there were no discernable differences between these two constructs in the different phases of the cell cycle (data not shown).  figure 5 figure 5 knockdown of c-cbl using an shrna increases cell proliferation. go to: discussion our results demonstrate that c-cbl is somatically mutated (or has loh) in lung cancers, and can significantly contribute to enhanced cell viability and motility. there was also a high prevalence of loh with respect to c-cbl in lung tumors that harbored met or egfr mutation.  in the present study, we have demonstrated the occurrence of c-cbl mutations in lung cancer patients, especially with different ancestral variations. mutations in c-cbl have been recently reported in juvenile myelomonocytic leukemia and myeloid malignancies. in the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">aml</span><span style=\"opacity: 0.80\"> study, the mutation r420q located in the junction of the ring finger and the linker region inhibited fms-like tyrosine kinase 3 (flt3) internalization and ubiquitination [20], thus contributing to the gain-in-function for the rtk. in addition, mutations such as h398y, c384r, and l380p mapped to the ring finger domain and the linker region of c-cbl that is required for its e3 activity [23], [24], [25], [26], [27]. additionally, homozygous mutations in the ring finger domain of the c-cbl gene were described as a result of acquired uniparental disomy (upd) [26]. it is important to note that our results indicate loh at 11q23 locus and these are mutually exclusive from missense mutations of c-cbl. the somatic mutations were all heterozygous. the mutations in </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">aml</span><span style=\"opacity: 0.80\"> led to abrogation of the e3 activity leading to prolonged rtk activation. in addition mutants located on the linker region surrounding the ring finger domain exhibited enhanced akt signaling in response to cytokine stimulation [26]. in addition, it was shown in nh3t3 cells, that neither mutations in the ring finger nor the linker region causes transformation, however while certain mutations perturbs the ubiquitination, others affect receptor recycling and </span><span style=\"background-color: hsl(120, 100.00%, 94.96%); opacity: 0.81\" title=\"0.045\">prolong</span><span style=\"opacity: 0.80\"> kinase activity [28].  we report here c-cbl mutations that mapped not only to the ring finger domain, but also to the tkb domain, proline-rich domain and the c-terminal region, but none mapped to the linker region as reported in the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">aml</span><span style=\"opacity: 0.80\"> studies described above [23], [24], [25], [26], [29]. in addition, 8 mutants that we detected were found in different ethnic backgrounds. for example, s80n/h94y, q249e, w802* were detected in caucasians, taiwanese and african-americans, respectively. the results point out not only the difference between lung cancer and other cancers, but also genetic polymorphism among different races in the same cancer. interestingly, there is a large disparity between african-american and other ethnic populations with lung cancer [30]. we have previously shown that there was a low frequency of egfr and met mutation in african-americans as compared with taiwanese and caucasians [31]. in this study, the number of african-american samples analyzed was relatively fewer and we found 3 mutations that are unique to this ethnicity. it would now behoove us to further study the genetic alterations that can occur and determine the targeted therapeutics for african-americans. our results provide evidence of the importance of c-cbl in tumorigenesis and potential signaling. our prediction, based on the </span><span style=\"background-color: hsl(0, 100.00%, 99.60%); opacity: 0.80\" title=\"-0.001\">aml</span><span style=\"opacity: 0.80\"> data, would be that the v391i ring finger domain mutation would affect the e3 activity. also, it will be important to determine the binding partners for c-cbl in the tkb domain and proline rich domain mutations. it has previously been shown that the tkb domain can bind to growth factor receptors and it will be important to determine the cross-binding of these mutants to met and or egfr. it would also be important in the future to look at fluorescence in situ hybridization/copy number changes in c-cbl in lung cancer.  c-cbl plays an important role in down regulating rtk-mediated signaling through k63 poly-ubiquitination and subsequent downregulation of rtks followed by lysosomal degredation [3]. mono-ubiquitination or ubiquitinated with k63-linked chains of substrates by c-cbl may lead to enhancement of biological and biochemical functions (reviewed in hermann et al, 2007 [32]). the mutations that we analyzed in our studies all point out to the fact that the e3 activity of c-cbl on egfr is intact; the egfr levels in the various mutants remain same (figure s2). multiple kinases, both rtks and non-rtks could be acted upon by c-cbl, including erbs, pdgfr, fms, met, c-kit, vegfr, flt-1, ron, fgfr, ir, as well as syk, fyn, lck, fgr, lyn and c-abl [3]. in lung cancers the relevant substrates of c-cbl in terms of degradation or signal transduction are yet to be identified.  the observation that c-cbl somatic mutations, especially s80n/h94y, q249e and w802* showed increased cell viability and cell motility that is in agreement with the physiological role for d-cbl in the regulation of apoptosis and differentiation identified in drosophila is very significant [33]. it has been previously shown that activating c-cbl mutation downregulates egfr signaling and decreases cellular proliferation and migration in breast cancer cell lines [34]. although the role of c-cbl in the negative regulation of rtks is well substantiated, thereby suggesting that it is a natural tumor suppressor, studies in cancer cells have revealed both tumor suppressor and tumor promoting activities depending on the type of c-cbl mutation and the number of alleles at the c-cbl locus [24]. in agreement with the above, the three c-cbl mutants described here appear to have tumor growth and metastasis promoting properties. although these mutants are outside of the ring finger or the linker region of c-cbl, their downstream effects are significant so as to cause increased proliferation and migration, but the substrate affected by these mutations are not known yet. this raises the possibility that some of the cellular functions of c-cbl are independent of its ubiquitin-ligase activity, an area that we are currently investigating. the oncogenic nature of rtks, addiction of cancers to growth signals and given the clustering of c-cbl, egfr and met mutations, it is possible that the transforming effect of c-cbl mutations is most likely a combinatorial effect of the three. we also show that loh for c-cbl was found in a significant number of samples that harbored met or egfr mutations. the fact that about 7% of lung tumor samples are likely to have c-cbl mutations and an additional 22% are likely to harbor c-cbl-related loh, makes c-cbl a highly mutated molecule in lung cancer. since loh alone is not enough to cause a transforming event [35], [36], [37], associated mutation in the met or egfr locus or yet another rtk discussed above may play a role in carcinogenesis. we predict that this loh in c-cbl results in haploinsufficiency that downplays rtk ubiquitination leading to hyperactivity of the rtks. however, whether this is sufficient cause for tumorigenesis remains to be determined. consistent with our hypothesis is the fact that c-cbl-/- mice have increased kinase activity in lymphocytes, but is not sufficient for tumor formation [35], [36], [37]. c-cbl loh could also lead to increased expression of c-cbl from the other allele to compensate for a loss of an allele. alternately, there could be some form of synergy working with reduced c-cbl levels and mutated receptors that exacerbate the phenotype of each alone.  previous studies from our lab and others have shown that east asians with lung cancers have relatively high frequencies of gain-of-function of mutations in rtks such as egfr and met [31]. in a cohort of japanese patients an activating met mutation was identified in the splice region that deletes the juxtamembrane domain that is involved e3 activity of c-cbl [38]. this study also found that activation of met is mutually exclusive of egfr, kras and her2 gene mutations [38]. we failed to detect such mutations in significant numbers in lung tumor samples obtained from african-americans (n = 29) and caucasian (n = 50) patients. one met mutation was identified in each of the groups whereas 1 and 3 egfr mutations were identified in the african-american and caucasian cohorts respectively. egfr mutations have earlier been identified as one of the key mutations affecting lung adenocarcinoma patients in a comprehensive study of 188 patients [39]. our study encompasses different histologies of nsclc. however, the published series did not find any mutations in c-cbl or met unlike our study that encompassed different subtypes of nsclc. it is important to note, we have recently shown that met mutations in lung cancer are in majority germline [31]. we have reported earlier c-cbl mutations in a small cohort of taiwanese lung cancer samples [40]. in our efforts to understand the ethnic differences in the lung oncogenome, we also looked at pax transcription factors such as pax5 and pax8 that are highly expressed in lung cancers; however there was no preferential expression or mutations of the above genes in lung tumor samples of african-americans. in this study, we show relatively high frequency of c-cbl mutations in lung cancers, especially in the large cell type among caucasians and particularly among african-americans. we therefore propose c-cbl as an efficacious target for lung cancers in african-americans that needs to be further substantiated. this is all the more important because the prognosis for african-americans with lung cancer, especially for men is much poorer compared to their caucasian counterparts [41].  in conclusion, the results presented in this study demonstrate that c-cbl is frequently mutated or even lost in lung cancers. our results support a role for c-cbl mutants that are independent of its ubiquitination activity. given the relatively high mutation rates in c-cbl as well as rtks such as met and egfr, it is likely that their combined effect could be synergistic in promoting tumorigenesis.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=9\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.002</b>, score <b>-4.367</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.37%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.095\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__phosphorylation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.97%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.000\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__trimethylation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.97%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.000\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__641\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.97%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.000\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__g9a\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.96%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.000\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__h3k27\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.96%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.000\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__y245\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__dimethylation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__orbitrap\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__srsf2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__aebp2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__chirality\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__anhydride\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__h3k9\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__functional\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__di\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__preference\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__onco\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__htert\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__u2af1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__analogous\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__idh\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__repression\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__seed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__package\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__biotin\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__2009\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__acetonitrile\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__differential\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__idh1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.019\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bound\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.45%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__alternative\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.44%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__composed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.39%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.023\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__interactions\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.38%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.024\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ms\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.37%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.024\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__methionine\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.37%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.024\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__spliceosome\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.32%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.027\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__implicated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.31%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.027\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ward\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.31%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.028\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__branch\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.25%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.031\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__snrnp\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.20%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.034\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__aberrantly\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.18%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.036\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__myelodysplastic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.08%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.042\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__recessive\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.02%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.046\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__zrsr2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.95%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.050\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__u2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.064\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__tetracycline\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.073\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__retention\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.075\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__represses\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.49%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.175\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__neomorphic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 94.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.525\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 82.00%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -2.923\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">w802*</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">cbl</span><br/><b>Text:</b> <span style=\"opacity: 0.80\"> abstract background  non-small cell lung cancer (nsclc) </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> a heterogeneous group of disorders with a number of genetic and proteomic alterations. c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> an e3 ubiquitin ligase and adaptor molecule important </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> normal homeostasis and cancer. we determined </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> genetic variations of c-cbl, relationship to receptor tyrosine kinases (egfr and met), and functionality </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> nsclc.  methods and findings  using archival formalin-fixed paraffin embedded (ffpe) extracted genomic dna, we show that c-cbl mutations occur </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> somatic fashion for lung cancers. c-cbl mutations were not mutually exclusive of met or egfr mutations; however they were independent of p53 and kras mutations. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> normal/tumor pairwise analysis, there was significant loss of heterozygosity (loh) for </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl locus (22%, n = 8/37) and none of these samples revealed any mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> remaining copy of c-cbl. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl loh also positively correlated with egfr and met mutations observed </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> same samples. using select c-cbl somatic mutations such as s80n/h94y, q249e and w802* (obtained from caucasian, taiwanese and african-american samples, respectively) transfected </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> nsclc cell lines, there was increased cell viability and cell motility.  conclusions  taking </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> overall mutation rate of c-cbl to </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> a combination as somatic missense mutation and loh, it </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> clear that c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> highly mutated </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers and may play an essential role </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung tumorigenesis and metastasis.  go to: introduction </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> us alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which more than 145,000 of them succumb to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> disease [1]. this number </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> roughly equivalent to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> combined mortality rates of cancers of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> breast, prostate, colon, liver, kidney and melanoma [1]. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> addition </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> prognosis </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> usually poor and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> five-year survival rate </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> less than 15%. there are also significant ethnic differences for lung cancer, and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> outcome </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> worse for blacks compared to whites. gender differences are also striking with women having significantly better prognosis as compared to men. there are a number of genetic alterations that can occur </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer. as an example, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> nsclc, mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> kras, p53, egfr and met have been identified. many of these pathways, especially receptor tyrosine kinases (rtks) are controlled by c-cbl.  cbl (casitas b-lineage lymphoma) </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> a mammalian gene located on human chromosome 11q23.3 [2] and </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> involved </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> cell signaling and protein ubiquitination [3]. cbl proteins belong to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ring finger class of ubiquitin ligases (e3) and there are three homologues c-cbl, cbl-b, cbl-3 [4]. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl and cbl-b genes are ubiquitously expressed with </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> highest levels </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> hematopoietic tissues [5]. c-cbl consists of four regions encoding for functionally distinct protein domains: </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> n-terminal tyrosine kinase binding (tkb) domain, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> linker region, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> catalytic ring finger domain, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> proline-rich region and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-terminal ubiquitin-associated (uba) domain that also overlaps with a leucine-zipper (lz) domain [3]. both tkb and ring finger domains are essential for ligand-induced ubiquitination of rtks [6], [7], [8], [9]. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ring finger domain </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> required for </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> recruitment of e2 ubiquitin-conjugating enzymes. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> tkb domain includes four-helix bundle (4h), a calcium-biding ef hand, and a modified sh2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> addition, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> proline-rich region of c-cbl can associate with </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> sh3 domain of grb2, which can indirectly recruit c-cbl to rtks via </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> grb2 adaptor protein [7], [13], [14].  c-cbl also binds to egfr and acts as </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> e3 that targets egfr for ubiquitination and degradation. furthermore, cbl desensitizes egf signaling and opposes cellular proliferation induced by egf [15]. egf </span><span style=\"background-color: hsl(120, 100.00%, 96.96%); opacity: 0.81\" title=\"0.022\">activation</span><span style=\"opacity: 0.80\"> also appears to activate </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> tyrosine kinase src, which phosphorylates c-cbl and </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> turn activates </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ubiquitination and degradation of egfr [16], [17], [18]. a recent study shows that defective endocytosis of egfr </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> characterized by a </span><span style=\"background-color: hsl(0, 100.00%, 95.89%); opacity: 0.81\" title=\"-0.034\">deletion</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.05%); opacity: 0.80\" title=\"-0.012\">mutant</span><span style=\"opacity: 0.80\"> and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> point mutation l858r, whereby its association with c-cbl and subsequent ubiquitination are impaired [19]. recently, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> first human c-cbl mutations were reported </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> acute myeloid leukemia (aml) patients [20]. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> mutation r420q inhibits fms-like tyrosine kinase 3 (flt3) internalization and ubiquitination [20].  not only can e3 activity </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> important </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> oncogenesis, c-cbl has a dual but separate function as a signal transduction molecule. we have previously shown that c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> important </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> binding crkl and bcr/abl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> hematopoietic cells. also, it can bind and modulate functions of cytoskeleton by binding to proteins like talin and paxillin. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> tkb domain </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> important </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> binding to a number of molecules, and they then function </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> signal transduction.  given </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> critical role of cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> normal homeostasis and cancer, we hypothesized that it might </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> mutated </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> this study, we report novel c-cbl somatic mutations s80n/h94y, q249e and w802* </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> caucasian, taiwanese and african-american lung cancer patients, respectively. expressing these mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> nsclc cell lines lead to increased proliferation and cell motility. we show that c-cbl mutations occur with or without met or egfr mutations but are mutually exclusive of a loh at </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl locus. additionally, c-cbl loh </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> associated with either met or egfr mutations. we thus hypothesize that c-cbl mutations might contribute to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> oncogenic potential of met and egfr </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer.  go to: methods ethics statement  written consent on all research on human subjects has been obtained from </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> institutional review board, university of chicago and covers all research performed </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> laboratory. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> following </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> their contact information: institutional review board, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> university of chicago, mcgiffert hall, 5751 s. woodlawn ave., 2nd floor, chicago, il 60637. written informed consents were received from all patients whose tissue samples were used for this study.  tissue samples  lung cancer tissue and paired adjacent normal lung tissues were obtained from 50 caucasian, 29 african-americans and 40 taiwanese nsclc patients who were recruited at </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> university of chicago hospital (chicago, usa) (caucasian and african-american patients) and taipei veterans general hospital of taiwan (taiwanese patients) after obtaining appropriate institutional review board permission and informed consent from </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> patients. out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 years. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> terms of tumor types, 53 were adenocarcinoma, 32 were squamous cell carcinoma and 34 were large cell carcinoma. 49 were stage i, 14 were stage ii, 34 were stage iii, and 13 were stage iv (table s1).  cell culture  human non-small cell lung carcinoma cells a549 and h358 were maintained </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> dmem and rpmi-1640, respectively. human embryonic kidney 293t cells were cultured </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> dmem. media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (invitrogen, carlsbad, ca). cells were cultured at 37°c </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a humidified incubator containing 5% co2.  c-cbl gene mutational analysis  exons 2 to 16 of c-cbl gene were individually amplified by polymerase chain reaction (pcr). primers are listed </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> table s2. pcr conditions were 1 cycle of 95°c for 5 minutes; 35 cycles of 94°c for 30 seconds, 58°c for 30 seconds and 72°c for 2 minutes; and one cycle of 72°c for 10 minutes. pcr products were treated with exosap-it (usb corporation, cleveland, oh) and sequenced by big-dye terminator chemistry (applied biosystems, foster city, ca). sequencing was performed on </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> forward coding strand with confirmation of c-cbl alterations performed by sequencing </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> reverse strand as well. chromatograms were analyzed for mutations using mutation surveyor v2.61 (softgenetics, state college, pa).  </span><span style=\"background-color: hsl(0, 100.00%, 96.98%); opacity: 0.80\" title=\"-0.022\">plasmid</span><span style=\"opacity: 0.80\"> constructs and site-directed mutagenesis  </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> wild-type c-cbl cdna insert was subcloned into </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> paltermax expression vector using xhoi and sali restriction enzyme sites (promega, madison, wi). using this parental </span><span style=\"background-color: hsl(0, 100.00%, 96.98%); opacity: 0.80\" title=\"-0.022\">plasmid</span><span style=\"opacity: 0.80\"> paltermax-c-cbl, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> tkb domain double mutation (s80n/h94y), </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> point mutation (q249e), and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-terminal point mutation w802* of c-cbl were created using </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> following primers: 5′-gctggcgctaaagaataacccaccttatatcttagac-3′ and 5′-ctaccagatacctaccagtatctccgtactatcttgtc-3′ for </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> double mutation s80n/h94y; 5′-ctttacccgactctttgagccctggtcctctttgc-3′ for q249e, and 5′-cagctcctcctttggctgattgtctctggatggtgatc-3′ for w802* along with their complementary primers using </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> quickchange site-directed mutagenesis xl kit (stratagene, la jolla, ca) according to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> manufacturer&#x27;s instructions. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> constructs were confirmed for </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> point mutations by standard dna sequencing of both strands.  loss of heterozygosity (loh) analysis  five microsatellites on chromosome 11 (3 on 11q at or within 200 kb up or downstream of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl gene and 2 control markers on 11p) were selected for analysis (table s3). established microsatellite markers and respective primer </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.005\">sequences</span><span style=\"opacity: 0.80\"> were selected from </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> geneloc database (http://genecards.weizmann.ac.il/geneloc/index.shtml, weizmann institute of science, rehovot, israel). primers were custom designed and each forward primer was fluorescently labeled at </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> 5′ end with fam, pet, ned, or vic (applied biosystems). primer annealing temperatures and duplex scores were evaluated with nist primer tools (http://yellow.nist.gov:8444/dnaanalysis/primertoolspage.do; national institute of standards and technology, gaithersburg, md). primers were verified by performing pcr with control dna (isolated from tk6 cells) and resolving </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> products on agarose gels. bands were visualized with an uv transilluminator. genomic dna was extracted from tumor samples and paired normal lung tissue. primers were grouped into multiplex combinations shown </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> table s4. marker d11s929 served as an internal control to check for consistency </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> pcrs and of peaks from capillary electrophoresis. multiplex pcrs were carried out </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a volume of 10 µl that contained 1 µl genomic dna (20–50 ng), 0.5 µm of each primer (1.0 µm total for each primer pair), 400 µm dntps, 1x pcr buffer containing mgcl2, and 0.2 u taq dna polymerase. pcr was performed on </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> abi geneamp 9700 pcr system under </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> following conditions: 5 min at 94°c; 30 cycles of 30 sec at 94°c, 1 min at 60°c, 1 min at 72°c; and 5 min at 72°c. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> pcr products were separated by capillary electrophoresis on an abi 3130xl dna analyzer. chromatograms were analyzed with peak scanner 1.0 and genemapper 3.7 software (applied biosystems) for allelic alterations. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> area of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> peaks produced by </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> dna pcr products was quantified for each allele. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ratio of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> allelic areas was calculated for each tumor and paired normal dna </span><span style=\"background-color: hsl(120, 100.00%, 97.53%); opacity: 0.80\" title=\"0.016\">sample</span><span style=\"opacity: 0.80\">. when </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> qloh (allelic ratio for </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> tumor peaks divided by </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> allelic ratio of paired normal </span><span style=\"background-color: hsl(120, 100.00%, 97.53%); opacity: 0.80\" title=\"0.016\">sample</span><span style=\"opacity: 0.80\">) was ≤0.5 or ≥2.0 for c-cbl and at least one other 11q marker </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> at least </span><span style=\"background-color: hsl(120, 100.00%, 99.26%); opacity: 0.80\" title=\"0.003\">two</span><span style=\"opacity: 0.80\"> separate experiments, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.53%); opacity: 0.80\" title=\"0.016\">sample</span><span style=\"opacity: 0.80\"> was considered as having an allelic imbalance and interpreted as loh. samples were evaluated </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> at least </span><span style=\"background-color: hsl(120, 100.00%, 99.26%); opacity: 0.80\" title=\"0.003\">two</span><span style=\"opacity: 0.80\"> separate experiments and samples showing prospective loh at c-cbl repeated a third time which included a new control marker at </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> bax locus (data not shown) on chromosome 19 to verify integrity of </span><span style=\"background-color: hsl(120, 100.00%, 97.53%); opacity: 0.80\" title=\"0.016\">sample</span><span style=\"opacity: 0.80\"> dna.  transfection of c-cbl constructs  </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> a549 cell line was transfected using </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> fugene hd (roche, nutley, nj) reagent according to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> manufacturer&#x27;s instructions. eight µg of </span><span style=\"background-color: hsl(0, 100.00%, 96.98%); opacity: 0.80\" title=\"-0.022\">plasmid</span><span style=\"opacity: 0.80\"> dna, containing either no insert (empty vector), wild-type c-cbl, s80n/h94y c-cbl, q249e c-cbl or w802* cbl was used for transfection </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a 6-well culture plate. cells were harvested 48 h after transfection and analyzed for expression.  c-cbl knockdown  c-cbl knockdown was performed using lentiviral transduction using mission lentiviral transduction particles (sigma-aldrich, st. louis, mo) as per manufacturer&#x27;s instructions. briefly, 1×105 h358 cells/well were seeded </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> 6-well plates and infected </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> following day with c-cbl lentiviral shrna constructs. to generate stable c-cbl knockdown cell lines, cells were selected for 2 days with 1 µg/ml puromycin. c-cbl levels were determined using whole cell lysates by immunoblotting with anti-cbl antibody (santa cruz biotechnologies, santa cruz, ca).  cell viability assay  cells were transfected as described above </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> transfection assay. forty-eight hours after transfection, viability of cells was assessed using trypan blue exclusion.  wound healing assay  a549 cells were seeded </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> 6-well plates and cultured for 48 h until 100% confluent. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> medium was then changed and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cells were transfected as described </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> transfection assay. twelve hours after transfection, a straight scratch was made across </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cell layer using a 1 ml pipette tip. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cells were then gently washed with 1× pbs to remove cellular debris and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> media was replaced. photographs were taken of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> wound region every 12 h until 48 h.  </span><span style=\"background-color: hsl(0, 100.00%, 97.21%); opacity: 0.80\" title=\"-0.019\">western</span><span style=\"opacity: 0.80\"> blot analysis  forty eight hours after transfection, cells were collected and washed twice </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> 1x pbs, then lysed </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> ice-cold lysis buffer (0.5m tris-hcl with ph 7.4, 1.5 m nacl, 2.5% deoxycholic acid, 10 mm edta, 10% np-40, 0.5 mm dtt, 1 mm phenylmethylsulfonyl fluoride, 5 µg/ml leupeptin, and 10 µg/ml aprotinin) for 5 minutes. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> lysate was centrifuged at 13,000 rpm for 20 minutes at 4°c, and protein content of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> supernatant was measured. total cell lysates (50 µg/well) were separated by sds-page electrophoresis and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> gels transferred onto nitrocellulose membranes (whatman, piscataway, nj). membranes were blocked with 5% non-fat dry milk </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> phosphate-buffered saline containing tween-20 (pbst) (1x pbs, 0.1% tween-20) for 1 h at room temperature and incubated with </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> appropriate primary antibody at 4°c overnight. membranes then were washed three times with pbst and probed with appropriate horseradish peroxidase (hrp)-conjugated secondary antibody for 1 h at room temperature. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> membranes were again washed three times </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> pbst and bands were visualized using </span><span style=\"background-color: hsl(0, 100.00%, 97.21%); opacity: 0.80\" title=\"-0.019\">western</span><span style=\"opacity: 0.80\"> blot chemiluminescence reagent (biorad, valencia, ca) on a chemidoc gel documentation system (biorad, valencia, ca). antibodies were obtained from santa cruz biotechnologies and used at </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> following dilutions (c-cbl, 1∶5000; c-met, 1∶5000; egfr, 1∶5000; ubiquitin, 1∶1000; ha, 1∶5000 and β-actin, 1∶10,000).  flow cytometry  cell cycle analysis was carried out by flow cytometry. approximately 2×106 cells were grown </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> media containing 10% fbs. cells were harvested by trypsin/edta treatment, washed with 1x pbs three times and fixed with ice-cold 70% ethanol for 2 h. cells were washed again with cold pbs and stained with a solution containing 25 µg/ml propidium iodide, 200 µg/ml rnase a, and 0.1% triton x-100 for 30 minutes </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> dark. cell cycle analysis was performed using a guava pca-96 flow cytometer (guava technologies, millipore, billerica, ma).  ubiquitin ligase activity  293t cells were maintained </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> culture </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> dmem supplemented with 10% fbs and 1% penicillin (100 units/ml) and streptomycin (100 µg/ml) were transfected with 0.2 µg egfr-pcdna3 and 2 µg ha-tagged c-cbl constructs as indicated using calcium phosphate according to manufacturer&#x27;s protocol (profection, promega, madison, wi). twenty-four hours post-transfection, cells were starved overnight </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> dmem supplemented with 0.5% fbs, and then treated with or without egf (100 ng/ml) for 15 min. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cells were collected and washed </span><span style=\"background-color: hsl(120, 100.00%, 99.26%); opacity: 0.80\" title=\"0.003\">two</span><span style=\"opacity: 0.80\"> times </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> ice-cold pbs containing 0.2 mm sodium orthovanadate then lysed </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> ice-cold lysis buffer (10 mm tris hcl, ph 7.5, 150 mm nacl, 5 mm edta, 1% triton x100, 10% glycerol, 2 mm sodium orthovanadate and protease inhibitors). lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4°c. egfr immunoprecipitations were performed on 200 µg of cleared lysate using 250 ng of rabbit-anti-egfr and protein a/g plus sepharose overnight at 4°c. precipitations were washed 5 times </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lysis buffer before boiling </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 97.04%); opacity: 0.80\" title=\"-0.021\">laemmli</span><span style=\"opacity: 0.80\"> buffer. elutions were immunoblotted with anti-ubiquitin and egfr. twenty micrograms of cleared lysate were immunoblotted for each of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl constructs using anti-ha.  statistical analysis  mutation rates between different groups were compared using fisher&#x27;s exact test. for continuous variables, group comparisons were performed using analysis of variance (anova) followed by sidak&#x27;s adjustment for multiple comparisons. experiments involving repeated measurements over time were analyzed using repeated measures anova with </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> greenhouse-geisser adjustment to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> degrees of freedom. analyses were conducted using stata (v10.1) software (stata corporation, college station, tx).  go to: results c-cbl gene mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer  to investigate </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> role of c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer, we analyzed its genomic dna </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> tumor and paired normal samples drawn from multiple ethnicities. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> lung tumor samples represented caucasians (n = 50), african-americans (n = 29), and taiwanese (n = 40) lung cancer patients. we designed 12 pairs of primers to sequence </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> coding region of c-cbl gene that spans exons 2 to 16 (table s2). we identified 8 unique somatic mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl exons among 8 different patients. a variation l620f, a known snp (rs2227988) </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> exon 11 was also detected. importantly, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> eight novel non-synonymous mutations were confirmed by sequencing both strands of c-cbl genomic dna obtained from lung tumor samples (table 1). moreover, none of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> 8 mutations were detected </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> corresponding normal tissue, indicating that these were somatic mutations. four synonymous single nucleotide variations (snvs) were also identified but were not used further </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> this study.  table 1 table 1 c-cbl mutation analysis </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> 119 lung cancer patient tumor tissues. three of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> 8 novel non-synonymous mutations were located </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> tkb (tyrosine kinase binding) domain (s80n, h94y, and q249e), one </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ring finger domain (v391i), one </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> proline-rich region (72515_72517 del atg), and three </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-terminal region (w802*, r830k, and a848t) of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl protein (figure 1a and figure s1). </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> figure 1b, we show model chromatograms of representative samples.  figure 1 figure 1 c-cbl mutations and loh </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> non-small cell lung cancer. 11q loh of c-cbl gene  paired lung tumor and normal lung tissue samples from taiwanese patients (n = 37) were investigated for loh. eight (</span><span style=\"background-color: hsl(0, 100.00%, 90.07%); opacity: 0.83\" title=\"-0.119\">21</span><span style=\"opacity: 0.80\">.6%) showed loh at </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl locus on chromosome 11 while 29 samples (78.4%) revealed normal allelic contribution at </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> microsatellite markers (figures 1c and d).  c-cbl mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> different ethnic groups  </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl double </span><span style=\"background-color: hsl(0, 100.00%, 98.05%); opacity: 0.80\" title=\"-0.012\">mutant</span><span style=\"opacity: 0.80\"> s80n/h94y was found </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> same patient and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> overall mutation rate for c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung tumors was 6.7% (8/119). </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> frequency of c-cbl mutation was highest </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> large cell carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> least was observed </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> adenocarcinoma (ad) (1.8%; 1 of 53 patients), although these rates were not statistically significant (p = 0.292). mutation rates were 6.0% among caucasians (0 of 20 </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> ad; 0 of 10 </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> sq; and 3 of 20 </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lc), 13.8% </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> african-americans (1 of 10 </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> ad; 1 of 10 </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> sq; and 2 of 9 </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lc), and 2.5% (0 of 23 </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> ad; 1 of 12 </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> sq; and 0 of 5 </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lc) </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> taiwanese population. additionally </span><span style=\"background-color: hsl(120, 100.00%, 99.26%); opacity: 0.80\" title=\"0.003\">two</span><span style=\"opacity: 0.80\"> taiwanese patients with lung cancer (one squamous and one adenocarcinoma) had </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> known snp l620f. ethnic differences were not statistically significant, however </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> power to detect differences was low.  mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> met and egfr can </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> co-associated with c-cbl alterations  since east asians with lung cancer have a higher frequency of egfr and met mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung tumors [</span><span style=\"background-color: hsl(0, 100.00%, 90.07%); opacity: 0.83\" title=\"-0.119\">21</span><span style=\"opacity: 0.80\">], [22], we also determined mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> egfr and met </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> same taiwanese cohort samples and compared </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> results with </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> observed c-cbl alterations (loh and/or mutations). </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> 37 samples tested, we did not find any overlap between c-cbl mutations and c-cbl loh (figure 2). of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> three c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> (including </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> known l620f snp, rs2227988), one of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> samples had a met mutation (n375s) and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> other had an egfr mutation (l858r). among </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> 8 samples that had a loh at </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl locus 5 had an additional mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> met (n375s) and 2 had an egfr exon 19 </span><span style=\"background-color: hsl(0, 100.00%, 95.89%); opacity: 0.81\" title=\"-0.034\">deletion</span><span style=\"opacity: 0.80\">. twenty-six samples had neither c-cbl mutation nor c-cbl loh (3 patients had a c-cbl mutation but no c-cbl loh). among these 26 samples 9 had a met mutation (8 n375s, 1 l211w), 13 had an egfr mutation (7 exon 9 </span><span style=\"background-color: hsl(0, 100.00%, 95.89%); opacity: 0.81\" title=\"-0.034\">deletion</span><span style=\"opacity: 0.80\">, 6 l858r) and 4 had no other met or egfr mutation. thus </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> rate of met or egfr mutations among patients with loh at </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl locus (7 of 8) was similar to that seen </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> patients without c-cbl mutation or loh (22 of 26 patients) (p = 0.99). these 4 patients with no identifiable mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl, met or egfr represented 10.8% of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> 37 patients analyzed </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> taiwanese patient cohort. conversely, 89.2% taiwanese lung cancer patients have an identifiable mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> either c-cbl, met or egfr or a combination of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> three genes (figure 2). additionally, we determined p53 and kras mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> these taiwanese cohorts. </span><span style=\"background-color: hsl(120, 100.00%, 99.26%); opacity: 0.80\" title=\"0.003\">two</span><span style=\"opacity: 0.80\"> p53 and 1 kras mutation were detected. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> single kras mutation overlapped with one p53 mutation. this patient also had </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> egfr exon 19 </span><span style=\"background-color: hsl(0, 100.00%, 95.89%); opacity: 0.81\" title=\"-0.034\">deletion</span><span style=\"opacity: 0.80\"> but had no c-cbl mutation. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> other p53 mutation </span><span style=\"background-color: hsl(120, 100.00%, 97.53%); opacity: 0.80\" title=\"0.016\">sample</span><span style=\"opacity: 0.80\"> had a c-cbl loh with concurrent met n375s mutation. thus, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> taiwanese samples analyzed, p53/kras mutations and c-cbl mutations were mutually exclusive (data not shown).  figure 2 figure 2 c-cbl mutations and relationship to met and egfr mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer. cellular functions of c-cbl alterations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> context of lung tumorigenesis  a. e3 activity </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> intact </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.05%); opacity: 0.80\" title=\"-0.012\">mutant</span><span style=\"opacity: 0.80\"> c-cbl proteins to investigate whether </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> different c-cbl mutations affect </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> e3 activity, egfr was chosen as a model substrate for c-cbl e3 function. all of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> tested enhanced ubiquitination of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> activated egfr similar to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> wild-type c-cbl protein. this result demonstrates that </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> catalytic activity of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> not impaired when egfr was </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> substrate. (figure 3a).  figure 3 figure 3 ubiquitination, viability, expression and cell cycle analysis of various c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\">. b. effect on lung cancer cell viability </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> effect of a representative c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.05%); opacity: 0.80\" title=\"-0.012\">mutant</span><span style=\"opacity: 0.80\"> from each of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> three ethnic backgrounds on lung cancer cell viability </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> cell lines was determined. s80n/h94y double mutation, q249e, and w802* were identified </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung tumor samples obtained from a caucasian, a taiwanese and an african-american, respectively. as described </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> methods, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl wild-type (</span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.009\">wt</span><span style=\"opacity: 0.80\">) and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> above three </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> were expressed after cloning them into paltermax vector </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a549 cells. these cells express relatively low basal levels of endogenous c-cbl (data not shown). transfection efficiency was comparable between different groups and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> number of cells transfected with c-cbl wild-type construct was about 70% compared to control cells that were transfected with </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> empty vector. cells transfected with s80n/h94y, q249e and w802* c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.05%); opacity: 0.80\" title=\"-0.012\">mutant</span><span style=\"opacity: 0.80\"> constructs resulted </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> increased number of viable cells that was 132.3%, 120.8% and 147.9% higher respectively, relative to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> empty vector control transfected cells and significantly different from </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> wild-type construct (p = 0.022, p = 0.049, and p = 0.008, respectively) (figure 3b). relative levels of c-cbl protein </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> whole cell lysates prepared from samples obtained from a parallel experiment was determined. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl protein levels </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> samples representing untransfected and empty vector transfected cells were comparable and those representing </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.009\">wt</span><span style=\"opacity: 0.80\"> and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> three c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> were comparable (figure 3c).  c. effect on cell cycle to investigate if </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> increases </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> cell viability </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> different c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> are due to increased cellular proliferation, a cell cycle analysis was performed. a549 cells were transfected with </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.009\">wt</span><span style=\"opacity: 0.80\"> or </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> three different </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\">: s80n/h94y, q249e and w802*. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> empty vector transfectant was used as a control. forty-eight hours after transfection cell cycle analysis was performed as described </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> materials and methods. there was no significant change </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> subg1, g1 or </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> s phase of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cell cycle among </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> different </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> compared to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.009\">wt</span><span style=\"opacity: 0.80\"> construct (p = 0.64, p = 0.40, and p = 0.28, respectively). </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> g2/m phase of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cell cycle showed an increase </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> cell numbers for </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> three </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\">, s80n/h94y, q249e and w802*, when compared to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.009\">wt</span><span style=\"opacity: 0.80\"> but again </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> difference was not statistically significant (p = 0.25) (figure 3d).  d. effect on cell motility to investigate </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> effect of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> expression of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> above three c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> on cell migration, we carried out wound healing assay as described </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> materials and methods. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> closing of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> scratch or </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> wound was monitored at 0, 12, 24, 36, and 48 h. (figure 4a). </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> all </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> samples, that represented cells transfected with </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> wound gap was much smaller than that seen </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.53%); opacity: 0.80\" title=\"0.016\">sample</span><span style=\"opacity: 0.80\"> that represented cells transfected with c-cbl </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.009\">wt</span><span style=\"opacity: 0.80\"> (p&lt;0.001). we also determined </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> rate of wound closure for all </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> five groups. at 48 h, wild-type c-cbl transfectants showed 61.1% open wound while </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> s80n/h94y, q249e and w802* </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> showed 18.7%, 23.9% and 34.3% open wound respectively (p&lt;0.001) (figure 4b).  figure 4 figure 4 c-cbl mutations affect wound healing </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a549 cells. e. c-cbl knockdown increases cell viability it </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> hypothesized that </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> loh seen </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> our samples could lead to decreased expression of c-cbl. thus we tested </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> effect of c-cbl knockdown </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer cells. compared to a549, h358 lung cancer cells express relatively high levels of endogenous c-cbl (data not shown). c-cbl expression was knocked down using lentiviral construct that expressed c-cbl specific shrna and compared </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> results with those that were transduced with scrambled shrna and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> results are shown </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> figure 5. we identified several clones that revealed varying degrees of c-cbl knockdown showing different sets of c-cbl lentiviral shrna knockdown efficiency (figure 5a). of all </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> clones tested, clone 27 was chosen for further experiments. equal amount of cells were seeded </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a 6-well plate and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cell proliferation was measured at various times and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> results are depicted </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> figure 5b. as expected, number of cells increased </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a time dependent fashion from 100 to 190% relative to scrambled shrna as control </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a span of 48 h (p = 0.0002) (figure 5b). </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cell cycle phases </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> h358 cells that were knocked down with c-cbl shrna were looked at and compared with </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> scrambled shrna. there were no discernable differences between these </span><span style=\"background-color: hsl(120, 100.00%, 99.26%); opacity: 0.80\" title=\"0.003\">two</span><span style=\"opacity: 0.80\"> constructs </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> different phases of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cell cycle (data not shown).  figure 5 figure 5 knockdown of c-cbl using an shrna increases cell proliferation. go to: discussion our results demonstrate that c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> somatically mutated (or has loh) </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers, and can significantly contribute to enhanced cell viability and motility. there was also a high prevalence of loh with respect to c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung tumors that harbored met or egfr mutation.  </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> present study, we have demonstrated </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> occurrence of c-cbl mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer patients, especially with different ancestral variations. mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl have been recently reported </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> juvenile myelomonocytic leukemia and myeloid malignancies. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> aml study, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> mutation r420q located </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> junction of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ring finger and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> linker region inhibited fms-like tyrosine kinase 3 (flt3) internalization and ubiquitination [20], thus contributing to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.53%); opacity: 0.80\" title=\"0.008\">gain</span><span style=\"opacity: 0.80\">-</span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\">-function for </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> rtk. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> addition, mutations such as h398y, c384r, and l380p mapped to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ring finger domain and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> linker region of c-cbl that </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> required for its e3 activity [23], [24], [25], [26], [27]. additionally, homozygous mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ring finger domain of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl gene were described as a result of acquired uniparental disomy (upd) [26]. it </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> important to note that our results indicate loh at 11q23 locus and these are mutually exclusive from missense mutations of c-cbl. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> somatic mutations were all heterozygous. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> aml led to abrogation of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> e3 activity leading to prolonged rtk </span><span style=\"background-color: hsl(120, 100.00%, 96.96%); opacity: 0.81\" title=\"0.022\">activation</span><span style=\"opacity: 0.80\">. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> addition </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> located on </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> linker region surrounding </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ring finger domain exhibited enhanced akt signaling </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> response to cytokine stimulation [26]. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> addition, it was shown </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> nh3t3 cells, that neither mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ring finger nor </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> linker region causes transformation, however while certain mutations perturbs </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ubiquitination, others affect receptor recycling and prolong kinase activity [28].  we report here c-cbl mutations that mapped not only to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ring finger domain, but also to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> tkb domain, proline-rich domain and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-terminal region, but none mapped to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> linker region as reported </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> aml studies described above [23], [24], [25], [26], [29]. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> addition, 8 </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> that we detected were found </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> different ethnic backgrounds. for example, s80n/h94y, q249e, w802* were detected </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> caucasians, taiwanese and african-americans, respectively. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> results point out not only </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> difference between lung cancer and other cancers, but also genetic polymorphism among different races </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> same cancer. interestingly, there </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> a large disparity between african-american and other ethnic populations with lung cancer [30]. we have previously shown that there was a low frequency of egfr and met mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> african-americans as compared with taiwanese and caucasians [31]. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> this study, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> number of african-american samples analyzed was relatively fewer and we found 3 mutations that are unique to this ethnicity. it would now behoove us to further study </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> genetic alterations that can occur and determine </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> targeted therapeutics for african-americans. our results provide evidence of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> importance of c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> tumorigenesis and potential signaling. our prediction, based on </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> aml data, would </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> that </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> v391i ring finger domain mutation would affect </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> e3 activity. also, it will </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> important to determine </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> binding partners for c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> tkb domain and proline rich domain mutations. it has previously been shown that </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> tkb domain can bind to growth factor receptors and it will </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> important to determine </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cross-binding of these </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> to met and or egfr. it would also </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> important </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> future to look at fluorescence </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> situ hybridization/copy number changes </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer.  c-cbl plays an important role </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> down </span><span style=\"background-color: hsl(0, 100.00%, 98.99%); opacity: 0.80\" title=\"-0.005\">regulating</span><span style=\"opacity: 0.80\"> rtk-mediated signaling through k63 poly-ubiquitination and subsequent downregulation of rtks followed by lysosomal degredation [3]. mono-ubiquitination or ubiquitinated with k63-linked chains of substrates by c-cbl may lead to enhancement of biological and biochemical functions (reviewed </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> hermann et al, 2007 [32]). </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> mutations that we analyzed </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> our studies all point out to </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> fact that </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> e3 activity of c-cbl on egfr </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> intact; </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> egfr levels </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> various </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> remain same (figure s2). multiple kinases, both rtks and non-rtks could </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> acted upon by c-cbl, including erbs, pdgfr, fms, met, c-kit, vegfr, flt-1, ron, fgfr, ir, as well as syk, fyn, lck, fgr, lyn and c-abl [3]. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> relevant substrates of c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> terms of degradation or signal transduction are yet to </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> identified.  </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> observation that c-cbl somatic mutations, especially s80n/h94y, q249e and w802* showed increased cell viability and cell motility that </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> agreement with </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> physiological role for d-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> regulation of apoptosis and differentiation identified </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> drosophila </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> very significant [33]. it has been previously shown that activating c-cbl mutation downregulates egfr signaling and decreases cellular proliferation and migration </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> breast cancer cell lines [34]. although </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> role of c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> negative regulation of rtks </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> well substantiated, thereby suggesting that it </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> a natural tumor suppressor, studies </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> cancer cells have revealed both tumor suppressor and tumor promoting activities depending on </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> type of c-cbl mutation and </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> number of alleles at </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> c-cbl locus [24]. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> agreement with </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> above, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> three c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> described here appear to have tumor growth and metastasis promoting properties. although these </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> are outside of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ring finger or </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> linker region of c-cbl, their downstream effects are significant so as to cause increased proliferation and migration, but </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> substrate affected by these mutations are not known yet. this raises </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> possibility that some of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> cellular functions of c-cbl are independent of its ubiquitin-ligase activity, an area that we are currently investigating. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> oncogenic nature of rtks, addiction of cancers to growth signals and given </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> clustering of c-cbl, egfr and met mutations, it </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> possible that </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> transforming effect of c-cbl mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> most likely a combinatorial effect of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> three. we also show that loh for c-cbl was found </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a significant number of samples that harbored met or egfr mutations. </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> fact that about 7% of lung tumor samples are likely to have c-cbl mutations and an additional 22% are likely to harbor c-cbl-related loh, makes c-cbl a highly mutated molecule </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer. since loh alone </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> not enough to cause a transforming event [35], [36], [37], associated mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> met or egfr locus or yet another rtk discussed above may play a role </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> carcinogenesis. we predict that this loh </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl results </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> haploinsufficiency that downplays rtk ubiquitination leading to hyperactivity of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> rtks. however, whether this </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> sufficient cause for tumorigenesis remains to </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> determined. consistent with our hypothesis </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> fact that c-cbl-/- mice have increased kinase activity </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lymphocytes, but </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> not sufficient for tumor formation [35], [36], [37]. c-cbl loh could also lead to increased expression of c-cbl from </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> other allele to compensate for a loss of an allele. alternately, there could </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> some form of synergy working with reduced c-cbl levels and mutated receptors that exacerbate </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> phenotype of each alone.  previous studies from our lab and others have shown that east asians with lung cancers have relatively high frequencies of </span><span style=\"background-color: hsl(120, 100.00%, 98.53%); opacity: 0.80\" title=\"0.008\">gain</span><span style=\"opacity: 0.80\">-of-function of mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> rtks such as egfr and met [31]. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a cohort of japanese patients an activating met mutation was identified </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> splice region that deletes </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> juxtamembrane domain that </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> involved e3 activity of c-cbl [38]. this study also found that </span><span style=\"background-color: hsl(120, 100.00%, 96.96%); opacity: 0.81\" title=\"0.022\">activation</span><span style=\"opacity: 0.80\"> of met </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> mutually exclusive of egfr, kras and her2 gene mutations [38]. we failed to detect such mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> significant numbers </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung tumor samples obtained from african-americans (n = 29) and caucasian (n = 50) patients. one met mutation was identified </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> each of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> groups whereas 1 and 3 egfr mutations were identified </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> african-american and caucasian cohorts respectively. egfr mutations have earlier been identified as one of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> key mutations affecting lung adenocarcinoma patients </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a comprehensive study of 188 patients [39]. our study encompasses different histologies of nsclc. however, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> published </span><span style=\"background-color: hsl(0, 100.00%, 99.21%); opacity: 0.80\" title=\"-0.003\">series</span><span style=\"opacity: 0.80\"> did not find any mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl or met unlike our study that encompassed different subtypes of nsclc. it </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> important to note, we have recently shown that met mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancer are </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> majority germline [31]. we have reported earlier c-cbl mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> a small cohort of taiwanese lung cancer samples [40]. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> our efforts to understand </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> ethnic differences </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> lung oncogenome, we also looked at pax transcription factors such as pax5 and pax8 that are highly expressed </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers; however there was no preferential expression or mutations of </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> above genes </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung tumor samples of african-americans. </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> this study, we show relatively high frequency of c-cbl mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers, especially </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> large cell type among caucasians and particularly among african-americans. we therefore propose c-cbl as an efficacious target for lung cancers </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> african-americans that needs to </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> further substantiated. this </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> all </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> more important because </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> prognosis for african-americans with lung cancer, especially for men </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> much poorer compared to their caucasian counterparts [41].  </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> conclusion, </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> results presented </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> this study demonstrate that c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> frequently mutated or even lost </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> lung cancers. our results support a role for c-cbl </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.007\">mutants</span><span style=\"opacity: 0.80\"> that are independent of its ubiquitination activity. given </span><span style=\"background-color: hsl(0, 100.00%, 99.97%); opacity: 0.80\" title=\"-0.000\">the</span><span style=\"opacity: 0.80\"> relatively high mutation rates </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> c-cbl as well as rtks such as met and egfr, it </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.001\">is</span><span style=\"opacity: 0.80\"> likely that their combined effect could </span><span style=\"background-color: hsl(0, 100.00%, 99.68%); opacity: 0.80\" title=\"-0.001\">be</span><span style=\"opacity: 0.80\"> synergistic </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">in</span><span style=\"opacity: 0.80\"> promoting tumorigenesis.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Analyzing individual predictions. Let's check some predictions from the validation set. You see a summary of various vectorizer's contribution at the top, and then below you can see features highlighed in text.\n",
    "eli5.show_prediction(model, doc=X.values[1], vec=vectorizer)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Do a kaggle submission "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>CHEK2</td>\n",
       "      <td>H371Y</td>\n",
       "      <td>The incidence of breast cancer is increasing i...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>AXIN2</td>\n",
       "      <td>Truncating Mutations</td>\n",
       "      <td>An unselected series of 310 colorectal carcino...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>3</td>\n",
       "      <td>WNT4</td>\n",
       "      <td>E216G</td>\n",
       "      <td>Mycosis fungoides and Sézary syndrome are prim...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>4</td>\n",
       "      <td>SUCLA2</td>\n",
       "      <td>G118R</td>\n",
       "      <td>Regulated progression through the cell cycle ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>T599insTT</td>\n",
       "      <td>Pilocytic astrocytoma (PA) is emerging as a tu...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   ID    Gene             Variation  \\\n",
       "0   1   CHEK2                 H371Y   \n",
       "1   2   AXIN2  Truncating Mutations   \n",
       "2   3    WNT4                 E216G   \n",
       "3   4  SUCLA2                 G118R   \n",
       "4   5    BRAF             T599insTT   \n",
       "\n",
       "                                                Text  \n",
       "0  The incidence of breast cancer is increasing i...  \n",
       "1  An unselected series of 310 colorectal carcino...  \n",
       "2  Mycosis fungoides and Sézary syndrome are prim...  \n",
       "3   Regulated progression through the cell cycle ...  \n",
       "4  Pilocytic astrocytoma (PA) is emerging as a tu...  "
      ]
     },
     "execution_count": 109,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Stage 2 test data\n",
    "df_test = pd.read_csv('../data/processed/stage2_test_variants_text.csv')\n",
    "df_test.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>CHEK2</td>\n",
       "      <td>H371Y</td>\n",
       "      <td>The incidence of breast cancer is increasing i...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    Gene Variation                                               Text\n",
       "0  CHEK2     H371Y  The incidence of breast cancer is increasing i..."
      ]
     },
     "execution_count": 110,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test =df_test.drop(['ID'],axis =1)\n",
    "test['Gene'] = test['Gene'].astype(str)\n",
    "test['Variation'] = test['Variation'].astype(str)\n",
    "test['Text'] = test['Text'].astype(str)\n",
    "test.head(1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(986,)"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "X_te = vectorizer.transform(test.values)\n",
    "y_te = model.predict(X_te)\n",
    "y_te.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Prepare submission file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>class1</th>\n",
       "      <th>class2</th>\n",
       "      <th>class3</th>\n",
       "      <th>class4</th>\n",
       "      <th>class5</th>\n",
       "      <th>class6</th>\n",
       "      <th>class7</th>\n",
       "      <th>class8</th>\n",
       "      <th>class9</th>\n",
       "      <th>ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0.158321</td>\n",
       "      <td>0.112634</td>\n",
       "      <td>0.020692</td>\n",
       "      <td>0.185344</td>\n",
       "      <td>0.087246</td>\n",
       "      <td>0.090857</td>\n",
       "      <td>0.340086</td>\n",
       "      <td>0.002392</td>\n",
       "      <td>0.002427</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0.856448</td>\n",
       "      <td>0.010093</td>\n",
       "      <td>0.000448</td>\n",
       "      <td>0.026435</td>\n",
       "      <td>0.001082</td>\n",
       "      <td>0.002280</td>\n",
       "      <td>0.102744</td>\n",
       "      <td>0.000204</td>\n",
       "      <td>0.000265</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0.112375</td>\n",
       "      <td>0.235962</td>\n",
       "      <td>0.008642</td>\n",
       "      <td>0.094381</td>\n",
       "      <td>0.096315</td>\n",
       "      <td>0.026855</td>\n",
       "      <td>0.420874</td>\n",
       "      <td>0.002436</td>\n",
       "      <td>0.002161</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0.051843</td>\n",
       "      <td>0.176138</td>\n",
       "      <td>0.002842</td>\n",
       "      <td>0.380138</td>\n",
       "      <td>0.014628</td>\n",
       "      <td>0.015741</td>\n",
       "      <td>0.354217</td>\n",
       "      <td>0.001768</td>\n",
       "      <td>0.002685</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0.013848</td>\n",
       "      <td>0.040591</td>\n",
       "      <td>0.004731</td>\n",
       "      <td>0.074646</td>\n",
       "      <td>0.014663</td>\n",
       "      <td>0.005102</td>\n",
       "      <td>0.844569</td>\n",
       "      <td>0.000898</td>\n",
       "      <td>0.000951</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     class1    class2    class3    class4    class5    class6    class7  \\\n",
       "0  0.158321  0.112634  0.020692  0.185344  0.087246  0.090857  0.340086   \n",
       "1  0.856448  0.010093  0.000448  0.026435  0.001082  0.002280  0.102744   \n",
       "2  0.112375  0.235962  0.008642  0.094381  0.096315  0.026855  0.420874   \n",
       "3  0.051843  0.176138  0.002842  0.380138  0.014628  0.015741  0.354217   \n",
       "4  0.013848  0.040591  0.004731  0.074646  0.014663  0.005102  0.844569   \n",
       "\n",
       "     class8    class9  ID  \n",
       "0  0.002392  0.002427   1  \n",
       "1  0.000204  0.000265   2  \n",
       "2  0.002436  0.002161   3  \n",
       "3  0.001768  0.002685   4  \n",
       "4  0.000898  0.000951   5  "
      ]
     },
     "execution_count": 81,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "y_pro = model.predict_proba(X_te)\n",
    "submission_df = pd.DataFrame(y_pro, columns=['class'+str(c+1) for c in range(9)])\n",
    "\n",
    "submission_df['ID'] = df_test['ID']\n",
    "submission_df.head()\n",
    "\n",
    "mysubset = [\"ID\"]\n",
    "mask = submission_df.columns.isin(mysubset)\n",
    "submission_df =pd.concat([submission_df.loc[:,mask], submission_df.loc[:,~mask]], axis=1)\n",
    "submission_df.to_csv('../data/processed/stage2_submission_3.2.csv', index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.2 Word2vec on Text + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_tr, X_val, y_tr, y_val = split_data(df,\n",
    "                                      'Text',\n",
    "                                      'Class',\n",
    "                                      0.1,\n",
    "                                      0,\n",
    "                                      stratify='Class')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "w2vmodel not found. training model\n",
      "Model done training. Saving to disk\n"
     ]
    }
   ],
   "source": [
    "# Use document df\n",
    "w2vec = get_word2vec(\n",
    "    MySentences(\n",
    "        df['Text'].values, \n",
    "    ),\n",
    "    'w2vmodel'\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "clf = Pipeline([('vect', MeanEmbeddingVectorizer(w2vec)),\n",
    "                ('clf',  xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42))])                           "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.6186186186186187\n"
     ]
    }
   ],
   "source": [
    "clf.fit(X_tr, y_tr)  \n",
    "predicted = clf.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.3 Google News Corpora Word2Vec + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "W0603 21:10:41.930970 140735731893120 smart_open_lib.py:385] this function is deprecated, use smart_open.open instead\n"
     ]
    }
   ],
   "source": [
    "vector_path = r\"/Users/j/Dropbox/Learn/kaggle_quora/src/data/embeddings/GoogleNews-vectors-negative300/GoogleNews-vectors-negative300.bin\"\n",
    "gmodel = KeyedVectors.load_word2vec_format (vector_path, binary=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "clf = Pipeline([('vect', MeanEmbeddingVectorizer(gmodel)),\n",
    "                ('clf',  xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42))])                           "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.5813253012048193\n"
     ]
    }
   ],
   "source": [
    "clf.fit(X_tr, y_tr)  \n",
    "predicted = clf.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.4 Medical Corpora Word2Vec + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: Logging before flag parsing goes to stderr.\n",
      "W0603 20:53:23.172096 140735731893120 smart_open_lib.py:385] this function is deprecated, use smart_open.open instead\n"
     ]
    }
   ],
   "source": [
    "vector_path = r\"../src/pubmed2018_w2v_200D/pubmed2018_w2v_200D.bin\"\n",
    "biomodel = KeyedVectors.load_word2vec_format (vector_path, binary=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "clf = Pipeline([('vect', MeanEmbeddingVectorizer(biomodel)),\n",
    "                ('clf',  xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42))])                           "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.6024096385542169\n"
     ]
    }
   ],
   "source": [
    "clf.fit(X_tr, y_tr)  \n",
    "predicted = clf.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/j/miniconda3/lib/python3.7/site-packages/matplotlib/cbook/__init__.py:424: MatplotlibDeprecationWarning: \n",
      "Passing one of 'on', 'true', 'off', 'false' as a boolean is deprecated; use an actual boolean (True/False) instead.\n",
      "  warn_deprecated(\"2.2\", \"Passing one of 'on', 'true', 'off', 'false' as a \"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AxesSubplot(0.125,0.125;0.62x0.755)\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAATIAAAEWCAYAAADl+xvlAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDMuMC4zLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvnQurowAAIABJREFUeJztnXecFFXWv58zM+TgkLMiCkOSMEQlKKAERcGAioqkBXP25+qa94VVX33VdV0D5oSYMBNFkWAgK0lRFCRJUjLIMJzfH1WDzQjd1dNVPV3DefzUZ7qrq773dlV7uPfWvecrqophGEaYSSvsChiGYSSKBTLDMEKPBTLDMEKPBTLDMEKPBTLDMEKPBTLDMEKPBbIihoiUEpEPRWSriLyVgM7FIjLJz7oVBiIyXkQGFnY9jGCxQFZIiMhFIjJHRHaIyDr3f7iOPkifB1QDKqlqv4KKqOprqtrdh/ochIicIiIqImPz7W/u7p/qUeceEXk11nGq2ktVXypgdY2QYIGsEBCRG4FHgX/hBJ2jgSeAPj7IHwMsU9V9PmgFxUbgJBGpFLFvILDMrwLEwX7fRwqqalsSN+AoYAfQL8oxJXAC3Vp3exQo4X52CrAauAnYAKwDBruf3QvsBXLcMoYC9wCvRmjXBRTIcN8PAn4CtgM/AxdH7J8Rcd5JwGxgq/v3pIjPpgL/A8x0dSYBlQ/z3fLq/xRwlbsv3d13FzA14th/A6uAbcBcoJO7v2e+7/lNRD1GuvXYDRzv7vub+/mTwNsR+g8AUwAp7N+FbYlt9i9W8jkRKAm8G+WY24H2QAugOdAWuCPi8+o4AbEWTrD6r4hUUNW7cVp5b6hqWVV9LlpFRKQM8BjQS1XL4QSrBYc4riLwsXtsJeBh4ON8LaqLgMFAVaA4cHO0soGXgUvd1z2AxThBO5LZONegIjAaeEtESqrqhHzfs3nEOQOA4UA5YGU+vZuAZiIySEQ64Vy7gepGNSO8WCBLPpWATRq963cx8E9V3aCqG3FaWgMiPs9xP89R1XE4rZKsAtZnP9BUREqp6jpVXXyIY84AflDVV1R1n6q+DnwHnBlxzAuqukxVdwNv4gSgw6KqXwAVRSQLJ6C9fIhjXlXVzW6Z/4fTUo31PV9U1cXuOTn59HYBl+AE4leBa1R1dQw9IwRYIEs+m4HKIpIR5ZiaHNyaWOnuO6CRLxDuAsrGWxFV3QlcAFwOrBORj0WkoYf65NWpVsT7XwtQn1eAq4EuHKKFKiI3ichS9wnsFpxWaOUYmquifaiqs3C60oITcI0igAWy5PMlsAfoG+WYtTiD9nkczV+7XV7ZCZSOeF898kNVnaiqpwE1cFpZz3ioT16d1hSwTnm8AlwJjHNbSwdwu35/B84HKqhqJs74nORV/TCaUbuJInIVTstuLXBLwatupBIWyJKMqm7FGdT+r4j0FZHSIlJMRHqJyP+6h70O3CEiVUSksnt8zKkGh2EB0FlEjhaRo4Db8j4QkWoicpY7VvYHThc19xAa44AG7pSRDBG5AGgMfFTAOgGgqj8DJ+OMCeanHLAP5wlnhojcBZSP+Hw9UDeeJ5Mi0gAYgdO9HADcIiJRu8BGOLBAVgio6sPAjTgD+BtxukNXA++5h4wA5gDfAguBee6+gpQ1GXjD1ZrLwcEnDWcAfC3wG05QufIQGpuB3u6xm3FaMr1VdVNB6pRPe4aqHqq1OREYjzMlYyVOKzay25g32XeziMyLVY7blX8VeEBVv1HVH4B/AK+ISIlEvoNR+Ig9sDEMI+xYi8wwjNBjgcwwjNBjgcwwjNBjgcwwjNATbVJm0ilfoaJWqVEnMP2jShULTDtZ7N57qNkR/lGyWHqg+hp9mlcoSBeJfVCKsnLlCjZt2pTQF0gvf4zqvt2ejtXdGyeqas9EyvNCSgWyKjXq8MDo8YHp92pcIzDtZLFo1dZA9bNqlAtUPzfgp+S5+4MPlOVD/A9ih3atE9bQfbspkXW+p2P3LPhvrJUYvpBSgcwwjDAgkGIZkiyQGYYRHwKkBTsEES8WyAzDiJ8UGye0QGYYRpxY19IwjKKAtcgMwwg1Qsq1yFKrNodh069ruGfYeVx/zsnccG4XPh79LABvPvV/DO/eipsvOI2bLziNedOn+FLeFcOHULd2Ndq0PMEXvWToj7j1anq1rc9FvU48sG/ZkoUMPfc0BpzZiUF9u7D4m7m+lLVnzx66dW5Px3bZnNi6GfeNuMcX3fzk5ubS5aTW9D/PD0+Wg2ndtD4nt29J1w6t6X5ye9/1J02cQLMmWTRpeDwP/u/9odOPjjgtMi9bkggskInI8yKyQUQWJaqVnp7BpTfezaNjP+dfL3/IxDdeZNVyx3Cn9yXDeOiNyTz0xmSyO3VLuN4AFw8YxHsfBjefLQj9M87pzyPPv33QvscfuJuh197CKx9OZ/j1t/H4A3f7UlaJEiV4f9wnzPh6HtO+nMuUyROZPesrX7QjefqJx6if1ch33TzGfjyZT2fOYdLn/tY9NzeX66+9ivc/HM/8b5fw1pjXWbpkSWj0PZGW7m1LVnUC1H4Rx+0mYSpUqUa9Rk7rpVSZstQ6tj6/bfw1xlkFp2OnzlSoUDFU+i3bdqB8ZoWD9okIO3dsB2DH9m1UqVb9UKfGjYhQtqyTyTonJ4ecnH2Iz//6rl2zmskTxnPJwCG+6iaD2bNmcdxxx3NsvXoUL16cfhdcyEcfvh8a/di4g/1etiQRWEmqOg0nWZ+vbFi7ip+/X0T9pi0BmDDmBW46/1SeuOdGdmzb4ndxoeb6O/7F4/ffxVkdm/Cf++/iipvv8k07NzeXTu1b0aBuDU7p2o3Wbdr5pg1w+y03cfeI+0hLC+gnKsIFfU/ntM7tePmFZ32VXrt2DbVr/7nUrlat2qxZk2hW8OTpx0Q4crqWXhGR4a7j9pxtWzZHPXb3rp08dPMwBt98L6XLlqN7v0v5z4df8OCYSWRWrsrLD/8zSbUOB2NHP891t/+LD2Ys5rp/jGTkbdf6pp2ens70r+ayeNlK5s2dzZLFCY8gHGDi+I+pXKUKLVq28k0zPx9Nmson02cx+p0PeeGZJ/ly5nTftA+VrNTPFmvQ+p7woUUmIlkisiBi2yYi14tIRRGZLCI/uH8rRBUiBQKZqo5S1daq2rp8ZqXDHrcvJ4f/u3kYnXqdTbtupwOQWakK6enppKWlceo5F/Pjor9YMh7RjBv7Ol16OI5t3U7vy5JvYmaEjpujMjPp2Olkpkye6JvmrK++YMK4j2jZ+HiGD7qYGZ9/xuVDL419YhxUr+GYUlWpUpXTe/dh/tzZvmnXqlWb1av/zMq9Zs1qatasGeWM1NKPjT9dS1X9XlVbqGoLoBWO+9a7wK3AFFWtj2OgfGusGhV6IPOCqvLkvTdR69jjOXPAZQf2/75x/YHXsz4dT53jCmrtWDSpXK0G876eCcCcL6dRp249X3Q3bdzI1i1ON3737t1M/WwK9bP8u/Z33juShctWMH/Jj4x68TU6ntyFp577i+1lgdm5cyc7tm8/8Hrqp5/QsFET3/Rbt2nDjz/+wIqff2bv3r289cYYzuh9Vmj0YyJAerq3zTvdgOWquhLoA7zk7n+J6I5jQEjmkX23YDbTPn6Ho+s34uYLTgPgoqtvZcbE91jx/RJEhCo1anPZHQ/4Ut6gARcxfdpUNm/aRIN6dbj9znsYOHioL9pB6d95/VDmfT2TLb9v5swOTRh23a3cNvJRHvmf28jN3UfxEiW5beSjvtT/11/XceXwIeTm5rJ//37OPvc8evbq7Yt2Mti4YT2DL+4HQO6+fZzd70K6ntbDN/2MjAwe+ffjnHlGD3Jzcxk4aAiNm/gXKIPW94T3rmxlEZkT8X6Uqo46xHEX4riHAVRT1XUAqrpORKrGrE5Q5iMi8jpwCo6h6nrgblV9Lto5xzVurpbGJzqWxieGvqXxiUqHdq2ZO3dOQgNqaeVra4m2V3s6ds+U2+aqatTcQSJSHMfJq4mqrheRLa6Pad7nv6tq1HGywFpkqto/KG3DMAoZfx8u9ALmqWreWNF6EanhtsZqABtiCYRijMwwjBTD33lk/fmzWwnwATDQfT0QiDlJzgKZYRjx4XUOmYdWm4iUBk4Dxkbsvh84TUR+cD+LuQYrFIP9hmGkGD4tP1LVXUClfPs24zzF9IwFMsMw4sTykRmGURSwfGSGYYSaFMxHllKBrFzJDLrUjzn3rcAEPQer+TGZsQ9KkKZ1jgpUPz0ttf6ljZep328MvIxTsqoEXkZqY11LwzCKAuaiZBhG6LExMsMwQo1Y19IwjKKAtcgMwwg7SU/kGAMLZIZhxIWT6doCmWEYYUYESbFpOqk1YhcHfnseJtMXMmhPwqB9OSF8vo0b163h74PPZviZHbisTyfee8XJ7ffsQ/cw7MyTuOLsk/nntQPZsc2fuYZhuz7xIiKetmQRpK9lHRH5TESWishiEbnOT32/PQ+T5QuZDE/CoH05w+jbmJ6RwbD/dy+jPpzJI6PH89GY51m5/HtanngyT707jSff/ZxadY/jjWf/nZL1T6a+F46YQAbsA25S1UZAe+AqEWnsh3AQnofJ8oVMhidh0L6cYfRtrFilGsc3bgZA6TJlqVOvAZvXr6NVhy6kZzgjLA2btWLT+rUpWf9k6nvhiAlkqrpOVee5r7cDS4FafmgH7nnoEoQvZKF7EvpA2H0b16/5heVLF5LV7GC7uUnvvk6bjom71Yf9+sRE4tiSRFLGyESkLtAS+DpRrWR4HuYRhC9kSngSJkiYfRt379rBiBuGcNnf/4cyZf/0J3j96UdIT0+nS+/zEi4jzNfHC4K31liRaJHlISJlgXeA61V12yE+P2DQu3nTpph6yfA8zCMIX8jC9yRMnLD6Nu7LyWHE9UPocsa5dDjtT9enye+PYda0SdzywJO+/M8X1usTD2lpaZ62pNUnSHERKYYTxF5T1bGHOibSoLdS5coxNYP2PIwkCF/IQvck9IEw+jaqKo/edT116jXgnIFXHNg/Z8anvPXc49z9n1coWap0olUHwnl94iXVWmSBzSMT51s8ByxV1YeDKscvkuULmQxPwqB9OcPo27h4/tdM+fAt6tZvxFXndgFg4HW389R9/yBn715uH+b4XDZs1opr7n4o5eqfTP2YJHn8ywtB+lp2BKYDC4H97u5/qOq4w53TIruVTpme8DDaYfnx1x2BaUNy8pEF7dto+chiE+Z8ZH74WmZUrqeZvf/l6djNL/WP6WvpB0H6Ws4g5eK2YRiJkjfY74uWSCbwLNAUUGAI8D3wBlAXWAGcr6q/R9MJ7cx+wzAKD0kTT5sH/g1MUNWGQHOcaVq3AlNUtT4wxX0fFQtkhmHEh/gz2C8i5YHOOGPpqOpeVd0C9AFecg97Cegbq0oWyAzDiJs4AlnlvOlV7jY8QqYesBF4QUTmi8izIlIGqKaq68CZWA/ENPKw7BeGYcRNHGNkm6IM9mcA2cA1qvq1iPwbD93IQ2EtMsMw4sLHmf2rgdWqmjdV4W2cwLZeRGoAuH83xBKyQGYYRvz4sNZSVX8FVolIlrurG7AE+AAY6O4bCMRcEZ9SXct0EcqUCK5KQc/zytm3P/ZBCRL4PLLiqWXzFS9t6laIfZCRGIKfy4+uAV4TkeLAT8BgnAbWmyIyFPgF6BdLJKUCmWEY4cCveWSqugA41BhaXGlILJAZhhE/KTbV3QKZYRhxk2qppyyQGYYRF8nObOEFC2SGYcSNBTLDMEJPqtnBWSAzDCNurEVmGEa4kdQLZKGc2R9289PVq1ZxRo9utG7RhLbZJ/DE44/5qr9nzx66dW5Px3bZnNi6GfeNuMdXfQj/PQD/TZ4jKQrX53AIIOJtSxZBGvSWFJFZIvKNa9B7rx+6RcH8NCMjg5H3P8icBYuZ8vkXPPP0E3y31L8ySpQowfvjPmHG1/OY9uVcpkyeyOxZX/mmXxTuAfhv8pxHUbk+h+fIclH6A+iqqs2BFkBPEWmfqGhRMD+tXqMGLVpmA1CuXDmyGjZk7Vr/fAlFhLJlywKQk5NDTs4+X39UReEeBGHynEdRuD6xSEsTT1vS6hOUsDrkJckv5m4JLxQsauanK1eu4NsFC2jdpp2vurm5uXRq34oGdWtwStduvuoXhXsQpMlzUbg+UfHYrSwSXUsAEUkXkQU4aTgmR6TriDzmgK/lxk2xjSOKkvnpjh07GNC/H/c/+DDly5f3VTs9PZ3pX81l8bKVzJs7myWLF/mmHfZ7ELTJc9ivTyyEI6hFBqCquaraAqgNtBWRpoc45oCvZZXKsd1pior5aU5ODpf0P4/zL7iIs/qe47t+HkdlZtKx08lMmTzRN82w34OgTZ7Dfn28cES1yPJw83BPBXomqlUUzE9Vlasu/xtZWY24+robfNUG2LRxI1u3bAFg9+7dTP1sCvWzsmKc5Z2w34OgTZ7Dfn28kGqD/UEa9FYBclR1i4iUAk4FHkhUtyiYn371xUzGjH6VJk1PoEM7Z9D/rntH0KPn6b7o//rrOq4cPoTc3Fz279/P2eeeR89evX3RhqJxD4KkyF+fJLe2vBCkQW8zHAeUdNxEaar6z2jntGrVWmd+PSeQ+iSDopBYsWTIEyvu/GNf4GUEmfwzaPww6C1dM0vrD3vC07Hf/vPU0Bv0fgu0DErfMIzCI9VaZOH9p8UwjEIj1ZYoWSAzDCM+UnCMzAKZYRhx4ay1TK1IZoHMMIy48SuOicgKYDuQC+xT1dYiUhF4A6gLrADOV9Xfo+mEMvuFYRiFi88z+7uoaouIp5u3AlNUtT4wBQ/u4xbIDMOIDwl8QmwfnKlbuH/7xjohpbqWu3NyWbJ6W2D6jWv7u54xP8lYWzZ49PxA9Z+9sEWg+sUygv23M+h5dsaf+cg8UllEIieHjlLVURHvFZgkIgo87X5WTVXXAajqOhGpGquQlApkhmGEgbhaW5tiTIjtoKpr3WA1WUS+K0iNrGtpGEbc+LVoXFXXun83AO8CbYH1IlLDKUdq4GTPiYoFMsMw4kP8GewXkTIiUi7vNdAdWAR8AAx0DxsIxMwaaV1LwzDiwsd5ZNWAd12tDGC0qk4QkdnAmyIyFPgF6BdLyAKZYRhx40cgU9WfgOaH2L8Z6BaPlgUywzDiJsUm9lsgMwwjflJtiVIoBvvvveUqTm19HOf3ONiEacyLT3NO11b0696Of993p2/lBe0ZeMXwIdStXY02LU/wVTdN4ME+jbjt1OOccjoew0N9G/F/fRtxU5d6lPRpDlfQvplBXZ9IWjetz8ntW9K1Q2u6n5ywuddfKMq+lkec+QgcMCCZLyIfFVTjzHMv4j8vvnPQvtlfTuPzTz5mzPgveGvS1wwYdm3CdYXkeAZePGAQ73043ldNgNMbV2X1lj0H3r/49Spufm8pN723lE0799KzcWxPBC8E7ZsZ1PXJz9iPJ/PpzDlM+ty/ukPR97UUvD2xLDLmIy7XAUsTEchu14GjMisctO/tV59j0OU3ULxECQAqejAu8UIyPAM7dupMhQoVfdWsWLoYreocxZRlmw7s253zZ8ba4ulpPpjxOQTtmxnE9UkmR4SvpYinLWn1CVJcRGoDZwDP+q39y8/LmT/7Sy7t25VhF5zO4m/m+qJb6J6BBWRwuzq8MnvNX6zCrux4DM/2b0atzJKMWxJzXqFngvTNTAoiXND3dE7r3I6XX/D351nkfS1Jva5l0IP9jwK3AOUOd4CIDAeGA1SvWedwh/2F3Nx9bNu6hZfencLib+Zx69WD+GDatwm3DArbM7AgtKpzFFv35PDT5l00qV72oM+emLGSNIEh7evQoV5FPvthsy9l5vlmbt2yhUv6n8uSxYto3OQvbn8py0eTplK9Rk02btzA+X16Ub9BFid26OSLdpH3tZTU+3/isC0yESkfbYslLCK9gQ2qGrWpFOlrWaFSJc8Vr1q9Jl17nomI0LRFKyQtjS2/Jf4/aSp4BsZLVtUytDk6kyf6NeX6U+rRtGZ5ru1c98Dn+xW++Pl32h+T6XvZQfhmJoPqNZx7WqVKVU7v3Yf5c2f7pn0k+FqmibctafWJ8tlinOUCiyO2RRF/Y9EBOMtNnDYG6CoiryZU2whO6X4Gs7+YBsDKn35kX04OmRW9B8LDkQqegfEyeu5aLntjIVe+tYhHp/7EorXbeGzaCqqXK3HgmNZ1jmLN1j1RVLwTtG9m0OzcuZMd27cfeD31009o2Mg/O7Ujwdcy1Qb7D9u1VFXv/bxDn38bcBuAiJwC3KyqlxRE6x/XDmHOVzPY8vtmep3YiMuuv40+/QZw7y1XcX6P9mQUK8Y9Dz3pS3M3GZ6BgwZcxPRpU9m8aRMN6tXh9jvvYeDgob6WIcDVnetSqlg6IrDyt12M+uIXX7SD9s0M+vps3LCewRc7q15y9+3j7H4X0vW0Hr7pF3VfS8F5cplKePK1FJELgXqq+i93AL9arC5jvvNPwQlkUX/tjZu11Fc/+NyrbNwEnY8sGbmwBr46L1D9sOcjS4avZflSxQIvIyj88LXMPKaRdr7dmzP7h5e1TYqvZcxflYg8DnQBBri7dgFPxVOIqk6NFcQMwwgJHrPDJvOBgJenlieparaIzAdQ1d9EpHjA9TIMI4VJsYeWngJZjoik4U6nFJFKwP7opxiGUVQRSOpkVy94CWT/Bd4BqojIvcD5wL2B1sowjJQmmU8kvRAzkKnqyyIyFzjV3dVPVb1MvzAMowiS7Fn7XvA6sz8dyMHpXoYiY4ZhGMGRal1LL08tbwdeB2oCtYHRInJb0BUzDCN1EY9bsvDSIrsEaKWquwBEZCQwF7jP78oUz0jjmCql/ZZNGulJGDd4ol+zQPW37MoJVL9S2WAfeCfjHhipt9bSSyBbme+4DOCnYKpjGEaq4zy1LOxaHMxhA5mIPIIzJrYLWCwiE9333YEZyameYRgph/i7jlJE0oE5wBpV7S0ix+Ksz64IzAMGqOreaBrRWmR5TyYXAx9H7Pc3naZhGKHD565lXvLVvDWEDwCPqOoYEXkKGAo8GU0g2qLx5/yqpWEYRQc/u5YRyVdHAjeKEyG7Ahe5h7wE3ENBA1lEQce5hTQGSubtV9UGBam4YRjhJ44WWWURmRPxfpSqjop4nz/5aiVgi6rmrf5fDdSKVYiXwf4XgRHAQ0AvYDC2RMkwjmjiaJBtOlz2i8jkq26GnMNJx0wr4yWQlVbViSLykKouB+4QkekezjMMowgi4ts0l7zkq6fj9PbK47TQMkUkw22V1QbWxhLyMkv/D7ffulxELheRM4GqBa+7P+Tm5tLlpNb0P6+P79pFwZMwaN/GF55+nB6dWtG9YzbPP/UfX7VXr1rFGT260bpFE9pmn8ATjz/mq34e9hsqOH6k8VHV21S1tqrWBS4EPlXVi4HPgPPcwwYCMS2ivASyG4CywLU4EXQYMMTDeYjIChFZKCIL8vWTE+bpJx6jflYjPyWBouVJGJRv4/dLFzPm1Rd4b+J0xk2dxaeTx/Pz8h9908/IyGDk/Q8yZ8Fipnz+Bc88/QTfLfX/GtlvqOAE7KL0d5yB/x9xxsxiPniMGchU9WtV3a6qv6jqAFU9S1VnxlGpLqraws8skWvXrGbyhPFcMtBTPI2LI8GTMFF+XPYdLVq1pVTp0mRkZND2pE5MHOffd6heowYtWmYDUK5cObIaNmTtWn/tzuw3VHAEb56W8azHjEy+qqo/qWpbVT1eVfup6h+xzo/movSuiIw93Oa5hgFw+y03cfeI+0hL83/9epHxJAzQtzGrURNmfTmD33/bzO5du5j6yQTWrVntaxl5rFy5gm8XLPDdN9N+QwngsTWWKr6Wj/ugr8AkEVHg6XyPXYGDfS1r1zk6puDE8R9TuUoVWrRsxYxp/uf3LyqehEH6Nh7foCGXX3MTA87rTekyZWjUpBkZGf5bpO7YsYMB/ftx/4MPU768f34L9htKnNCstVTVKT7od1DVtSJSFZgsIt+p6rR85YwCRgG0yG4V8zHrrK++YMK4j/hk0gT+2LOH7du3cfnQS3nqOW9mCLEoKp6Eh/Jt9CuQAVxwySAuuGQQAA+OuIvqNWNO9YmLnJwcLul/HudfcBFn9T3HV237DSWGAOkpFsgCzS2mqmvdvxuAd4G2iWreee9IFi5bwfwlPzLqxdfoeHIX336AUDQ8CYP2bQTYtHEDAGtW/8KEj9/nrHPO901bVbnq8r+RldWIq6+7wTfdPOw3lDipZtDrf3/ARUTKAGmqut193R34Z1Dl+UVR8CQM2rcR4IrB/dny+29kFCvGPx94lKMyK/im/dUXMxkz+lWaND2BDu2cQf+77h1Bj56n+1ZGkBSF31AsUi37hSdfSwARKeHl6UHE8fVwWmHgBMzRqjoy2jktslvplOlfey0ibsqUCCxuJ41tu4PNF7brj9xA9YPOR7Y3N/hFJ2H+Hfnha1m9flO9+OF3PB378FkNk+Jr6WWtZVuceRxHAUeLSHPgb6p6TbTzVPUnoLkvtTQMI6VItRaZlzGyx4DewGYAVf0Gx7DXMIwjlDBNv8gjTVVX5nvcGmz/wzCMlEWAjBR7auklkK1yu5fqZnK8BlgWbLUMw0hlUiyOeQpkV+B0L48G1gOfuPsMwzgCkTiXHyUDLwa9G3BWphuGYQAhbJGJyDMcIrGZqg4PpEaGYaQ8qfbU0kvX8pOI1yWBs4FVhzk2IVQhZ1+A84BKBCedLMqXKhaoftBzpOb8/Hug+u2OqxiovuEuUUqxSOala/lG5HsReQWYHFiNDMNIbZK8/MgLBfnn91jgGL8rYhhGeJB4svYnAS9jZL/z5xhZGvAbcGuQlTIMI3UJldM4gJurvzmQl7Vtv3pdnGkYRpEl1QJZ1CVKbtB6V1Vz3c2CmGEYvpiP+ImXtZazRCQ78JoYhhEKHDs4b1uyOGzXMsJXriMwTESWAztxusiqqhbcDOMIxY+Z/SJSEpiGMzEqA3hbVe8WkWOBMUBFYB4wQFX3Rq1PlM9muX/7AlnA6UA/HL+5fgl9gwTZumULf7v0Qjq2OYFObZsxZ5a/dmdFwZMwyDKuGD6EurWr0ablCb5pbli3husv7cOlp7dnUO+TePvlpwHYtuV3bhpyDhfF+nZqAAAWg0lEQVT3aMNNQ85h+9YtvpQX9ntcmL6WeYP9PmSI/QPoqqrNgRZATxFpDzwAPKKq9YHfgaGxhKIFMgFQ1eWH2mJ/XRCRTBF5W0S+E5GlInKil/NiceetN9Hl1O7MmL2QKTPmUL9BQz9kgaLhSRh0GRcPGMR7H473TQ8gPT2dK//+T14e9xVPjJnIe689x4ofv2P0M/8mu31nXps4m+z2nRn9zKMJlxX2e1xUfC3VYYf7tpi7KdAVeNvd/xJOYyoq0QJZFRG58XBbLGGXfwMTVLUhztPPpR7POyzbt23jqy+mc9GAwQAUL16cozIzE5U9QFHwJAy6jI6dOlOhgr8z6CtVrU6DJk4eztJly3HMcfXZtH4dM6eMo2dfZ6lvz74XMuOTcQmXFfZ7XNi+liCkedyAyiIyJ2I7aGmjiKSLyAJgA85E++XAFndYC2A1ENPZJlogS8dxGC93mC36VxUpD3TGdQlW1b2qmnC/YOWKn6lUuQrXXzmM0zq15aZrLmfXzp2Jyh6gKHgSFrrvYYKsW/0LPyxdSKPmrfht80YqVa0OOMHu9982Jawf9ntc2PdXiKtFtklVW0dsB1lCurMhWgC1ccyJDmX9HnO2RLR5ZOtUNRGzkHrARuAFNz32XOA6VT0o6kT6Wtby4Gu5L3cfC7+Zz8j/fYTs1m254+838p9HHuTvd9yTQFX/pCh4EqaC72FB2bVzB3dfO4irbxtJmbL+eVlGEvZ7XOj3VyDD54lkqrpFRKYC7YHMiIeNtYG1sc6POUaWABlANvCkqrbEeeL5lxUBqjoqL1pXqlQ5pmjNmrWoUbM22a0dZ7nefc5h4bfzE6zqnxQFT8LC9j0sKPtycrj72kGceuZ5dO5+JgAVK1Vh84ZfAdi84VcqVIz9G4lF2O9xYd/fOFtkh9cRqSIime7rUsCpOMNPn+E8VAQYCMTsN0cLZN1if6WorAZWq2qeLdLbOIEtIapWq07N2rX58YfvAZjx+Wc0yDpUa7RgFAVPwlTwPYwXVeV/77iWo49rwPmDrzyw/6SuvZjw3hgAJrw3hg7dEreEC/s9ToX7m+YmV4y1xaAG8JmIfAvMBiar6kfA34EbReRHoBLu8FQ0ojmN/+b5Wx36/F9FZJWIZKnq9ziB0ZdHKyMfeISrhg0iZ+9ejq57LI8+8YwfskDR8CQMuoxBAy5i+rSpbN60iQb16nD7nfcwcHDMJ+RRWTjvaya9/yb1GjRmaN+TARh2wx1cNOw67r1hCOPeeY1qNWpxz6MvJFz/sN/jVPC19KMnq6rfAi0Psf8n4jTz9uxrWRBEpAXwLFAc+AkYrKqHTUjVvGUrnTj1y8Dqk1kmWE/FokDu/mBXoVk+ssLFD1/LYxs107tf/sjTsYPbHpMavpaJoKoLgMC/hGEYSUT8mdnvJ+G1TDYMo1BwZvZbIDMMI+SkVhizQGYYRgFIsQaZBTLDMOIlubnGvGCBzDCMuBC8JTJMJhbIDMOIGxvsL0Q274iamy1hyhRPD1Q/GZQM+DsEPc+rQpurA9UH+H3244HqBzmXzxdlSb21u0dUIDMMI3Gsa2kYRpHAWmSGYYSe1ApjFsgMw4gTAdKtRWYYRthJsThmgcwwjHgRJMU6lxbIDMOIm1RrkaXaU1RPBOlrufyH7+neqc2BreHRlXn2ycd80wfYs2cP3Tq3p2O7bE5s3Yz7RtwTKn0In29j/WOq8tWYWw9s66c/yNUXncLtl53O8okjDuzv0bGxD7UP/voE4S3qFWf6hWcXpaQQWItMRLKANyJ21QPuUtWEjQnzfC2ffXkMe/fuZfeuXYlKHuC4+llMmj4bcPwDWzc+lp5n9PFNH6BEiRK8P+4TypYtS05ODr1O7cyp3XvSpm37UOjn+Sp+PH4ytWrXpmP7NvTufRaNGvsTBILQ/2HlBtpf6ASUtDRh+cSRfPDZNww460T+8+pnPPrKFF/qDsFfH3C8RS+74mqGDRnom6ZnPOTjTzaBtchU9XtVbeFaPbUCdgHvJqobtK9lJDM+/5Rj6taj9tHH+KorIpQtWxaAnJwccnL2+TovJ2j9sPs2dmmbxc+rN/LLumCy1SbDdzIIb9F48Clnv3/1SVI53YDlqroyUaGgfS0j+WDsW/Q59/xAtHNzc+nUvhUN6tbglK7daN2mXWj0w+7b2K9HK96cMPfA+8sv7MysN27jqbsvJrNcqYT1C9t3MmicxIretmSRrEB2IfD6oT4QkeF5LsSbN8c2X83ztRw4dDiTp8+iVOnS/OeRB/2uL3v37mXS+I/o3fdc37UB0tPTmf7VXBYvW8m8ubNZsnhRaPTD7NtYLCOdM04+gbGTHQvBZ96aTuMz76Hdhffz66Zt3H/jOQmXUei+k0lAPP4XVUOkjoh8JiJLRWSxiFzn7q8oIpNF5Af3b4VY9Qk8kIlIceAs4K1DfZ5qvpZ5fPbJBE5o3oIqVav5rh3JUZmZdOx0MlMmTwyNfph9G3t0bMyC71ax4bftAGz4bTv79yuqyvNjZ9K6aeLDCIXtO5kM/PC1BPYBN6lqIxxj3qtEpDGO/+0UVa0PTOEQfrj5SUaLrBcwT1XX+yEWtK9lHu+//SZ9zr3Ad12ATRs3snXLFgB2797N1M+mUD8rKzT6YfZtPL9n64O6ldUr/+lm3qdrc5YsX5dwGangOxk0frTIVHWdqs5zX2/HMeetBfQBXnIPewnoG6s+yZhH1p/DdCsLSpC+lgC7d+1i2tQp3P/If33VzePXX9dx5fAh5Obmsn//fs4+9zx69uodGv2w+jaWKlmMru0acvWIP3+OI6/rS7Os2qgqK9f9xjUjEv+pJsN3MghvUa/kjZF5pLKIzIl4P0pVR/1FU6Qujsfl10A1VV0HTrATkaox6xSwr2VpYBVQT1W3xjo+aF/L3GAtGy0fWQpg+cii0+nENsxL0Ney4Qkt9dmxn3orr0HFmL6WIlIW+BwYqapjRWSLqmZGfP67qkYdJwva13IXjuW5YRhFCL8eXYhIMeAd4DVVHevuXi8iNdzWWA1gQyydUM7sNwyj8MjztUx0Hpk4j3KfA5aq6sMRH30A5M30HQjEnIRnay0Nw4gbn1pkHYABwEIRWeDu+wdwP/CmiAwFfgH6xRKyQGYYRvz4EMlUdUYUpW7xaFkgMwwjbsxFyTCM0JNaYcwCmWEYBSHFIpkFMsMw4kLAMsRGIyNNyCxTvLCrYYSYDV/6mwSzMEgPMG2EL8opmI8spQKZYRjhIMXimAUywzDiRVIuLZEFMsMw4ibF4pgFMsMw4kOwrqVhGEWBFItkFsgMw4ibVJt+EcrsF2HzVEy2fjLKCLP+6lWrOKNHN1q3aELb7BN44nH/p2yE+fp4wadU1/6hqoFtwA3AYmARTpbYktGOz85upbtzNOq2Y88+PbZePV3y/XLduvMPPeGEZjrvm8Uxz/O6hV2/KHyHRPS37c6NuS37abVO+2K2btudq2s2bNHjjq+vs+Yt9HTutt25ob4+2dmtNNH/rxs3a6kLV2/3tAFzgowxeVtgLTIRqQVcC7RW1aZAOo6bUkKE3VMxGZ6HYf8OQetXr1GDFi2zAShXrhxZDRuydq1/dm1hvz5e8CNnv58E3bXMAEqJSAZQGlibqGDYPRWT4XkY9u+QTF/IlStX8O2CBeb7GQdC6nUtg3QaXwM8hJMYbR2wVVUn+aD7l31h8VRMhn4yygi7fh47duxgQP9+3P/gw5QvXz72CR4pKtcnGuJxSxZBdi0r4Ng6HQvUBMqIyCWHOO6AQe/GTRtj6obZUzEZ+skoI+z6ADk5OVzS/zzOv+AizuqbuClvJEXh+sQkxSJZkF3LU4GfVXWjquYAY4GT8h8UadBbpXKVmKJh9lRMhn4yygi7vqpy1eV/IyurEVdfd4NvunmE/fp4wY+c/X4S5DyyX4D2riXcbpzUtXOinxKbsHoqJks/GWWEXf+rL2YyZvSrNGl6Ah3aOYP+d907gh49T/dFP+zXxwupNYsseF/Le4ELcKzR5wN/U9U/Dnd8q1atdebXCcc64wgmZ9/+wMsolhHK6ZcAdGjXmrkJ+lo2bZ6tYyfN8HRsVvUyMX0t/SDQO6Kqd6tqQ1VtqqoDogUxwzDCQV5iRT+mX4jI8yKyQUQWReyrKCKTReQH929Uc14I6cx+wzAKEY9TLzwOkb0I9My371ZgiqrWB6a476NigcwwjLjx66Glqk4Dfsu3uw/wkvv6JaBvLB1bNG4YRpzElVixsohEDnyPUtVRMc6ppqrrAFR1nYhUjVWIBTLDMOImjpkVm0I/2G8YRtHDa7cygUej60WkBoD7d0OsEyyQGYYRP8FGsg+Age7rgUDMFfEWyAzDiBsfp1+8DnwJZInIahEZCtwPnCYiPwCnue+jklJjZEqwExrDPJExj9z9wU1ghmA9FZOB3ePo+KXs1+ojVe1/mI+6xaOTUoHMMIwQIJBq/95ZIDMMowCkViSzQGYYRlzkJVZMJSyQGYYRNykWxyyQGYYRP9YiMwwj9CQ7tXYsLJAZhhE3qRXGQjgh1sxVY3PF8CHUrV2NNi1P8F07j7Bfo7DrJ+MeHw6vKXyKhIsSgIhcJyKLRGSxiFzvh2ZGRgYj73+QOQsWM+XzL3jm6Sf4bukSP6QByM3N5fprr+L9D8cz/9slvDXmdZYuCY8+wMUDBvHeh+N91Ywk7Nco7PoQ/D2OxRHjaykiTYFhQFugOdBbROonqmvmqrHp2KkzFSpU9FUzkrBfo7DrQ/D3OCZHkItSI+ArVd2lqvuAz4Gz/SzAzFULh7Bfo7DrpwIpFscCDWSLgM4iUsl1UjodqJP/oEhfy00bY/ta5mHmqoVH2K9R2PULH29WcMm0gwvSaXwp8AAwGZgAfIPjppT/uAO+lpWrxPa1BDNXLWzCfo3Crl/Y5M3sP2IG+1X1OVXNVtXOOHm5f/BB08xVC5mwX6Ow6xt/JeinllXdv0cD5wCvJ6qZZ6467fPP6NAumw7tspk4YVyisgeIND9tcUIjzu13fmDmqkHoAwwacBFdTz6JH5Z9T4N6dXjphed81Q/7NQq7PgR/j2ORai2yoA16pwOVgBzgRlWdEu347Fat9fOZswKrj+Wqik3Y85EVBYK8x51ObMO8BA16W2a31qke/z/NLJ2eFIPeQGf2q2qnIPUNwygEktza8oItUTIMIy4sjY9hGEWCZM7a94IFMsMw4ibVWmThH/02DCPp+DWzX0R6isj3IvKjiNxa0PpYIDMMI358iGQikg78F+gFNAb6i0jjglTHAplhGHEh4NcSpbbAj6r6k6ruBcYAfQpSp5QaI5s/b+6m8qXSV8ZxSmVgU1D1Mf0ir5+MMlJN/5hEC5w3b+7EUsWkssfDS4rInIj3o1R1lPu6FrAq4rPVQIEyQKRUIFNVb4stXURkTpCT7Uy/aOsno4yw6x8KVe3pk9ShmmwFmg1sXUvDMAqL1RycEac2sLYgQhbIDMMoLGYD9UXkWBEpDlwIfFAQoZTqWhaAUbEPMX3TL9Qywq4fGKq6T0SuBiYC6cDzqrq4IFqBLho3DMNIBta1NAwj9FggMwwj9IQykInI8yKyQUQWBaBdR0Q+E5Glro3ddQGUUVJEZonIN24Z9wZQRrqIzBeRj/zWdvVXiMhCEVmQb56QX/qZIvK2iHzn3osTfdTOcuudt23zy64woowb3Hu7SEReF5GSPuv7brUYalQ1dBvQGcgGFgWgXQPIdl+XA5YBjX0uQ4Cy7utiwNdAe5/LuBEYDXwU0D1YAVQO8B6/BPzNfV0cyAyonHTgV+AYHzVrAT8Dpdz3bwKDfNRvimPuUxrngd0nQP2g7kUYtlC2yFR1Go4HQBDa61R1nvt6O7AU54fpZxmqqjvct8XczbenLiJSGzgDeNYvzWQiIuVx/rF6DkBV96rqloCK6wYsV9V4VpR4IQMoJSIZOAGnQPOjDkPgVothI5SBLFmISF2gJU6LyW/tdBFZAGwAJquqn2U8CtwC7PdRMz8KTBKRuSIy3GftesBG4AW3e/ysiJTxuYw8LsQHL4lIVHUN8BDwC7AO2Kqqk3wswpPV4pGEBbLDICJlgXeA61V1m9/6qpqrqi1wZjO3dZ3ZE0ZEegMbVHWuH3pR6KCq2TiZC64Skc4+amfgDB08qaotgZ1AgVO8HA53EuZZwFs+61bAWfx8LFATKCMil/ilrx6tFo8kLJAdAhEphhPEXlPVsUGW5XaZpgJ+rV/rAJwlIitwsgl0FZFXfdI+gKqudf9uAN7FyWTgF6uB1RGt1LdxApvf9ALmqep6n3VPBX5W1Y2qmgOMBU7yswANwGoxzFggy4c4ltDPAUtV9eGAyqgiIpnu61I4P/zv/NBW1dtUtbaq1sXpNn2qqr61BgBEpIyIlMt7DXTH6e74gqr+CqwSkSx3VzdgiV/6EfTH526lyy9AexEp7f6euuGMtfpGEFaLYSaUS5RE5HXgFKCyiKwG7lZVv4z9OgADgIXuGBbAP1TVP/NM58noS25iuTTgTVUNZJpEQFQD3nX+HyUDGK2qE3wu4xrgNbf79xMw2E9xd2zpNOAyP3UBVPVrEXkbmIfT5ZuP/0uJ3hGRPKvFq1T1d5/1Q4UtUTIMI/RY19IwjNBjgcwwjNBjgcwwjNBjgcwwjNBjgcwwjNBjgSxEiEium61hkYi85U4hKKjWKXmZMUTkrGjmqG4miisLUMY9InKz1/35jnlRRM6Lo6y6QWRDMcKBBbJwsVtVW6hqU2AvcHnkh+IQ9z1V1Q9U9f4oh2QCcQcyw0gWFsjCy3TgeLclslREnsCZgFlHRLqLyJciMs9tuZWFA/b034nIDJzZ4Lj7B4nI4+7raiLyrpsr7RsROQm4HzjObQ0+6B73/0Rktoh8G5lPTURuF5HvReQTIIsYiMgwV+cbEXknXyvzVBGZLiLL3DWkeYvtH4wo2/cJrUb4sEAWQtzUML2Ahe6uLODliAXWdwCnuou65wA3uon9ngHOBDoB1Q8j/xjwuao2x1nfuBhnwfZytzX4/0SkO1AfZ31lC6CViHQWkVY4y6Ja4gTKNh6+zlhVbeOWtxQYGvFZXeBknJRET7nfYShONok2rv4wETnWQzlGESaUS5SOYEpFLJuajrMmtCawUlW/cve3BxoDM90lRMWBL4GGOAuZfwBwF5IfKv1OV+BScDJ0AFvdbA6RdHe3+e77sjiBrRzwrqrucsvwYu3VVERG4HRfy+I46uTxpqruB34QkZ/c79AdaBYxfnaUW/YyD2UZRRQLZOFit5v65wBusNoZuQsnv1n/fMe1wL/kjQLcp6pP5yvj+gKU8SLQV1W/EZFBOGto88ivpW7Z16hqZMDLyx1nHKFY17Lo8RXQQUSOB2dxtIg0wMmucayIHOce1/8w508BrnDPTRcnW+t2nNZWHhOBIRFjb7XcbAzTgLNFpJSbHeNMD/UtB6xzUyddnO+zfiKS5ta5HvC9W/YV7vGISAMJLumiERKsRVbEUNWNbsvmdREp4e6+Q1WXiZPJ9WMR2QTMwMn9np/rgFEiMhTIBa5Q1S9FZKY7vWG8O07WCPjSbRHuAC5R1Xki8gawAFiJ0/2NxZ04GXhX4oz5RQbM73HSOFcDLlfVPSLyLM7Y2Tw3Rc5GoK+3q2MUVSz7hWEYoce6loZhhB4LZIZhhB4LZIZhhB4LZIZhhB4LZIZhhB4LZIZhhB4LZIZhhJ7/D/x5GRRpc/QwAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 432x288 with 2 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(skplt.plot_confusion_matrix(y_val, predicted))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
